---
about_this_resource_text: <p><strong>Topics covered:</strong> Recombinant DNA 2</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: Video-YouTube-Stream
  media_location: qObvbkcU838
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Video-YouTube-Stream
  type: Video
  uid: 3983cabef38dbc1ff415c96a0703f229
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 6ede988d0b3e72d2e510afb5c834e5e5
- id: lec16.jpg
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-16-recombinant-dna-2/lec16.jpg
  title: lec16.jpg
  type: null
  uid: cc096e1ed71e5bcbd6460ea6f0e50213
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/qObvbkcU838/default.jpg
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 296e9803c3c05cda051b8baefd91c4e0
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec16-18oct2004-220k.mp4
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Video-Internet Archive-MP4
  type: Video
  uid: 1defb6253a399717f5ddf51960a31c23
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Video-iTunes U-MP4
  type: Video
  uid: 8da82fc1b9f024c3e9f5de151638fbb7
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Audio-iTunes U-MP3
  type: Audio
  uid: a093dd8840e93b69b60acd748fc4e76c
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-18oct2004-1000.mp3
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Video-Internet Archive-MP3
  type: Video
  uid: d928cc16eede50e3c20cdba0d28d4063
- id: Thumbnail-OCW-JPG
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 9bbe992acbbe4d224905416427938efe
- id: 3Play-3PlayYouTubeid-MP4
  media_location: qObvbkcU838
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: cf32eda332f818f9889a4c3c2ab9ad40
- id: qObvbkcU838.srt
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-16-recombinant-dna-2/qObvbkcU838.srt
  title: 3play caption file
  type: null
  uid: 7954a3bc12ea1ffe17c1bb05f8282391
- id: qObvbkcU838.pdf
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-16-recombinant-dna-2/qObvbkcU838.pdf
  title: 3play pdf file
  type: null
  uid: 21a9adf03765a5cccb9497a3516f16f1
- id: Caption-3Play YouTube id-SRT
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: 5db6692926220b58a89dfa7d9cd54732
- id: Transcript-3Play YouTube id-PDF
  parent_uid: b48b7f97aa23237e5dec4f1f9ff42e71
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: a28c37f7e1c803005bd9ec82aec9be37
inline_embed_id: 32292108lecture16:recombinantdna265036307
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-16-recombinant-dna-2
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-16-recombinant-dna-2
template_type: Tabbed
title: 'Lecture 16: Recombinant DNA 2'
transcript: '<p><span m=''1000''>Good morning. So,</span> <span m=''5000''>we are
  going to see if my voice holds up through this lecture today.</span> </p><p><span
  m=''9000''>It is a casualty of having been at Foxborough yesterday,</span> <span
  m=''13000''>and then staying up rather late watching the Red Sox game.</span> </p><p><span
  m=''17000''>On the whole, both seemed to have come through successfully,</span>
  <span m=''21000''>but my voice is a bit of a casualty of the events.</span> </p><p><span
  m=''25000''>So, we''ll see. But I''m going to sound a lot</span> <span m=''29000''>scratchier
  than normal.</span> </p><p><span m=''35000''>So, how many of you stayed up to the
  end of the game last night?</span> </p><p><span m=''41000''>Good, excellent. I approve.</span>
  </p><p><span m=''49000''>OK,last time, we spoke about the idea of cloning DNA,</span>
  <span m=''54000''>to create libraries of molecules.</span> </p><p><span m=''59000''>And
  again, I think this is just one of the most clever inventions</span> <span m=''63000''>because
  it''s a completely new way to think about purifying molecules.</span> </p><p><span
  m=''67000''>Rather than purifying molecules, by separating them based on their</span>
  <span m=''71000''>biochemical properties, it''s purifying molecules by diluting</span>
  <span m=''75000''>them into single components, and then amplifying each back up</span>
  <span m=''80000''>from its own source. It''s really quite a beautiful idea.</span>
  </p><p><span m=''84000''>And just to go over it, we take, say, human DNA,</span>
  <span m=''88000''>or we could take drosophila DNA, or we could take yeast DNA, or
  we</span> <span m=''92000''>could take any other DNA we feel like.</span> </p><p><span
  m=''97000''>We cut it up in some fashion with a restriction enzyme.</span> </p><p><span
  m=''102000''>We''ll use our favorite restriction enzyme here, echo R1,</span> <span
  m=''108000''>which cuts a defying side, GAATTC. We take that. We add our</span>
  <span m=''114000''>insert DNA. These are referred to as inserts because they''re
  going to</span> <span m=''120000''>be inserted into a plasmid. We take a plasmid
  vector.</span> </p><p><span m=''125000''>The plasmid vector here is a naturally
  occurring,</span> <span m=''131000''>although sometimes modified, piece of DNA that
  bacteria have that</span> <span m=''136000''>take an origin of replication that
  allow it to grow autonomously when</span> <span m=''142000''>put in a bacterial
  cell, a selectable marker.</span> </p><p><span m=''148000''>The selectable marker,
  for example, ampicillin resistance,</span> <span m=''152000''>or some other resistance,
  we add these and then we seal up the pieces</span> <span m=''157000''>of the DNA
  using the enzyme ligase. Ligase joins and joins producing</span> <span m=''162000''>for
  us molecules of this sort. We make zillions of them in</span> <span m=''167000''>parallel
  in one test tube. We then transform them by adding</span> <span m=''172000''>these
  molecules to bacterial cells that have been appropriately</span> <span m=''177000''>prepared
  to be transformed, that is, their membranes have been</span> <span m=''182000''>treated
  in such a way that they''re going to be most likely to</span> <span m=''187000''>suck
  up pieces of DNA. We then plate them on a plate at a</span> <span m=''191000''>density
  so that individual bacterial cells are well separated from each</span> <span m=''195000''>other.
  You try a bunch of different densities so you get one right.</span> </p><p><span
  m=''199000''>And, you let them grow up. And, every colony here, as we discussed,</span>
  <span m=''203000''>is the descendant of a single bacterial cell,</span> <span m=''207000''>carrying
  ideally a single plasmid.</span> </p><p><span m=''211000''>And, that single plasmid,
  we know it''s carrying a single</span> <span m=''215000''>plasmid because we were
  clever enough to put ampicillin or other</span> <span m=''219000''>selectable marker
  on this plate. And so, only bacteria that have</span> <span m=''224000''>picked
  up the plasmid are ampicillin resistant. And there you go.</span> </p><p><span m=''228000''>This
  is called a library. And, at the end of the day, you may have</span> <span m=''233000''>a
  library that contains one plate of clones or a library containing</span> <span m=''237000''>hundreds
  of plates of clones. We''re going to see how we last</span> <span m=''242000''>through
  this. Now, a few people asked me at the end of</span> <span m=''248000''>the last
  lecture, well, OK, but what about the details.</span> </p><p><span m=''253000''>Is
  it really going to work like this? How come some of these</span> <span m=''259000''>plasmid
  molecules don''t automatically get closed back up by</span> <span m=''264000''>ligase?
  Why is it that there''s always an insert in the plasmid?</span> </p><p><span m=''270000''>What''s
  the answer to that question? Sorry? There''s not an answer</span> <span m=''274000''>because
  sometimes ligase might close up that molecule.</span> </p><p><span m=''278000''>Now,
  that would be unfortunate because it would mean that a bunch</span> <span m=''282000''>of
  the things in your library just had the vector without any insert.</span> </p><p><span
  m=''286000''>So, and these are details, but over the course of years,</span> <span
  m=''290000''>recombinant DNA specialists have worked out lots of cute tricks to</span>
  <span m=''294000''>make better and better libraries. I''ll just give you an example
  of</span> <span m=''298000''>the kinds of things. Remember that in order to ligate
  DNA,</span> <span m=''303000''>we had a five prime here. We have a phosphate group
  here,</span> <span m=''309000''>three prime hydroxyl phosphate here, double strand
  of DNA here. We have</span> <span m=''316000''>a phosphate here. We have a hydroxyl
  here,</span> <span m=''323000''>phosphate five prime, three prime.</span> </p><p><span
  m=''330000''>If ligase is going to come along, it turns out that ligase needs the</span>
  <span m=''337000''>phosphate there in order to seal it up and make a chain.</span>
  </p><p><span m=''345000''>So, for example, suppose we were to arrange that the plasmid
  vector</span> <span m=''353000''>didn''t have phosphates on its two ends. Then ligase
  would not be able</span> <span m=''361000''>to re-seal the plasmid vector. That''s
  a cute trick.</span> </p><p><span m=''366000''>This is just cooking, but I''m giving
  you an idea of the</span> <span m=''370000''>kind of cooking tricks we use in all
  this. So, ideally,</span> <span m=''373000''>you would like an enzyme that can remove
  phosphate groups from the end</span> <span m=''377000''>of DNA. How are you going
  to invent such an enzyme?</span> </p><p><span m=''381000''>It already exists is
  the answer to all these questions.</span> </p><p><span m=''384000''>And, bacteria
  have such an enzyme that can remove phosphate groups.</span> </p><p><span m=''388000''>So,
  just remove phosphate groups. And of course these enzymes are</span> <span m=''392000''>developed
  by bacteria because they need them in the course of DNA</span> <span m=''396000''>metabolism.
  And, what do you think the enzyme is</span> <span m=''401000''>called? Phosphotase,
  of course. That''s what happens,</span> <span m=''405000''>use phosphotase, and
  you treat that, and it doesn''t seal back up. Now,</span> <span m=''409000''>somebody
  will say to me, well, OK, but now I''ve got my vector</span> <span m=''414000''>here,
  and I don''t have a phosphate on it, and so this is</span> <span m=''418000''>my
  vector DNA. And then, I''ve got my insert DNA,</span> <span m=''423000''>and sorry,
  my insert DNA here, it has a hydroxyl here and a phosphate</span> <span m=''428000''>here.
  So, the vector has no phosphate. But,</span> <span m=''433000''>when ligase wants
  to attach an insert, it''s got a phosphate here</span> <span m=''439000''>but not
  here. What''s going to happen? Well,</span> <span m=''444000''>it turns out that
  ligase will seal up this because it''s got a phosphate,</span> <span m=''449000''>but
  it''ll leave this one open. Now, is that a problem?</span> </p><p><span m=''454000''>It
  turns out, if you just transform it into the bacteria with that hole</span> <span
  m=''458000''>there on one strand but not both strands, it''s still a covalently</span>
  <span m=''462000''>closed circle on one of its strands. The bacteria will repair
  it. So,</span> <span m=''467000''>you can take advantage of the bacteria''s own
  DNA repair mechanisms</span> <span m=''471000''>to just throw the molecule in sealed
  up on one strand and let its repair</span> <span m=''475000''>mechanism; all these
  tricks we play to our advantage.</span> </p><p><span m=''480000''>Someone else asked
  after class, what happens if the gene I''m</span> <span m=''484000''>interested
  in studying has, here''s my gene let''s say that I''m</span> <span m=''488000''>interested
  in studying. I take human DNA. I cut it with</span> <span m=''493000''>echo R1.
  So, I have cut it at all the echo sites.</span> </p><p><span m=''497000''>Well,
  golly, what happens if my gene happened to have an echo site</span> <span m=''502000''>in
  it? Then my gene''s going to be cut up into two pieces.</span> </p><p><span m=''506000''>Isn''t
  that bad? What do I do about that?</span> </p><p><span m=''510000''>Do I know in
  advance if my gene has an echo site? Well,</span> <span m=''514000''>no, I don''t,
  because I don''t know what my gene is.</span> </p><p><span m=''518000''>I''m making
  a library of everything in the genome.</span> </p><p><span m=''522000''>So, some
  genes will have it, and some won''t. And, I might not</span> <span m=''526000''>know
  the gene I''m looking for. So, how do I avoid that? Sorry?</span> </p><p><span m=''530000''>Oh,
  you''ve tried another enzyme. You''ve tried BAM and Hindi,</span> <span m=''534000''>and
  make a library with different enzymes. That''s one</span> <span m=''538000''>way.
  That works. Another way, just to give you a</span> <span m=''542000''>sense of how
  fast molecular biologists are with this.</span> </p><p><span m=''546000''>Supposed
  when we add echo R1 we don''t let the reaction go to</span> <span m=''550000''>completion.
  Suppose we run the reaction under conditions where it''s</span> <span m=''554000''>somewhat
  inefficient, and instead of managing to cleave</span> <span m=''558000''>every echo
  site, on average it cleaves,</span> <span m=''562000''>say, one out of every three-echo
  sites. You can do that.</span> </p><p><span m=''566000''>So, that means you can
  arrange just by your reaction conditions to on</span> <span m=''570000''>average
  randomly cleave some but not others.</span> </p><p><span m=''575000''>And, these
  are called partial digestions. So,</span> <span m=''578000''>it turns out that all
  of the kinds of things that people were asking me</span> <span m=''581000''>about
  afterwards, I was very glad people were thinking</span> <span m=''584000''>about
  would this really work? There are tricks to get around all</span> <span m=''587000''>of
  it, and there''s a whole fat book of protocols about if you want to</span> <span
  m=''590000''>make a library really, really carefully, how you would do</span> <span
  m=''594000''>that, how you make sure the vector doesn''t re-close,</span> <span
  m=''597000''>how you make sure that you don''t cut every site but random sites,</span>
  <span m=''600000''>and things like that. And, all of these rely on lots of</span>
  <span m=''604000''>enzymes and things that bacteria have already invented.</span>
  </p><p><span m=''608000''>So, I''m just going to put these down as cooking tips.</span>
  </p><p><span m=''612000''>These are not really necessarily, I don''t care whether
  you know the</span> <span m=''616000''>details or not, rather that there exists
  a whole 15</span> <span m=''620000''>years, 20 years worth of ways to make the best
  possible libraries.</span> </p><p><span m=''624000''>And so, it''s quite routine
  now to be able to make good libraries.</span> </p><p><span m=''628000''>All right,
  so, having made a library,</span> <span m=''634000''>the challenge is finding your
  clone. How to find your clone, the clone</span> <span m=''640000''>of interest.
  So, I need to describe a number of ways</span> <span m=''646000''>that people have
  for finding a clone of interest. And here,</span> <span m=''652000''>of course,
  up to this point, the DNA could be zebra DNA, and it</span> <span m=''658000''>could
  be human DNA and yeast DNA, and it could be something that is an</span> <span m=''664000''>enzyme
  for arginine, or this, or that.</span> </p><p><span m=''671000''>But now we have
  to be specific. So, let''s suppose we go back to a</span> <span m=''678000''>problem
  we talked about before about, say, auxotrophy for a nutrient.</span> </p><p><span
  m=''685000''>So, let''s suppose that I have a bacteria, maybe even E coli itself,</span>
  <span m=''692000''>where I have selected mutants that are auxotrophic for arginine.</span>
  </p><p><span m=''700000''>So, arginine auxotrophs will grow on rich medium, but
  on minimal medium</span> <span m=''712000''>they don''t grow. But, they would grow
  if I added</span> <span m=''720000''>arginine to that medium. They don''t grow because
  they have a</span> <span m=''724000''>mutation in a gene. We know it''s a gene because
  we</span> <span m=''729000''>crossed together the mutant and the wild type. We show
  that we can</span> <span m=''733000''>define this phenotype to be a recessive phenotype.</span>
  </p><p><span m=''738000''>We can map it in the yeast genome by showing it has linkage
  to other</span> <span m=''742000''>phenotypes. That''s all great. We can do classical
  genetics, a la</span> <span m=''747000''>Mendel, a la Morgan, a la Sturtevant.</span>
  </p><p><span m=''752000''>But, how are we going to find the gene? How are we going
  to,</span> <span m=''757000''>now, use our tools of recombinant DNA to get physically
  in our hand</span> <span m=''762000''>the piece of DNA that encodes the gene that
  is defective in the strand?</span> </p><p><span m=''767000''>So, have a mutant bacteria.
  It can''t make arginine. It can''t</span> <span m=''772000''>grow in minimal medium.
  Somewhere in there, you know</span> <span m=''777000''>there''s a mutation in the
  DNA sequence.</span> </p><p><span m=''782000''>How do we find it? What should we
  do?</span> </p><p><span m=''787000''>This is the whole point of recombinant DNA,</span>
  <span m=''793000''>to make this abstract notion of, there exists genes, they transmit</span>
  <span m=''798000''>all this kind of stuff, concrete. How are you going to find</span>
  <span m=''804000''>it? Any takers? Sorry? Run a gel.</span> </p><p><span m=''810000''>So,
  I take DNA, cut it up, run a gel.</span> </p><p><span m=''814000''>I have all the
  DNA from the bacteria schmeered (sic) out.</span> </p><p><span m=''818000''>And
  somewhere in that schmeer is the gene. So,</span> <span m=''822000''>I take normal
  DNA from normal bacteria. I take mutant DNA.</span> </p><p><span m=''826000''>One
  nucleotide is different in the mutant DNA. I run them out,</span> <span m=''830000''>and
  I assure you, they just look like a schmeer.</span> </p><p><span m=''834000''>It''s
  just a big schmeer of DNA. It''s hard to see one nucleotide</span> <span m=''838000''>difference
  out of the 4 million nucleotides.</span> </p><p><span m=''843000''>The E coli say,
  how are we going to get that?</span> </p><p><span m=''847000''>This is good. We''re
  thinking practically here.</span> </p><p><span m=''851000''>What else? Sorry? Sorry?
  Cut it up. I''m assuming</span> <span m=''855000''>she wanted it cut up and run
  out on the gel. It still will look like a</span> <span m=''859000''>schmeer. Forget
  the gel. Cut it up. Make a library.</span> </p><p><span m=''863000''>OK, so we''re
  going to make a library. Let''s assume now we have a</span> <span m=''867000''>library
  of different E coli cells containing individual plasmids,</span> <span m=''871000''>containing
  random bits of E coli. How''s that going to help?</span> </p><p><span m=''877000''>Splice
  it back in. How do I know if I spliced it back in?</span> </p><p><span m=''886000''>Ooh,
  that''s an interesting thought. Suppose I were to make my library</span> <span m=''895000''>using
  wild type DNA, DNA from the wild type strain.</span> </p><p><span m=''904000''>So,
  I''m going to make a library containing lots and lots of</span> <span m=''909000''>fragments
  of normal E coli DNA. This is my library. I''m going to</span> <span m=''914000''>transform
  it into, what kind of bacteria should I</span> <span m=''920000''>transform it into,
  wild type or mutant?</span> </p><p><span m=''925000''>Who votes mutant? Who votes
  wild type?</span> </p><p><span m=''931000''>We''ll go with mutant, then. Mutant.
  We''ll put it in</span> <span m=''938000''>mutant. So now, all of these mutant cells,</span>
  <span m=''945000''>each one is going to suck up a plasmid. We then are going to
  plate</span> <span m=''952000''>this, and let colonies grow up. One of these colonies
  contained,</span> <span m=''959000''>so this mutant is arge minus. And, one of these
  colonies is going</span> <span m=''966000''>to contain the ARG plus gene here. How
  are we going to know which one?</span> </p><p><span m=''972000''>Sorry? How are
  we going to know which one has the arge plus gene?</span> </p><p><span m=''978000''>Yes?
  So, plate it on minimal medium. If I plate it on minimal</span> <span m=''984000''>medium,
  what will happen to most of my mutant bacteria?</span> </p><p><span m=''991000''>They''re
  not going to grow. But, what''s going to happen to the</span> <span m=''994000''>bacteria
  that happens to be lucky enough to have picked up the plasmid</span> <span m=''998000''>that
  contains the ARG plus gene? It''ll grow. So, whatever grows on</span> <span m=''1001000''>minimal
  medium has been rescued. In fact, we''ve complemented the</span> <span m=''1005000''>defect.
  Remember, we talked about complementation</span> <span m=''1009000''>tests? In a
  way, it would be the plasmid is</span> <span m=''1012000''>complementing the defect.
  Bingo, that''s it. So, we can</span> <span m=''1016000''>actually find that gene
  functionally.</span> </p><p><span m=''1020000''>We plate on minimal median, and
  we look for growth. The only</span> <span m=''1029000''>things that will grow have
  been rescued. So, this is called cloning</span> <span m=''1038000''>by complementation
  because we are complementing the defect</span> <span m=''1047000''>in this strand.
  All right. So,</span> <span m=''1054000''>any time I have a functional defect in
  my bacteria,</span> <span m=''1058000''>I can find the gene for that functional
  defect by simply taking a</span> <span m=''1063000''>total library for normal from
  wild type bacteria,</span> <span m=''1068000''>transforming it into a mutant bacteria,
  and looking for rich</span> <span m=''1072000''>bacteria has suddenly been rescued.
  Then I''ll purify that bacterium,</span> <span m=''1077000''>and I''ll purify out
  the plasmid. And that plasmid will contain the</span> <span m=''1085000''>DNA for
  the gene. That''s pretty cool.</span> </p><p><span m=''1096000''>Let''s try another
  one. Suppose, yes? OK, great.</span> </p><p><span m=''1108000''>I''ve got my plate
  here, and I''ve said only one of these</span> <span m=''1112000''>bacteria will
  grow. It''s the one that happens to have</span> <span m=''1116000''>within it the
  plasmid containing the ARG gene. And,</span> <span m=''1121000''>you''re fine with
  that, but you''re saying, but how would I</span> <span m=''1125000''>get that plasmid
  back out of the bacteria because the bacteria''s got</span> <span m=''1130000''>its
  own chromosome, and I''m making this big deal about</span> <span m=''1134000''>how
  we purified stuff away from all this other DNA.</span> </p><p><span m=''1139000''>But,
  I''ve thrown this plasmid back into a bacteria that has all</span> <span m=''1143000''>its
  chromosomal DNA. So, who am I kidding?</span> </p><p><span m=''1148000''>How are
  we going to purify out just that plasmid? If I could purify the</span> <span m=''1153000''>plasmid,
  it would be OK right? It turns out I can. Plasmids are</span> <span m=''1158000''>little
  circles of DNA. Chromosomes are big pieces of DNA.</span> </p><p><span m=''1162000''>It
  turns out that the coiling of the plasmid as a little circle gives</span> <span
  m=''1167000''>it different densities and different physical chemical properties
  to big</span> <span m=''1172000''>chunks of DNA which get broken up. And so, there
  are a bunch of tricks</span> <span m=''1177000''>that allow me to get a pretty high
  purification of a plasmid away from</span> <span m=''1181000''>chromosomal DNA based
  on the different physical properties of a</span> <span m=''1186000''>small circle
  versus big chromosome. But, good question. Otherwise, how</span> <span m=''1190000''>would
  I get that plasmid out? But it turns out, you can purify</span> <span m=''1195000''>plasmids.
  Good question. OK, so now, let''s try another one.</span> </p><p><span m=''1200000''>Next
  cloning expedition: we''re going to go to the library,</span> <span m=''1205000''>and
  we want to withdraw a volume from the library.</span> </p><p><span m=''1210000''>And,
  I want now, instead of bacteria that can''t make arginine,</span> <span m=''1215000''>let''s
  go with human DNA. Let''s try human DNA. And,</span> <span m=''1220000''>I would
  like you to now please find the gene that encodes beta-globin.</span> </p><p><span
  m=''1225000''>Beta globin, of course, is one of the two proteins in hemoglobin.</span>
  </p><p><span m=''1230000''>Hemoglobin is a tetramer. It has alpha-globin and beta-globin.</span>
  </p><p><span m=''1234000''>This tetramer is the oxygen carrier in your blood.</span>
  </p><p><span m=''1239000''>It carriers oxygen. Beta-globin happens to be the site</span>
  <span m=''1243000''>of some very important mutations. We know that sickle cell anemia
  is</span> <span m=''1248000''>caused by mutations in beta-globin. We know that diseases
  like</span> <span m=''1252000''>thalassemia are caused by mutations in beta-globin.</span>
  </p><p><span m=''1257000''>And, people knew this before they had recombinant DNA
  because they</span> <span m=''1261000''>could study red blood cells. There''s lots
  of beta-globin in red</span> <span m=''1266000''>blood cells. They could see that
  something was funny about the</span> <span m=''1270000''>protein. They could even
  see that in sickle cell anemia the protein</span> <span m=''1274000''>had a different
  net charge, and it would run differently.</span> </p><p><span m=''1279000''>So,
  they knew something was funny with the beta globin protein.</span> </p><p><span
  m=''1283000''>All I want you to do now is clone beta-globin for me.</span> </p><p><span
  m=''1287000''>Could we do the same thing? Why not?</span> </p><p><span m=''1292000''>Bacteria
  don''t make beta-globin. So, what can we do? Well, we could</span> <span m=''1300000''>make
  a library of human DNA. And, we could throw it into the</span> <span m=''1309000''>bacteria.
  So, why don''t we just select for a</span> <span m=''1318000''>bacteria that makes
  beta-globin? Could we do that?</span> </p><p><span m=''1325000''>I don''t know,
  how? Do you see how? How would we</span> <span m=''1331000''>select for that? I
  mean, there, we could see who</span> <span m=''1336000''>grows without arginine.
  But how are we going to tell which</span> <span m=''1341000''>bacteria has picked
  up beta-globin? I don''t know. Yeah? Use</span> <span m=''1347000''>mammals. We
  could take a mouse that did not</span> <span m=''1352000''>make beta globin, a mouse
  that had, say,</span> <span m=''1357000''>thalassemia, isolate a naturally occurring
  mouse with a defect in</span> <span m=''1361000''>beta-globin. Then, do injections
  of plasmids into mouse</span> <span m=''1366000''>eggs, grow up the mouse eggs by
  implanting them back into</span> <span m=''1371000''>pseudo-pregnant females, do
  this for 108 individual plasmids</span> <span m=''1375000''>with 108 individual
  mice, and look for the mouse that is</span> <span m=''1380000''>rescued. Intellectually,</span>
  <span m=''1384000''>you''re absolutely right, it works. So, that''s exactly the</span>
  <span m=''1388000''>cloning by complementation we talked about for bacteria,</span>
  <span m=''1392000''>and you''re dead-on right. That would work. Getting it funded</span>
  <span m=''1396000''>is another matter because it''s a hugely expensive experiment
  to shoot</span> <span m=''1399000''>up each egg with this, but it could work. So,</span>
  <span m=''1403000''>we need another solution because we can''t rescue the function
  in mice</span> <span m=''1407000''>because it''s just not practical to do so.</span>
  </p><p><span m=''1411000''>Of course, if we could do this in mouse cells, maybe
  we could make it</span> <span m=''1415000''>work in cell culture in mice. But, let''s
  suppose we don''t have a</span> <span m=''1420000''>cell culture phenotype. We just
  have an organism phenotype.</span> </p><p><span m=''1424000''>So, it''s not going
  to work to just do this by complementation.</span> </p><p><span m=''1429000''>But,
  good thinking guys. This is good. So, next trick we might have</span> <span m=''1433000''>at
  our disposal is suppose because beta-globin is so abundant in red</span> <span m=''1438000''>blood
  cells we have purified beta-globin, and we''ve done amino</span> <span m=''1442000''>acid
  sequencing of the protein. By end degradation,</span> <span m=''1449000''>you can
  work out the sequence of globin. And, you can learn that</span> <span m=''1457000''>beta-globin
  has, here at its amino terminal,</span> <span m=''1465000''>val, leu, ser, pro,
  ala, asp, lys, threonine dot, dot, dot, dot,</span> <span m=''1473000''>dot off
  to the carboxy terminal, OK?</span> </p><p><span m=''1481000''>If I knew that this
  was the amino acid sequence of the beginning,</span> <span m=''1486000''>just the
  beginning of beta-globin, couldn''t I figure out what that</span> <span m=''1491000''>initial
  portion of the DNA sequence must be?</span> </p><p><span m=''1497000''>Wouldn''t
  this give me a clue? If I knew a little bit of the</span> <span m=''1501000''>protein
  sequence, wouldn''t this give me a clue about</span> <span m=''1505000''>the nucleotide
  sequence that must be there in the human genome to encode</span> <span m=''1509000''>this
  protein? So, a biochemist has purified the</span> <span m=''1513000''>protein. Biochemists
  have studied the protein well enough to know some</span> <span m=''1517000''>of
  its amino acid sequence. Can I infer the DNA sequence from</span> <span m=''1521000''>the
  amino acid sequence, or at least a little snippet of it?</span> </p><p><span m=''1525000''>Sorry?
  Multiple possibilities,</span> <span m=''1530000''>but an infinite number? No. Why
  do you encode valine? Well,</span> <span m=''1535000''>GT something; something could
  be actually A, T,</span> <span m=''1540000''>C, or G. What about luecine. Well,
  it''s either a T and a C,</span> <span m=''1545000''>or is T in the first place?
  There''s always a T there.</span> </p><p><span m=''1550000''>There you go, and it
  can be either of those. There''s a T,</span> <span m=''1555000''>C, anything, or
  an A, G, and a T, or a C.</span> </p><p><span m=''1561000''>Here, we have C, C anything.
  Here we have a G,</span> <span m=''1567000''>C anything. We have a G, A, T, or a
  C. For leucine it''s an A, an A,</span> <span m=''1573000''>either an A or a G.
  Here, it''s an A, a C,</span> <span m=''1579000''>an anything. Here, it''s an A,
  an A, a T, or a C, here a G, a T,</span> <span m=''1585000''>anything, an A, an
  A, A or a G. You''re right. There are</span> <span m=''1591000''>multiple possibilities.
  But, it''s not an infinite number,</span> <span m=''1596000''>right? There are certain
  possible DNA sequences that might be encoded</span> <span m=''1601000''>here. If
  I just work it out, it''s either two choices here. There</span> <span m=''1606000''>are
  four choices here. There''s two choices here.</span> </p><p><span m=''1612000''>There''s
  four choices here. There''s two choices here, two</span> <span m=''1617000''>choices,
  etc. If I just look at,</span> <span m=''1622000''>let''s take a segment of this.
  Let''s try one, two, three, these</span> <span m=''1628000''>six amino acids. Four
  choices here,</span> <span m=''1634000''>how many possible DNA sequences could encode
  these six amino acids</span> <span m=''1640000''>in this order? Four times four
  times two times two</span> <span m=''1646000''>times four times two, what is that?</span>
  </p><p><span m=''1652000''>256, let''s see, two, two, to the two,</span> <span m=''1658000''>to
  the four, to the five, to the six, to the seven, eight,</span> <span m=''1664000''>512.
  I think it''s about 512 possibilities.</span> </p><p><span m=''1670000''>So, 512
  possible nucleotide sequences could work here.</span> </p><p><span m=''1676000''>Well,
  512''s not infinite. There''s 18 bases of sequence,</span> <span m=''1682000''>512
  possible 18 base long nucleotide sequences.</span> </p><p><span m=''1689000''>Just
  suppose that you knew which one it was. Now, you have to suspend</span> <span m=''1694000''>your
  disbelief for a second. I''m not going to tell you how you</span> <span m=''1699000''>might
  know, but suppose you knew which of the 512 it was.</span> </p><p><span m=''1704000''>OK,
  could we use that little fact of knowing a stretch from about 18</span> <span m=''1709000''>bases
  of the sequence to find the clone?</span> </p><p><span m=''1715000''>How could we
  find that clone in our library that has that 18 bases of</span> <span m=''1719000''>sequence?
  Google. [LAUGHTER] And, of course,</span> <span m=''1723000''>you are totally right
  because as we''ll come back to,</span> <span m=''1727000''>that is the way you would
  do it today if it''s the human genome</span> <span m=''1731000''>because the entire
  sequence of the human genome''s on the web.</span> </p><p><span m=''1735000''>But,
  you might have an organism where it''s not on the web.</span> </p><p><span m=''1740000''>But,
  we''ll come back because, of course, the human genome project</span> <span m=''1744000''>changes
  everything as to how you would approach this.</span> </p><p><span m=''1749000''>Google
  is how you would do it today. But, in the absence of Google or</span> <span m=''1753000''>the
  absence of the entire sequence of the human genome,</span> <span m=''1758000''>but
  I''m glad you raise it because it''s absolutely right,</span> <span m=''1763000''>how
  could I find the clone that has that specific 18 base pair sequence?</span> </p><p><span
  m=''1767000''>Who has my 18 base sequence. Well, here''s a trick.</span> </p><p><span
  m=''1773000''>I could chemically synthesize an oligonucleotide that matches my</span>
  <span m=''1781000''>sequence: an 18 base pair long ologonucleotide encoding my sequence.</span>
  </p><p><span m=''1788000''>What I''d like to do is use this ologonucleotide as a
  chemical probe</span> <span m=''1796000''>to wash over my library. And, by washing
  it over my library,</span> <span m=''1802000''>I''d like to see where it sticks.
  Now, that''s kind of interesting.</span> </p><p><span m=''1807000''>What do I mean
  by that? What I''d really like to do would be to kind</span> <span m=''1812000''>of
  crack open all the cells of my library, and then the DNA would be</span> <span m=''1818000''>sitting
  there. And, I''d like to take my</span> <span m=''1823000''>ologonucleotide probe
  for a little snippet of the gene and wash it over</span> <span m=''1828000''>the
  library. And then, by the amazing powers of</span> <span m=''1833000''>Crick and
  Watson base pairing, it should stick to the right place.</span> </p><p><span m=''1839000''>Could
  it do that? Turns out DNA, given time to wash around,</span> <span m=''1844000''>will
  stick to its own complement. So that''s the idea. How in the</span> <span m=''1849000''>world
  do I do this in practice? So, here''s what you do in practice.</span> </p><p><span
  m=''1855000''>In practice, let us grow our</span> <span m=''1860000''>bacteria.
  Let''s plate the bacteria on an agar plate on which we have</span> <span m=''1866000''>put
  a membrane a nitrocellulose filter or some other kind of filter.</span> </p><p><span
  m=''1872000''>Just imagine it being a piece of filter paper. And,</span> <span m=''1878000''>I''m
  going to plate my bacteria on the filter paper that''s here.</span> </p><p><span
  m=''1884000''>I''ll let them grow up because there''s nutrients here.</span> </p><p><span
  m=''1890000''>The nutrients diffuse through the filter paper. And then,</span> <span
  m=''1895000''>I have a piece of filter paper that I can pick up with my tweezers,</span>
  <span m=''1900000''>and on that filter paper are bacterial colonies growing.</span>
  </p><p><span m=''1905000''>So, this is a filter. Then, what I''m going to do is
  I''m going to</span> <span m=''1910000''>take this filter with these glistening
  bacterial colonies,</span> <span m=''1915000''>and I''m going to stick it in the
  autoclave.</span> </p><p><span m=''1920000''>And, I''m going to heat it up in the
  presence of wet heat,</span> <span m=''1924000''>and the bacterial cells will crack
  open. And, under these conditions,</span> <span m=''1929000''>the DNA will tend
  to stick to the filter because I''ve picked the</span> <span m=''1933000''>filter
  that the DNA tends to stick to. And, I''m going to wash this</span> <span m=''1938000''>filter
  in a certain way that all the usual junk, some of the proteins and</span> <span
  m=''1943000''>cell surface junk washes off. And, the DNA from each bacterial</span>
  <span m=''1947000''>colony will stick. So now, I have the DNA from each</span> <span
  m=''1953000''>colony sticking to that spot. Then, what I''m going to do is I''m</span>
  <span m=''1959000''>going to take my filter and I''m going to add my ologoprobe.</span>
  </p><p><span m=''1965000''>This thing is now called a probe. I''m going to add the
  probe to the</span> <span m=''1971000''>filter, and I''m going to put this in a,
  I need some sort of a</span> <span m=''1977000''>hybridization device in which the
  probe and the ologonucleotide and a</span> <span m=''1983000''>little water can
  swish around. And here, we use a technical device</span> <span m=''1987000''>called
  a baggy, or some other kind of,</span> <span m=''1991000''>basically, a Ziploc bag
  or you can heat seal it or something like a</span> <span m=''1995000''>freeze meal.
  In fact that''s actually what''s used in the lab is</span> <span m=''1998000''>Freeze-a-Meal.
  You get these Freeze-a-Meal bags,</span> <span m=''2002000''>you toss your filter
  in, you squirt a little bit of your probe in,</span> <span m=''2006000''>and you
  put it in the Freeze-a-Meal bag, and then you put</span> <span m=''2010000''>it
  in a water bath. And, it switches back and forth.</span> </p><p><span m=''2014000''>And,
  the probe just goes washing all over the place.</span> </p><p><span m=''2020000''>And,
  wherever the probe finds its corresponding cognate sequence by</span> <span m=''2026000''>Crick
  and Watson, it''ll stick. And there you go.</span> </p><p><span m=''2031000''>That
  clone contains your sequence. Now, we have a few problems here,</span> <span m=''2037000''>don''t
  we? What are some of the problems with this? Yeah?</span> </p><p><span m=''2043000''>Sorry,
  what if it sticks what? So, the probe, I thought this</span> <span m=''2047000''>filter
  likes DNA. So, why won''t the probe just stick</span> <span m=''2052000''>nonspecifically
  everywhere? We treat it in some way so that</span> <span m=''2057000''>after we''ve
  got the DNA adhering to it it''s now not going to stick</span> <span m=''2062000''>everywhere.
  Good, next problem. Well,</span> <span m=''2067000''>even before that, yes? No,
  we''ll take the whole library.</span> </p><p><span m=''2071000''>We''ve gotten the
  library scattered out on this filter.</span> </p><p><span m=''2075000''>Good, so
  hang on to that one for a second. First off,</span> <span m=''2079000''>do we even
  know where that clone is? How did we know where the piece of</span> <span m=''2082000''>DNA
  stuck? I mean, I drew it as red. But,</span> <span m=''2086000''>how do we know
  where that red spot is? Yeah? Oh yeah,</span> <span m=''2090000''>you see the problem
  is if I just wash it over there,</span> <span m=''2093000''>unless you have, you
  know, Superman vision, you''re not going to</span> <span m=''2097000''>know where
  that probe is. So, you''re proposing, the first</span> <span m=''2101000''>thing
  we better do is radioactively label the probe.</span> </p><p><span m=''2105000''>So,
  let''s put a radioactive label on the probe, OK?</span> </p><p><span m=''2108000''>Radio
  label, and it turns out you can radio label probes by using</span> <span m=''2112000''>these
  enzymes that can add a radioactive phosphate group,</span> <span m=''2115000''>etc.
  So, now, when it''s radioactive, we put it here.</span> </p><p><span m=''2119000''>And
  now we have a radioactive signal here. How are we going to</span> <span m=''2122000''>find
  our radioactive signal? We put it up against x-ray films.</span> </p><p><span m=''2126000''>We
  take our filter. We dry it off.</span> </p><p><span m=''2130000''>We slap it onto
  a piece of x-ray film. We let it expose overnight.</span> </p><p><span m=''2133000''>We
  develop the x-ray film. And, we''ll see a black dot.</span> </p><p><span m=''2136000''>We''d
  better actually have taken some care to take a little</span> <span m=''2139000''>radioactive
  pen and make a couple of fiducial marks around the corners.</span> </p><p><span
  m=''2143000''>Otherwise, we''re not going to know where our black dot corresponds
  to.</span> </p><p><span m=''2146000''>But, assume we''ve made a couple of dots and
  we know how to line up our</span> <span m=''2149000''>x-ray film to our filter.
  Now, we go back to our filter.</span> </p><p><span m=''2153000''>We say, uh-huh,
  there is a black dot corresponding to the location of</span> <span m=''2156000''>the
  radioactive probe right there.</span> </p><p><span m=''2160000''>That was, as you
  said, where the colony used to be that we</span> <span m=''2166000''>wished we still
  had [LAUGHTER] because we cooked it in the</span> <span m=''2172000''>autoclave,
  which is too bad. So, what should we do about that?</span> </p><p><span m=''2179000''>Yep?
  So, if I did it one colony at a time, I would know exactly which</span> <span m=''2185000''>one
  it came from. But, it could take a long time.</span> </p><p><span m=''2192000''>Sorry?
  So, plate it first onto a plate of agar.</span> </p><p><span m=''2195000''>Take
  a filter, and press the filter up against the</span> <span m=''2199000''>plate and
  make a copy of it. Replicaplate (sic) that. It turns</span> <span m=''2203000''>out,
  that''ll work. There are two different approaches</span> <span m=''2206000''>and
  both of you were right. One approach is to replicaplate it.</span> </p><p><span
  m=''2210000''>Plate it first on a normal plate, and lay a piece of filter on top
  of</span> <span m=''2214000''>it, and a little bacteria will stick in the same patterns.</span>
  </p><p><span m=''2218000''>Peel it off, and you now have it. Alternatively,</span>
  <span m=''2221000''>now in the presence of robotics, you can use a robot to take
  these</span> <span m=''2225000''>colonies into microtiter plates, and you can screen
  the individual</span> <span m=''2228000''>wells by stamping them onto a filter,
  things like that.</span> </p><p><span m=''2232000''>And frankly, that''s how we
  do it now. If you want to screen the</span> <span m=''2235000''>human genome, at
  least set up a library with a few tens of thousands</span> <span m=''2239000''>or
  hundreds of thousands such things. And, we can read off from a grid</span> <span
  m=''2243000''>which one it was, and we go back to our master</span> <span m=''2246000''>microtiter
  plates where we have. But, either way, we need to have a</span> <span m=''2250000''>living
  copy of the library. But, that''s how you do it.</span> </p><p><span m=''2254000''>So
  now, we''re in business. We have a living copy of the</span> <span m=''2259000''>library.
  We make a filter containing that.</span> </p><p><span m=''2263000''>We cook the
  filter in the autoclave. We add a radioactive probe.</span> </p><p><span m=''2268000''>Wherever
  it sticks, it matches by the wonders of Crick-Watson base</span> <span m=''2273000''>pairing.
  We''re in business. Yes? So now, there was this issue.</span> </p><p><span m=''2278000''>I
  mean, how do I know that that sequence doesn''t appear multiple</span> <span m=''2283000''>times
  in the human genome? That''s one issue. So, I''m going to</span> <span m=''2288000''>have
  to pull out each of the positive hits I get and check it out.</span> </p><p><span
  m=''2293000''>I''m going to have to analyze the clone because just knowing that
  it</span> <span m=''2298000''>hybridized to that might not tell me it''s the beta-globin
  gene,</span> <span m=''2303000''>but at least it''s probably a good start, right?
  I''ve narrowed it down.</span> </p><p><span m=''2308000''>But, yes? Wait a second,
  right. We said there were 512 possibilities,</span> <span m=''2313000''>and I said,
  bear with me, let''s suppose we knew which one it</span> <span m=''2319000''>was
  and we used it. Well, how are we going to know</span> <span m=''2325000''>which
  one it is? Well, we could do the experiment</span> <span m=''2331000''>512 times,
  and one of them would work. That''s lousy.</span> </p><p><span m=''2337000''>We
  could go and make 512 ologotes and simultaneously throw them in the</span> <span
  m=''2343000''>same seal-a-meal bag. That actually works.</span> </p><p><span m=''2347000''>How
  do you make 512 ologotes? How do you make an ologote, by the</span> <span m=''2350000''>way?
  To make an ologonucleotide, there''s very fancy chemistry that''s</span> <span m=''2353000''>been
  developed, which someone won a Nobel Prize.</span> </p><p><span m=''2356000''>Nowadays,
  of course, if you need an ologote made, how do you do it?</span> </p><p><span m=''2360000''>Go
  to the catalog, that''s right. In fact, you can go on the web,</span> <span m=''2363000''>type
  in the sequence you want, and there''s a machine that will make</span> <span m=''2366000''>it.
  You can have it tomorrow. So, it turns out, that''s how you</span> <span m=''2369000''>make
  ologonucleotides today. There are good machines for it.</span> </p><p><span m=''2372000''>And,
  it turns out that if you wanted to, so what you do is you</span> <span m=''2376000''>type
  into the computer the following. You type in, please make me an</span> <span m=''2380000''>ologote
  that starts, put a C in the first position,</span> <span m=''2384000''>a C in the
  second position. And, what are you going to put in the</span> <span m=''2387000''>third
  position? Just tell the computer to put in a</span> <span m=''2391000''>random mix
  of all four. Then, a G in this position,</span> <span m=''2395000''>a C in that
  position, and then a random mix of all four.</span> </p><p><span m=''2399000''>Then,
  put in a G and an A, and then put in a 50/50 mix of T and</span> <span m=''2403000''>A.
  In fact, in one synthesis,</span> <span m=''2406000''>by telling the computer to
  just add a mixture at certain steps,</span> <span m=''2409000''>it''ll simultaneously
  synthesize a mixture of all 512 possibilities for</span> <span m=''2412000''>you.
  So actually, a single synthesis will suffice to</span> <span m=''2416000''>get a
  mixture of 512. You take your mixture of 512,</span> <span m=''2419000''>wash it
  over the filter, etc. Now, your point still stands. How do we</span> <span m=''2422000''>know
  that there''s not something else in the genome that has this,</span> <span m=''2425000''>etc.?
  But at least we can find all the specific positives associated</span> <span m=''2428000''>with
  this, and we can analyze them further as we''ll talk about next</span> <span m=''2431000''>time
  more about how you actually analyze them.</span> </p><p><span m=''2435000''>And,
  of course, whether 18 is the right number of</span> <span m=''2438000''>bases, or
  you might prefer to have a longer probe or shorter probes,</span> <span m=''2441000''>or
  two probes, these are all the cooking tips molecular biologists</span> <span m=''2444000''>worry
  about. But, given a sequence of an amino acid</span> <span m=''2448000''>sequence,
  you can infer, although with redundancy,</span> <span m=''2451000''>a nucleotide
  sequence. Given a nucleotide sequence,</span> <span m=''2454000''>you can make an
  ologonucleotide probe. Given a nucleotide probe,</span> <span m=''2458000''>you
  can wash it over the filter. You can find the colonies that have</span> <span m=''2462000''>it,
  and therefore you could clone by hybridization.</span> </p><p><span m=''2467000''>So,
  we''ll call this one cloning by hybridization,</span> <span m=''2472000''>or cloning
  by sequence. OK, now, there are other ways to do</span> <span m=''2477000''>it,
  or by sequence here. Of course, as someone correctly</span> <span m=''2481000''>noted,
  if the entire sequence of the human genome has been already</span> <span m=''2486000''>sequenced
  as it has right now, if you knew the amino acid sequence,</span> <span m=''2491000''>you
  could do this hybridization not using filters and radioactive probes,</span> <span
  m=''2496000''>but just doing it in silico. You can do it in the computer,</span>
  <span m=''2502000''>and that will work as well. So now, let''s do the next one.
  Last cloning</span> <span m=''2510000''>expedition: I''d like to clone the gene
  for Huntington''s disease or</span> <span m=''2517000''>cystic fibrosis or something
  like that. Cloning a disease gene,</span> <span m=''2525000''>such as Huntington''s
  disease, is a dominantly inherited disorder</span> <span m=''2533000''>passed to
  some of the offspring, causes a brain degeneration that</span> <span m=''2543000''>onsets
  typically in the fifth decade of life.</span> </p><p><span m=''2553000''>Let''s
  clone that gene. Can we do it by method number one,</span> <span m=''2556000''>cloning
  by complementation? No, because we don''t have a bacteria</span> <span m=''2559000''>that
  has Huntington''s disease. We don''t have mice that have</span> <span m=''2562000''>Huntington''s
  disease. And, we can''t certainly shoot up</span> <span m=''2566000''>people and
  try to rescue the phenotype and all that.</span> </p><p><span m=''2569000''>That''s
  not going to work. Number two, how about doing it by</span> <span m=''2572000''>number
  two? Let''s just get the protein for Huntington''s disease,</span> <span m=''2576000''>get
  its amino acid sequence, and then find its nucleotide</span> <span m=''2579000''>sequence.
  Pretty good. What''s the protein for Huntington''s</span> <span m=''2583000''>disease?
  Huntase. No, it''s actually called Huntington</span> <span m=''2587000''>it turns
  out. But, at the time that people went off</span> <span m=''2591000''>trying to
  find the gene for Huntington''s disease,</span> <span m=''2595000''>I''m afraid
  they didn''t know. They had no idea what the gene was</span> <span m=''2599000''>that
  caused Huntington''s disease. That was the point. They wanted to</span> <span m=''2603000''>use
  molecular biology to find the gene when they didn''t even</span> <span m=''2607000''>know
  the protein. So, we can''t use our method number</span> <span m=''2612000''>two.
  So, how are we going to find it? The disease does lead to</span> <span m=''2617000''>degeneration
  of nervous cells. Study nerve cells. So, we could</span> <span m=''2622000''>take
  brain biopsies from patients who have died of Huntington''s</span> <span m=''2627000''>disease,
  and people did that. But, nerve cells that die, a lot of</span> <span m=''2632000''>stuff
  goes on. All sorts of proteins go wrong,</span> <span m=''2637000''>and it''s stuff.
  The problem with studying tissue</span> <span m=''2642000''>from people who have
  a disease is that it''s diseased tissue.</span> </p><p><span m=''2646000''>And,
  just because you see something wrong doesn''t mean it''s a cause</span> <span m=''2650000''>rather
  than the effect of the disease. That''s why we really want</span> <span m=''2655000''>to
  find the gene and find its mutation because we know then that''s</span> <span m=''2659000''>the
  primary cause. But, how are we going to do that?</span> </p><p><span m=''2664000''>We
  don''t know its sequence. We can''t rescue it by complementation.</span> </p><p><span
  m=''2668000''>As a pure geneticist, what can we do?</span> </p><p><span m=''2673000''>Yeah,
  we know the sequence of the human genome. So,</span> <span m=''2676000''>we just
  sequence the entirety of the genome of somebody with Huntington''s</span> <span
  m=''2680000''>disease and compare it to normal. That actually may become a</span>
  <span m=''2683000''>reasonable way to do things, but the first sequence of the human</span>
  <span m=''2687000''>genome costs a couple of billion dollars. Doing it again would
  be</span> <span m=''2690000''>cheaper. We''d spend about $30 million or so,</span>
  <span m=''2694000''>but it''s pricey. Also, there would be a lot of</span> <span
  m=''2697000''>genetic variation, just random, meaningless</span> <span m=''2701000''>polymorphism
  between individuals. The human genome differs between any</span> <span m=''2705000''>two
  people by about one letter or 1, 00. So, we would see about 3</span> <span m=''2711000''>million
  differences between the person with Huntington''s and the</span> <span m=''2716000''>wild
  type reference sequence on Google. We wouldn''t know which one</span> <span m=''2722000''>causes
  it. Suppose you have a family tree. How could we use it?</span> </p><p><span m=''2727000''>Compare
  the children and the parents.</span> </p><p><span m=''2733000''>That''s all right.
  What does a geneticist do with a</span> <span m=''2737000''>family tree? What did
  Sturtevant teach us: genetic mapping.</span> </p><p><span m=''2742000''>Suppose
  we were to study a family tree of individuals with</span> <span m=''2747000''>Huntington''s
  disease. And suppose on the chromosome where</span> <span m=''2752000''>the Huntington''s
  disease gene lives, we were to look at genetic markers.</span> </p><p><span m=''2757000''>Could
  we do genetic linkage analysis?</span> </p><p><span m=''2763000''>Genetic linkage
  analysis that would allow us to know that there was a</span> <span m=''2769000''>marker
  here, some kind of a marker, a DNA marker, a DNA variation that</span> <span m=''2775000''>was
  co-inherited with that showed linkage with Huntington''s disease?</span> </p><p><span
  m=''2781000''>We could do that just by finding that across a family,</span> <span
  m=''2787000''>there tended to be very little genetic recombination between this</span>
  <span m=''2793000''>marker and Huntington''s disease. Now, how would we know to
  look here?</span> </p><p><span m=''2798000''>You wouldn''t. We''d try markers all
  over the genome.</span> </p><p><span m=''2804000''>Next chromosome, next chromosome;
  if we tried genetic</span> <span m=''2810000''>variations all over the human genome,
  we would eventually find that some</span> <span m=''2816000''>genetic markers in
  the human genome tended to be co-inherited along with</span> <span m=''2822000''>Huntington''s
  disease. It turns out that that''s enough.</span> </p><p><span m=''2827000''>This
  will tell us approximately where this unknown gene must live.</span> </p><p><span
  m=''2832000''>Here''s a portion of the chromosome where the unknown Huntington''s</span>
  <span m=''2837000''>disease gene lives. Here''s a genetic variant,</span> <span
  m=''2842000''>and here''s a genetic variant, a marker, that shows correlation.</span>
  </p><p><span m=''2847000''>Maybe there''s only 1% recombination here, and 1% recombination
  here.</span> </p><p><span m=''2852000''>And, that''s the powerful thing about Sturtevant''s
  idea.</span> </p><p><span m=''2856000''>It works in fruit flies. It works in humans.
  If I have any</span> <span m=''2861000''>genetic variation and it''s 99% correlated,
  or only recombines 1% of</span> <span m=''2865000''>the time, it tells me that this
  unknown gene must be nearby.</span> </p><p><span m=''2870000''>So, I could use this
  genetic marker as a DNA probe to wash over a</span> <span m=''2875000''>library
  to get a big piece of DNA from this region.</span> </p><p><span m=''2880000''>I
  can take this piece of DNA and use it as a probe,</span> <span m=''2884000''>a radioactive
  probe, to get an overlapping piece of DNA.</span> </p><p><span m=''2889000''>I can
  use the end of this DNA as a probe to wash over a library and get</span> <span m=''2893000''>the
  next piece of DNA. And, I can do the same thing here.</span> </p><p><span m=''2898000''>Once
  I have any piece of DNA that''s even vaguely in the neighborhood,</span> <span m=''2902000''>I
  can use it as a probe to wash over a library and get a piece of DNA,</span> <span
  m=''2907000''>use it to get the next piece, the next piece, the next piece,</span>
  <span m=''2911000''>in a process that was called chromosomal walking.</span> </p><p><span
  m=''2916000''>That gives me a series of clones that I know must cover the region</span>
  <span m=''2920000''>for this unknown gene. I then begin to analyze them and I</span>
  <span m=''2925000''>say, let''s look at some more genetic markers, a genetic marker
  a little</span> <span m=''2930000''>closer and a little closer and a little closer.</span>
  </p><p><span m=''2935000''>Which ones show perfect correlation with Huntington''s
  disease?</span> </p><p><span m=''2940000''>And, that narrows me down to a small
  number of clones that must contain</span> <span m=''2944000''>the gene, even though
  I had no idea in advance what that gene was.</span> </p><p><span m=''2949000''>This
  is called cloning by position. And, that''s a very powerful</span> <span m=''2954000''>technique
  of genetics because you don''t need to know in advance what''s</span> <span m=''2959000''>wrong
  with a diseased gene. You first figure out where it is,</span> <span m=''2964000''>and
  then you get the clones to figure out what it is. So,</span> <span m=''2969000''>this
  actually works. Now, the process of getting the next</span> <span m=''2973000''>piece,
  and the next clone, and the next clone, is unbelievably</span> <span m=''2977000''>boring
  and tedious. And, for Huntington''s disease,</span> <span m=''2981000''>this process
  took nine years. Of course, now, how would you do it?</span> </p><p><span m=''2985000''>Go
  to the web because with all of this process of the human genome,</span> <span m=''2989000''>you''ve
  got all these clones laid out already. And so,</span> <span m=''2993000''>the work
  that used to take years now is, once you have a genetic marker</span> <span m=''2997000''>that''s
  close to Huntington''s you can just look up all the clones in the</span> <span m=''3001000''>neighborhood
  and actually all the sequences in the neighborhood.</span> </p><p><span m=''3005000''>So,
  this process has gone from nine years to, if you have do this again,</span> <span
  m=''3009000''>you could get that region for Huntington''s disease in a couple</span>
  <span m=''3014000''>weeks. Now the question is, how do you analyze that region?</span>
  </p><p><span m=''3018000''>How do you know what''s in that region? How do you know
  what the</span> <span m=''3023000''>genes are that are in that region? And that''s
  what we''ll talk about</span> <span m=''3027000''>next time.</span> </p>'
type: course
uid: b48b7f97aa23237e5dec4f1f9ff42e71

---
None