---
about_this_resource_text: <p><strong>Topics covered:</strong> Nervous System 3</p>  <p><strong>Instructors:</strong>
  Dr. Andrew Chess, Guest Lecturer</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec28.jpg
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-28-nervous-system-3/lec28.jpg
  title: lec28.jpg
  type: null
  uid: 95134cf91d6ea03b4f547fb52e30b6c6
- id: Video-YouTube-Stream
  media_location: rWG1hLvoP-U
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Video-YouTube-Stream
  type: Video
  uid: 4cc3b360f68f852b4a9f288549bb0015
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/rWG1hLvoP-U/default.jpg
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 4adb61d269d0fc8c56108f05b5253da9
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Video-iTunes U-MP4
  type: Video
  uid: 868254b8f35c737b2e863c6f6632b7cb
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec28-19nov2004-220k.mp4
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Video-Internet Archive-MP4
  type: Video
  uid: f5309cb6066e1877566e1f031c91f5cc
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 275e29b3ff6c669c29a78b64e01edd81
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-19nov2004-1000.mp3
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Video-Internet Archive-MP3
  type: Video
  uid: fc60b4badea4eb313fe1e4d7450efd77
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: ba8ca3760651ec8de892c126b0d56e15
- id: Thumbnail-OCW-JPG
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 7246786bb38dbf4420fc4dfda439b1a1
- id: 3Play-3PlayYouTubeid-MP4
  media_location: rWG1hLvoP-U
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 12dcb2d30fe46b36ce8e7554d3ea1a2a
- id: rWG1hLvoP-U.srt
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-28-nervous-system-3/rWG1hLvoP-U.srt
  title: 3play caption file
  type: null
  uid: ceca13193bb679efa6e6afe012df3d4c
- id: rWG1hLvoP-U.pdf
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-28-nervous-system-3/rWG1hLvoP-U.pdf
  title: 3play pdf file
  type: null
  uid: 46c78c4ca461b8b2a1611518690dc3aa
- id: Caption-3Play YouTube id-SRT
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: 8f2f3555c05c86ca520e7351801fac9b
- id: Transcript-3Play YouTube id-PDF
  parent_uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: 99a9ec24b6381d81f7be24e74318925e
inline_embed_id: 56864855lecture28:nervoussystem381645611
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-28-nervous-system-3
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-28-nervous-system-3
template_type: Tabbed
title: 'Lecture 28: Nervous System 3'
transcript: <p><span m='1000'>Hello, everybody. Can we get started?</span> </p><p><span
  m='5000'>So my name is Andrew Chess. And I'm lecturing today replacing</span> <span
  m='10000'>Eric Lander for the day. He had to be out of town,</span> <span m='15000'>something
  that he could not reschedule. He really tries to</span> <span m='20000'>arrange
  his very busy schedule so that he is here,</span> <span m='25000'>but this was something
  that could not be rearranged. Anyway.</span> </p><p><span m='30000'>So I am a professor
  at Harvard Medical School,</span> <span m='34000'>but I have a long history here
  at MIT, including being on the faculty</span> <span m='38000'>here for a number
  of years. I used to teach undergraduates.</span> </p><p><span m='42000'>And even
  going back further than that, I actually took this class.</span> </p><p><span m='46000'>It
  was called 7.01 without any extra number at that time.</span> </p><p><span m='50000'>Now
  it is what, 7.012 or 7. 13? 0-1-2. Anyway. So it was</span> <span m='54000'>called
  7.01. And it was an extremely interesting introduction</span> <span m='58000'>to
  biology back then. It was some time over 20 years ago.</span> </p><p><span m='61000'>I'm
  not sure exactly how many years. I kind of stopped counting at 20.</span> </p><p><span
  m='65000'>So when Eric called me, when Professor Lander called me up to ask</span>
  <span m='69000'>me to give this lecture, I talked to him for a while and I</span>
  <span m='72000'>agreed to do it. And then I was thinking about what</span> <span
  m='76000'>to present. And I went over the material that he has presented to</span>
  <span m='80000'>earlier this week. And I discussed with him some of</span> <span
  m='83000'>the things that he likes to do for the third lecture on neurobiology.</span>
  </p><p><span m='87000'>And I also thought about some of the things that I would
  like to do.</span> </p><p><span m='91000'>And so one of the things that occurred
  to me,</span> <span m='94000'>so it's probably occurred to you from hearing Eric
  talk about</span> <span m='98000'>neurobiology earlier this week that he is very
  enthusiastic about the</span> <span m='102000'>subject. He always talks about it
  as being one of the driving forces</span> <span m='105000'>that led him to enter
  biology from the realm of mathematics where he</span> <span m='109000'>started his
  academic career. Anyway. So he's always talked to</span> <span m='113000'>me, in
  the years I've known him, about how he loves neurobiology.</span> </p><p><span m='117000'>And
  I was thinking about it. It is, in some ways,</span> <span m='121000'>kind of ironic
  that Eric loves neurobiology so much because in some</span> <span m='125000'>ways
  he's been a big trouble-maker for neurobiology.</span> </p><p><span m='129000'>Let
  me explain. So Eric, Professor Lander is, of course,</span> <span m='133000'>as
  you all know, was an instrumental driving force behind the sequencing</span> <span
  m='137000'>of the human genome. Before all those efforts,</span> <span m='141000'>over
  the last decade, people used to go around, biologists used to go</span> <span m='145000'>around
  to each other and they would talk.</span> </p><p><span m='150000'>And they would
  say there are around 100,000 genes in the human genome,</span> <span m='153000'>or
  100,000 genes in a mouse genome. Mammalian genomes have around</span> <span m='156000'>100,000
  genes. Sometimes some people would say 90,</span> <span m='159000'>00 genes. Sometimes
  they would say 110,000 genes.</span> </p><p><span m='162000'>But generally it was
  around 100, 00 genes. And everybody was</span> <span m='165000'>comfortable with
  that. In fact, right around the turn of</span> <span m='168000'>the century when
  the stock market was going way up,</span> <span m='172000'>the Internet bubble and
  biotech bubble and everything,</span> <span m='175000'>estimates of the number of
  genes in a human genome actually</span> <span m='178000'>went higher also. They
  went up as high as 120,</span> <span m='182000'>00, 150,000. And this, I think,
  was because various companies were</span> <span m='186000'>competing to have the
  most genes on their kind of micro array or</span> <span m='190000'>whatever they
  were selling. They wanted to say we have the most,</span> <span m='194000'>and so
  they kept saying more. The academic scientist usually</span> <span m='198000'>still
  stayed around 100, 00 in terms of their thinking.</span> </p><p><span m='202000'>OK.
  The other thing that was of a lot of</span> <span m='206000'>use to neurobiologists
  in terms of them feeling that their problem of</span> <span m='210000'>trying to
  figure out how the brain is set up as a tractable problem was</span> <span m='214000'>that
  they thought there were going to be lots of genes available.</span> </p><p><span
  m='218000'>So there were 100,000 genes in the genome, and around half of them,</span>
  <span m='222000'>people would say, are probably brain-specific.</span> </p><p><span
  m='226000'>So there were a variety of pieces of evidence that people would think</span>
  <span m='230000'>that a lot of the genes in the genome would be brain-specific.</span>
  </p><p><span m='234000'>And that still remains the case. But, as you know from Eric
  Landers</span> <span m='238000'>and other people's work sequencing the genome and
  mouse genome and</span> <span m='242000'>other genomes now, it looks like mammals
  now have only</span> <span m='246000'>around 30,000 to 40,000 genes. Now, if Eric
  told you a different</span> <span m='250000'>number listen to his number. What did
  he say? [20,000 to 25,</span> <span m='256000'>00?]. OK. Anyway. So many fewer than
  100,000. OK?</span> </p><p><span m='262000'>A small number. A fly is thought to
  have only around 15,000 genes.</span> </p><p><span m='268000'>So Lander is now saying
  that humans don't have that many more genes than</span> <span m='271000'>flies.
  So this presented a problem for neurobiologists,</span> <span m='274000'>because
  even if you have half of them brain-specific,</span> <span m='277000'>you still
  don't have nearly as many genes to play with to make the</span> <span m='281000'>complex
  structure of the brain as you did when there were 100,</span> <span m='284000'>00
  genes in the genome. So the brain, of course,</span> <span m='287000'>is an extremely
  complicated structure. There are thought to be</span> <span m='291000'>somewhere
  between 100 billion and a trillion different neurons in the</span> <span m='294000'>brain,
  and they also fall into many, many different neural types.</span> </p><p><span m='298000'>And
  so the developmental process, developmental biology is something</span> <span m='301000'>that
  some of you will study in future courses and you'll have had</span> <span m='305000'>some
  introduction to here, that's how you get from a single</span> <span m='309000'>fertilized
  egg, a single cell to the complex</span> <span m='313000'>organism. In the brain
  it's a particularly difficult problem</span> <span m='316000'>because you have so
  many different types of cells and so many cells.</span> </p><p><span m='320000'>And
  each cell then makes all of these complex connections.</span> </p><p><span m='324000'>A
  given neuron might connect to 1, 00 or a few thousand different other</span> <span
  m='328000'>cells as a normal process. So forming all these different kinds</span>
  <span m='332000'>of neurons and wiring up is a very daunting problem.</span> </p><p><span
  m='336000'>So what I thought I would do today would be to focus on two examples</span>
  <span m='340000'>where in each case, starting with either one gene or a</span> <span
  m='344000'>small number of genes you get a lot of complexity.</span> </p><p><span
  m='348000'>So this would then allow the smaller number of total gene number</span>
  <span m='352000'>in the human genome to allow a lot of different kinds of proteins
  and</span> <span m='356000'>maybe provide some explanations for certain parts of
  the complexity</span> <span m='360000'>of the brain. Now, by no means am I going
  to</span> <span m='364000'>attempt to explain all of how the brain develops.</span>
  </p><p><span m='368000'>That would take, well, at least one course,</span> <span
  m='372000'>more likely a few different courses to actually get a good appreciation</span>
  <span m='376000'>of that, but I'm going to go through a couple of very intriguing
  examples</span> <span m='380000'>that are approachable at the level of this class.</span>
  </p><p><span m='384000'>OK. So the standard way that we</span> <span m='403000'>think
  about how genetic information gets made into proteins is that</span> <span m='409000'>there
  is DNA and then RNA and then proteins. I hope at this point in 7.</span> </p><p><span
  m='415000'>1 this is all familiar to you. Good. OK. So the DNA sequence</span> <span
  m='421000'>gets transcribed into an RNA, and then there's a splicing event</span>
  <span m='426000'>which takes bits and pieces of the RNA, puts them together,</span>
  <span m='432000'>and then there's an area of the RNA that tells the ribosome to
  start</span> <span m='438000'>making protein. And so you get the protein</span>
  <span m='444000'>synthesis. Everything is following from the</span> <span m='449000'>blueprint
  that was in the DNA. So what I'm going to talk about</span> <span m='453000'>today
  are two different examples. One of them involves alternative</span> <span m='457000'>splicing.</span>
  </p><p><span m='469000'>And that will be causes where instead of there being a static</span>
  <span m='474000'>always reproduced way of going from DNA to RNA to a protein,</span>
  <span m='479000'>that there are different alternative splicing events that can occur.</span>
  </p><p><span m='484000'>And this can allow one gene to make multiple different protein
  products.</span> </p><p><span m='488000'>And then I'm going to go over another way
  that you can violate</span> <span m='493000'>this central dogma, this DNA, RNA to
  protein,</span> <span m='498000'>which is something called RNA editing. So, as you
  might imagine</span> <span m='502000'>from the common usage of the word editing,
  by editing what we mean is</span> <span m='507000'>that the RNA sequence itself
  is actually changed so that it no</span> <span m='512000'>longer reflect the exact
  nucleotide sequence of the DNA.</span> </p><p><span m='517000'>This can also add
  diversity to the number of potential encoded proteins.</span> </p><p><span m='521000'>I
  want to make sure that I, I'm going to talk about</span> <span m='526000'>neural-specific
  examples, but I want to mention that these</span> <span m='530000'>processes, alternative
  splicing and RNA editing are used also by other</span> <span m='535000'>parts of
  the developing animal, and also in other plants and other</span> <span m='540000'>organisms,
  but not just by the brain to generate diversity.</span> </p><p><span m='545000'>So
  these are mechanisms that are widely used. But some of the most</span> <span m='548000'>striking
  examples, as you'll see from my lecture and in</span> <span m='552000'>further reading
  that you might do in the future, some of the most</span> <span m='556000'>striking
  examples come from the nervous system.</span> </p><p><span m='559000'>And that's
  not surprising given the complexity of the nervous system and</span> <span m='563000'>the
  fact that there are so many genes out there ready to help with</span> <span m='567000'>this
  complexity. So the first I'll turn to</span> <span m='571000'>alternative splicing.</span>
  </p><p><span m='584000'>So first, before getting into the extremely complex case
  that I'm</span> <span m='587000'>going to focus on, I'm going to just briefly,</span>
  <span m='591000'>by way of introduction, go over a standard alternative</span> <span
  m='595000'>splicing scenario.</span> </p><p><span m='615000'>OK. So in a gene for
  which there is no alternative splicing,</span> <span m='623000'>if I draw the exons
  as boxes and the introns as lines,</span> <span m='631000'>what winds up happening
  is you have the first exon spliced to the second</span> <span m='640000'>one, second
  to the third, third to the forth. And so what you</span> <span m='648000'>wind up
  with is a messenger RNA.</span> </p><p><span m='660000'>So here is the messenger
  RNA which has been spliced from the primary</span> <span m='663000'>transcript.
  The primary transcript, of course, reflects the actual</span> <span m='667000'>structure
  of the DNA in terms of sequence also,</span> <span m='671000'>because in the genomic
  DNA you'd also have areas that are going to be</span> <span m='674000'>exon, intron,
  exon intron exactly like this. So this is just a</span> <span m='678000'>general
  example of alternative splicing, I'm sorry,</span> <span m='682000'>of regular splicing.
  So then alternative splicing would involve</span> <span m='686000'>something like
  this. You'd have 1, 2,</span> <span m='692000'>3A, 3B, 4. So then what happens is
  you have normal splicing from 1 to 2.</span> </p><p><span m='701000'>And then 2
  could either go to 3A, which will then go to 4 leaving 3B</span> <span m='711000'>out.
  Or the alternative is that 2 can skip 3A, go to 3B,</span> <span m='720000'>which
  will then splice to 4. So this allows then two different</span> <span m='727000'>messengers
  to be formed.</span> </p><p><span m='753000'>So this 3A and 3B might encode a slightly
  different sequence and</span> <span m='757000'>might then allow two distinct proteins
  with different functions to</span> <span m='761000'>be forwarded from one message.
  So this is an example, a simple</span> <span m='765000'>example, a general example
  of alternative splicing.</span> </p><p><span m='769000'>So from one gene you have
  two proteins.</span> </p><p><span m='774000'>The example that I'm going to focus</span>
  <span m='782000'>on today, instead of going from one gene to two proteins,</span>
  <span m='787000'>allows you to go from one gene to 38, 00 different possibilities.</span>
  </p><p><span m='793000'>It's actually 38,016 to be exact, and I'll explain to you
  why, but 38,</span> <span m='798000'>00 will have occurred to you that this is larger
  than the number of</span> <span m='803000'>genes that Eric Lander says are in the
  human genome.</span> </p><p><span m='809000'>It's certainly larger than the number
  of genes in the fly genome.</span> </p><p><span m='814000'>And this example I'm
  giving you is from a single gene in the fruit fly.</span> </p><p><span m='819000'>This
  one gene can come in 38,000 different forms.</span> </p><p><span m='825000'>The
  gene is called drosophila DSCAM.</span> </p><p><span m='839000'>It's named for a
  human gene which was cloned first and characterized</span> <span m='843000'>first
  which was called just plain DSCAM. What DSCAM stands for is</span> <span m='848000'>Down
  syndrome cell adhesion molecule.</span> </p><p><span m='853000'>And let me just
  explain briefly why</span> <span m='873000'>it has this name. I don't think that
  this name is</span> <span m='876000'>actually relevant so much to the biology, and
  it certainly not</span> <span m='879000'>relevant to the alternative splicing because
  the human gene and the mouse</span> <span m='882000'>gene, neither of them have
  a lot of alternative splicing.</span> </p><p><span m='885000'>This is something
  that is particular to the fly.</span> </p><p><span m='888000'>That's something I
  will return to later in lecture.</span> </p><p><span m='891000'>This name Down syndrome
  cell adhesion molecule came about because</span> <span m='894000'>this was cloned
  first from human and it's located on human</span> <span m='897000'>chromosome 21.</span>
  </p><p><span m='905000'>Chromosome 21 is normally present in two copies in every
  individual.</span> </p><p><span m='910000'>In individuals who wind up with three
  copies of chromosome 21,</span> <span m='915000'>something called trisomy 21, trisomy
  21 causes Down syndrome,</span> <span m='921000'>which is a syndrome that has some
  brain manifestations like mental</span> <span m='926000'>retardation, and also has
  a number of other problems associated</span> <span m='932000'>with it. When the
  people who found this gene</span> <span m='937000'>found it they named it. So they
  gave the first part of its</span> <span m='942000'>name, the DS comes from Down
  syndrome. Cell adhesion molecule</span> <span m='947000'>comes from the fact that
  this gene is similar in structure to a lot of</span> <span m='952000'>known cell
  adhesion molecules that are encoded by many different</span> <span m='957000'>loci
  in the genome. And they initially thought that</span> <span m='962000'>perhaps,
  and this gene is expressed in the brain. And they initially</span> <span m='966000'>thought
  that perhaps having an extra copy, a third copy of this gene</span> <span m='970000'>might
  be what's causing a lot of the brain phenotypes.</span> </p><p><span m='974000'>Subsequent
  work has not provided further evidence for that.</span> </p><p><span m='979000'>So
  at this point what I would say is that the name of the gene is Down</span> <span
  m='983000'>syndrome cell adhesion molecule, DSCAM. It's on human chromosome 21.</span>
  </p><p><span m='987000'>It may play a role in Down syndrome but there isn't --</span>
  <span m='992000'>The name is really the best evidence that it plays a role,</span>
  <span m='997000'>just the fact that they named it that. OK. But in the fly this
  is</span> <span m='1002000'>an extremely interesting molecule because, as I mentioned,</span>
  <span m='1007000'>it can come in 38,000 different forms. OK. So as for why you would</span>
  <span m='1012000'>have a cell adhesion molecule in the brain, I just want to</span>
  <span m='1017000'>mention briefly. So Professor Lander went over with</span> <span
  m='1022000'>you the structure of a neuron. That neurons have cell bodies and</span>
  <span m='1026000'>axons and growth cones which allow them to get to wherever they're</span>
  <span m='1030000'>supposed to connect. One of the types of molecules that</span>
  <span m='1034000'>allows an axon as it's growing, the growth cone to lead an axon</span>
  <span m='1038000'>along a complex path is to interact with various structures that
  it</span> <span m='1042000'>encounters. And so cell adhesion molecule is one of
  the kinds of</span> <span m='1046000'>molecules that can allow the growth cone and
  then the rest of the axon</span> <span m='1050000'>to interact with various other
  cells or other extracellular substrates,</span> <span m='1054000'>proteins that
  have been deposited by other kinds of cells as they make</span> <span m='1058000'>their
  way and make the appropriate connections in the brain.</span> </p><p><span m='1063000'>So
  cell adhesion molecules are one of the mechanisms.</span> </p><p><span m='1066000'>There
  are also mechanisms that allow cells to respond to gradients</span> <span m='1070000'>of
  chemical signaling messengers. Question? Yes. Could you explain</span> <span m='1074000'>what
  a growth cone is? Oh, I'm sorry. That was not</span> <span m='1078000'>covered?
  OK. So the neuron has a cell body</span> <span m='1082000'>with a nucleus and all
  the other stuff that's in regular cells that</span> <span m='1086000'>you learned
  about. It then has an axon which then</span> <span m='1090000'>allows it to connect.
  And this connection could be very</span> <span m='1094000'>far away. In the case
  of, for example, a motor neuron in the</span> <span m='1099000'>spinal cord that's
  intermating a muscle in the foot,</span> <span m='1103000'>that single cell would
  have its cell body in the spinal cord and its axon</span> <span m='1107000'>go all
  the way out to the foot. OK? So that's just an example of</span> <span m='1112000'>one
  very long neuron. Some of them are very long.</span> </p><p><span m='1116000'>Some
  of them are shorter. So this is the axon. At the tip of the axon</span> <span m='1120000'>is
  this thing which looks sort of like my son's mitten when it's cold</span> <span
  m='1124000'>outside. But this is the growth cone. And basically what's going on</span>
  <span m='1128000'>is as the axon is growing out in this direction it's feeling its
  way.</span> </p><p><span m='1132000'>And there might be cell adhesion molecules
  on these different</span> <span m='1136000'>protrusions that if they attach really
  well --</span> <span m='1140000'>Like let's say that this area over here is stickier
  for this particular</span> <span m='1146000'>growth cone than this area over here.
  Then this growth axon is more</span> <span m='1152000'>likely to grow in that direction.
  OK. So cell adhesion molecules are</span> <span m='1158000'>well known to play important
  roles in axon guidance.</span> </p><p><span m='1164000'>It's how axons grow in different
  directions.</span> </p><p><span m='1170000'>So I will tell you know about the DSCAM
  gene, and that will give some</span> <span m='1174000'>insight into how the 38,
  00 different forms might be used</span> <span m='1179000'>because they're going
  to provide different kinds of stickiness.</span> </p><p><span m='1183000'>Let me
  explain.</span> </p><p><span m='1194000'>So this is a drawing of the genomic organization
  of DSCAM that allows</span> <span m='1200000'>the extensive alternative splicing.</span>
  </p><p><span m='1213000'>So this diagram is similar to the diagram that I drew by
  hand for the</span> <span m='1217000'>more simple cases. But basically in DSCAM
  what you</span> <span m='1221000'>start out with is exon 1 gets spliced to exon
  2 gets spliced to</span> <span m='1226000'>exon 3, and then when you reach exon
  4 there are 12 distinct</span> <span m='1230000'>possibilities. And only one of
  the 12 is chosen.</span> </p><p><span m='1235000'>In this case the diagram shows
  it choosing, I don't know,</span> <span m='1239000'>the ninth one perhaps. Then
  exon 5 is regular so that always gets</span> <span m='1243000'>included. And exon
  6 there are 48 distinct choices.</span> </p><p><span m='1247000'>And again only
  one is chosen. Here, in this example, this one has</span> <span m='1251000'>been
  chosen at the expense of all of these other ones.</span> </p><p><span m='1255000'>Exon
  7 and 8 are normal. And exon 9 there are 33 choices.</span> </p><p><span m='1259000'>Exon
  17 there are two choices. And if you multiple 2 x 33 x 48 x 12</span> <span m='1263000'>you
  wind up with 38, 16. There is evidence from a number</span> <span m='1267000'>of
  different types of studies, including coning and sequencing,</span> <span m='1272000'>lots
  of different messenger RNAs that are already spliced that</span> <span m='1277000'>basically
  almost all of these forms can be made. So what this structure</span> <span m='1281000'>allows
  is for there to be diversity generated in important areas of the</span> <span m='1286000'>cell
  adhesion part of this molecule.</span> </p><p><span m='1291000'>The DSCAM molecule
  starts out with a</span> <span m='1313000'>number of domains which are called immunoglobulin-like
  domains.</span> </p><p><span m='1319000'>Immunoglobulin domains are named for</span>
  <span m='1328000'>immunoglobulins which is another name for antibodies.</span> </p><p><span
  m='1332000'>Antibodies help you fight infection. I don't know if that's been covered.</span>
  </p><p><span m='1336000'>Has it been covered? Yes. And the particular fold that</span>
  <span m='1340000'>they form allows recognition of foreign antigens and it also allows</span>
  <span m='1344000'>stickiness of molecules in general. So cell adhesion molecules
  often</span> <span m='1348000'>have these immunoglobulin domains. The DSCAM starts
  out with,</span> <span m='1353000'>from the N-terminus towards the C-terminus it
  starts out,</span> <span m='1359000'>the first nine domains are these Ig type domains.
  That's then followed</span> <span m='1365000'>by another kind of domain called a
  fiberonectin type domain,</span> <span m='1371000'>which that's not important. All
  of the diversity is in this</span> <span m='1377000'>nine immunoglobulin domain.
  The exon 4 diversity allows</span> <span m='1382000'>diversity of the second of
  the nine. The exon 6 alternative splicing</span> <span m='1388000'>affects the third
  out of the nine immunoglobulin folds.</span> </p><p><span m='1394000'>And the exon
  9 diversity affects the seventh. So of these nine</span> <span m='1399000'>domains
  of immunoglobulin folds that allow for different kinds of</span> <span m='1405000'>stickiness,
  a lot of them are the same.</span> </p><p><span m='1411000'>One is the same. 4,
  5 and 6 are the same.</span> </p><p><span m='1415000'>And 8 and 9 are the same.
  But 2, 3 and 7 have these</span> <span m='1420000'>differences which are encoded
  by this striking kind of genomic</span> <span m='1424000'>structure and alternative
  splicing.</span> </p><p><span m='1451000'>So how is this diversity used?</span>
  </p><p><span m='1469000'>So the early models for how DSCAM would be used stipulated
  that</span> <span m='1474000'>individual different kinds of neurons might express
  vastly reduced</span> <span m='1480000'>subsets out of the 38, 00. So let's say
  that one</span> <span m='1485000'>particular neuron type, of which there might be
  many</span> <span m='1491000'>different neurons, might express maybe ten out of
  the</span> <span m='1497000'>38,000 or even one out of 38,000. These were the different
  kinds of</span> <span m='1502000'>models that were tossed around by people who were
  thinking about this</span> <span m='1506000'>problem. But then people started to
  study it. And what turned out to be</span> <span m='1511000'>the case is that it
  looks like every kind of neuron population at first</span> <span m='1516000'>approximately
  expresses almost all of the different forms.</span> </p><p><span m='1520000'>OK?
  So these are wrong, these models. And each different</span> <span m='1525000'>neuron
  type expresses a slightly different repertoire.</span> </p><p><span m='1530000'>But
  at first approximation over 10, 00 or 20,000 forms are possible for</span> <span
  m='1535000'>each different neuron type. So that then caused people to</span> <span
  m='1541000'>scratch their heads and wonder, well, how is this used then? How is</span>
  <span m='1547000'>this used to make different kinds of neurons different from one
  another</span> <span m='1553000'>or anything in the function of them?</span> </p><p><span
  m='1559000'>So the answer to this question has emerged in part from analyses of</span>
  <span m='1565000'>individual single cells. So it turns out that an individual</span>
  <span m='1572000'>cell, and I'm using the word cell and neuron interchangeably because</span>
  <span m='1578000'>neurons are cells. And not all cells are neurons but</span> <span
  m='1585000'>all neurons are cells. So for one cell or one neuron it</span> <span
  m='1592000'>makes somewhere in the range of 10 to 50 forms.</span> </p><p><span
  m='1599000'>These are randomly chosen, apparently from the data that's</span> <span
  m='1604000'>available, from the tens of thousands of forms that are possible.</span>
  </p><p><span m='1609000'>So you can imagine that two neighboring cells that are
  otherwise</span> <span m='1614000'>identical, that each are picking, let's say ten
  just to make it easy,</span> <span m='1619000'>ten different forms of DSCAM, are
  going to wind up with very different</span> <span m='1624000'>repertoires of DSCAM
  than an adjacent cell.</span> </p><p><span m='1630000'>So what this allows is each
  individual cell to have a unique</span> <span m='1633000'>identity. The whole idea
  that individual neurons might need to</span> <span m='1637000'>have a unique identity
  actually is a new concept that's really been</span> <span m='1641000'>enlightened
  by this molecule. Because the way that people used to</span> <span m='1644000'>think
  of neurons is that they would wind up with unique identities based</span> <span
  m='1648000'>on the connections or experience, what they were exposed to in terms</span>
  <span m='1652000'>of different stimuli. But what this indicates is that from</span>
  <span m='1656000'>the splicing of an individual gene and the fact that each time
  this</span> <span m='1662000'>gene gets spliced you can wind up with a different
  form that at any</span> <span m='1668000'>given time each cell will have a unique
  set of messenger RNAs,</span> <span m='1674000'>and therefore proteins encoding
  this DSCAM gene.</span> </p><p><span m='1680000'>OK. So I mentioned earlier that
  the human DSCAM does not have</span> <span m='1685000'>alternative splicing. We
  all like to think of ourselves,</span> <span m='1690000'>humans and other mammals
  as having brains that are on the level of</span> <span m='1695000'>complexity, at
  least on par with the fly. And so it's odd to think of</span> <span m='1701000'>all
  this complexity that's there for flies and other insects but why is</span> <span
  m='1706000'>it not there for humans? Well, it turns out that there are</span> <span
  m='1711000'>other kinds of genes that do have extensive alternative splicing in</span>
  <span m='1716000'>mammals. So one of them is called neurexins. These are genes that
  are</span> <span m='1720000'>involved in synapse, how the different kinds of cells</span>
  <span m='1725000'>communicate with each other at the interface. There are genes
  called</span> <span m='1730000'>protocadherins. And there are also other kinds of</span>
  <span m='1735000'>genes that all have extensive alternative splicing in mammals.</span>
  </p><p><span m='1740000'>Interestingly, these genes tend not to have</span> <span
  m='1744000'>extensive alternative splicing in flies. It's as if in each lineage</span>
  <span m='1748000'>certain genes have been chosen to get a lot of diversity by this</span>
  <span m='1752000'>mechanism of alternative splicing and other genes are left with
  their</span> <span m='1756000'>just standard single function where it's one gene,</span>
  <span m='1760000'>one RNA, one protein. OK. So I'm going to switch now to</span>
  <span m='1764000'>the second example which is RNA editing.</span> </p><p><span m='1769000'>So,
  as I mentioned earlier,</span> <span m='1781000'>RNA editing involves an actual
  change in the RNA sequence so that</span> <span m='1785000'>it no longer reflects
  the exact DNA sequence. Now,</span> <span m='1789000'>this is different than splicing.
  Splicing takes different pieces of</span> <span m='1793000'>RNA and splices them
  together leaving out intervening sequences or</span> <span m='1797000'>introns.
  But in RNA editing you actually change the nucleotide</span> <span m='1801000'>sequence
  so that it no longer is identical to the DNA.</span> </p><p><span m='1805000'>This
  is used in a number of parts of the brain. Most of the examples are</span> <span
  m='1810000'>brain-specific. There are some non-brain specific</span> <span m='1815000'>parts.
  Most of the time the editing event changes in adenosine,</span> <span m='1820000'>an
  A in the ACGT nomenclature, into an inosine.</span> </p><p><span m='1825000'>This
  is read differently by the</span> <span m='1834000'>ribosome than the adenosine.
  So this leads, for example, in an</span> <span m='1840000'>important kind of channel
  called a glutamate receptor.</span> </p><p><span m='1845000'>And the specific subtype
  that I'm talking about is something called an</span> <span m='1851000'>AMPA glutamate
  receptor. That's for a chemical ligand that</span> <span m='1856000'>activates this
  particular kind of glutamate receptor.</span> </p><p><span m='1862000'>This leads
  to an important change of a glutamine to an arginine in the</span> <span m='1870000'>protein.
  So let me draw a quick diagram of what the protein looks</span> <span m='1879000'>like
  so we can see what the importance of this glutamine to</span> <span m='1887000'>arginine
  switch in this glutamate receptor is.</span> </p><p><span m='1896000'>So in the
  absence of detailed</span> <span m='1907000'>structural information about different
  kinds of neurotransmitter</span> <span m='1911000'>receptors people often draw a
  diagram like this where this is the</span> <span m='1915000'>outside of the cell,
  this is the inside of the cell,</span> <span m='1919000'>this represents the cell
  membrane. And here's the amino terminus.</span> </p><p><span m='1925000'>And then
  they draw a transmembrane portion. And this is what's called</span> <span m='1932000'>a
  reentrant loop. It doesn't quite pass through the</span> <span m='1938000'>membrane,
  but then it passes the membrane again. And here's</span> <span m='1945000'>the carboxy
  terminus. The glutamine to arginine change is</span> <span m='1953000'>here. It's
  in this area which is involved in making the pore.</span> </p><p><span m='1961000'>So
  the pore of the channel has this change.</span> </p><p><span m='1970000'>Glutamine
  to arginine change.</span> </p><p><span m='1984000'>This vastly changes the properties
  of the channel,</span> <span m='1991000'>a channel that doesn't undergo this editing
  event.</span> </p><p><span m='1998000'>So let me just state that for the GluR2 AMPA
  receptor,</span> <span m='2005000'>which is one of four different genes, it is 99%
  edited in adults.</span> </p><p><span m='2013000'>So where over 99% of the time
  this adenosine is made into an inosine</span> <span m='2017000'>which leads to a
  glutamine becoming an arginine in the protein.</span> </p><p><span m='2022000'>What
  this does to channel is it changes its permeability.</span> </p><p><span m='2027000'>So
  this is a kind of channel that is mostly designed to let sodium in,</span> <span
  m='2032000'>but if it doesn't get edited, if the glutamine is there it also</span>
  <span m='2037000'>lets calcium in. So whether or not calcium gets into</span> <span
  m='2042000'>the cell is very important because they're both, both sodium and</span>
  <span m='2046000'>calcium are cat ions and can lead to membrane, potential disturbances</span>
  <span m='2050000'>like you learned about earlier this week, leading to an action
  potential.</span> </p><p><span m='2054000'>But calcium also has other effects. It
  can lead ultimately to the turn</span> <span m='2058000'>on and off of genes and
  phosphorylation of various proteins</span> <span m='2062000'>to other kinds of effects
  in the neuron. So it has to be</span> <span m='2066000'>regulated very tightly.
  So these channels are designed to</span> <span m='2070000'>just let sodium through,
  to be involved in the transmission</span> <span m='2074000'>of an action potential
  from one neuron to the next.</span> </p><p><span m='2078000'>So if you had a perturbation
  in this process you would then also let</span> <span m='2082000'>calcium in because
  the glutamine containing channel lets calcium in.</span> </p><p><span m='2086000'>In
  fact, early in development it's probably true that you don't edit</span> <span m='2090000'>100%
  and you let a little bit of calcium in.</span> </p><p><span m='2095000'>And there
  are also other glutamate receptors that are not part of the</span> <span m='2100000'>AMPA
  family but they are related. And they're encoded by genes called,</span> <span m='2105000'>so
  this GluR1 through 4 encodes AMPA receptors, a gene called GluR5 and 6,</span> <span
  m='2111000'>they have editing which is more regulated. And by regulated I mean</span>
  <span m='2116000'>that the editing is sometimes present and sometimes not.</span>
  </p><p><span m='2122000'>So even within a given neuron you might have some channels
  that have</span> <span m='2127000'>the glutamine and some channels that have the
  arginine.</span> </p><p><span m='2133000'>There are also other sites.</span> </p><p><span
  m='2141000'>I've talked about the main site. This is the one that has the most</span>
  <span m='2145000'>profound impact on the function of the protein. There are also
  other</span> <span m='2149000'>sites in the molecule that are edited. And then they
  also have</span> <span m='2153000'>important but slightly less prominent roles in
  the regulation of</span> <span m='2157000'>these channels. So what leads this gene
  to become</span> <span m='2162000'>edited? Why do most genes not become edited and
  this gene becomes</span> <span m='2167000'>edited? Well, people are starting to
  pursue that</span> <span m='2172000'>kind of mechanisms. And one of the mechanisms
  that's</span> <span m='2177000'>become very clear is that, for example, in the Q
  to R change in</span> <span m='2183000'>the pore, or the adenosine to inosine change
  in the messenger RNA</span> <span m='2188000'>that leads to the Q to R change in
  the pore, if you look at</span> <span m='2193000'>the exon where that -- I'll draw
  it as an A.</span> </p><p><span m='2198000'>Where that A is present. There's an
  area, and then here's</span> <span m='2204000'>the intron, of the intronic sequence
  which actually loops back and then</span> <span m='2209000'>allows base pairing
  to form between the messenger RNA and the intron.</span> </p><p><span m='2215000'>And
  then the enzymes that are involved in mediating this adenosine</span> <span m='2221000'>to
  inosine change recognize the base pairing of this short area and some</span> <span
  m='2227000'>sequence specificity to the RNA sequence that's around here.</span>
  </p><p><span m='2233000'>It's not just any base pairing, but the base pairing is
  critical.</span> </p><p><span m='2239000'>And then the enzymes, which are called
  adenosine deaminase,</span> <span m='2245000'>can mediate this change of an adenosine
  to an inosine.</span> </p><p><span m='2252000'>So this gene has been selected to
  have parts of its intron,</span> <span m='2257000'>in addition to allowing splicing
  to occur, to actual be able to base</span> <span m='2262000'>pair and allow this
  editing function to happen. And so this allows an</span> <span m='2267000'>individual
  gene then to make more than one form of protein.</span> </p><p><span m='2272000'>OK.
  One other example that I'll tell you about of RNA editing</span> <span m='2277000'>involves
  the serotonin system. It involves a serotonin receptor</span> <span m='2283000'>which
  has RNA editing. And this is a serotonin receptor</span> <span m='2290000'>whose
  name is serotonin receptor 2C, so it's often written as 5-HT or 5</span> <span m='2298000'>hydroxtryptamine
  2C serotonin receptor. This receptor is a member</span> <span m='2305000'>of the
  G protein-coupled receptor super family.</span> </p><p><span m='2313000'>G protein-coupled
  receptors are 7-transmembrane domain receptors.</span> </p><p><span m='2319000'>And
  the editing of the serotonin receptor occurs in the second</span> <span m='2326000'>intracellular
  loop and affects the coupling to G protein.</span> </p><p><span m='2333000'>So the
  way that these G protein-coupled receptors --</span> <span m='2340000'>Did they
  do this already? G protein-coupled receptors</span> <span m='2344000'>transduce
  the signal through a G protein which often binds to the</span> <span m='2348000'>intracellular
  loops, particularly the second loop.</span> </p><p><span m='2352000'>And the editing
  event which changes adenosines to inosines,</span> <span m='2356000'>there are actually
  a few of them in this region here,</span> <span m='2360000'>a few different sites
  which can get changed. That affects the</span> <span m='2364000'>efficiency of the
  transduction, when serotonin is present the</span> <span m='2368000'>transduction
  of a signal inside the cell.</span> </p><p><span m='2373000'>There are other serotonin
  receptors that are ligand gated channels but</span> <span m='2377000'>they don't
  appear to have the same kind of editing as this serotonin</span> <span m='2381000'>receptor
  that is of the G protein-coupled variety has.</span> </p><p><span m='2386000'>So
  I'll end with just one final note</span> <span m='2399000'>about this serotonin
  receptor which is that a drug called fluoxetine or</span> <span m='2403000'>Prozac,
  did Eric go over that this year? Prozac?</span> </p><p><span m='2407000'>No? He
  mentioned it. OK, he mentioned it. So it's</span> <span m='2411000'>mostly known
  as something which blocks reuptake of serotonin.</span> </p><p><span m='2416000'>So
  one cell releases serotonin, which is a neurotransmitter.</span> </p><p><span m='2420000'>Another
  cell responds to it. If you block the reuptake the</span> <span m='2424000'>neurotransmitter
  is present in the synapse for a longer amount of time</span> <span m='2428000'>leading
  to increased signaling. Prozac is widely thought to have its</span> <span m='2433000'>main
  effect, and that probably is its main effect to just block the</span> <span m='2437000'>reuptake
  leading to increased serotonin signaling.</span> </p><p><span m='2441000'>Someone
  has studied the serotonin receptor in individuals who are</span> <span m='2445000'>taking
  Prozac versus individuals who are not taking Prozac.</span> </p><p><span m='2449000'>And
  what they found was that there were differences in the amount of</span> <span m='2453000'>editing
  of various sites in this key area in individuals taking Prozac</span> <span m='2457000'>versus
  individuals not taking Prozac.</span> </p><p><span m='2462000'>And the direction
  of the difference, whether it's switching from edited</span> <span m='2466000'>to
  unedited or back, and it varies for the different</span> <span m='2470000'>sites,
  the direction was the opposite of what was seen in a</span> <span m='2475000'>comparison
  of brains of victims of suicide versus brains of other</span> <span m='2479000'>accident
  victims. So it looks like Prozac is having</span> <span m='2484000'>an effect which
  is in the opposite effect to the skewing of editing</span> <span m='2488000'>that
  one sees in certain cases of depression.</span> </p><p><span m='2493000'>So I bring
  that example up because it's important to know that,</span> <span m='2498000'>you
  know, if you can impact on some of these subtle differences between</span> <span
  m='2503000'>different kinds of messages, like whether it's edited or not or</span>
  <span m='2508000'>whether the splicing is more towards one kind of alternative splicing
  or</span> <span m='2513000'>another kind of alternative splicing, these might provide
  very interesting</span> <span m='2518000'>pharmacologic targets for therapies that
  might impact on a variety of</span> <span m='2523000'>different human diseases.
  So what I've hoped to do is give you</span> <span m='2528000'>a sense of a couple
  of different examples where you can take a single</span> <span m='2532000'>gene
  and make more than one protein, and this can lead to increases in</span> <span m='2537000'>the
  diversity of neurons, and therefore increases in the</span> <span m='2541000'>complexity
  of the brain. Thank you. [APPLAUSE] Are there</span> <span m='2546000'>questions?</span>
  </p>
type: course
uid: 5c24cbb88dc19cfb0c3d05626f4e9e4f

---
None