---
about_this_resource_text: <p><strong>Topics covered:</strong> Human Polymorphisms
  and Cancer Classification</p>  <p><strong>Instructors:</strong> Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec34.jpg
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-34-human-polymorphisms-and-cancer-classification/lec34.jpg
  title: lec34.jpg
  type: null
  uid: ce8f094d42a875e86970fdf2d3c88752
- id: Video-YouTube-Stream
  media_location: Eqom7VcaEKI
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Video-YouTube-Stream
  type: Video
  uid: 579455737e099c905f29a90fdb88d90a
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/Eqom7VcaEKI/default.jpg
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: b444236bc89393fa42c2fca606639f99
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Video-iTunes U-MP4
  type: Video
  uid: 20e5b59698032bc64e0586ddc307b7b4
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec34-06dec2004-220k.mp4
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Video-Internet Archive-MP4
  type: Video
  uid: 1ab95b00758a54838b943b4b941bb637
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 7171c79240832d0636d3bfdc7ed6891a
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-06dec2004-1000.mp3
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Video-Internet Archive-MP3
  type: Video
  uid: 756bd6641e1bae4132bdaa99d068ab6c
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 9e0d032b109dac20f3e5cda385f70ddf
- id: Thumbnail-OCW-JPG
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 9b5922c106d3e20fb7e7c428b31eab11
- id: 3Play-3PlayYouTubeid-MP4
  media_location: Eqom7VcaEKI
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 8682e34f8306edfde877c72bcef039c5
- id: Eqom7VcaEKI.srt
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-34-human-polymorphisms-and-cancer-classification/Eqom7VcaEKI.srt
  title: 3play caption file
  type: null
  uid: 08f01d94f3bf4e47a98e14a72bb20101
- id: Eqom7VcaEKI.pdf
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-34-human-polymorphisms-and-cancer-classification/Eqom7VcaEKI.pdf
  title: 3play pdf file
  type: null
  uid: bee7da23a1818bc8a8285eb42bc23586
- id: Caption-3Play YouTube id-SRT
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: d6ff303bb2f06f9dc1def715799cb00f
- id: Transcript-3Play YouTube id-PDF
  parent_uid: b82c7be20d0a354b9a36ccd2e3ee48ef
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: 682dc7431f5a9fa9094131f34fbbb2b2
inline_embed_id: 97304052lecture34:humanpolymorphismsandcancerclassification81248407
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-34-human-polymorphisms-and-cancer-classification
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-34-human-polymorphisms-and-cancer-classification
template_type: Tabbed
title: 'Lecture 34: Human Polymorphisms and Cancer Classification'
transcript: '<p><span m=''1000''>Today is my last class with you. Awe, I''m sorry,
  too. You guys are</span> <span m=''4000''>a lot of fun. This has actually been the
  most interactive 7.</span> </p><p><span m=''8000''>1 I''ve ever had. Usually there
  are a couple of people who perk up and</span> <span m=''12000''>say things, but
  you guys are great because all sorts of people are</span> <span m=''16000''>willing
  to contribute. So, I''ve had a wonderful time and</span> <span m=''20000''>it certainly
  seems like you guys have learned a lot.</span> </p><p><span m=''24000''>What I''d
  like to do for my last lecture is pick up again a little</span> <span m=''28000''>bit
  like I did with genomics and try to give you a sense of where</span> <span m=''32000''>things
  are going. I always like doing this because I</span> <span m=''36000''>get to talk
  about things that are in none of the textbooks that,</span> <span m=''40000''>well,
  I mean, it''s just stuff that many people working in the field</span> <span m=''44000''>don''t
  necessarily know. And that''s what''s so much fun about</span> <span m=''48000''>teaching
  introductory biology is because it only takes a semester for</span> <span m=''52000''>you
  guys to get up to the point of at least being able to understand</span> <span m=''56000''>what''s
  getting done on the cutting-edge.</span> </p><p><span m=''61000''>Even if you might
  not yet be able to go off and practice it,</span> <span m=''65000''>you might need
  a little more experience for that,</span> <span m=''69000''>but you''d be surprised,
  it''s not that much more.</span> </p><p><span m=''73000''>Take maybe Project Lab
  and you''ll be able to start doing it already.</span> </p><p><span m=''77000''>It''s
  really wonderful that it''s possible to grasp what''s going on.</span> </p><p><span
  m=''81000''>And, in many ways, you guys may have an advantage in grasping what''s</span>
  <span m=''85000''>going on because, as I''ve already hinted,</span> <span m=''89000''>biology''s
  undergoing this remarkable transformation from being a purely</span> <span m=''93000''>laboratory-based
  science where each individual works on his or her own</span> <span m=''97000''>project
  to being an information-based science that</span> <span m=''101000''>involves an
  integration of vast amounts of data across the whole</span> <span m=''105000''>world
  and trying to learn things from this tremendous dataset.</span> </p><p><span m=''110000''>And,
  in that sense, I think the new students coming into</span> <span m=''112000''>the
  field have a distinct advantage over those who have been in it.</span> </p><p><span
  m=''115000''>And certainly the students who know mathematical and physical and</span>
  <span m=''118000''>chemical and other sorts of things, and aren''t scared to write
  computer</span> <span m=''121000''>code when they need to write computer code have
  a really</span> <span m=''124000''>great advantage. So, anyway, all that by way
  of</span> <span m=''127000''>introduction. I want to talk about two subjects today
  of great interest</span> <span m=''131000''>to me. One is DNA variation and one
  is RNA variation.</span> </p><p><span m=''134000''>The variation of DNA sequence
  between individuals within a</span> <span m=''138000''>population, and in particular
  our population, and the other is RNA</span> <span m=''141000''>variation, the variation
  in RNA expression between different cell</span> <span m=''145000''>types, different
  tissues. And the work I''m going to talk</span> <span m=''148000''>about today is
  work that I, and my colleagues, have all been</span> <span m=''152000''>involved
  in. And it''s stuff I know and love.</span> </p><p><span m=''156000''>So, feel free
  to ask questions about it. I may know the answers,</span> <span m=''160000''>but
  what''s reasonably fun about these lectures is if I don''t know</span> <span m=''164000''>the
  answers it''s probably the case that the answers aren''t known.</span> </p><p><span
  m=''168000''>So, that''s good fun because it''s stuff I really do know well,</span>
  <span m=''172000''>and I love. So, anyway, here''s some DNA sequence. It''s pretty
  boring.</span> </p><p><span m=''176000''>This is a chunk of sequence from, let''s
  say, the human genome.</span> </p><p><span m=''180000''>How much does this differ
  between any two individuals?</span> </p><p><span m=''184000''>If I were to sequence
  any two chromosomes, any two copies of the</span> <span m=''189000''>chromosome
  from an individual in this class or two individuals on</span> <span m=''194000''>this
  planet, how much would they differ? The answer is that much.</span> </p><p><span
  m=''199000''>That''s the average amount of difference between any two people on</span>
  <span m=''205000''>this planet. Not a lot. If you counted up, it is on average</span>
  <span m=''210000''>one nucleotide difference out of 1, 00 nucleotides on average,
  somewhat</span> <span m=''215000''>less than one part in 1, 00 or better than 99.9%
  identity</span> <span m=''221000''>between any two individuals. Now, that is a very
  small amount,</span> <span m=''226000''>not just in absolute terms, 99.9% identity
  is a lot,</span> <span m=''231000''>but in comparative terms with other species.
  If I take two chimpanzees</span> <span m=''237000''>in Africa, on average they will
  differ by about twice as much as any</span> <span m=''242000''>two random humans.
  And if I take two orangutans in</span> <span m=''247000''>Southeast Asia, they will
  on average differ by about</span> <span m=''252000''>eight times as much as any
  two humans on this planet.</span> </p><p><span m=''257000''>You guys think the orangutans
  all look the same.</span> </p><p><span m=''261000''>They think you all look the
  same, and they''re right. So, why is this?</span> </p><p><span m=''266000''>Why
  are humans amongst mammalian</span> <span m=''271000''>species relatively limited
  in the amount of variation?</span> </p><p><span m=''276000''>Well, it''s a direct
  result of our population history.</span> </p><p><span m=''280000''>It turns out
  that the amount of variation that can be sustained in a</span> <span m=''285000''>population
  depends on two things. At equilibrium, if population has</span> <span m=''290000''>constant
  size N for a very long time and a certain mutation rate,</span> <span m=''295000''>Mu,
  you can just write a piece of arithmetic that says,</span> <span m=''299000''>well,
  mutations are always arising due to new mutations in the</span> <span m=''304000''>population
  and mutations are being lost by genetic drift,</span> <span m=''309000''>just by
  random sampling from generation to generation.</span> </p><p><span m=''314000''>And
  those two processes, the creation of new mutations and</span> <span m=''317000''>the
  loss of mutations just due to random sampling in each generation,</span> <span m=''321000''>sets
  up an equilibrium, and the equilibrium defines an equation</span> <span m=''325000''>there,
  Pi equals one over one plus four and Mu reciprocal which</span> <span m=''329000''>equation
  you have no need to memorize whatsoever and possibly</span> <span m=''333000''>even
  no need to write down. The important point is the concept,</span> <span m=''336000''>that
  if you know the number of organisms in the population and you</span> <span m=''340000''>know
  the mutation rate, those set up the bounds of mutation</span> <span m=''343000''>and
  drift, and you can write down how polymorphic,</span> <span m=''347000''>how heterozygous
  random individuals should be at equilibrium.</span> </p><p><span m=''351000''>That
  is if the population has been at size N for a very long time.</span> </p><p><span
  m=''354000''>Well, the expected amount of heterozygosity for the</span> <span m=''358000''>human
  population -- Sorry. For a population of size 10,</span> <span m=''362000''>00 would
  be about one nucleotide in 1300. We have exactly the amount of</span> <span m=''366000''>heterozygosity
  you would expect for a population of about 10,</span> <span m=''371000''>00 individuals.
  Yeah, but wait, we''re not a population of 10,000</span> <span m=''375000''>individuals.
  Why do we have the heterozygosity you would expect from</span> <span m=''380000''>a
  population of 10, 00 individuals? We''re six billion.</span> </p><p><span m=''385000''>It''s
  a reflection of our history.</span> </p><p><span m=''391000''>Because remember I
  said that was the statement about what the</span> <span m=''395000''>population
  heterozygosity should be at equilibrium?</span> </p><p><span m=''398000''>We haven''t
  been six billion people except very recently.</span> </p><p><span m=''402000''>The
  human population has undergone an exponential expansion.</span> </p><p><span m=''405000''>It
  used to be a relatively small size, and then it very recently</span> <span m=''409000''>underwent
  this huge exponential expansion. If you actually write</span> <span m=''412000''>down
  the equations, the amount of variation in our</span> <span m=''416000''>population
  was determined by that constant size for a very long time.</span> </p><p><span m=''420000''>And
  then a rapid exponential expansion that''s basically taken</span> <span m=''423000''>place
  in a mere 3, 00 generations, it''s much too rapid</span> <span m=''427000''>to have
  any affect on the real variation in our population.</span> </p><p><span m=''431000''>What
  do I mean by that? What''s the mutation rate per nucleotide in the</span> <span
  m=''435000''>human genome? It''s on the order of two times ten to the minus eighth</span>
  <span m=''438000''>per generation. In a mere 3,000 generations,</span> <span m=''442000''>a
  tiny mutation rate like two times ten to the minus eighth is not going</span> <span
  m=''446000''>to be able to build up much more variation.</span> </p><p><span m=''450000''>So
  you might as well ignore the last 100,000 years or so.</span> </p><p><span m=''452000''>They''re
  irrelevant to how much variation we have.</span> </p><p><span m=''454000''>The variation
  we have was set by our ancestral population size.</span> </p><p><span m=''456000''>Now,
  don''t get me wrong. Eventually it will equilibrate.</span> </p><p><span m=''458000''>A
  couple million years from now we will have a much higher variation in</span> <span
  m=''460000''>the human population as a function of our size, but the population</span>
  <span m=''462000''>variation we have today is set by the fact that humans derive
  from a</span> <span m=''464000''>founding population of about 10, 00 individuals
  or so.</span> </p><p><span m=''467000''>And that means that the variation that you
  see in the human population</span> <span m=''472000''>is mostly ancestral variations,
  the variation that we all walked</span> <span m=''477000''>around with in Africa.
  And, in fact, that makes a</span> <span m=''483000''>prediction. That would say
  that if most of the variation in the human</span> <span m=''488000''>population
  is from the ancestral African founding population then if</span> <span m=''493000''>I
  go to any two villages around this world, in Japan or in Sweden or in</span> <span
  m=''499000''>Nigeria, the variance that I see will largely be identical.</span>
  </p><p><span m=''504000''>And that prediction has been well satisfied.</span> </p><p><span
  m=''510000''>Because when you go and look and you collect variation in Japan or
  Sweden</span> <span m=''514000''>or Africa and you compare it, 90% of the variance
  are common</span> <span m=''518000''>across the entire world. Most variation is
  common ancestral</span> <span m=''522000''>variation around the world, and only
  a minority of the variance</span> <span m=''526000''>are new local mutations restricted
  to individual populations.</span> </p><p><span m=''530000''>This is so contrary
  to what people think because there''s a natural</span> <span m=''534000''>tendency
  to kind of xenophobia, to imagine that world populations</span> <span m=''538000''>are
  very different in their genetic background.</span> </p><p><span m=''542000''>But,
  in point of fact, they''re extremely similar.</span> </p><p><span m=''545000''>So,
  anyway, there''s a limited amount of variation.</span> </p><p><span m=''549000''>That''s
  why we have such little variation in the human population.</span> </p><p><span m=''553000''>Now,
  that variation, humans have a low rate of genetic variation.</span> </p><p><span
  m=''557000''>Most of the variance that are out there are due to common genetic</span>
  <span m=''560000''>variance, not rare variance. If I take your genome and I find
  a</span> <span m=''564000''>site of genetic variation at the point of heterozygosity
  in your</span> <span m=''568000''>genome, what''s the probability that somebody
  else in this class also is</span> <span m=''572000''>heterozygous for that spot?
  It turns out that the odds are about</span> <span m=''576000''>95% that someone
  else in this class will also share that variance.</span> </p><p><span m=''580000''>So
  that the variance are not mostly rare, they''re mostly common.</span> </p><p><span
  m=''584000''>And it turns out that some of this common variation,</span> <span m=''588000''>that
  is most of this variation is likely to be important in the risk</span> <span m=''592000''>of
  human genetic diseases. So human geneticists have gotten</span> <span m=''596000''>very
  excited about the following paradigm.</span> </p><p><span m=''600000''>If there''s
  only a limited amount of genetic variation in the human</span> <span m=''603000''>population,
  actually, if you do the arithmetic,</span> <span m=''606000''>there are only about
  ten million sites of common variation in the</span> <span m=''609000''>human population,
  where common might be defined as</span> <span m=''612000''>more than about 1% in
  the population. There are only ten million sites.</span> </p><p><span m=''615000''>Folks
  are saying, well, why not enumerate them all?</span> </p><p><span m=''618000''>Let''s
  just know them all, and then let''s test each one for its</span> <span m=''622000''>risk
  of, say, confirming susceptibility of diabetes or heart</span> <span m=''625000''>disease
  or whatever? After all, ten million is not as</span> <span m=''628000''>big a number
  as it used to be. We now have the whole sequence of</span> <span m=''632000''>the
  human genome. Why not layer on the sequence of</span> <span m=''636000''>the human
  genome all common human genetic polymorphism?</span> </p><p><span m=''640000''>Now,
  that''s a fairly outrageous idea but could be a very useful one.</span> </p><p><span
  m=''644000''>Some of these variance are important, by the way.</span> </p><p><span
  m=''648000''>We know that there are two nucleotides that vary in the gene</span>
  <span m=''652000''>apolipoprotein E on chromosome number 19. Apolipoprotein E is
  also</span> <span m=''656000''>an apolipoprotein like we talked about before with
  familiar</span> <span m=''660000''>hypercholesterolemia. But, in fact, it turns
  out that</span> <span m=''664000''>apolipoprotein E is expressed in the brain. And
  it turns out,</span> <span m=''668000''>amongst other tissues, that it comes in
  three variances,</span> <span m=''673000''>the spelling T-T, T-C and C-C at those
  two particular spots.</span> </p><p><span m=''678000''>And if you happen to be homozygous
  for the E4 variant,</span> <span m=''682000''>homozygous for the E4 variant, you
  have about a 60% to 70% lifetime</span> <span m=''687000''>risk of Alzheimer''s
  disease. In this class 13 of you are</span> <span m=''692000''>homozygous for E4
  and have a high lifetime risk of Alzheimer''s.</span> </p><p><span m=''697000''>And
  it would be fairly trivial to go across the street to anybody''s</span> <span m=''702000''>lab
  and test that. Now, I don''t particular recommend</span> <span m=''707000''>it,
  and I haven''t tested myself for this variant because there happens</span> <span
  m=''712000''>to be no particular therapy available today to delay the onset</span>
  <span m=''717000''>of Alzheimer''s disease. And, therefore,</span> <span m=''721000''>I
  don''t recommend finding out about that. But a number of</span> <span m=''725000''>pharmaceutical
  companies, knowing that this is a very</span> <span m=''728000''>important gene
  in the pathogenesis of Alzheimer''s disease,</span> <span m=''731000''>are working
  on drugs to try to delay the pathogenesis using this</span> <span m=''735000''>information.
  And it may be the case that five or ten years from now</span> <span m=''738000''>people
  will begin to offer drugs that will delay the onset of</span> <span m=''741000''>Alzheimer''s
  disease by delaying the interaction of apolipoprotein E with</span> <span m=''745000''>a
  target protein called towe, etc. So, this is an example of where a</span> <span
  m=''749000''>common variant in the population points us to the basis of a common</span>
  <span m=''753000''>disease and has important therapeutic implications.</span> </p><p><span
  m=''757000''>There are some other ones, for example. 5% of you carry a</span> <span
  m=''761000''>particular variant in your factor 5 gene which is the clotting cascade.</span>
  </p><p><span m=''765000''>It''s called the leiden variant. Those 5% of you are going
  to account</span> <span m=''769000''>for 50% of the admissions to emergency rooms
  for deep venous</span> <span m=''773000''>clots, for example. The much higher risk
  of deep venous</span> <span m=''777000''>clots. And, in particular,</span> <span
  m=''782000''>there are significant issues if you have that variant and you are a</span>
  <span m=''786000''>woman with taking birth control pills. Some of you were at higher</span>
  <span m=''791000''>risk for diabetes, type 2 adult onset diabetes.</span> </p><p><span
  m=''796000''>There''s a particular variant in the population that increased your
  risk</span> <span m=''800000''>for type 2 diabetes by about 30%. 85% of you have
  the high-risk</span> <span m=''805000''>factor, so you might as well figure you
  do.</span> </p><p><span m=''810000''>15% of you have a lower risk, et cetera. And
  one I''m particularly</span> <span m=''815000''>interested in here, it turns out
  that HIV virus gets</span> <span m=''820000''>into cells with a co-receptor encoded
  by a gene called CCR5.</span> </p><p><span m=''826000''>Well, it turns out that
  if we go across the European population,</span> <span m=''831000''>10% of all chromosomes
  of European ancestry have a deletion</span> <span m=''837000''>within the CCR5 gene.
  If 10% of all chromosomes have that</span> <span m=''842000''>deletion then 10%
  times 10%, 1% of all individuals are homozygous</span> <span m=''846000''>for that
  deletion. Those individuals are essentially</span> <span m=''850000''>immune to
  infection from HIV. They are not susceptible. It''s not</span> <span m=''855000''>through
  immunity, it''s through lack of a receptor.</span> </p><p><span m=''859000''>Yes?
  You certainly can. It''s not hard. It''s a specific known variant.</span> </p><p><span
  m=''863000''>You could test for it. Absolutely.</span> </p><p><span m=''868000''>Now,
  of course, that only helps the 1% of people who</span> <span m=''871000''>have that
  variant. But what it did do was point to the</span> <span m=''874000''>pharmaceutical
  industry that the interaction between the virus and</span> <span m=''877000''>that
  variant is essential. And now companies are developing</span> <span m=''881000''>drugs
  to block the interaction with that particular protein.</span> </p><p><span m=''884000''>And
  that tells you that it''s an important protein. Yes?</span> </p><p><span m=''888000''>Over
  the whole world?</span> </p><p><span m=''896000''>I just specified European population
  for that one.</span> </p><p><span m=''900000''>That one, interestingly, is not found
  at as high a frequency</span> <span m=''903000''>outside of Europe, and no one knows
  why,</span> <span m=''906000''>whether that might have been due to an ancient selective
  event or a</span> <span m=''909000''>genetic drift. By contrast, the apolipoprotein
  E</span> <span m=''913000''>variant, at that frequency of about 3% of people being
  homozygous and</span> <span m=''916000''>being at risk for Alzheimer''s, is about
  the same frequency</span> <span m=''919000''>everywhere in the world. So, there''s
  a little bit of</span> <span m=''923000''>population variation in frequency. Now,
  the HIV variant is found</span> <span m=''926000''>elsewhere but at considerably
  lower frequencies there.</span> </p><p><span m=''930000''>And that''s an interesting
  question as to what causes that variation.</span> </p><p><span m=''933000''>So the
  notion would be, I''ve given you a couple of interesting examples,</span> <span
  m=''936000''>but, look, there''s only ten million variants. Just write them all
  down.</span> </p><p><span m=''940000''>Make one big Excel spreadsheet with ten million
  variants along the top</span> <span m=''943000''>and all the diseases along the
  rows, and let''s just fill in the matrix</span> <span m=''947000''>and then we''ll
  really, you know, this is the way people</span> <span m=''950000''>think in a post-genomic
  era. Now, could you do something like</span> <span m=''954000''>that? You would
  have to enumerate all of the single nucleotide</span> <span m=''957000''>polymorphisms,
  or SNPs we call them,</span> <span m=''961000''>single nucleotide polymorphisms.
  Now, to give you an idea of the</span> <span m=''965000''>magnitude of this problem,
  as recently as 1998, the number of</span> <span m=''969000''>SNPs that were known
  in the human genome was a couple hundred.</span> </p><p><span m=''973000''>But then
  a project has taken off. In 1998 an initial SNP map of the</span> <span m=''977000''>human
  genome was built here at MIT that had about 4,</span> <span m=''981000''>00 of these
  variants. Then within the next year or so an</span> <span m=''985000''>international
  consortium was organized here and elsewhere to</span> <span m=''989000''>begin to
  collect more of these genetic variants.</span> </p><p><span m=''994000''>The goal
  was going to be to find 300, 00 of them within a period of two</span> <span m=''998000''>years.
  In fact, that goal was blown away and within</span> <span m=''1002000''>three years
  two million of the SNPs in the human population were found.</span> </p><p><span
  m=''1006000''>And as of today, if you go on the Web, you''ll find the database with</span>
  <span m=''1011000''>about 7.8 million of the roughly ten million SNPs in the human
  population</span> <span m=''1015000''>already known. Now, that isn''t all ten million.</span>
  </p><p><span m=''1020000''>And it takes a while to collect the last ones, you know,</span>
  <span m=''1023000''>collecting the last ones are hard, but we''re already the hump
  of</span> <span m=''1027000''>knowing the majority of common variation in the human
  population.</span> </p><p><span m=''1030000''>Not just a sequence of the genome,
  but a database that already contains</span> <span m=''1034000''>more than half of
  all common variation in the population.</span> </p><p><span m=''1037000''>So, we
  could start building that Excel spreadsheet.</span> </p><p><span m=''1041000''>Now,
  it turns out that it''s even a little bit better than that because</span> <span
  m=''1044000''>if we look at many chromosomes in the population,</span> <span m=''1048000''>here
  are chromosomes in the population, it turns out that the</span> <span m=''1051000''>common
  variance on each of those chromosomes tend to be correlated</span> <span m=''1055000''>with
  each other. If I know your genotype at one</span> <span m=''1058000''>variant, like
  over at this locus, I know your genotype at the next</span> <span m=''1061000''>locus
  with reasonably high probability. There''s a lot of local</span> <span m=''1065000''>correlation.
  So, instead of looking like a scattered</span> <span m=''1068000''>picture like
  that, it''s more like this.</span> </p><p><span m=''1071000''>If I know that you''re
  red, red, red you''re probably red,</span> <span m=''1075000''>red, red over here.
  In other words, these variations occur in blocks</span> <span m=''1078000''>that
  we called haplotypes. Here''s real data.</span> </p><p><span m=''1081000''>Across
  111 kilobases of DNA there''s a bunch of variants,</span> <span m=''1084000''>but
  it turns out that the variants come in two basic flavors.</span> </p><p><span m=''1088000''>98%
  of all chromosomes are either this, this, this,</span> <span m=''1091000''>this,
  this or this, this, this, this, this.</span> </p><p><span m=''1094000''>Then there
  tends to be sites of recombination that are actually</span> <span m=''1098000''>hotspots
  of recombination where most of the recombination of the</span> <span m=''1101000''>population
  is concentrated. And you get a couple of</span> <span m=''1104000''>possibilities
  here. So, the human genome can kind of be</span> <span m=''1108000''>broken up into
  these haplotypes. Blocks that might be 20,</span> <span m=''1111000''>30, 40, sometimes
  100 kilobases long in which within the block you tend</span> <span m=''1115000''>to
  have a small number of haplotypes, or flavors as you might think of</span> <span
  m=''1119000''>them, that define most of the chromosomes in the population.</span>
  </p><p><span m=''1123000''>So, in fact, I don''t actually need to know all the variants.</span>
  </p><p><span m=''1126000''>If they''re so well correlated within a block,</span>
  <span m=''1130000''>if I knew this block structure I would be able to pick a small
  number</span> <span m=''1134000''>of SNPs that would serve as a proxy for that entire
  block of inheritance</span> <span m=''1138000''>in the population. So, what you
  might want to do is</span> <span m=''1141000''>determine that entire haplotype block
  structure of hwo they''re</span> <span m=''1144000''>related to each other, and
  pick out tag snips.</span> </p><p><span m=''1148000''>And it turns out that in theory,
  a mere 300,000 or so of them would</span> <span m=''1151000''>suffice to proxy for
  most of the genome. So, you might want to</span> <span m=''1154000''>declare an
  international project, and international haplotype map</span> <span m=''1158000''>project
  to create a haplotype map of the human genome.</span> </p><p><span m=''1161000''>And
  indeed, such a project was declared about a year and a half ago</span> <span m=''1164000''>through
  some instigation of scientists and a number of places,</span> <span m=''1168000''>including
  here. And this is $100 million project</span> <span m=''1171000''>involving six
  different countries. And, it is already more than</span> <span m=''1175000''>halfway
  done with the task, and it''s very likely that by the</span> <span m=''1179000''>middle
  of next year, we will have a pretty good haplotype</span> <span m=''1182000''>map,
  not just knowing all the variation, but knowing the</span> <span m=''1186000''>correlation
  between that variation, being able to break up the genome</span> <span m=''1190000''>into
  these blocks. By the next time I teach 701,</span> <span m=''1193000''>I should
  be able to show a haplotype map of the whole human genome</span> <span m=''1197000''>already.
  That will allow you to start undertaking systematic studies</span> <span m=''1201000''>of
  inheritance for different diseases across populations.</span> </p><p><span m=''1205000''>And
  in fact, people are already doing things like that.</span> </p><p><span m=''1208000''>Here''s
  an example of a study done here at MIT like this,</span> <span m=''1212000''>where
  to study inflammatory bowel disease, there was evidence that</span> <span m=''1215000''>there
  might be a particular region of the genome that contained it,</span> <span m=''1219000''>and
  haplotypes were determined across this, and blah,</span> <span m=''1222000''>blah,
  blah, blah, blah, blah, blah. And this red haplotype</span> <span m=''1226000''>here
  turns out to confer high risk, about a two and a half or higher</span> <span m=''1229000''>risk
  of inflammatory bowel disease.</span> </p><p><span m=''1233000''>And it sits over
  some genes involved in immune responses,</span> <span m=''1236000''>certain cytokine
  genes and all that. And, things like this have been</span> <span m=''1240000''>done
  for type 2 diabetes, schizophrenia, cardiovascular</span> <span m=''1244000''>disease,
  just right now at the moment, a dozen or two examples.</span> </p><p><span m=''1247000''>But
  I think we''re set for an explosion in this kind of work.</span> </p><p><span m=''1251000''>In
  addition, you can use this information to do things beyond</span> <span m=''1255000''>medical
  genetics. You can use it for history and</span> <span m=''1259000''>anthropology
  as well. It turns out rather interestingly,</span> <span m=''1263000''>that since
  the human population originated in Africa and spread out</span> <span m=''1267000''>from
  Africa all the way around the world arriving at different places</span> <span m=''1272000''>in
  different times, you can trace those migrations by</span> <span m=''1277000''>virtue
  of rare genetic variants that arose along the way,</span> <span m=''1281000''>and
  let you, like a trail of break crumbs, see the migrations.</span> </p><p><span m=''1286000''>So,
  for example, there are certain rare genetic variants that we can</span> <span m=''1290000''>see
  in a South American Indian tribe, and we can actually see that they</span> <span
  m=''1295000''>came along this route because we can see that residual of that.</span>
  </p><p><span m=''1300000''>In fact, we can do things with this like take a look
  at Native American</span> <span m=''1305000''>individuals and determine that they
  cluster into three distinct genetic</span> <span m=''1310000''>groups that represent
  three distinct migrations over the land bridge.</span> </p><p><span m=''1315000''>And,
  you can assign them to these different migrations.</span> </p><p><span m=''1320000''>You
  can do this on the basis of mitochondrial genotype,</span> <span m=''1323000''>etc.
  You can also, for example, determine when people talk about the</span> <span m=''1326000''>out
  of Africa migration, there''s now increasing evidence that</span> <span m=''1329000''>there
  really were two, one that went this way over the land,</span> <span m=''1333000''>and
  one that went this way following along the coast into southeast Asia.</span> </p><p><span
  m=''1336000''>And, it looks like we''re now beginning to get enough evidence of</span>
  <span m=''1339000''>these two separate migrations by virtue of the genetic breadcrumbs</span>
  <span m=''1342000''>that they have left along the way.</span> </p><p><span m=''1346000''>So,
  it''s really a very fascinating thing of how much you can</span> <span m=''1350000''>reconstruct
  from looking at genetic variation, both the common variation</span> <span m=''1354000''>that
  allows us to recognize medical risk, and the rare genetic variation</span> <span
  m=''1358000''>that provides much more individual trails of things.</span> </p><p><span
  m=''1363000''>None of this is perfect yet. There''s lots to learn. But I think</span>
  <span m=''1367000''>anthropologists are finding that the existing human population
  has a</span> <span m=''1371000''>tremendous amount of its own history embedded in
  pattern of genetic</span> <span m=''1375000''>variation across the world. You can
  do other things.</span> </p><p><span m=''1380000''>I won''t spend much time on this.
  Well, I''ll take a moment on this,</span> <span m=''1384000''>right? There''s some
  very interesting work of a post-doctoral</span> <span m=''1389000''>fellow here
  at MIT named Pardese Sebetti who has been trying to ask,</span> <span m=''1393000''>can
  we see in the genetic variation in the population,</span> <span m=''1398000''>signatures,
  patterns of ancient selection, or even recent selection</span> <span m=''1402000''>in
  the human population? Now, hang onto your seats,</span> <span m=''1407000''>because
  this will get just slightly tricky.</span> </p><p><span m=''1412000''>But, hang
  on. It''s only a couple of slides. Here was her idea.</span> </p><p><span m=''1415000''>You
  see, when a mutation arises in the population,</span> <span m=''1419000''>it usually
  dies out, right? Any new mutation just</span> <span m=''1423000''>typically dies
  out. But, sometimes by chance it drifts</span> <span m=''1427000''>up to a high
  frequency. Random events happen. But it</span> <span m=''1430000''>usually takes
  a long time to do that. If some random mutation happens,</span> <span m=''1434000''>and
  it happens to drift up to high frequency with no selection on it,</span> <span m=''1438000''>then
  on average it takes a long time to do so.</span> </p><p><span m=''1442000''>If you
  want, I could write a stochastic differential equation</span> <span m=''1445000''>that
  would say that, but just take your gut feeling that</span> <span m=''1449000''>if
  something has no selection on it and it''s a rare event that''ll drift</span> <span
  m=''1452000''>up, when it drifts up it''s kind of a slow process. It was a slow
  process.</span> </p><p><span m=''1456000''>Then over the course of time that it
  took to drift to high frequency,</span> <span m=''1460000''>a lot of genetic recombination
  would have had to have occurred many</span> <span m=''1463000''>generations. And
  the correlation between the genotype at that spot</span> <span m=''1467000''>and
  genotypes at other loci would break down.</span> </p><p><span m=''1471000''>And
  there would only be short-range correlation. So,</span> <span m=''1474000''>in other
  words, the amount of correlation between knowing the</span> <span m=''1478000''>genotype
  here and the genotype here, maybe allele A here and a C here.</span> </p><p><span
  m=''1481000''>That is an indication of time. It''s a clock almost. It''s like</span>
  <span m=''1485000''>radioactive decay, right, that genetic recombination</span>
  <span m=''1489000''>scrambles up the correlations. And, if something''s old, the</span>
  <span m=''1492000''>correlations go over short distances. But suppose that something
  happened.</span> </p><p><span m=''1496000''>Some mutation happened that was very
  advantageous.</span> </p><p><span m=''1500000''>Then, it would have risen to high
  frequency quickly because it was</span> <span m=''1503000''>under selection. If
  it did so quickly,</span> <span m=''1507000''>then the long-range correlations would
  not have had time to break</span> <span m=''1511000''>down, and we''d have a smoking
  gun. A smoking gun would be that there</span> <span m=''1515000''>would be a long-range
  correlation around that locus,</span> <span m=''1518000''>much longer than you would
  expect across the genome.</span> </p><p><span m=''1522000''>Things even out of this
  distance would show correlation with that,</span> <span m=''1526000''>indicating
  that this was a recent event.</span> </p><p><span m=''1530000''>So, we just measure
  across the genome, and look for this telltale</span> <span m=''1534000''>sign of
  common variance that have very long range correlation that</span> <span m=''1539000''>indicate
  that they''re very recent. So, a plot of the allele frequency,</span> <span m=''1544000''>common
  variance, sorry, if something has a common high frequency and</span> <span m=''1549000''>long-range
  correlation, you wouldn''t expect that by chance.</span> </p><p><span m=''1554000''>So,
  something that was common in its</span> <span m=''1558000''>frequency and had long-range
  correlation would be a signature of</span> <span m=''1562000''>positive selection.
  So anyway, Pardise had this idea,</span> <span m=''1566000''>and she tried it out
  with some interesting mutations,</span> <span m=''1569000''>some mutations that
  confer resistance to malaria,</span> <span m=''1573000''>one well-known mutation
  causing resistance to malaria called G6 PD</span> <span m=''1577000''>and another
  one that she herself had proposed as a mutation causing</span> <span m=''1581000''>resistance
  to malaria, variants in the CD4 ligand gene.</span> </p><p><span m=''1584000''>And
  to make a long story short, both the known and her newly</span> <span m=''1588000''>predicted
  variant showed this telltale property of having a high</span> <span m=''1592000''>frequency
  and very long range correlation.</span> </p><p><span m=''1596000''>Well that''s
  very interesting because she was able to show that each of</span> <span m=''1600000''>these
  mutations probably were the result of positive selection.</span> </p><p><span m=''1604000''>But
  what you could do in principle is test every variant in the human</span> <span m=''1609000''>genome
  this way: take any variant, look at its frequency, and compare</span> <span m=''1613000''>it
  to the long range correlation around it, and test every single</span> <span m=''1618000''>variant
  in the human population to see which ones might be the result</span> <span m=''1622000''>of
  long range correlation. Now, when she proposed this,</span> <span m=''1626000''>this
  was about a year and a half ago or two years ago,</span> <span m=''1629000''>this
  was a pretty nutty idea because you would need all the variants in</span> <span
  m=''1632000''>the human population, and you would need all this</span> <span m=''1635000''>correlation
  information. But in fact, as I say, that</span> <span m=''1638000''>information''s
  almost upon us, and I believed that this experiment,</span> <span m=''1641000''>this
  analysis to look for all strong positive selection in the human</span> <span m=''1644000''>genome
  will in fact be done in the course of the next 12 months.</span> </p><p><span m=''1647000''>So,
  I''m hoping by next year I can actually report on a genome-wide</span> <span m=''1650000''>search
  for all the signatures of positive selection.</span> </p><p><span m=''1653000''>Now,
  this doesn''t detect all positive selection.</span> </p><p><span m=''1656000''>It
  will detect sufficiently strong positive selection going back pretty</span> <span
  m=''1659000''>much only over the 10, 00 years. When you do the</span> <span m=''1662000''>arithmetic,
  that''s how much power you have. Of course,</span> <span m=''1665000''>10,000 years
  has been a pretty interesting time for the human</span> <span m=''1668000''>population,
  right? The time of civilization and</span> <span m=''1672000''>population density,
  and infectious diseases,</span> <span m=''1675000''>and all that, and I think we''ll
  have an interesting window into</span> <span m=''1678000''>the change in diet. All
  of that should come out of</span> <span m=''1682000''>something like this. So, there''s
  a lot of really cool</span> <span m=''1686000''>information in DNA variation to
  be had. All right,</span> <span m=''1690000''>that''s one half. The other half of
  what I would like to talk about is</span> <span m=''1694000''>totally different.
  It''s not about inherited DNA</span> <span m=''1698000''>variation. It''s about
  somatic differences between tissues in RNA</span> <span m=''1702000''>variation.
  So, let''s shift gears.</span> </p><p><span m=''1706000''>RNA variation: let me
  start by giving you an example here.</span> </p><p><span m=''1710000''>These are
  cells from two different patients with acute leukemia.</span> </p><p><span m=''1716000''>Can
  you spot the difference between these? Yep? More like bunches of</span> <span m=''1723000''>grapes
  and all that. Yeah, it turns out that''s just a</span> <span m=''1729000''>reflection
  of the field of view you have if you move over</span> <span m=''1736000''>to look
  like that. But I mean, that''s good.</span> </p><p><span m=''1742000''>It''s just
  that it turns out that that isn''t actually a distinction</span> <span m=''1747000''>when
  you look at more fields. Anything else? Yep? White blood</span> <span m=''1752000''>cells
  like different. They look broken. There''s more of</span> <span m=''1756000''>them
  in this field of view. But you look at 100 fields of view</span> <span m=''1761000''>and
  it turns out that''s not either. Well, the reason you''re having</span> <span m=''1766000''>trouble
  spotting any difference is that highly trained pathologists</span> <span m=''1771000''>can''t
  find any difference either. I generally agree there''s no</span> <span m=''1775000''>difference
  between these two if you look at enough fields of view.</span> </p><p><span m=''1779000''>But
  you can convince yourself if you look that you see things there.</span> </p><p><span
  m=''1783000''>But these actually are two very different kinds of leukemia.</span>
  </p><p><span m=''1786000''>And, these patients have to be treated very differently.</span>
  </p><p><span m=''1790000''>But, pathologists cannot determine which leukemia it
  is just by looking</span> <span m=''1794000''>at the microscope, it turns out. This
  is the work of this man,</span> <span m=''1797000''>Sydney Farber, namesake of the
  Dana Farber Cancer Institute here in</span> <span m=''1801000''>Boston, who in the
  1950s began noticing that patients with</span> <span m=''1805000''>leukemias, some
  of them seemed different in the way they responded</span> <span m=''1808000''>to
  a certain treatment, and he said, look, I think there''s</span> <span m=''1812000''>some
  underlying classification of these leukemias,</span> <span m=''1816000''>but I can''t
  get any reliable way to tell it in the microscope.</span> </p><p><span m=''1819000''>And
  he put many years into working this out, first by noticing certain</span> <span
  m=''1823000''>difference in enzymes in the cells, and then people noticed certain</span>
  <span m=''1827000''>things in cell surface markers, and some chromosomal rearrangements.</span>
  </p><p><span m=''1831000''>And nowadays, there are a bunch of test that can be done
  by a</span> <span m=''1834000''>pathologist when a patient comes in with acute leukemia
  to determine</span> <span m=''1838000''>whether they have AML or ALL. But it turns
  out that you can''t do</span> <span m=''1842000''>it by looking. You have to do
  some kind of</span> <span m=''1846000''>immunohystochemical test of some sort in
  order to do that.</span> </p><p><span m=''1850000''>So this is a triumph of diagnosis.
  After 40 years of work, we can now</span> <span m=''1854000''>correctly classify
  patients as AML or ALL. And they get the</span> <span m=''1858000''>appropriate
  treatment. And if they don''t get the right</span> <span m=''1862000''>treatment,
  they have a much higher chance of dying.</span> </p><p><span m=''1866000''>And if
  they do get the right treatment, they have a much higher</span> <span m=''1870000''>chance
  of living. So, this is great.</span> </p><p><span m=''1874000''>There''s only one
  problem with the story. It took 40 years,</span> <span m=''1878000''>40 years to
  sort this out. That''s a long time. Couldn''t we do</span> <span m=''1882000''>better?
  Surely these cells know what they are.</span> </p><p><span m=''1886000''>Surely
  we could just ask them if they are. Well, here''s the idea.</span> </p><p><span
  m=''1890000''>Suppose we could ask each cell, please tell us every gene that you</span>
  <span m=''1893000''>have turned on, and the level to which you have that</span>
  <span m=''1897000''>gene expressed. In other words,</span> <span m=''1900000''>let
  us summarize each cell, each tumor by a description of its</span> <span m=''1904000''>complete
  pattern of gene expression to 22,000 genes on the human genome.</span> </p><p><span
  m=''1907000''>Let''s write down the level of expression, X1 up to X22,</span> <span
  m=''1911000''>00 for each of the 22,000 genes of the genome. So,</span> <span m=''1914000''>ever
  tumor becomes a point in 22, 00 dimensional space, right?</span> </p><p><span m=''1918000''>Now
  clearly, if we had every tumor described as a point in 22,</span> <span m=''1921000''>00
  dimensional space, we ought to be able to sort out which tumors are</span> <span
  m=''1925000''>similar to each other, right? Well, it turns out you can</span> <span
  m=''1929000''>do that now. These are gene chips, one of several technologies by
  which</span> <span m=''1933000''>on a piece of glass are put little spots, each
  of which contains a</span> <span m=''1937000''>piece of DNA, a unique DNA sequence.
  Actually, many copies of that DNA</span> <span m=''1941000''>sequence are there.
  Each of these is a 25 base long DNA</span> <span m=''1945000''>sequence, and I can
  design this so whatever DNA sequence you</span> <span m=''1949000''>want is in each
  spot. The way that''s done is with the same</span> <span m=''1952000''>photolithographic
  techniques that are used to make microprocessors.</span> </p><p><span m=''1956000''>People
  have worked out a chemistry where through a mask,</span> <span m=''1960000''>you
  shine a light, photodeprotect certain pixels; the pixels that are</span> <span m=''1964000''>photodeprotected
  you can chemically attach an A, then re-protect the</span> <span m=''1968000''>surface.
  Use a light. Chemically photodeprotect certain</span> <span m=''1972000''>spots.
  Wash on a C. And in this fashion,</span> <span m=''1976000''>since you can randomly
  address the spots by light,</span> <span m=''1980000''>and then chemically add bases
  to whatever spots are deprotected,</span> <span m=''1984000''>you can simultaneously
  construct hundreds of thousands of spots each</span> <span m=''1988000''>containing
  its own unique specified oligonucleotide sequence.</span> </p><p><span m=''1992000''>And
  you can get them in little plastic chips.</span> </p><p><span m=''1996000''>And
  then if you want, all you do is you take a tumor.</span> </p><p><span m=''2000000''>You
  grind it up. You prepare RNA. You fluorescently label the RNA</span> <span m=''2004000''>with
  some appropriate fluorescent dye. You squirt it into the chip.</span> </p><p><span
  m=''2008000''>You wash it back and forth. You rock it back and forth,</span> <span
  m=''2011000''>wash it out, and stick it in a laser scanner. And it''ll see how much</span>
  <span m=''2015000''>fluorescence is stuck to each spot. And bingo: you get a readout
  of the</span> <span m=''2018000''>level of gene expression. I guess each spot, you
  should</span> <span m=''2022000''>design it so that this spot has an oligonucleotide
  complementary to</span> <span m=''2025000''>gene number one. And the next one,</span>
  <span m=''2029000''>an oligonucleotide matching by Crick-Watson base pairing</span>
  <span m=''2033000''>complementary to gene number two and gene number three.</span>
  </p><p><span m=''2036000''>So, if I knew all the genes in the genome, I could make
  a detector spot</span> <span m=''2040000''>for each gene in the genome. And of course
  we know essentially</span> <span m=''2043000''>all the genes in the genome. So you
  can make those detector</span> <span m=''2047000''>spots and you can buy them. So,
  you can now get a readout of</span> <span m=''2050000''>all the, I mean, this is
  like so cool because when I</span> <span m=''2053000''>started teaching 701, which
  wasn''t that long ago because I</span> <span m=''2057000''>ain''t (sic) that old
  still, the way people did an analysis of</span> <span m=''2060000''>gene expression
  is they used primitive technologies where they</span> <span m=''2063000''>would
  analyze one gene at a time, certain things called northern blots</span> <span m=''2067000''>and
  things like that, right? And, you know,</span> <span m=''2070000''>you''d put in
  a lot of work and you get the expression level of a gene,</span> <span m=''2074000''>whereas
  now you can get the expression of all the genes</span> <span m=''2077000''>simultaneously,
  and it''s pretty mind boggling that</span> <span m=''2081000''>you can do that.
  How do you analyze data like that?</span> </p><p><span m=''2084000''>So, we still
  use northern blots. It''s true. So,</span> <span m=''2088000''>every tumor becomes
  a vector, and we get a vector corresponding to</span> <span m=''2091000''>each tumor.
  So, this line here is the first tumor,</span> <span m=''2095000''>the second tumor,
  the third tumor, the fourth tumor.</span> </p><p><span m=''2099000''>The columns
  here correspond to genes. There are 22,</span> <span m=''2102000''>00 columns in
  this matrix, and I''ve shown a certain subset of</span> <span m=''2106000''>the
  columns because these genes here have the interesting property that</span> <span
  m=''2110000''>they tend to be high red in the ALL tumors, and they tend to be low
  blue</span> <span m=''2114000''>in the AML tumors, whereas these genes here have
  the</span> <span m=''2118000''>opposite property. They tend to be low blue in the
  ALL</span> <span m=''2122000''>tumors and high red in the AML tumors. These genes
  do a pretty</span> <span m=''2126000''>good job of telling apart these tumors.</span>
  </p><p><span m=''2130000''>So, here''s a new tumor. Patient came in. We analyzed
  the</span> <span m=''2135000''>RNA, squirted it on the chip. Can somebody classify
  that? Louder?</span> </p><p><span m=''2140000''>AML. Next? Next? Congratulations,
  you''re</span> <span m=''2145000''>pathologists. Very good. That''s right, you can
  do that.</span> </p><p><span m=''2150000''>It works. And in fact, in the study that
  was done that was</span> <span m=''2156000''>published about this, the computer
  was able to get it</span> <span m=''2161000''>right 100% of the time. Not bad. So
  now you say,</span> <span m=''2165000''>wait, wait, wait, but you''re cheating.</span>
  </p><p><span m=''2169000''>You''re giving it a whole bunch of knowns. Once I have
  a whole bunch</span> <span m=''2172000''>of knowns it''s not so hard to classify
  a new tumor.</span> </p><p><span m=''2175000''>What Sydney Farber did was he discovered
  in the first place that</span> <span m=''2179000''>there existed two subtypes. Surely
  that''s harder than</span> <span m=''2182000''>classifying when you''re given a
  bunch of knowns. And</span> <span m=''2186000''>that''s true. So, suppose instead,</span>
  <span m=''2189000''>I didn''t tell you in advance which were AML''s and which were
  ALL''s,</span> <span m=''2193000''>and I just gave you vectors corresponding to
  a large number of</span> <span m=''2197000''>tumors, do you think you would be able
  to sort out that they actually</span> <span m=''2201000''>fell into two clusters?</span>
  </p><p><span m=''2209000''>Could you by computer tell that there''s one class and
  the other</span> <span m=''2213000''>class? Turns out that you can. Now, I''ve made
  it a little easier</span> <span m=''2217000''>by not listing most of the 22,000
  columns here.</span> </p><p><span m=''2222000''>But think about it. Every tumor
  is a point in 22,</span> <span m=''2226000''>00 dimensional space. If some of the
  tumors are similar,</span> <span m=''2230000''>what can you say about those points
  in 22,000 dimensional space?</span> </p><p><span m=''2234000''>They''re going to
  be clumped together. They''re near each other.</span> </p><p><span m=''2238000''>So,
  just plot every tumor as a point in 22,000 dimensional space,</span> <span m=''2242000''>and
  your question is, do the points tend to lie in two clumps up in 22,</span> <span
  m=''2246000''>00 dimensional space? And there''s simple arithmetic you</span> <span
  m=''2250000''>can learn using linear algebra to get some separating hyperplane and</span>
  <span m=''2254000''>ask, do tumors lie on one side or the other? And,</span> <span
  m=''2258000''>it turns out the procedures like that will quickly tell you that</span>
  <span m=''2262000''>these tumors clump into two very clear clumps. They''re not
  randomly</span> <span m=''2266000''>distributed. And so, if you get these tumors,</span>
  <span m=''2270000''>and you do gene expression on them and put the data into a computer,</span>
  <span m=''2274000''>the amount of time it takes the computer to discover that there
  were</span> <span m=''2278000''>actually two types of acute leukemia is about three
  seconds marked down</span> <span m=''2282000''>from 40 years. That''s good. So,
  you can reproduce the discovery</span> <span m=''2286000''>of AML and ALL in three
  seconds. Now you know what the pathologists</span> <span m=''2290000''>say about
  this. They say, oh, give me a break.</span> </p><p><span m=''2294000''>It''s shooting
  fish in a barrel. We know there was a distinction.</span> </p><p><span m=''2298000''>Big
  deal that the computer can find the distinction.</span> </p><p><span m=''2302000''>We
  knew that there was distinction there. I know the computer didn''t</span> <span
  m=''2306000''>know it and all that. Tell us something we don''t know.</span> </p><p><span
  m=''2310000''>That''s a fair question. So it turns out that you can ask</span> <span
  m=''2315000''>some more questions. You can say, suppose I take now</span> <span
  m=''2320000''>just the ALL''s. Are they a homogeneous class,</span> <span m=''2325000''>or
  did they fall into two classes? It turns out that extending this</span> <span m=''2330000''>work,
  folks here were able to show that we can further split that ALL</span> <span m=''2335000''>class.
  There was a hint that you might be able to do so because</span> <span m=''2340000''>there''s
  some ALL patients who have disruptions of a gene called MLL.</span> </p><p><span
  m=''2346000''>And this tends to be a little more common in infants,</span> <span
  m=''2349000''>and tends to be associated with a poor prognosis.</span> </p><p><span
  m=''2353000''>But it was really very unclear whether this was simply one of a</span>
  <span m=''2356000''>zillion factoids about some leukemia patients, whether this
  was a</span> <span m=''2360000''>fundamental distinction. So, what happened was
  folks took a</span> <span m=''2364000''>lot of ALL patients, got their expression
  profiles,</span> <span m=''2367000''>and lo and behold it turned out that ALL itself
  broke into two very</span> <span m=''2371000''>different clusters. This is an artist''s
  rendition of a</span> <span m=''2374000''>22,000 dimensional space. We can''t afford
  a 22,000</span> <span m=''2378000''>dimensional projector here, so we''re just using
  two dimensions.</span> </p><p><span m=''2382000''>But, the two forms of ALL were
  quite distinct from each other,</span> <span m=''2386000''>and so actually ALL itself
  should be split up into two classes,</span> <span m=''2390000''>ALL plus and minus,
  or ALL one and two, or MLL and ALL.</span> </p><p><span m=''2394000''>And it turns
  out that these forms are quite different.</span> </p><p><span m=''2398000''>They
  have different outcomes and should be treated differently.</span> </p><p><span m=''2402000''>It
  also turns out that a particularly good distinction</span> <span m=''2407000''>between
  these two subtypes of ALL is found by looking at this particular</span> <span m=''2412000''>gene
  called the flit-3 kinase. The flit-3 kinase gene, whatever</span> <span m=''2417000''>that
  is, was of great interest because people know that they can</span> <span m=''2423000''>make
  inhibitors against certain kinases. And so,</span> <span m=''2428000''>it turned
  out that an inhibitor against flit-3 kinases,</span> <span m=''2433000''>against
  this flit-3 kinase gene product.</span> </p><p><span m=''2439000''>If you treat
  cells with that inhibitor, cells of this type die,</span> <span m=''2444000''>and
  cells of this type are not affected. So in fact,</span> <span m=''2449000''>there''s
  a potential drug use of flit-3 kinases in the MLL class of</span> <span m=''2454000''>these
  leukemias, and folks are trying some clinical</span> <span m=''2460000''>trials
  now. So, not only did the analysis of the</span> <span m=''2465000''>gene expression
  point to two important sub-types of leukemias,</span> <span m=''2469000''>but the
  analysis of the gene expression even suggested potential</span> <span m=''2474000''>targets
  for therapy. So, I''ll give you a bunch more</span> <span m=''2479000''>examples.
  I have a bunch more examples like that there.</span> </p><p><span m=''2483000''>They
  are examples of taking lymphomas and showing that they can</span> <span m=''2488000''>be
  split into two different categories, examples of taking</span> <span m=''2493000''>breast
  cancers into several categories, colon cancers.</span> </p><p><span m=''2498000''>Basically
  what''s going on right now is an attempt to reclassify cancers</span> <span m=''2502000''>based
  not on what they look like in the microscope,</span> <span m=''2507000''>and based
  not on what organ in the body they affect,</span> <span m=''2511000''>but based
  on, molecularly, what their description is, because</span> <span m=''2516000''>the
  molecular description, as Bob talked to you about with CML</span> <span m=''2521000''>and
  with Gleveck, turns out to be a tremendously</span> <span m=''2525000''>powerful
  way of classifying cancers because you''re able to see what is</span> <span m=''2530000''>the
  molecular defect and can make a molecular targeted therapy.</span> </p><p><span
  m=''2535000''>So, these sorts of tools are quite cool, and I''ve got to say,</span>
  <span m=''2540000''>in the last year we''ve begun using these expression tools not
  just to</span> <span m=''2545000''>classify cancers, but to classify drugs.</span>
  </p><p><span m=''2550000''>We''ve begun an interesting and somewhat crazy project
  to take all</span> <span m=''2554000''>the FDA approved drugs, put them onto cell
  types,</span> <span m=''2558000''>and see what they do, that is, get a signature,
  a fingerprint,</span> <span m=''2562000''>a gene expression description of the action
  of a drug.</span> </p><p><span m=''2566000''>And then we hope, here''s the nutty
  idea,</span> <span m=''2570000''>that we can look up in the computer which drugs
  do which things and</span> <span m=''2574000''>might be useful for which diseases,
  because we''d put the diseases and</span> <span m=''2578000''>the drugs on an equal
  footing. All of them would be described in</span> <span m=''2582000''>terms of their
  gene expression patterns. So,</span> <span m=''2586000''>I''ll tell you one interesting
  example, OK? This is an interesting</span> <span m=''2590000''>enough example. I
  don''t even have slides for it yet.</span> </p><p><span m=''2594000''>It turns out
  that these patients with ALL that I''ve been talking</span> <span m=''2598000''>about,
  some of the patients with ALL will respond to the drug</span> <span m=''2603000''>dexamethasone.
  Some won''t. If you take patients</span> <span m=''2607000''>who respond to dexamethasone,
  and patients who are resistant to</span> <span m=''2611000''>dexamethasone, and
  you get their gene expression</span> <span m=''2615000''>patterns, you can ask are
  there some genes that explain the difference?</span> </p><p><span m=''2620000''>And
  you can get a certain gene signature, a list of,</span> <span m=''2624000''>say,
  a dozen or so genes that do a pretty good job of classifying who''s</span> <span
  m=''2628000''>sensitive and who''s resistant. Then you can go to this database I</span>
  <span m=''2633000''>was telling you about of the action of many drugs and say,</span>
  <span m=''2637000''>do we see any drugs whose effect would be to produce a signature</span>
  <span m=''2641000''>of sensitivity? If we found a drug X,</span> <span m=''2646000''>which
  when we put it on cells turned on those genes that correlate with</span> <span m=''2650000''>being
  sensitive to dexamethasone, you could hallucinate the following</span> <span m=''2654000''>really
  happy possibility that when you added that drug together with</span> <span m=''2658000''>dexamethasone,
  you might be able to treat resistant</span> <span m=''2662000''>patients because
  that drug could make them sensitive to dexamethasone,</span> <span m=''2666000''>and
  that you could find that drug just by looking it up in</span> <span m=''2670000''>a
  computer database. So, we tried it and we hit a drug.</span> </p><p><span m=''2675000''>There
  was a certain drug that came up on the screen,</span> <span m=''2680000''>yes? That''s
  very much in the idea too. We found a drug that produced</span> <span m=''2685000''>the
  signature sensitivity, and tested it in vitro. In vitro,</span> <span m=''2689000''>if
  you take cells that are resistant and you add dexamethasone,</span> <span m=''2694000''>nothing
  happens because they''re resistant. If you add drug X,</span> <span m=''2699000''>nothing
  happens. But if you add both drug X plus dexamethasone,</span> <span m=''2704000''>the
  cells drop dead. It''s now going into clinical trials</span> <span m=''2708000''>in
  human patients. It turns out drug X is already a</span> <span m=''2712000''>well
  FDA approved drug, so it can be tested in human</span> <span m=''2715000''>patients
  right away, so it''s going to be tested.</span> </p><p><span m=''2719000''>So, the
  gene expression pattern was able to tell us to use a drug which</span> <span m=''2722000''>actually
  had nothing to do with cancer uses in a cancer setting</span> <span m=''2726000''>because
  it might do something helpful.</span> </p><p><span m=''2730000''>Now, what''s the
  point of all this? We can turn up the lights because I</span> <span m=''2733000''>think
  I''m going to stop the slides there. The point of all of this,</span> <span m=''2737000''>which
  is what I''ve made again, and I will make again,</span> <span m=''2741000''>because
  you are the generation that''s going to really live this,</span> <span m=''2745000''>is
  that biology is becoming information. Now,</span> <span m=''2748000''>don''t get
  me wrong. It''s not stopping being</span> <span m=''2752000''>biochemistry. It''s
  going to be biochemistry. It''s not stopping</span> <span m=''2756000''>being molecular
  biology. It''s not stopping any of the things</span> <span m=''2760000''>it was
  before. 45:57 But it is also becoming</span> <span m=''2763000''>information, that
  for the first time we''re entering a world where we can</span> <span m=''2767000''>collect
  vast amounts of information: all the genetic variants in a</span> <span m=''2771000''>patient,
  all of the gene expression pattern in a cell,</span> <span m=''2775000''>or all
  of the gene expression pattern induced by a drug,</span> <span m=''2778000''>and
  that whatever question you''re asking will be informed by being</span> <span m=''2782000''>able
  to access that whole database. In no way does it decrease the role</span> <span
  m=''2786000''>of the individual smart scientist working on his or her problem.</span>
  </p><p><span m=''2790000''>To the contrary, the goal is to empower the</span> <span
  m=''2792000''>individual smart scientist so that you have all of that information
  at</span> <span m=''2795000''>your fingertips. There are databases scattered</span>
  <span m=''2798000''>around the web that have sequences from different species,</span>
  <span m=''2801000''>variations from the human population, all of these drug database,</span>
  <span m=''2804000''>etc., etc., etc., etc. It''s a time of tremendous ferment,</span>
  <span m=''2807000''>a little bit of chaos. You talk to people in the field,</span>
  <span m=''2810000''>they say, we''re getting deluged by data. We''re getting crushed
  by the</span> <span m=''2813000''>amount of data. I don''t'' know what to do with
  all</span> <span m=''2816000''>the data. There''s only one solution for a</span>
  <span m=''2819000''>field in that condition, and that is young scientists because</span>
  <span m=''2822000''>the young scientists who come into the field are the ones who
  take for</span> <span m=''2825000''>granted, of course we''re going to have all
  these data.</span> </p><p><span m=''2828000''>We love having all these data. This
  is just great, couldn''t be</span> <span m=''2831000''>happier to have all these
  data. We''re not put off by it in the</span> <span m=''2834000''>least. That''s
  what''s going on. That''s what''s so important about</span> <span m=''2837000''>your
  generation, and that''s why I think it''s really</span> <span m=''2840000''>important
  that even though it''s 701 and we''re supposed to be teaching</span> <span m=''2843000''>you
  the basics, it''s important that you see this</span> <span m=''2846000''>stuff because
  this is the change that''s going on,</span> <span m=''2849000''>and we''re counting
  on this very much to drive a revolution in health,</span> <span m=''2852000''>a
  revolution in biomedical research, and we''re counting on you guys very</span> <span
  m=''2855000''>much to drive that revolution. It has been a pleasure to teach you</span>
  <span m=''2859000''>this term. I hope many of you will stay in touch,</span> <span
  m=''2863000''>and some of you will go into biology, and even those of you who don''t
  will</span> <span m=''2868000''>know lots about it and enjoy it. Thank you very
  much. [APPLAUSE]</span> </p>'
type: course
uid: b82c7be20d0a354b9a36ccd2e3ee48ef

---
None