---
about_this_resource_text: <p><strong>Topics covered:</strong> Recombinant DNA 1</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec15.jpg
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-15-recombinant-dna-1/lec15.jpg
  title: lec15.jpg
  type: null
  uid: a3ed861fa813cd4ef08baa6107a67d22
- id: Video-YouTube-Stream
  media_location: _TdJBLu6hPc
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Video-YouTube-Stream
  type: Video
  uid: a1d0016796379b82f151a67b1b19cd59
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/_TdJBLu6hPc/default.jpg
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 68a255dd761d77c3fcb366f390abde74
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Video-iTunes U-MP4
  type: Video
  uid: 8f7d865b84d6435004e334725fe88286
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec15-15oct2004-220k.mp4
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Video-Internet Archive-MP4
  type: Video
  uid: 0c8d0b45fe24c1399df204ec5fa2d4b5
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 8e39bc66ae6300584eb992271310ef4c
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-15oct2004-1000.mp3
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Video-Internet Archive-MP3
  type: Video
  uid: 3660d6a426e8312caf2e1ec50f724d64
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 7110bba9a07cae24b4b307336f888d71
- id: Thumbnail-OCW-JPG
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 247c5e8cc3ab9c725489026b6bdfa184
- id: 3Play-3PlayYouTubeid-MP4
  media_location: _TdJBLu6hPc
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 17346fa49b196c43305d2099ca4d30c7
- id: TdJBLu6hPc.srt
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-15-recombinant-dna-1/TdJBLu6hPc.srt
  title: 3play caption file
  type: null
  uid: 1eb71bb7c17534d2faee8a5e75b6760d
- id: TdJBLu6hPc.pdf
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-15-recombinant-dna-1/TdJBLu6hPc.pdf
  title: 3play pdf file
  type: null
  uid: 8a3bf11196b6f453ae2784274352173f
- id: Caption-3Play YouTube id-SRT
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: f2cd386491eb8f4cabe80cb8c054fe4c
- id: Transcript-3Play YouTube id-PDF
  parent_uid: 04338efad747f6be4bd2a10a6662563b
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: f5a78bc30f07b569a45969381623af72
inline_embed_id: 96161898lecture15:recombinantdna154101542
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-15-recombinant-dna-1
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-15-recombinant-dna-1
template_type: Tabbed
title: 'Lecture 15: Recombinant DNA 1'
transcript: "<p><span m='9000'>Good morning. So, I see we have a lot of parents here.</span>\
  \ </p><p><span m='13000'>How many parents have we got here?</span> </p><p><span\
  \ m='17000'>Welcome to the parents. How many of the parents have done</span> <span\
  \ m='20000'>the reading for today? [LAUGHTER] Good, because we'll call</span> <span\
  \ m='23000'>the parents too, right? We'll see what happens.</span> </p><p><span\
  \ m='27000'>All right, so, where are we? We've talked about this diagram that I</span>\
  \ <span m='31000'>keep coming back to. If you want to</span> <span m='36000'>study\
  \ biological function the two traditional ways to do that were to</span> <span m='40000'>look\
  \ at genetics or to look at biochemistry: genetics, the</span> <span m='44000'>study\
  \ of an organism with one broken component, those components being</span> <span\
  \ m='48000'>genes; biochemistry: the study of the purification of</span> <span m='53000'>individual\
  \ components from an organism away from the organism,</span> <span m='55000'>particularly\
  \ the most important such components being proteins.</span> </p><p><span m='58000'>What\
  \ do they have to do to each other?</span> </p><p><span m='61000'>The unification\
  \ in molecular biology that occurred in the middle</span> <span m='65000'>of the\
  \ century from the 1950s into the \x9160s and really</span> <span m='69000'>up to\
  \ 1970 or so, we came to a conceptual understand</span> <span m='73000'>that genes\
  \ encode proteins,</span> <span m='76000'>and therefore these two different ways\
  \ of looking at the organism:</span> <span m='79000'>organism minus a component,\
  \ components</span> <span m='83000'>minus an organism were complementary points\
  \ of view,</span> <span m='86000'>and in theory, you could go from a gene sequence\
  \ to a protein</span> <span m='89000'>Sequence, a protein sequence back to a gene\
  \ sequence,</span> <span m='92000'>to go for a gene sequence to its function, its\
  \ function</span> <span m='95000'>to a protein, except for one to a tiny detail.\
  \ This was all just</span> <span m='100000'>conceptual. Conceptually we understood\
  \ by about 1970</span> <span m='106000'>that the DNA made the RNA made the protein.\
  \ The protein carried</span> <span m='111000'>out the function but as of then,</span>\
  \ <span m='116000'>you couldn't individually work with or purify the DNA corresponding\
  \ to</span> <span m='120000'>any particular gene. All of the inferences had been</span>\
  \ <span m='124000'>indirect inferences: indirect inferences from bacterial genetics,</span>\
  \ <span m='127000'>bacterial regulation or Meselson-Stahl experiments,</span> <span\
  \ m='131000'>and all sorts of interesting indirect ways</span> <span m='135000'>working\
  \ out the genetic code, but it didn't let you read anything.</span> </p><p><span\
  \ m='139000'>This was a problem. Some people in the</span> <span m='143000'>late\
  \ 1960s said, great, molecular biology is over.</span> </p><p><span m='146000'>We\
  \ understand in principle how life works. Now let's go understand</span> <span m='149000'>how\
  \ the brain works. And there was an exodus of some</span> <span m='151000'>people\
  \ from molecular biology into neurobiology to now go</span> <span m='154000'>nail\
  \ the brain, figured that would be worth another</span> <span m='156000'>ten years\
  \ or so. But in fact, remarkably,</span> <span m='159000'>people began to focus\
  \ on how you could get to</span> <span m='162000'>work with individual specific\
  \ genes. Now, what's so hard about that? I</span> <span m='166000'>mean, it's not\
  \ very hard to crack open a</span> <span m='169000'>red blood cell and purify different\
  \ proteins. You can purify hemoglobin.</span> </p><p><span m='173000'>You can purify\
  \ different enzymes. Biochemistry allows you to purify</span> <span m='176000'>different\
  \ components from each other. I want to purify an enzyme: let's</span> <span m='179000'>crack\
  \ open a yeast cell, separate the</span> <span m='182000'>proteins over some column\
  \ that separates them based on their size</span> <span m='184000'>or their charge,\
  \ and I'll get purer and purer</span> <span m='186000'>fractions. I'll assay each\
  \ fraction to see</span> <span m='189000'>which one has the enzymatic activity.\
  \ But basically I use the</span> <span m='191000'>physical chemical properties of\
  \ the proteins to</span> <span m='194000'>separate them into different buckets.\
  \ Why not do that with,</span> <span m='197000'>say, the human DNA and purify out\
  \ the gene for beta</span> <span m='201000'>globin, that encodes the beta component\
  \ of hemoglobin?</span> </p><p><span m='205000'>What would be the problem of just\
  \ using physical chemical purification</span> <span m='210000'>to purify one human\
  \ gene from another? Well, I mean, it's one</span> <span m='216000'>very big molecule.\
  \ Well, I could sheer it up.</span> </p><p><span m='219000'>Maybe I'll just break\
  \ it up. Now, let's purify the beta</span> <span m='223000'>globin containing part.\
  \ It all looks the same. It's just</span> <span m='227000'>DNA. It's one chemical\
  \ polymer with pretty boring properties,</span> <span m='231000'>and they're not\
  \ very different.</span> </p><p><span m='234000'>Any particular DNA sequence in\
  \ any other DNA sequence basically about</span> <span m='236000'>the same molecular\
  \ weight, same charges,</span> <span m='240000'>there's nothing to separate them\
  \ by. How are you going to purify beta</span> <span m='244000'>globin? That was\
  \ the problem. That's where</span> <span m='248000'>recombinant DNA came in was\
  \ recombinant DNA was a remarkable</span> <span m='251000'>and totally different\
  \ way of purifying individual</span> <span m='255000'>components. And the basis\
  \ of it was this notion of cloning.</span> </p><p><span m='259000'>If I want to\
  \ purify out from the human</span> <span m='263000'>genome, how big is the human\
  \ genome? The human genome is about three</span> <span m='267000'>billion bases\
  \ long. If I want to purify a particular</span> <span m='271000'>gene, let's say\
  \ beta globin or some other gene,</span> <span m='277000'>typical gene, might be\
  \ on the order of 30,000 letters long.</span> </p><p><span m='286000'>This is one\
  \ part in 105 purification I've got to achieve.</span> </p><p><span m='289000'>Any\
  \ given gene is only one part in</span> <span m='293000'>105 of the human genome.\
  \ And then, what about a typical</span> <span m='296000'>mutation? Maybe the mutation\
  \ that causes</span> <span m='299000'>sickle cell anemia by changing a single nucleotide\
  \ and beta globe,</span> <span m='305000'>well, that's one base pair. So that means\
  \ I'm trying to identify</span> <span m='310000'>something that's on the order of\
  \ one part in 109 actually,</span> <span m='314000'>a little less than one part\
  \ in 109 of the whole genome.</span> </p><p><span m='317000'>Carrying out purifications\
  \ like: really kind of hard</span> <span m='321000'>to imagine. But the way it was\
  \ done was by the invention of cloning.</span> </p><p><span m='325000'>Let me briefly\
  \ overview of the idea</span> <span m='329000'>of cloning, and then we'll dive into\
  \ the details. The idea of cloning</span> <span m='337000'>was, the way to purify\
  \ individual molecules would just be</span> <span m='342000'>to take the molecules\
  \ and just dilute them so that there</span> <span m='346000'>was only one of each\
  \ model. That's very pure,</span> <span m='349000'>isn't it? The problem is it's\
  \ not very much, so you need</span> <span m='352000'>a way to take a single copy\
  \ of a molecule,</span> <span m='356000'>and then make many copies of it. So purification's\
  \ not hard. You</span> <span m='360000'>just dilute it down so you work with</span>\
  \ <span m='363000'>single molecules but then you need to copy it back again</span>\
  \ <span m='365000'>and again and again, and no biochemical technique</span> <span\
  \ m='368000'>involves, say, fractionating a cell and replicating</span> <span m='370000'>some\
  \ enzyme, you know, copying some enzyme.</span> </p><p><span m='373000'>You can't\
  \ copy enzymes, but you can copy DNA, and that was</span> <span m='376000'>the basis\
  \ of it. So here's the way it goes. The</span> <span m='379000'>basic overview we'll\
  \ look at is take your DNA and cut your DNA</span> <span m='382000'>of interest,\
  \ maybe the human genome,</span> <span m='388000'>into pieces at defined sites Then,\
  \ paste your DNA, which is more</span> <span m='396000'>technically ligate, the\
  \ word we use.</span> </p><p><span m='404000'>Paste your DNA to some other DNA called\
  \ a vector. So,</span> <span m='413000'>cut your DNA and paste your DNA. Each piece\
  \ of your, say, human DNA</span> <span m='417000'>gets stuck to some piece of vector.</span>\
  \ </p><p><span m='426000'>Insert this DNA into vectors that can replicate in bacteria.</span>\
  \ </p><p><span m='438000'>So, I'm going to actually take my piece of human DNA</span>\
  \ <span m='445000'>and not just ligate it to any piece of DNA.</span> </p><p><span\
  \ m='448000'>I'm going to take my human DNA, and I'm going to ligate it to a</span>\
  \ <span m='452000'>vector that has all of the machinery,</span> <span m='455000'>all\
  \ of the ability to be copied in a bacteria. Then what</span> <span m='459000'>I'm\
  \ going to do is I'm going to transform my DNA into a</span> <span m='465000'>host\
  \ cell, a host bacterial cell. Transform means introduce. When we</span> <span m='473000'>talk\
  \ about transforming DNA,</span> <span m='479000'>we're not talking about changing\
  \ it. It's the word that's used for</span> <span m='482000'>taking my DNA, stuck\
  \ into a vector,</span> <span m='486000'>and introducing it into bacterial cells.\
  \ Ideally,</span> <span m='490000'>each bacterial cell would carry one such</span>\
  \ <span m='494000'>DNA molecule, and then what I want to do is I want to plate my\
  \ cells,</span> <span m='502000'>and select those that carry human DNA, my DNA;\
  \ DNA I've put on</span> <span m='511000'>it. So, I'm going to put them on a Petri\
  \ plate and I want only the</span> <span m='518000'>bacteria that happen to have\
  \ picked an individual piece of human DNA</span> <span m='524000'>to grow. So, that's\
  \ the trick. It's a very simple trick. Take</span> <span m='527000'>total human\
  \ DNA, cut it up into pieces, glue</span> <span m='530000'>it to a vector that's\
  \ able to be copied so that it's able to be</span> <span m='533000'>replicated in\
  \ bacteria, put the vectors into bacterial</span> <span m='536000'>cells; every\
  \ bacterial cell picks up no more than one vector.</span> </p><p><span m='539000'>You\
  \ plate it out, and you simply arrange so that the</span> <span m='543000'>only\
  \ cells that grow are those that picked up the piece of human DNA.</span> </p><p><span\
  \ m='546000'>And then, every one of these colonies</span> <span m='549000'>is the\
  \ descendent of a single bacterial cell that picked up a</span> <span m='552000'>single\
  \ human molecule, but is obligingly</span> <span m='555000'>copying that molecule\
  \ for you again and again and again and again.</span> </p><p><span m='558000'>And\
  \ thus, you have what we refer</span> <span m='561000'>to; this whole collection\
  \ here is called a library of clones.</span> </p><p><span m='566000'>This is called\
  \ a recombinant library because</span> <span m='571000'>every piece of the human\
  \ genome is somewhere in here.</span> </p><p><span m='575000'>You know, this one\
  \ here probably is active, and maybe</span> <span m='579000'>this one here maybe\
  \ is collagen-11 and that one there might,</span> <span m='583000'>ah, there's beta\
  \ globin. OK, actually</span> <span m='587000'>when you look at the plate there's\
  \ no way to tell but in principle</span> <span m='589000'>they're all there. So,\
  \ there will be this question of,</span> <span m='592000'>how do we look at a library\
  \ and pull out what the right one</span> <span m='594000'>is? But somewhere in there\
  \ should be a bacterial</span> <span m='597000'>colony that has pure beta globin\
  \ gene, the DNA for beta globin.</span> </p><p><span m='601000'>The next lecture\
  \ will be about how you actually find it.</span> </p><p><span m='603000'>But today\
  \ let's just build this library. So our</span> <span m='605000'>goal is to be able\
  \ to build a library like this.</span> </p><p><span m='609000'>So, we have to figure\
  \ out how to cut DNA,</span> <span m='613000'>paste, DNA, vectors, etc., etc. So\
  \ that's what our</span> <span m='615000'>subject will be today. Let's dive in.\
  \ First,</span> <span m='618000'>cutting DNA, how do you cut DNA? Restriction enzymes,</span>\
  \ <span m='626000'>etc. It turns out that the way you could cut</span> <span m='640000'>DNA\
  \ at particular places is as follows. Let me take a piece of DNA.</span> </p><p><span\
  \ m='643000'>Here's a double-stranded piece of DNA. We'll go A,</span> <span m='647000'>G,\
  \ C, T, A, G, A, A, T, T, C, T, T, A, C, C,</span> <span m='652000'>hydroxyl there,\
  \ three primad. Let's go back on the</span> <span m='657000'>other strand. What\
  \ do we have? G,</span> <span m='663000'>G, T, A, A, G, A, A, T, T, C, T, A, G,\
  \ C, T,</span> <span m='670000'>hydroxyl there, three prime. There's my double</span>\
  \ <span m='674000'>stranded piece of DNA. It turns out that there exists an</span>\
  \ <span m='679000'>enzyme that recognizes that exact sequence: G, A,</span> <span\
  \ m='689000'>A, T, T, C. The enzyme goes by the name Eco R1.</span> </p><p><span\
  \ m='699000'>This protein, this enzyme, scans along the the DNA, and it</span> <span\
  \ m='702000'>finds this sequence: G, A, A, T, T, C.</span> </p><p><span m='705000'>Actually\
  \ it's on this strand. What about on the other strand does</span> <span m='708000'>it\
  \ say? Same thing. But it's a reverse palindrome.</span> </p><p><span m='712000'>It's\
  \ symmetric. That's very good.</span> </p><p><span m='716000'>And it turns out most\
  \ restriction enzymes do that. OK,</span> <span m='720000'>so what it does when\
  \ it finds that,</span> <span m='724000'>with the benefit of colored chalk that\
  \ has just shown up here</span> <span m='728000'>is it cleaves the DNA fragment\
  \ like that.</span> </p><p><span m='734000'>And what it gives you then is a broken\
  \ double strand with</span> <span m='743000'>an overhang, T, T, A, A, five prime,</span>\
  \ <span m='749000'>three prime, three prime, five prime. This has a hydroxyl</span>\
  \ <span m='755000'>here this. This has a phosphate there. And then this other fragment</span>\
  \ <span m='758000'>here is A, A, T, T, C, T, T, A, C, C, G, G, T,</span> <span m='767000'>A.\
  \ So, what happens is, and this has a hydroxyl five</span> <span m='777000'>prime,\
  \ three prime, three prime, five prime I get into</span> <span m='784000'>two fragments\
  \ of DNA that have been</span> <span m='790000'>broken there and have it over. The\
  \ overhang is complementary.</span> </p><p><span m='796000'>Those two sequences\
  \ match each other. There's what's called a five prime</span> <span m='803000'>overhang\
  \ and they're complementary So, we have complementary,</span> <span m='810000'>that\
  \ is matching, five prime over X. This is called Eco R1 because it's</span> <span\
  \ m='816000'>purified, this particular enzyme,</span> <span m='823000'>from E coli\
  \ strain R and it's the number one such enzyme that</span> <span m='829000'>was\
  \ purified from it. So, it is very simple nomenclature</span> <span m='835000'>here.\
  \ Now, here's a question. Why do bacteria have an enzyme like</span> <span m='841000'>this?\
  \ There are some people who feel that the reason</span> <span m='845000'>is that\
  \ this enzyme is here precisely to allow molecular</span> <span m='849000'>biologists\
  \ to cut and paste DNA, and this represents</span> <span m='854000'>impressions\
  \ likely, me among them.</span> </p><p><span m='860000'>How did anybody find this\
  \ stuff? Well, shaggy dog story, I have to</span> <span m='864000'>tell you the\
  \ following shaggy dog story.</span> </p><p><span m='868000'>So, this is a fun shaggy\
  \ dog story, and it's an MIT shaggy dog story</span> <span m='871000'>because it\
  \ comes from the work of Salvador</span> <span m='874000'>Luria, who is a very famous\
  \ biologist who worked</span> <span m='877000'>here at MIT. So, Salvador Luria was\
  \ studying</span> <span m='881000'>bacteriophage. Remember, bacteriophage are the</span>\
  \ <span m='886000'>viruses that infect bacteria. So, he was studying bacteriophage,</span>\
  \ <span m='892000'>and he took his bacteriophage and used it to infect a strain\
  \ of</span> <span m='900000'>bacteria, strain A, and he also used it to infect a\
  \ strain of</span> <span m='907000'>bacteria, strain B. So when he did that,</span>\
  \ <span m='913000'>what you do is you plate a lawn of bacterial cells.</span> </p><p><span\
  \ m='917000'>You kind of have a slush of bacterial cells that you</span> <span m='921000'>plate\
  \ here with virus mixed in, and wherever there's a</span> <span m='926000'>virus,\
  \ the virus grows, replicates, and either kills or</span> <span m='932000'>slows\
  \ down the growth of the cells so that bacterial cells grow</span> <span m='935000'>everywhere\
  \ else, but where a viral particle landed</span> <span m='939000'>there's an absence\
  \ of bacterial cells and that hole in the lawn,</span> <span m='941000'>this whole\
  \ thing is called a lawn of</span> <span m='944000'>bacteria, and the holes in the\
  \ lawn are called plaques.</span> </p><p><span m='947000'>So, when he did this,\
  \ he found that when he did it on</span> <span m='951000'>strain A he got a bunch\
  \ of plaques and when he did it</span> <span m='956000'>on strain B, he didn't,\
  \ no plaques. So what what's the</span> <span m='962000'>simplest explanation for\
  \ this?</span> </p><p><span m='966000'>Strain B is different somehow. It's resistant\
  \ to the virus. I</span> <span m='970000'>don't know, the virus has to come in and\
  \ do</span> <span m='974000'>various things, and strain b isn't compatible with</span>\
  \ <span m='978000'>the virus or something like that. No big deal. So it's a resistant</span>\
  \ <span m='983000'>strain. But, occasionally you'd get a plaque.</span> </p><p><span\
  \ m='988000'>Very occasionally, you'd have an occasional plaque.</span> </p><p><span\
  \ m='993000'>So now, how would this be? I said the strain was</span> <span m='998000'>resistant.\
  \ How could there be an occasional plaque?</span> </p><p><span m='1004000'>Mutation\
  \ in, could it be imitation in the bacteria?</span> </p><p><span m='1010000'>Sorry.\
  \ Well, if it was a mutation in the bacteria</span> <span m='1016000'>there would\
  \ be one bacteria that had the mutation. It was now</span> <span m='1019000'>susceptible,\
  \ and it would die. But, the lawn would</span> <span m='1023000'>kind of grow because\
  \ the cells around it wouldn't have a mutation.</span> </p><p><span m='1026000'>So\
  \ it's probably not a mutation in the bacteria</span> <span m='1030000'>but what\
  \ could be? Maybe a mutation of the virus: what</span> <span m='1033000'>if it was\
  \ a mutation in the virus that was able to overcome</span> <span m='1037000'>the\
  \ resistance? Ah, so that's OK.</span> </p><p><span m='1041000'>So, what this must\
  \ be is the existence of a resistant</span> <span m='1045000'>virus that is a virus\
  \ that can overcome the resistance of</span> <span m='1050000'>the bacteria. So\
  \ far: perfectly normal, no problem. Now, let's</span> <span m='1056000'>do the\
  \ following experiment. Let's take this resistant virus,</span> <span m='1063000'>and\
  \ grow it, again, on strain A and grow it on</span> <span m='1070000'>strain b.\
  \ What do you think is going to happen when I grow it on</span> <span m='1076000'>strain\
  \ A? It'll grow lots of plaques. It still grows</span> <span m='1082000'>on strain\
  \ A, and now what's going to happen when I grow</span> <span m='1088000'>it on strain\
  \ B? If this was really a mutation that</span> <span m='1093000'>made it able to\
  \ grow on strain b then it gets lots of plaques because</span> <span m='1097000'>it's\
  \ now gained the ability to grow on strain B, and sure</span> <span m='1101000'>enough,\
  \ that's what happens. So,</span> <span m='1107000'>there's nothing funky yet. But\
  \ now, suppose I take one of</span> <span m='1115000'>these resistant viruses that\
  \ I isolated here on strain B,</span> <span m='1121000'>I grow it again here on\
  \ strain A. It grows. I grow it on strain B.</span> </p><p><span m='1124000'>It\
  \ grows. If I take it again from strain B and</span> <span m='1127000'>I repeat\
  \ this, it'll still grow on strain A and</span> <span m='1130000'>still grow on\
  \ strain B. Let's take one,</span> <span m='1134000'>though, from strain A. It's\
  \ the resistant one which we have just now</span> <span m='1140000'>happened to\
  \ have grown on strain A. And now,</span> <span m='1148000'>let's grow it again\
  \ on strain A versus on strain B. And sure</span> <span m='1159000'>enough, it continues\
  \ to grow on strain A, no problem.</span> </p><p><span m='1164000'>And we grow it\
  \ now on strain B. And, what shall we</span> <span m='1169000'>get? Well, it should\
  \ grow on strain B, right, because it was a mutant</span> <span m='1172000'>virus,\
  \ and it gained the ability to grow on either.</span> </p><p><span m='1176000'>We\
  \ passage it through B, it grows. We passage it through A.</span> </p><p><span m='1182000'>But\
  \ the answer was nothing, no growth. How can that be?</span> </p><p><span m='1188000'>We\
  \ had a virus. We agreed that was a mutant virus that</span> <span m='1193000'>had\
  \ picked up the ability to grow on strain B, and we demonstrated</span> <span m='1195000'>it\
  \ has now on either A or B.</span> </p><p><span m='1198000'>We then reached in,\
  \ and grabbed a copy of it here from</span> <span m='1202000'>strain A, having grown\
  \ on strain A, and we try it again and it now</span> <span m='1206000'>won't grow\
  \ on strain B. If this was a mutation,</span> <span m='1210000'>I mean, maybe the\
  \ mutation reverted, right?</span> </p><p><span m='1215000'>It was a reversion of\
  \ the mutation. It mutated back. Is that plausible?</span> </p><p><span m='1220000'>No,\
  \ come on. The chance that all of the copies there would mutate</span> <span m='1224000'>back,\
  \ come on. I mean, you could repeat this</span> <span m='1226000'>several times\
  \ and this is always what happens.</span> </p><p><span m='1230000'>What does that\
  \ tell you about this mutation in the virus?</span> </p><p><span m='1234000'>It\
  \ can't be a mutation of the virus because</span> <span m='1239000'>if it was a\
  \ mutation, it would be transmitted through.</span> </p><p><span m='1242000'>But,\
  \ passing through strain A makes it lose</span> <span m='1246000'>its ability to\
  \ grow on strain B. But as long as you keep passing it</span> <span m='1250000'>through\
  \ strain B, it can grow on strain B. This is</span> <span m='1255000'>not your typical\
  \ genetics. So, Salvador Luria loved this.</span> </p><p><span m='1259000'>And,\
  \ he really worked out what was going on. And somehow,</span> <span m='1264000'>well,\
  \ so anyway, they referred to this as strain B</span> <span m='1272000'>having the\
  \ ability to restrict the growth of the</span> <span m='1280000'>virus. Strain B\
  \ can restrict the growth of the virus.</span> </p><p><span m='1285000'>That's where\
  \ this word restriction enzyme</span> <span m='1290000'>comes from. What's really,\
  \ truly going on here underneath the</span> <span m='1294000'>shaggy dog story?\
  \ It took a long time before</span> <span m='1298000'>the shaggy dog story that\
  \ Salvador Luria was the one to really</span> <span m='1302000'>demonstrate is fully\
  \ worked out. But, what turns out to</span> <span m='1307000'>be the case is that\
  \ strain B has a restriction enzyme.</span> </p><p><span m='1314000'>That's how\
  \ it restricts the growth. It has one of</span> <span m='1322000'>these enzymes\
  \ that can cut DNA at a specific place.</span> </p><p><span m='1330000'>When the\
  \ virus comes into strain B, it injects its DNA,</span> <span m='1337000'>and the\
  \ enzyme comes along and cuts the virus's DNA, protecting the</span> <span m='1345000'>bacteria.\
  \ It's got its own little defense mechanism: pretty cool,</span> <span m='1351000'>pretty\
  \ cool. So, any DNA that's introduced,</span> <span m='1358000'>if it has the sequence\
  \ here, it'll take G, A, A, T, T, C, the</span> <span m='1363000'>bacteria cuts\
  \ it. Wait a second,</span> <span m='1369000'>the bacteria has its own DNA. Why\
  \ doesn't it chop up its own</span> <span m='1375000'>chromosome? Well, I mean,\
  \ so one simple possibility would be</span> <span m='1381000'>that if this thing\
  \ is looking for the sequence, G, A, A, T, T, C</span> <span m='1385000'>in the\
  \ genome, maybe it's the case that the</span> <span m='1390000'>bacteria has arranged\
  \ that its own DNA</span> <span m='1395000'>never has a G, A, A, T, T, C. That would\
  \ be a</span> <span m='1399000'>simple solution, right? But is it a plausible</span>\
  \ <span m='1404000'>solution? Why not? But just statistically,</span> <span m='1409000'>how\
  \ often do I expect to encounter a G, A, A, T, T, C? What's</span> <span m='1413000'>the\
  \ frequency of any given six letter word in a four letter</span> <span m='1419000'>alphabet?\
  \ It's about one in 46. So, about one in 46 positions will</span> <span m='1424000'>be\
  \ a G, A, A, T, T, C, and that's about 4,000</span> <span m='1428000'>letters. So,\
  \ every 4, 00 letters, I expect to encounter a</span> <span m='1432000'>G, A, A,\
  \ T, T, C. How big is the E coli genome?</span> </p><p><span m='1436000'>4 million\
  \ letters. So, how many G, A, A,</span> <span m='1439000'>T, T, Cs will there be?\
  \ About 1,000</span> <span m='1442000'>of them. It's just not plausible to imagine\
  \ that it doesn't have the</span> <span m='1446000'>sites. So, your idea is that\
  \ if it has these sites,</span> <span m='1450000'>it's got to arrange to protect\
  \ its own sites. So, how is</span> <span m='1452000'>it going to protect its own\
  \ sites?</span> </p><p><span m='1458000'>Covers it or something. You could imagine\
  \ something covers</span> <span m='1466000'>it or something, but you want to alter\
  \ your own,</span> <span m='1472000'>so it turns out you're exactly right. What\
  \ happens is there is</span> <span m='1475000'>an enzyme that comes along, and at\
  \ this position,</span> <span m='1480000'>attaches a methyl group. It modifies the\
  \ DNA</span> <span m='1487000'>by attaching a methyl group. It turns out that that\
  \ methyl group</span> <span m='1493000'>is enough to prevent the restriction enzyme\
  \ from binding.</span> </p><p><span m='1499000'>So, this blocks the restriction\
  \ enzyme. So, that way the bacteria</span> <span m='1507000'>is able to distinguish\
  \ between its own DNA,</span> <span m='1514000'>which is methylated, and the viral\
  \ DNA. So, wait a second, how does</span> <span m='1521000'>that explain my virus\
  \ that manage to grow? How did my virus manage to</span> <span m='1527000'>grow?\
  \ It would need to have gotten itself modified also to be protected.</span> </p><p><span\
  \ m='1533000'>Could that happen by chance? What if the methylation enzyme,</span>\
  \ <span m='1540000'>the methylase, which is floating around in the cell,</span>\
  \ <span m='1548000'>\x93accidentally\x94 methylated the virus's DNA? What would</span>\
  \ <span m='1556000'>happen then? The virus would become immune.</span> </p><p><span\
  \ m='1562000'>So, suppose the bacteria was pretty clever, and had a</span> <span\
  \ m='1568000'>lot more restriction enzyme, and only a little bit of methylase?</span>\
  \ </p><p><span m='1571000'>Well, you'd imagine that most of the</span> <span m='1574000'>time\
  \ the restriction enzyme would cut up the viral DNA first.</span> </p><p><span m='1577000'>But\
  \ every once in a while, the methylase</span> <span m='1580000'>would get there\
  \ first and protect the virus's DNA.</span> </p><p><span m='1584000'>That becomes\
  \ an immune virus because it can't</span> <span m='1588000'>be cut by the enzyme\
  \ anymore. And, if I take that, and I grow it</span> <span m='1592000'>again on\
  \ strain B, it'll now produce lots of plaques because</span> <span m='1596000'>it\
  \ was methylated. And, if I grow it</span> <span m='1602000'>again on strain B,\
  \ it remains methylated because once</span> <span m='1604000'>it's methylated and\
  \ comes into the cell, it's not</span> <span m='1606000'>cut. And so, its descendants\
  \ will get methylated.</span> </p><p><span m='1610000'>But, what happens if I ever\
  \ grow that methylated virus</span> <span m='1614000'>on strain A? Strain A doesn't\
  \ have the restriction enzyme,</span> <span m='1619000'>and it doesn't have the\
  \ methylase. So, the</span> <span m='1624000'>progeny phage that grew up on strain\
  \ A aren't methylated.</span> </p><p><span m='1628000'>They're no longer protected.\
  \ The protection that the</span> <span m='1633000'>virus has is the protection that\
  \ comes from this methylation enzyme.</span> </p><p><span m='1636000'>It's not the\
  \ sequence of the DNA. It's</span> <span m='1640000'>the attachment to these methyl\
  \ groups. And so,</span> <span m='1643000'>it turns out that if you ever pass this\
  \ virus through strain A,</span> <span m='1646000'>passage through strain A, the\
  \ resulting DNA loses is</span> <span m='1659000'>unmethylated. And now, it can\
  \ be cut.</span> </p><p><span m='1668000'>And it can be cut. Well, this explained\
  \ the weird</span> <span m='1672000'>results of Luria, that somehow bacteria had\
  \ a complex</span> <span m='1676000'>defense mechanism of a restriction enzyme</span>\
  \ <span m='1679000'>and a cognate methylase. The restriction enzyme would cut</span>\
  \ <span m='1682000'>the sequence. The chromosome would be protected by</span> <span\
  \ m='1686000'>methylating that site, and usually it would work fine.</span> </p><p><span\
  \ m='1689000'>Occasionally the bacterial virus would get methylated.</span> </p><p><span\
  \ m='1692000'>It would be protected as long as it continues to go through strains\
  \ that</span> <span m='1695000'>have this restricted methylation system. That was\
  \ it. Now, this</span> <span m='1698000'>shaggy dog story took a couple of decades\
  \ to work out,</span> <span m='1701000'>and eventually led to Nobel prizes for the\
  \ discovery</span> <span m='1705000'>of restriction enzymes. They're extremely important\
  \ because</span> <span m='1708000'>although bacteria do this to protect themselves,\
  \ they have also</span> <span m='1712000'>given us the perfect tool to now cut DNA\
  \ where we want to</span> <span m='1715000'>cut DNA. Now, what if you wanted to\
  \ cut at a G,</span> <span m='1718000'>A, A, T, T, C? You've got Eco R1.</span>\
  \ </p><p><span m='1722000'>But what if you wanted to cut it cut it in another sequence?</span>\
  \ </p><p><span m='1727000'>Well, it turns out that if you want to cut it at G, G,\
  \ A, T, C, C</span> <span m='1737000'>there's an enzyme called Bam H1. If you want\
  \ to cut it at</span> <span m='1749000'>A, A, G, C, T, T or A, A, G, C,</span> <span\
  \ m='1754000'>T, T, there's an enzyme called Hmd 3. If you</span> <span m='1760000'>want\
  \ to cut it at just G, A, T, C like this, C, T, A,</span> <span m='1766000'>G, an\
  \ enzyme called Mbo 1. And, there are enzymes that</span> <span m='1774000'>cut\
  \ it this way, enzymes that cut it this way,</span> <span m='1781000'>enzymes that\
  \ cut it this way, enzymes that recognize four bases,</span> <span m='1785000'>six\
  \ bases. There are even enzymes that recognize eight bases.</span> </p><p><span\
  \ m='1789000'>It turns out that bacteria have elaborated zillions of different</span>\
  \ <span m='1793000'>restriction enzymes that recognize different sequences. This</span>\
  \ <span m='1797000'>perfect for molecular biologists. Bacteria,</span> <span m='1800000'>of\
  \ course, are much smarter than we are, having been out this much</span> <span m='1802000'>longer,\
  \ have developed all of these tools for engineering.</span> </p><p><span m='1806000'>All\
  \ we have to do is borrow them. So how do you get Eco R1?</span> </p><p><span m='1810000'>We\
  \ grow out that strain of E coli; you purify Wco R1.</span> </p><p><span m='1813000'>And\
  \ how do you get Hmd 3? You grow up</span> <span m='1817000'>strain of haemophilus\
  \ influenza. You purify the enzyme. At least,</span> <span m='1820000'>that's how\
  \ primitive molecular biologists did it. If you</span> <span m='1824000'>wanted\
  \ to work with a restriction enzyme, you'd grow up the bacteria.</span> </p><p><span\
  \ m='1827000'>You'd purify the enzyme yourself, and you would just use</span> <span\
  \ m='1830000'>it in your laboratory. Of course today what does a modern</span> <span\
  \ m='1834000'>molecular biologist do if he or she should want Hmd 3?</span> </p><p><span\
  \ m='1839000'>It's in the catalog. So the catalog has 200 restriction</span> <span\
  \ m='1847000'>enzymes. Yup, PSI-1 is new, on sale, 500 units for $400.</span> </p><p><span\
  \ m='1855000'>Let's see what Eco R1 is going for.</span> </p><p><span m='1861000'>Eco\
  \ R1: look at this, 50,000 units $200. That's a good</span> <span m='1864000'>price\
  \ for Eco R1 because it's a very famous</span> <span m='1867000'>enzyme here. So\
  \ all you have to do is you give them your credit card</span> <span m='1870000'>number\
  \ and you have it tomorrow by FedEx. So that's how restriction</span> <span m='1875000'>enzymes\
  \ are obtained today. So, next up, we can cut DNA any</span> <span m='1881000'>place\
  \ we want to. We now need to glue DNA together.</span> </p><p><span m='1886000'>Suppose\
  \ I cut DNA, human DNA, and I'm</span> <span m='1892000'>going to cut it. I'll just\
  \ take human DNA,</span> <span m='1894000'>your DNA, which I've purified, and I'm\
  \ going to cut</span> <span m='1897000'>it at all its Eco R1 sites. I can take any\
  \ other DNA I want.</span> </p><p><span m='1906000'>I don't know, I could take zebra\
  \ DNA. I could take anything and I could</span> <span m='1910000'>also cut it at\
  \ Eco R1 sites. I could mix them together,</span> <span m='1916000'>and after mixing\
  \ them together the fragments will float around and</span> <span m='1922000'>remember\
  \ this down here has T, T,</span> <span m='1928000'>A, A. This fragment over here\
  \ from some other piece</span> <span m='1936000'>T, T, A, A, this could be human\
  \ DNA. This could be zebra DNA if you want</span> <span m='1943000'>to. It doesn't\
  \ matter. It could be bacterial DNA.</span> </p><p><span m='1949000'>These fragments\
  \ overlap. They'll hydrogen bond a little bit,</span> <span m='1953000'>but that\
  \ of course won't introduce a covalent bond here.</span> </p><p><span m='1957000'>I'd\
  \ really like to make a covalent bond. I would like to attach the</span> <span m='1962000'>piece\
  \ of DNA from one source to the piece of DNA from the</span> <span m='1964000'>other\
  \ source by doing the opposite of the</span> <span m='1967000'>restriction enzyme.\
  \ The restriction enzyme cut at these</span> <span m='1969000'>locations. I would\
  \ now like to catalyze</span> <span m='1972000'>the rejoining of the sugar phosphate\
  \ backbone here.</span> </p><p><span m='1975000'>So I would like to rejoin the sugar\
  \ phosphate backbone. I</span> <span m='1979000'>have a hydroxyl here. I have a\
  \ phosphate here,</span> <span m='1983000'>and I would like to ligate them together.\
  \ So</span> <span m='1988000'>how I manage to ligate? What kind of fancy chemistry\
  \ do I</span> <span m='1991000'>do to ligate these pieces of DNA together? I don't\
  \ do any fancy</span> <span m='1995000'>chemistry. I again sit at the feet of bacteria\
  \ who have solved all</span> <span m='2000000'>these problems before. And I ask\
  \ bacteria, how do you do</span> <span m='2002000'>this? And they say, well, we\
  \ have an enzyme called</span> <span m='2004000'>ligase. So, you purify ligase from\
  \ bacteria,</span> <span m='2006000'>you add that, and ligase ligates the fragments\
  \ together. Why</span> <span m='2009000'>do bacteria have an enzyme ligase?</span>\
  \ </p><p><span m='2013000'>For a pair of their own DNA. Things go wrong this is\
  \ part of the</span> <span m='2018000'>DNA maintenance scheme of bacteria. They\
  \ have an enzyme</span> <span m='2022000'>ligase to appear their own breaks in DNA\
  \ and, obligingly, you</span> <span m='2026000'>can purify DNA ligase. So you add\
  \ ligase, today,</span> <span m='2030000'>of course, if you need a ligase,</span>\
  \ <span m='2034000'>how do you get it? It's in the catalog,</span> <span m='2036000'>absolutely.\
  \ So, you can glue together any of those things you</span> <span m='2039000'>want.\
  \ All right, next up, what DNA do I want to stick</span> <span m='2044000'>together?\
  \ I mean, here I made a silly example.</span> </p><p><span m='2050000'>I'm going\
  \ to stick some human DNA to some zebra</span> <span m='2055000'>DNA. Why do that?\
  \ I mean, just to show you that I can</span> <span m='2059000'>doing it, right?\
  \ I'm just demonstrating that I could stick any</span> <span m='2062000'>DNA to\
  \ any DNA. Remember, once I've</span> <span m='2065000'>got a piece of DNA it doesn't\
  \ know whether it came from a human or a</span> <span m='2067000'>zebra. It's just\
  \ the molecule. You can stick the molecules together,</span> <span m='2071000'>right?\
  \ But what do I really want to attach my human DNA to?</span> </p><p><span m='2076000'>I\
  \ want to attach it to attach it to some other DNA that</span> <span m='2085000'>has\
  \ the ability to grow on its own within bacteria. Vectors: I need</span> <span m='2099000'>to\
  \ make, here's what I would really like. I would like to have</span> <span m='2105000'>a\
  \ piece of DNA that has some sequences that contain the</span> <span m='2113000'>recognition\
  \ sites for replication. I'd like to have some replication</span> <span m='2123000'>initiation\
  \ sites here. So, a piece of DNA that,</span> <span m='2131000'>remember, because\
  \ the bacterial chromosome itself,</span> <span m='2136000'>here's my bacteria,\
  \ bacteria'chromosome replicates</span> <span m='2142000'>itself, and it has the\
  \ ability to start DNA replication at multiple</span> <span m='2148000'>sites called\
  \ origins of replication. But, what I</span> <span m='2155000'>would really like\
  \ is to be able to construct in the laboratory a</span> <span m='2159000'>synthetic\
  \ piece of DNA that also would function as an</span> <span m='2167000'>origin of\
  \ replication because then what I could do is in vitro take</span> <span m='2179000'>my\
  \ piece of DNA, attach it to this vector,</span> <span m='2184000'>and it would\
  \ now have the ability to grow</span> <span m='2189000'>the bacteria. How am I going\
  \ to make a piece of DNA?</span> </p><p><span m='2193000'>What kind of engineering\
  \ tricks can we do to create</span> <span m='2197000'>a small piece of DNA that\
  \ has all the machinery needed to</span> <span m='2201000'>be able to be copied\
  \ and replicated just like bacterial</span> <span m='2205000'>chromosomes? That's\
  \ a pretty fancy feat of engineering.</span> </p><p><span m='2210000'>How are you\
  \ going to do that? Sorry? OK,</span> <span m='2215000'>so who are you going to\
  \ ask? If you wanted to do this, you're</span> <span m='2219000'>going to ask the\
  \ experts. Who are the experts? Viruses or</span> <span m='2224000'>bacteria, or\
  \ basically, if you want to do anything,</span> <span m='2227000'>the place to ask\
  \ is the folks who have the most</span> <span m='2230000'>experience. And, the folks\
  \ who have the most</span> <span m='2232000'>experience are almost always prokaryotic\
  \ organisms because they</span> <span m='2235000'>are by far the most evolved things\
  \ on</span> <span m='2238000'>this planet. Anything that can replicate itself and\
  \ grow every 20</span> <span m='2241000'>minutes or something like that has had\
  \ a lot more</span> <span m='2244000'>generations of evolution than you have. And\
  \ therefore,</span> <span m='2246000'>they are much more optimized than we are.\
  \ And so you go ask and say,</span> <span m='2249000'>has any bacteria worked out\
  \ how to do this? Turns out bacteria have</span> <span m='2253000'>worked out how\
  \ to do this just fine. In</span> <span m='2257000'>fact, most bacteria, at least\
  \ many bacteria,</span> <span m='2263000'>contain within them, in addition to their\
  \ own chromosome,</span> <span m='2269000'>small circles of DNA. These are called\
  \ episomes.</span> </p><p><span m='2277000'>This is the chromosome. Epi means on\
  \ top of</span> <span m='2286000'>or in addition to. So in addition to the chromosome,</span>\
  \ <span m='2290000'>there's an episome. The episome is in fact an autonomously replicating</span>\
  \ <span m='2294000'>piece of DNA that has an origin. And it replicates. Why do bacteria</span>\
  \ <span m='2302000'>have episomes? It turns out episomes</span> <span m='2310000'>often\
  \ contain genes. One fo the genes they contain,</span> <span m='2314000'>or some\
  \ of the types of genes they contain, are resistance</span> <span m='2318000'>genes.\
  \ There might be, for example, a penicillin resistance</span> <span m='2324000'>gene\
  \ contained on an episome, or a streptomycin resistance gene.</span> </p><p><span\
  \ m='2330000'>It turns out the bacteria have these</span> <span m='2336000'>episomes\
  \ containing resistance genes, and they're not in the chromosome.</span> </p><p><span\
  \ m='2342000'>They're separate. Now, why would they do that?</span> </p><p><span\
  \ m='2348000'>It turns out when a bacterium dies and a cell cracks open, the</span>\
  \ <span m='2355000'>DNA spills out. The next door neibhored bacteria has</span>\
  \ <span m='2361000'>mechanismis to suck up DNA from the environment. You never</span>\
  \ <span m='2365000'>know. It might find something interesting out there.</span>\
  \ </p><p><span m='2369000'>So, it turns out that bacteria are rather promiscuously\
  \ exchanging</span> <span m='2373000'>pieces of DNA all the time. And so,</span>\
  \ <span m='2377000'>a bacteria that has an episome that has a penicillin resistance\
  \ gene</span> <span m='2383000'>can spread it to other bacteria, and it's very nice.\
  \ It's compact.</span> </p><p><span m='2387000'>It's on its own little episome,\
  \ autonomously replicating</span> <span m='2391000'>piece of DNA. This is great\
  \ for bacteria wanting to spread drug</span> <span m='2396000'>resistance. It's\
  \ not good for human populations,</span> <span m='2400000'>for example, because\
  \ this is how drug resistance spread through</span> <span m='2403000'>populations.\
  \ This is why we have spreads of</span> <span m='2406000'>penicillin resistance.\
  \ Now, of course, wait a second,</span> <span m='2410000'>this whole mechanism of\
  \ spreading drug resistance, we've only had</span> <span m='2415000'>antibiotics\
  \ since the 1940s. How did bacteria devise this so</span> <span m='2420000'>quickly?\
  \ Sorry? Many generations since 1945?</span> </p><p><span m='2425000'>That would\
  \ be very impressive.</span> </p><p><span m='2430000'>Yeah, but, I mean, why do\
  \ they have this episome mechanism, the</span> <span m='2434000'>ability to spread\
  \ DNA and all that? That's an awful lot</span> <span m='2439000'>to evolve in 50\
  \ years? Yeah? Something natural like</span> <span m='2445000'>penicillin. It turns\
  \ out, we didn't</span> <span m='2451000'>think of penicillin. Who thought of penicillin?</span>\
  \ </p><p><span m='2454000'>Fungi. Right, again, we learn from the lower organisms.\
  \ Penicillin</span> <span m='2458000'>comes from fungi. Bacteria have been fighting</span>\
  \ <span m='2461000'>off penicillin for millions and tens of millions of years. So,</span>\
  \ <span m='2464000'>we may be very proud of our penicillin and all that.</span>\
  \ </p><p><span m='2467000'>But, they've been at this for a very long time.</span>\
  \ </p><p><span m='2469000'>This is about war between bacteria and fungi.</span>\
  \ </p><p><span m='2471000'>That's what this is, OK? So, that's why these things\
  \ are</span> <span m='2474000'>here. They're here so that bacteria can have these</span>\
  \ <span m='2477000'>resistance genes against fungi and things like that that make</span>\
  \ <span m='2480000'>antibiotics. Antibiotics are natural. We've made a few</span>\
  \ <span m='2484000'>new ones, but most of the antibiotics have been made by nature.</span>\
  \ </p><p><span m='2487000'>And so, if I wanted to replicate DNA,</span> <span m='2490000'>if\
  \ I wanted to attach my human DNA to a piece of DNA that's</span> <span m='2495000'>capable\
  \ of autonomous replication, autonomously replicating circles of</span> <span m='2501000'>DNA,\
  \ these autonomously replicating circles of DNA are also called</span> <span m='2507000'>plasmids.\
  \ And that's the word we'll mostly use for them,</span> <span m='2510000'>plasmids.\
  \ All I need to do is purify a plasmid from</span> <span m='2514000'>a bacteria.\
  \ So, I find a bacteria that has plasmids.</span> </p><p><span m='2517000'>I purify\
  \ the plasmid, and then I can cut open the plasmid</span> <span m='2521000'>at the\
  \ Eco R1 site, OK? So, this plasmid will have an</span> <span m='2527000'>ORI, an\
  \ origin of replication. I'll cut</span> <span m='2533000'>it open at the Eco R1\
  \ sight. I'll take human DNA fragments that</span> <span m='2539000'>I've cut with\
  \ Eco R1. I'll mix them with plasmid DNA that has</span> <span m='2544000'>been\
  \ opened up, has an origin. Ligase will come</span> <span m='2550000'>along, join\
  \ this up, and now I have a circle of DNA that</span> <span m='2554000'>has all\
  \ the machinery to autonomously replicate,</span> <span m='2558000'>plus my human\
  \ DNA. Now, if I wanted to get a vector, or an</span> <span m='2561000'>honest to\
  \ goodness plasmid, I can go to a bacteria,</span> <span m='2565000'>grow it up,\
  \ purify the plasmid, and cut it. Or alternatively,</span> <span m='2570000'>if\
  \ I needed the plasmid, say, tomorrow,</span> <span m='2575000'>it's in the catalog.\
  \ The next section of the catalog has</span> <span m='2577000'>a long list of plasmids\
  \ here. There's</span> <span m='2579000'>a plasmid there, right? It's a nice plasmid.</span>\
  \ </p><p><span m='2584000'>Oh yes, let's see, puck is a very good plasmid.</span>\
  \ </p><p><span m='2589000'>PBR 322 is a good plasmid. The whole section, all this\
  \ purple</span> <span m='2592000'>stuff are the plasmids. So, you can get the plasmids\
  \ too.</span> </p><p><span m='2595000'>You place one order, you get the restriction\
  \ enzymes,</span> <span m='2597000'>you get the ligases, you get the plasmids, no</span>\
  \ <span m='2600000'>problem. So, I can then take total human DNA, cut up, cut</span>\
  \ <span m='2606000'>up, cut up, cut up, add in plasmid,</span> <span m='2612000'>and\
  \ I'm going to ligate together. And then, having ligated my human</span> <span m='2618000'>DNA\
  \ to my plasmids, I'm going to mix with</span> <span m='2625000'>bacteria. I take\
  \ some bacterial cells. I add my mixture of these</span> <span m='2635000'>plasmids\
  \ containing human DNA. And now all I have to do is</span> <span m='2638000'>persuade\
  \ the bacteria to suck up my plasmids containin human DNA.</span> </p><p><span m='2642000'>How\
  \ do I teach bacteria to suck up DNA? They do that for</span> <span m='2648000'>a\
  \ living. That's what they do. They're always spreading material.</span> </p><p><span\
  \ m='2651000'>They have that ability. All we're doing is we're using</span> <span\
  \ m='2654000'>their ability. So you get the sense that the kind</span> <span m='2656000'>of\
  \ engineering that really works in biology is engineering that</span> <span m='2659000'>exploits\
  \ what nature has been doing for a very long time.</span> </p><p><span m='2662000'>Rather\
  \ than butting your head up against the problem, usually</span> <span m='2664000'>somebody\
  \ has solved it, and it's almost always</span> <span m='2667000'>bacteria. So, you've\
  \ transformed the bacteria.</span> </p><p><span m='2669000'>Now, there are a few\
  \ tricks you can</span> <span m='2672000'>use to make them a little more transformable.</span>\
  \ </p><p><span m='2675000'>You can add calcium phosphate, and blah, blah, blah,</span>\
  \ <span m='2679000'>but you can sort of persuade them to take up the DNA. And then</span>\
  \ <span m='2683000'>all you have to do is plate them out on a plate.</span> </p><p><span\
  \ m='2688000'>Plate them out fairly dilutely so there are a lot of single bacterial</span>\
  \ <span m='2695000'>cells that land on the plate, and wait for them to grow up.\
  \ Each</span> <span m='2703000'>one of these had a single plasmid, a different plasmid\
  \ than</span> <span m='2710000'>the next guy over. Wait a second, each one?</span>\
  \ </p><p><span m='2716000'>How do I guarantee that every bacteria in my test</span>\
  \ <span m='2722000'>tube took up a plasma? Is that plausible? I mean,</span> <span\
  \ m='2728000'>I can't guarantee that every bacteria is going to take up a</span>\
  \ <span m='2734000'>plasmid. Maybe I'll add so much plasmid that every bacteria\
  \ will</span> <span m='2738000'>take one up. Oh, but that's a bad idea</span> <span\
  \ m='2743000'>because why? Because then a lot of them will take up more than one.</span>\
  \ </p><p><span m='2745000'>You don't want to do that. You really only</span> <span\
  \ m='2747000'>want to have at most one. So, if you were going to arrange so</span>\
  \ <span m='2750000'>that at random you only have about one, you've got to</span>\
  \ <span m='2753000'>have a lot that are zero. So, this is a problem. I mean,</span>\
  \ <span m='2756000'>it's a real waste. My library is going to have large numbers\
  \ of</span> <span m='2759000'>bacteria that don't have any plasmid. In fact, this\
  \ transformation</span> <span m='2762000'>process is not so efficient. It's not\
  \ so efficient. So, we</span> <span m='2766000'>have a little bit of a problem here\
  \ is</span> <span m='2769000'>that some of these guys will have human DNA.</span>\
  \ </p><p><span m='2773000'>But, most of them won't. So, what can I do to arrange\
  \ that</span> <span m='2779000'>any bacteria that did not pick up a plasmid</span>\
  \ <span m='2786000'>was incapable of growing? Add a resistance gene to the</span>\
  \ <span m='2794000'>plasmid. Suppose I were so clever as</span> <span m='2802000'>to\
  \ add to that plasmid, penicillin resistance. So,</span> <span m='2806000'>not just\
  \ an origin of replication, but suppose I also had a</span> <span m='2811000'>resistance\
  \ gene here, say, for penicillin resistance or</span> <span m='2817000'>streptomycin\
  \ resistance, or ampicillin tends to be a very big</span> <span m='2823000'>favorite,\
  \ ampicillin resistance. Then,</span> <span m='2828000'>my plasmid would have ampicillin\
  \ resistance gene encoded on it,</span> <span m='2832000'>an enzyme that can, say,\
  \ break down ampicillin,</span> <span m='2836000'>so on and one way to to my perch\
  \ you plate I just said ampa cell</span> <span m='2842000'>and now i'll even though\
  \ most of the bacteria have not</span> <span m='2846000'>picked up a plasmid, only\
  \ those bacteria that have picked</span> <span m='2849000'>up a plasmid have the\
  \ ampicillin</span> <span m='2852000'>resistance gene and can grow on an ampicillin\
  \ containing plate.</span> </p><p><span m='2856000'>Now, how do I get a plasmid\
  \ with an ampicillin</span> <span m='2859000'>resistance gene? It's in the catalog.\
  \ It's</span> <span m='2863000'>all there, right? In fact, these occur naturally.</span>\
  \ </p><p><span m='2866000'>You can, with restriction enzymes, move the ampicillin\
  \ resistance</span> <span m='2869000'>gene to your favorite plasmid. If you don't\
  \ like that, you can put</span> <span m='2871000'>in kanamycin resistance, etc.,\
  \ etc.,</span> <span m='2873000'>etc. So, that's how you do it. So, we've got the\
  \ big picture here.</span> </p><p><span m='2879000'>We have now gotten a library,\
  \ the Library of Human</span> <span m='2886000'>Fragments contained in E coli. The\
  \ library is a big Petri plate or</span> <span m='2899000'>many Petri plates, each\
  \ one of which is a colony.</span> </p><p><span m='2908000'>Each colony has a single\
  \ vector with an origin,</span> <span m='2914000'>a resistance marker, and a distinct\
  \ piece of</span> <span m='2920000'>human DNA. In this library lives somewhere the\
  \ gene for Huntington's</span> <span m='2927000'>disease. Over here is a gene for\
  \ cystic fibrosis,</span> <span m='2933000'>over here a gene for Duchenne muscular\
  \ dystrophy,</span> <span m='2937000'>over here a gene for diastrophic dysplasia,</span>\
  \ <span m='2942000'>over here a gene for etc. etc. The only detail, now,</span>\
  \ <span m='2948000'>you've got a library. You've managed to purify each</span> <span\
  \ m='2955000'>piece of human DNA away from every other piece of human DNA.</span>\
  \ </p><p><span m='2959000'>The only question now is how do you use the library?</span>\
  \ </p><p><span m='2961000'>How do you go to the library and withdraw the correct</span>\
  \ <span m='2964000'>volume from the shelf? How do you find the one</span> <span\
  \ m='2968000'>you're looking for? So, we have converted the problem</span> <span\
  \ m='2972000'>of purification, which in every other form of biochemistry starts\
  \ by</span> <span m='2975000'>saying, I'm going to purify something based on its\
  \ distinctive</span> <span m='2978000'>properties, to I'm going to randomly purify\
  \ everything.</span> </p><p><span m='2982000'>Everything would be purified in its\
  \ own bacteria, and I've now converted</span> <span m='2986000'>to the problem of\
  \ finding the one that I want in my</span> <span m='2991000'>library. Next time,\
  \ we'll talk about how you go to the</span> <span m='2995000'>library and find what\
  \ you want.</span> </p><p><span m='2999000'>See you then.</span> </p>"
type: course
uid: 04338efad747f6be4bd2a10a6662563b

---
None