---
about_this_resource_text: <p><strong>Topics covered:</strong> Recombinant DNA 4</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec18.jpg
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-18-recombinant-dna-4/lec18.jpg
  title: lec18.jpg
  type: null
  uid: 3ab73a352fa1f09ff28731b874d71025
- id: Video-YouTube-Stream
  media_location: T5d5PvPjUlU
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Video-YouTube-Stream
  type: Video
  uid: 2bb4ef2a6e7fbdc7e35b5ac12060a512
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/T5d5PvPjUlU/default.jpg
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: c647d2901eb7e8daa1a2a0a068db1693
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Video-iTunes U-MP4
  type: Video
  uid: 3605061083c71481e42be5badbba748d
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec18-22oct2004-220k.mp4
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Video-Internet Archive-MP4
  type: Video
  uid: 2dae00d288e9d9e3d3a6a290fa821906
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Audio-iTunes U-MP3
  type: Audio
  uid: cf9df156b252e8a2a6e1ffe74f158ae1
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-22oct2004-1000.mp3
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Video-Internet Archive-MP3
  type: Video
  uid: ea994c3106f42a969c02f7bda0dad1c1
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 7f3020bfc9292268153bf30a35ceb4ea
- id: Thumbnail-OCW-JPG
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: a236a7e1fce6e04550ddea27bf187439
- id: 3Play-3PlayYouTubeid-MP4
  media_location: T5d5PvPjUlU
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: db5e835989d0e14d0ea64b6efbf51a8e
- id: T5d5PvPjUlU.srt
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-18-recombinant-dna-4/T5d5PvPjUlU.srt
  title: 3play caption file
  type: null
  uid: d23b7f77680d2f326104d1604215a563
- id: T5d5PvPjUlU.pdf
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-18-recombinant-dna-4/T5d5PvPjUlU.pdf
  title: 3play pdf file
  type: null
  uid: 8da39419e76d9a2365e4eecfbf4fa4ab
- id: Caption-3Play YouTube id-SRT
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: 36c2cdcc8acbb192329e39d32e805fbb
- id: Transcript-3Play YouTube id-PDF
  parent_uid: eea5db9b7a8ab27b00c9fa42e1e289aa
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: d415bcdad1ad18550c23410a61a938f0
inline_embed_id: 36068689lecture18:recombinantdna410878661
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-18-recombinant-dna-4
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-18-recombinant-dna-4
template_type: Tabbed
title: 'Lecture 18: Recombinant DNA 4'
transcript: <p><span m='3000'>Good morning.</span> </p><p><span m='17000'>It can't
  go without at least some acknowledgement dimension.</span> </p><p><span m='23000'>If
  you should ever find yourself in</span> <span m='29000'>life in a situation where
  you have or are about to give up all hope,</span> <span m='35000'>you think things
  are utterly impossible and there's no way,</span> <span m='42000'>you will remember
  this week that nothing is impossible.</span> </p><p><span m='49000'>It is possible
  to come back three games down in the bottom of the</span> <span m='53000'>ninth
  inning, you've got to believe you can do it,</span> <span m='58000'>and remember
  to have Dave Roberts pinch run.</span> </p><p><span m='63000'>Just a general bit
  of good advice. What an amazing week, just</span> <span m='66000'>absolutely amazing
  week. Wow. There are lessons in life to</span> <span m='69000'>be taken from it.
  Please do take them.</span> </p><p><span m='72000'>You know, there really are. I
  mean I'd given up hope by that</span> <span m='75000'>point, I confess. I wish I
  could say oh,</span> <span m='78000'>I knew they were going to pull it out, but
  I didn't.</span> </p><p><span m='82000'>And, boy, they pulled it out one game at
  a time.</span> </p><p><span m='85000'>So, all of you think good thoughts this week.
  This could be a historic</span> <span m='88000'>week, you know, you were here. Anyway,
  onward.</span> </p><p><span m='93000'>We were talking last time about how</span>
  <span m='105000'>to analyze your clone. The notion of cloning random pieces</span>
  <span m='109000'>of DNA, identifying your clone within a library,</span> <span m='114000'>purifying
  the DNA from a clone, doing some preliminary analysis by</span> <span m='119000'>maybe
  cutting with a restriction enzyme, then sequencing it using</span> <span m='123000'>these
  techniques that I'd described would allow you to take the clone</span> <span m='128000'>that
  was, say, able to rescue the yeast that</span> <span m='133000'>couldn't grow without
  arginine and figure out what its</span> <span m='137000'>DNA sequence was. You could
  take the clone that you</span> <span m='142000'>had obtained by hybridizing with
  the DNA sequence corresponding to the</span> <span m='146000'>protein sequence for
  beta-globin and sequence it and see the beta-globin</span> <span m='151000'>gene
  sequence perhaps. This is very powerful.</span> </p><p><span m='156000'>I want to
  take a brief moment, we'll come back to it in more detail</span> <span m='160000'>in
  a subsequent lecture, but I really described how you would</span> <span m='164000'>sequence
  one clone. I just want to make a note,</span> <span m='169000'>because someone asked
  about it last time, about how you would sequence</span> <span m='173000'>an entire
  genome. Someone asked about this.</span> </p><p><span m='178000'>Remember before
  we pulled out our clone, we sequenced it,</span> <span m='184000'>we got its DNA
  sequence. What if I wanted to sequence the</span> <span m='190000'>entirety of a
  genome? Yeah. Do a lot of this,</span> <span m='197000'>right, basically if I got
  a whole genome. Well,</span> <span m='203000'>somebody asked could I put a primer
  here and just sequence?</span> </p><p><span m='210000'>It would take a very long
  time. And it turns out that it wouldn't</span> <span m='214000'>work because the
  separation that you can achieve through gels is a</span> <span m='218000'>function,
  the separation between N and N plus 1 in length goes like the</span> <span m='223000'>logarithm
  of the ratio. So, it turns out that when N and N</span> <span m='227000'>plus 1
  get to like about a thousand, you can achieve very little physical</span> <span
  m='232000'>separation between them. And so, DNA sequencing runs cannot</span> <span
  m='236000'>go much past the thousand bases. So, the problem with sequencing a</span>
  <span m='241000'>genome by putting down a primer on an extraordinarily long piece
  of DNA,</span> <span m='245000'>a hundred million bases, is you cannot separate
  the little</span> <span m='249000'>fragments like that. So, what you do is you break
  up</span> <span m='254000'>your genome into lots of pieces. One strategy, break
  it up into a</span> <span m='258000'>library of some very big pieces. It turns out
  you can make pieces at</span> <span m='262000'>random of a hundred thousand base
  pairs.</span> </p><p><span m='267000'>Cloning these in bacterial artificial chromosomes,</span>
  <span m='271000'>as we talked about before. Take a library of bacterial</span> <span
  m='276000'>artificial chromosomes and then begin sequencing them.</span> </p><p><span
  m='280000'>And take any given bacterial artificial chromosome and break it</span>
  <span m='285000'>up into a whole lot of pieces that are maybe a thousand bases long,</span>
  <span m='289000'>and you could sequence all of those. How do you arrange to get
  just a</span> <span m='294000'>perfect overlapping set of thousand based pair clones
  that perfectly</span> <span m='298000'>tile across the sequence with no redundancy?</span>
  </p><p><span m='303000'>You don't. That's the correct answer. That's how you do
  it,</span> <span m='306000'>you don't. Instead you just randomly take a bunch of
  things.</span> </p><p><span m='310000'>And, in fact, typically you might take clones
  that give you six or</span> <span m='313000'>eight-fold redundancy. You just sequence
  a lot of clones</span> <span m='317000'>and then you ask the computer to reassemble
  it.</span> </p><p><span m='320000'>And, in fact, all that overlap is very good for
  being able to stick</span> <span m='324000'>these pieces together. Sometimes people
  do such things as</span> <span m='327000'>take pieces that might be four thousand
  bases long and sequence a</span> <span m='331000'>thousand bases here and a thousand
  bases here by using a primer that</span> <span m='335000'>starts there and a primer
  that starts there. And then you can get</span> <span m='339000'>DNA sequences from
  two ends of a clone. And if you had that for</span> <span m='342000'>zillions of
  clones your computer program might do an even better job</span> <span m='346000'>of
  linking things up. It's one very big crossword puzzle</span> <span m='350000'>of
  putting together all of these pieces, a jigsaw puzzle of putting</span> <span m='354000'>together
  all these pieces. But, in effect, this is how you</span> <span m='358000'>sequence
  a big piece of DNA. You chop it up into medium-sized</span> <span m='361000'>pieces
  of DNA and then tinny pieces of DNA, you sequence them,</span> <span m='365000'>and
  you use computational science to reassemble it.</span> </p><p><span m='369000'>Some
  people, for some genomes, take the whole big genome and</span> <span m='373000'>immediately
  go to lots of little pieces. That can work,</span> <span m='376000'>too. I depends
  on exactly how complicated your genome is.</span> </p><p><span m='380000'>In the
  human genome, there are some parts of a human genome that are</span> <span m='384000'>almost
  identical that might be like 99.91% identical in two different</span> <span m='388000'>parts
  of the genome. And so, if you do that,</span> <span m='392000'>you may have trouble
  telling those pieces apart. So,</span> <span m='396000'>for really complicated genomes
  people like sometimes breaking it up</span> <span m='400000'>into intermediate-sized
  pieces. But basically the idea of</span> <span m='405000'>sequencing a big piece
  of DNA by this process is referred to as</span> <span m='409000'>shotgun sequencing.
  Shotgun sequencing,</span> <span m='413000'>in fact, was developed in about 1980
  by Fred Sanger,</span> <span m='418000'>the same guy who developed the DNA sequencing
  technique that I told you</span> <span m='422000'>about using polymerase and dideoxynucleotides.</span>
  </p><p><span m='427000'>Sanger very quickly wanted to go from sequencing a single
  piece to</span> <span m='429000'>sequencing pieces, and so he developed the shotgun</span>
  <span m='432000'>technique there. And it's now been applied in many</span> <span
  m='435000'>different forms of intermediate shotguns, whole genome shotguns,</span>
  <span m='438000'>et cetera. So, that's in reply to the question someone asked last
  time</span> <span m='441000'>about, well, how would you do a whole genome? And,</span>
  <span m='444000'>as a matter of fact, this is not theoretical because,</span> <span
  m='447000'>in fact, people do hold genomes this way. And we do this at MIT.</span>
  </p><p><span m='450000'>Lots of genomes get done here in this fashion. Someone else
  asked</span> <span m='455000'>how would you analyze your clone. And, again, I'll
  just make a brief</span> <span m='460000'>remark on that in response to the question.
  So,</span> <span m='466000'>analyzing some DNA sequence. So, suppose we got some
  DNA</span> <span m='471000'>sequence, A-A-T-A, don't bother writing this down.</span>
  </p><p><span m='477000'>I'm just making up letters here. How would we make any sense
  of it?</span> </p><p><span m='483000'>Suppose I give you the ones and zeros from
  your hard-drive,</span> <span m='489000'>how would you make any sense out of them?
  This is about as interesting</span> <span m='495000'>as the ones as yours from your
  hard-drive, right?</span> </p><p><span m='501000'>It's got four letters, not two,
  but this is actually what</span> <span m='507000'>you get out of any project. You
  want to sequence beta-globin?</span> </p><p><span m='511000'>You'll get something
  like this. You want to sequence the arginine</span> <span m='515000'>gene? You want
  to sequence the human genome? You get a very long</span> <span m='519000'>string
  of four letters. What do you do with it?</span> </p><p><span m='523000'>Oh, well,
  you could compare it to a normal copy of the gene.</span> </p><p><span m='526000'>And
  if I did that I might find a bunch of differences.</span> </p><p><span m='530000'>But
  how would I even know where the beta-globin gene was within this</span> <span m='534000'>sequence?
  This clone contains beta-globin. How would I even find</span> <span m='538000'>the
  exons? Yes? Or whatever? Look at codons.</span> </p><p><span m='542000'>So, let's
  start looking at the codons. This codon here?</span> </p><p><span m='548000'>Well,
  or this, maybe it's this codon here. Sorry?</span> </p><p><span m='553000'>Find
  it. Do you see any start codons here? Oh,</span> <span m='559000'>there's an ATG
  there. So, maybe that's the start codon or</span> <span m='564000'>maybe not. How
  often do we expect to find an ATG in some</span> <span m='570000'>reading frame?
  You know, it could happen fairly</span> <span m='575000'>easily. Also, how do we
  know it's going this way?</span> </p><p><span m='579000'>Maybe we should look for
  an ATG, we'll put it there,</span> <span m='584000'>going this way. Sorry? I drew
  the arrow there?</span> </p><p><span m='588000'>Well, that's because it's where
  the sequence started out on my page.</span> </p><p><span m='593000'>It doesn't tell
  me my gene runs that way. Yes? From five</span> <span m='597000'>prime to three
  prime. Ah, but it's a double-stranded piece</span> <span m='602000'>of DNA. You
  see, if it's five prime to three prime on</span> <span m='606000'>this strand, the
  genome has a, another strand that reads the other</span> <span m='611000'>way. What
  did I get? C-A-T-A, right, C-C-T,</span> <span m='616000'>et cetera. And the gene
  could be encoded on this strand.</span> </p><p><span m='620000'>This could be the
  coding strand, that could be the coding strand, and</span> <span m='625000'>looking
  for a mere ATG in one of three possible reading frames on one</span> <span m='630000'>of
  two possible strands I'll find all sorts of stuff.</span> </p><p><span m='635000'>So,
  sorry? Guess. Guess. Guess is good.</span> </p><p><span m='638000'>They don't, don't,
  won't you, you remember we talked about getting</span> <span m='642000'>papers accepted.
  If you were to write up the paper</span> <span m='646000'>that way the reviewing
  would probably ding it and say,</span> <span m='650000'>you know, the guess isn't,
  isn't good enough. So, that's</span> <span m='654000'>actually very interesting.
  How do you actually find the gene</span> <span m='658000'>sequence? Well, it turns
  out to be a</span> <span m='662000'>non-trivial problem which often gets glossed
  over in the textbooks.</span> </p><p><span m='666000'>What you might do is if something
  really were exonic,</span> <span m='671000'>if this were any exon, does it have
  any properties that you</span> <span m='676000'>can think of? It shouldn't have
  a stop codon. No stop codon.</span> </p><p><span m='680000'>How often does a stop
  codon occur at random in a given reading frame?</span> </p><p><span m='685000'>How
  many stop codons are there? Three out of 64 possible codons.</span> </p><p><span
  m='690000'>There's about one in 20 codons in any given reading frame is a stop</span>
  <span m='693000'>codon. So, that means if I read for about 20 codons,</span> <span
  m='697000'>and I don't encounter a stop, it's beginning to get more likely</span>
  <span m='701000'>that that's not random. If I read for, say, 60 codons,</span> <span
  m='705000'>180 bases and I've encountered no stop codon in that reading frame,</span>
  <span m='709000'>that chances of that occurring is about either the minus three
  or so,</span> <span m='713000'>right? Because if I went through three characteristic
  lengths,</span> <span m='717000'>either the minus three, you know, and I don't know,
  about 5% or</span> <span m='721000'>something like that. If I went for thousands
  of bases</span> <span m='725000'>without any stop codon, would you be impressed?
  That's</span> <span m='729000'>pretty impressive. So, all I have to do is find the</span>
  <span m='734000'>few thousands of basis with no stop codon. The problem with that
  is</span> <span m='738000'>that in bacteria there are some genes that are a thousand
  bases long</span> <span m='743000'>and you, there, you can read them and they have
  no</span> <span m='747000'>stop codon. What's the problem with the human</span>
  <span m='753000'>genome? Introns. It turns out that because the</span> <span m='759000'>coding
  sequences are broken up into small exons, if I found a thousand</span> <span m='766000'>bases
  with no stop codons then it's very likely coding sequence.</span> </p><p><span m='773000'>But
  a typical human exon is on the order of a 150 to 200 bases.</span> </p><p><span
  m='780000'>Very inconvenient because, you know, it's a typical exon</span> <span
  m='784000'>encodes 50, 60, 70 codons. So, it turns out that</span> <span m='788000'>even
  that is not so easy to do. Well, the answer is it's not a</span> <span m='792000'>trivial
  problem. People do all sorts of things to</span> <span m='797000'>figure out how
  to decode sequences of genomes. You do run computer</span> <span m='801000'>filters
  across there that say, look, there are a bunch of</span> <span m='805000'>consecutive
  codons without stop codons.</span> </p><p><span m='810000'>There tend to be little
  preferences, like amongst the synonymous choices</span> <span m='813000'>of stop
  codons, humans tend to prefer one stop codon,</span> <span m='816000'>one codon
  for a specific amino acid over others. So,</span> <span m='819000'>there are some
  biases as to which codons get used.</span> </p><p><span m='823000'>And the computer
  can kind of take a little bit of account of that.</span> </p><p><span m='826000'>Then
  you can also have made a library of seed DNAs and sequence</span> <span m='829000'>seed
  DNA, the mRNA which will help you a lot and look for where they</span> <span m='833000'>match
  up. Then you can take sequences from the human</span> <span m='836000'>and the mouse.
  And it turns out that the sequences</span> <span m='839000'>in the mouse and the
  sequences in the human, if you line them up,</span> <span m='843000'>the exons tend
  to match up better than the introns because evolution</span> <span m='846000'>cares
  a lot about the exons. But it turns out this is not a</span> <span m='849000'>trivial
  problem. And even today, if I give you a</span> <span m='853000'>random stretch
  of human DNA, it's not, there is no simple</span> <span m='856000'>computer program
  that it's on, that on its own, not even a</span> <span m='860000'>complicated computer
  program, but on its own would be able,</span> <span m='863000'>without axillary
  data, to accurately pick out all the genes.</span> </p><p><span m='867000'>Even
  for simple bacteria, we cannot nail perfectly all the</span> <span m='870000'>genes.
  Although, the lack of these introns means that</span> <span m='874000'>the exons
  tend to be pretty big, it means the coding are pretty big</span> <span m='878000'>and
  we can kind of do it. So, I just wanted to point out that,</span> <span m='882000'>that
  there's a lot still to be done there. The cell manages,</span> <span m='886000'>thank
  you, to read this just fine, but we're not as smart yet as the</span> <span m='890000'>cell,
  and so we're not totally able to read out all this stuff.</span> </p><p><span m='894000'>We'll
  come back to genomics in a, in a further lecture. Yes? Yeah,</span> <span m='898000'>wouldn't
  -- What a cool idea.</span> </p><p><span m='901000'>Yes. There are actually some
  experiments, which maybe if you</span> <span m='904000'>remind me we can, I can
  work it into a subsequent</span> <span m='908000'>lecture, but people have some
  experiments where they can randomly</span> <span m='911000'>mutagenize zillions
  of bacteria and determine which ones will grow and</span> <span m='914000'>which
  ones won't. And they can do it all in parallel</span> <span m='918000'>in a single
  test-tube. And thereby you can tell which,</span> <span m='921000'>which nucleotides
  in the genome matter and which don't.</span> </p><p><span m='924000'>It's a kind
  of cool procedure. OK. Anyway, I just wanted to sort</span> <span m='928000'>of
  tie up that bit here. Now, let's move on to re-sequencing</span> <span m='931000'>a
  gene. So, let's suppose we've managed to</span> <span m='935000'>sequence, I don't
  know, the human genome, the entirety of</span> <span m='939000'>the human genome
  we have before us, OK? Actually, next week in the</span> <span m='943000'>journal
  Nature will appear a paper reporting, in fact,</span> <span m='947000'>today, yester,
  no, yesterday, in fact, it was yesterday, yesterday</span> <span m='951000'>appeared
  in the journal Nature a paper reporting the finished</span> <span m='955000'>sequence
  of the human genome.</span> </p><p><span m='959000'>http://www.nature. om/nature/journal/v431/n7011/pdf/nature03001.pdf</span>
  <span m='963000'>And so, anybody who wants to go</span> <span m='967000'>online,
  in fact, we can get, we'll get copies for the</span> <span m='971000'>class. Why
  don't we get you copies of the paper? It's not as long as</span> <span m='974000'>the
  last one. But, in fact, I didn't realize it was</span> <span m='977000'>yesterday.
  I thought it was next week. Yesterday came out the final</span> <span m='981000'>report
  on the finished sequence of the human genome,</span> <span m='984000'>which a number
  of us have been laboring on for quite a long time.</span> </p><p><span m='988000'>And
  it just appeared. So, we actually have that now.</span> </p><p><span m='991000'>It
  actually, it's been on the Web for a while, but the paper</span> <span m='995000'>describing
  it took a while to write up and it came out yesterday.</span> </p><p><span m='999000'>So,
  we'll get you a copy of that. But now you've got that whole</span> <span m='1003000'>sequence
  of the human genome here. I've been, you know, I've been</span> <span m='1006000'>working
  on this paper with people for so long that,</span> <span m='1010000'>you know, I
  hadn't actually paid attention to the fact that it just</span> <span m='1014000'>came
  out. You don't want to know how long it took to write this.</span> </p><p><span
  m='1018000'>The paper, actually, is unusual. It's the only paper that Nature has</span>
  <span m='1021000'>ever published where the author list is sufficiently long that
  we don't</span> <span m='1025000'>have it in the Journal. There's a website that
  contains the</span> <span m='1029000'>author list. There are, I believe, I don't
  have the final</span> <span m='1033000'>count, but something in the neighborhood
  of about 200 authors to</span> <span m='1036000'>the paper. We decided that everybody
  who'd worked on it should</span> <span m='1040000'>be a co-author of the paper,
  and we just put it all on a website.</span> </p><p><span m='1044000'>So, anyway,
  I digress. So, suppose we have the beta-globin gene here.</span> </p><p><span m='1048000'>So,
  I've got that in -- I've got the normal form of the</span> <span m='1051000'>beta-globin
  gene, or I've got one person's form,</span> <span m='1054000'>in any case, in the
  human genome sequence. Now I want to take a</span> <span m='1057000'>patient with
  sickle cell anemia and I want to re-sequence their gene.</span> </p><p><span m='1061000'>Now,
  remember what we said, we would, we would make a library from</span> <span m='1064000'>that
  person, right? So, we'd get that person's blood,</span> <span m='1067000'>we'd purify
  DNA, we'd cut it, we'd clone it, we'd probe the library</span> <span m='1070000'>with
  a radioactive probe for the beta-globin gene,</span> <span m='1073000'>we'd pull
  out the gene and we'd re-sequence it.</span> </p><p><span m='1076000'>Suppose we
  wanted to do that to a hundred patients.</span> </p><p><span m='1080000'>For every
  patient we'd get blood, we'd make DNA, we'd clone in a</span> <span m='1083000'>plasma,
  we'd made a whole plasma library, plate it out on filter,</span> <span m='1087000'>probe
  it with a radioactive probe, pull out the clone and sequence it.</span> </p><p><span
  m='1090000'>Now, for any such library you probably need to look through a</span>
  <span m='1094000'>couple hundred thousand clones to find beta-globin.</span> </p><p><span
  m='1098000'>So, for your DNA and your DNA and your DNA and your DNA,</span> <span
  m='1101000'>we're going to make libraries of a hundred thousand clones,</span> <span
  m='1105000'>that's a couple, that's a lot of plates, right?</span> </p><p><span
  m='1109000'>We're going to put them all on, on nylon filters in these</span> <span
  m='1112000'>Seal-a-Meal bags with these radioactive probes,</span> <span m='1115000'>and
  we're going to look for your beta-globin clone,</span> <span m='1118000'>your beta-globin
  clone, your beta-globin clone, your</span> <span m='1122000'>beta-globin clone,
  et cetera, et cetera,</span> <span m='1125000'>et cetera. This is really boring.
  Do you realize how off putting it</span> <span m='1128000'>would be to study sickle
  cell anemia if we had to do that for each</span> <span m='1131000'>successive patient,
  make a whole library?</span> </p><p><span m='1135000'>But that was what you had
  to do in molecular biology because that was</span> <span m='1138000'>how you got
  the gene. You build a whole library,</span> <span m='1142000'>you withdraw it from
  the library. However, if you wanted to do this,</span> <span m='1147000'>could you
  manage to get the beta-globin sequence from your</span> <span m='1152000'>genome
  without having to make the whole library?</span> </p><p><span m='1156000'>It turns
  out, and I know it's been covered at least in some of the</span> <span m='1161000'>sections,
  there's a cool technique to do that. And what is</span> <span m='1166000'>that technique?
  PCR. So, it turns out that the next</span> <span m='1172000'>really great advance
  in molecular biology was the technique of PCR.</span> </p><p><span m='1179000'>And
  what PCR was a way, is, is a way to obtain a piece of DNA</span> <span m='1187000'>corresponding
  to an already known gene, you have to already know the</span> <span m='1194000'>gene,
  and what it allows you to do is then obtain that piece of DNA</span> <span m='1201000'>based
  on knowing at least some of its sequence.</span> </p><p><span m='1209000'>It allows
  you to amplify just that DNA from a, from any individual.</span> </p><p><span m='1214000'>So,
  as compared to the experiment where I make a library for you and a</span> <span
  m='1220000'>library for you and a library from you and a library from you,</span>
  <span m='1225000'>each of which could take a month, PCR would allow us to do it
  in</span> <span m='1231000'>principle in five minutes. And, actually,</span> <span
  m='1236000'>there are machines that would let you do it in five minutes.</span>
  </p><p><span m='1241000'>So, let's discuss how this PCR works. Nobody uses the five
  minute</span> <span m='1246000'>machines because you usually will then wait an hour
  or so,</span> <span m='1252000'>but anyway. Suppose I take my DNA sequence here
  from</span> <span m='1257000'>the human genome. Five prime to three prime.</span>
  </p><p><span m='1264000'>Five prime to three prime. This sequence here beta-globin.</span>
  </p><p><span m='1274000'>I want to obtain that sequence. The first thing I do is
  I'm going to</span> <span m='1283000'>heat my DNA sample to maybe 97 degrees Celsius
  to denature.</span> </p><p><span m='1293000'>Denaturing means, of course, breaking
  the hydrogen</span> <span m='1302000'>bonds that separate the two strands so that
  the strands come apart,</span> <span m='1311000'>five prime to three prime, five
  prime to three prime.</span> </p><p><span m='1320000'>Now, what I then do is I take
  a specific DNA primer matching this</span> <span m='1329000'>stretch just before
  the beta-globin gene starts.</span> </p><p><span m='1339000'>Or just before where
  I'm interested in. How do I make a primer that</span> <span m='1342000'>matches
  just that sequence? I order, well, how do I know what</span> <span m='1346000'>to
  order? I know the sequence, right?</span> </p><p><span m='1350000'>I've got the
  sequence already. I just look at it and I say I want</span> <span m='1354000'>that
  sequence. And then how do I get it?</span> </p><p><span m='1358000'>I order it.
  I type it into the Web and the machine will synthesize me</span> <span m='1362000'>this,
  this primer. Typically a 20-base stretch will</span> <span m='1367000'>suffice.
  So, I'll get me a twentymer, a 20 base oligonucleotide</span> <span m='1371000'>complimentary
  to the sequence on this side of the gene.</span> </p><p><span m='1375000'>What I'm
  also going to do is the same thing over here.</span> </p><p><span m='1380000'>I'm
  going to get a second primer. This is primer number one. This is</span> <span m='1386000'>primer
  number two. OK? Now, let's see.</span> </p><p><span m='1392000'>Five prime. This
  is five prime, five prime. Now what I'd like to do</span> <span m='1399000'>is add
  polymerase, I'd like to add dNTPs.</span> </p><p><span m='1405000'>So, plus DNA
  polymerase plus dNTPs.</span> </p><p><span m='1412000'>And what will happen? Polymerase
  will come along and</span> <span m='1416000'>start copying my DNA, but it will only
  copy it starting</span> <span m='1421000'>from the primers. Now, this will keep
  going,</span> <span m='1425000'>of course, but DNA polymerase doesn't go forever,</span>
  <span m='1430000'>you know, the reactions sort of stops at some point.</span> </p><p><span
  m='1435000'>And so you'll get a strand going off here and a strand</span> <span
  m='1439000'>going off there. Now, notice what I've done.</span> </p><p><span m='1444000'>I
  started with an entire human genome, and the number of copies of</span> <span m='1450000'>beta-globin
  was one per genome. When I'm done with this process,</span> <span m='1455000'>how
  many copies, how many double-stranded copies of</span> <span m='1461000'>beta-globin
  do I have? Two. That's still very little,</span> <span m='1467000'>but it's more
  than I had before. So, what do I do next?</span> </p><p><span m='1473000'>Repeat.
  So, let's heat up that sample again. We'll denature at 97</span> <span m='1481000'>degrees,
  and now we have our initial strand here, we have our strand that</span> <span m='1488000'>came
  off this primer that runs to here and maybe goes forward,</span> <span m='1496000'>we
  have this strand here. We have this strand here.</span> </p><p><span m='1505000'>And
  this was five prime, five prime, five prime, five prime.</span> </p><p><span m='1515000'>Now
  what do we do? We repeat. We'll take our primer, this is</span> <span m='1525000'>primer
  number one, let's see. It matches over there.</span> </p><p><span m='1533000'>Primer
  number two over here. Number one over here. Number two.</span> </p><p><span m='1541000'>Have
  I got this right? Yes. Good. Then where does this guy stop?</span> </p><p><span
  m='1548000'>Right at the end where my other primer was. This guy</span> <span m='1556000'>runs
  along here. That guy stops right at the end.</span> </p><p><span m='1562000'>That
  guy might go a little further. How many copies of the beta-globin</span> <span m='1567000'>gene
  do I have now? Four. Two of which,</span> <span m='1572000'>by the way, perfectly
  sit between my pink primers. What's going to</span> <span m='1577000'>happen if
  I do this again? How many copies will I get?</span> </p><p><span m='1582000'>Eight,
  six of which will sit perfectly and two might be a little</span> <span m='1587000'>ragged
  as to where they go. So, initially,</span> <span m='1592000'>after cycle number
  zero, that is initial conditions,</span> <span m='1598000'>the number of copies
  relative to the genome was one.</span> </p><p><span m='1604000'>After one cycle
  it's two. After two cycles it's four.</span> </p><p><span m='1610000'>After N cycles
  it's two to the N copies. Is that clear</span> <span m='1616000'>how the PCR works?
  And that on every round you're</span> <span m='1621000'>doubling. And, with the
  exception of those two</span> <span m='1627000'>white things that go off to the
  side, they're going back and forth and</span> <span m='1633000'>back and forth between
  the two primers you chose to put in.</span> </p><p><span m='1638000'>What is when
  N equals ten, what do you got? A thousand copies.</span> </p><p><span m='1644000'>What
  happens when N equals 20? A million copies.</span> </p><p><span m='1650000'>What
  is the copy number of beta-globin? Beta,</span> <span m='1654000'>let's suppose
  beta-globin, for the sake of the argument,</span> <span m='1658000'>sake of argument
  is about one thousand bases.</span> </p><p><span m='1662000'>What fraction of the
  human genome does beta-globin represent?</span> </p><p><span m='1666000'>Yeah, about
  a millionth for the genome. No, actually,</span> <span m='1670000'>one three millionth,
  but we'll call it a millionth.</span> </p><p><span m='1674000'>So, after I've made
  a million-fold amplification of beta-globin,</span> <span m='1678000'>beta-globin
  now represents half of the stuff that's in the tube.</span> </p><p><span m='1683000'>What
  would happen if I go another ten rounds? How many copies do I</span> <span m='1688000'>have?
  A billion copies. So, in other words, I started with</span> <span m='1693000'>something
  that was only present at about one one-millionth of what was</span> <span m='1698000'>in
  my test tube. If I could make a billion copies of</span> <span m='1703000'>that
  specific molecule, now it so dominates the mixture that</span> <span m='1708000'>it
  is a thousand times more abundant than the rest of the genome.</span> </p><p><span
  m='1713000'>It works. That's the remarkable thing, this works.</span> </p><p><span
  m='1719000'>Any questions about the technique? Now, yes? Well, I need two primers</span>
  <span m='1725000'>in their sequence. How many copies do I need of each</span> <span
  m='1732000'>of those primers? Well, I, I obviously need a lot of</span> <span m='1738000'>copies
  of those primers. So, primer number one,</span> <span m='1743000'>it's a single
  sequence, but when I order it from the company,</span> <span m='1746000'>I'm going
  to order me a boat load, a lot of that primer. So, I'm</span> <span m='1750000'>adding,
  I better add a billion molecules of that primer because I'm</span> <span m='1753000'>going
  to make a billion copies starting from such primers.</span> </p><p><span m='1756000'>But
  if I have a billion copies of, of number one and a billion copies</span> <span m='1760000'>of
  number two and, you know, these days,</span> <span m='1763000'>billions aren't such
  big, you know, molecules are Avogadro's</span> <span m='1766000'>number and all
  that. It's not hard to get things.</span> </p><p><span m='1770000'>So, you throw
  in huge excess, a massive excess of primer number</span> <span m='1774000'>one,
  a massive excess of primer number two, and you just do this.</span> </p><p><span
  m='1778000'>Now, I mean, what does it cost to make such a massive excess of a</span>
  <span m='1782000'>primer? It's about ten cents a base, so it's two bucks,</span>
  <span m='1786000'>two bucks per primer give or take. You know, so I can get you
  a better</span> <span m='1790000'>price if you want, but, you know, anyway.</span>
  </p><p><span m='1794000'>It's not a bad price to, to buy primer. So, you can just
  go</span> <span m='1798000'>out and order a pair of primers. You can have them tomorrow.</span>
  </p><p><span m='1801000'>And then all you have to do is add the primers to the,</span>
  <span m='1805000'>so I take DNA. Do I, I, I need DNA from you.</span> </p><p><span
  m='1809000'>It turns out I could draw your blood and purify DNA and all that.</span>
  </p><p><span m='1813000'>But it turns out that if all I wanted to do was amplify
  one locus,</span> <span m='1816000'>I could actually take a Popsicle stick and ask
  you to scrape the</span> <span m='1820000'>inside of your cheek. That'll get enough
  cells off from</span> <span m='1824000'>the inside of your cheek, stick it in a
  test-tube, and it'll</span> <span m='1828000'>actually have enough DNA there. It
  turns out this is a very</span> <span m='1831000'>sensitive and powerful technique,
  so, but before we get to that notice</span> <span m='1835000'>what we had to do.
  We had to heat our DNA to 97</span> <span m='1839000'>degrees and add polymerase.
  Then we heat again to 97, add</span> <span m='1843000'>polymerase, heat again to
  97, add polymerase. Why do I have to</span> <span m='1846000'>keep adding polymerase?
  Because polymerase gets roomed at</span> <span m='1850000'>97 degrees so it's denatured.
  So, the nuisance about PCR is I</span> <span m='1854000'>have to go to my Eppendorf
  plastic tube, pop open the lid,</span> <span m='1857000'>stick in some DNA polymerase,
  close it up, stick it back in a</span> <span m='1861000'>heating bath, let it go
  for a while, take it out, pop it open, add some</span> <span m='1865000'>more polymerase,
  put it back in the heating bath,</span> <span m='1869000'>pop it out. And this is
  actually the way</span> <span m='1873000'>primitive scientists did PCR not so long
  ago, OK? Wouldn't it be cool</span> <span m='1877000'>if we could engineer a DNA
  polymerase that didn't denature at</span> <span m='1882000'>97 degrees? Because
  then what we could do is just add the polymerase,</span> <span m='1886000'>close
  up the tube, put it in a machine that goes heat,</span> <span m='1891000'>cool,
  heat, cool, heat, cool, heat, cool, but you would have,</span> <span m='1895000'>so
  how do we, what kind of cleaver biological engineering do we use to</span> <span
  m='1900000'>modify polymerase so it won't denature at 97 degrees?</span> </p><p><span
  m='1905000'>Yes? Get it from a bacteria. What kind of a bacteria would you</span>
  <span m='1909000'>ask for an enzyme that could work in, in basically boiling water?</span>
  </p><p><span m='1913000'>Bacteria that basically live in boiling water. Where would</span>
  <span m='1917000'>you look for such? Thermal vents.</span> </p><p><span m='1923000'>You'll,
  geysers, things like that. Life lives</span> <span m='1929000'>everywhere. What
  you go is you find yourself a bacterium,</span> <span m='1935000'>so you find bacteria
  that lived in geysers or in thermal vents and you</span> <span m='1941000'>purify
  their DNA polymerase. The most famous one comes from the</span> <span m='1947000'>organism,
  the bacterium called thermos aquaticus, aquaticus.</span> </p><p><span m='1954000'>Which
  of course means hot water, right? That's what the bacteria is</span> <span m='1958000'>called,
  thermos aquaticus. And, or, and its enzyme is called</span> <span m='1963000'>tack,
  tack. So, we'll refer to it often,</span> <span m='1967000'>Taq polymerase, meaning
  from this bacteria thermos aquaticus,</span> <span m='1972000'>OK? So, that's Taq.
  So, it turns out that you can do this now without</span> <span m='1977000'>having
  to open and close the test-tubes.</span> </p><p><span m='1982000'>Oops, I meant
  to put that here. How sensitive is PCR? It's very</span> <span m='1991000'>sensitive.
  You could do, so applications of PCR. Very</span> <span m='2000000'>versatile. First
  let's just re-sequence a gene.</span> </p><p><span m='2010000'>Gene from yeast or
  from human. You just need, you know, any DNA</span> <span m='2016000'>sample. Get
  my gene, get my primers, and as I was</span> <span m='2022000'>indicating with a
  Popsicle stick, I don't have to have it very pure,</span> <span m='2028000'>although
  in a laboratory you go to the trouble of making it pure</span> <span m='2034000'>because
  you want it to be pure and all that. Yes?</span> </p><p><span m='2040000'>Correct.
  Yeah. So, remember I was making a fuss</span> <span m='2046000'>over the accuracy
  of replication, right? And I said that on its own a</span> <span m='2052000'>polymerase
  might have an accuracy to only about ten to the minus five.</span> </p><p><span
  m='2059000'>So, now, what were the two mechanisms for,</span> <span m='2065000'>for
  repairing DNA, for proofreading DNA?</span> </p><p><span m='2072000'>One was a built-in
  proofreading activity that the enzyme had.</span> </p><p><span m='2075000'>The enzyme
  would have put in a base, would check the base,</span> <span m='2078000'>and that
  actually helped by an order of magnitude or two.</span> </p><p><span m='2082000'>And
  some of these polymerases have a proofreading activity.</span> </p><p><span m='2085000'>But
  then we also discussed the mismatch repair activity that would</span> <span m='2089000'>later
  come along and detect mismatches. You're absolutely right,</span> <span m='2092000'>PCR
  is not as accurate as cells because it doesn't have that</span> <span m='2096000'>mismatch
  repair activity. So, when you take a PCR product,</span> <span m='2100000'>if I
  were to clone, so if I were to take all the PCR product,</span> <span m='2105000'>say,
  from my beta-globin gene, so I'm going to take my test-tube,</span> <span m='2110000'>I'm
  going to add my primers and everything, I'm going to PCR,</span> <span m='2115000'>I'm
  going to PCR, and then I'm going to get a lot of copies of</span> <span m='2120000'>beta-globin.
  If I were to take that beta-globin and just directly</span> <span m='2125000'>sequence
  the DNA in the test-tube. Here's my pieces of beta-globin.</span> </p><p><span m='2130000'>I
  can now sequence it by adding a primer and doing my fluorescent</span> <span m='2134000'>sequencing
  and running it on a sequencer and all that.</span> </p><p><span m='2139000'>Sorry,
  going the other way. I'll run a sequencing reaction.</span> </p><p><span m='2144000'>I
  could actually do it, and what I do it on is the whole population of</span> <span
  m='2148000'>a million or a billion molecules. If any one of them is wrong it's</span>
  <span m='2153000'>going to be swamped out by others, OK? Because I could do my</span>
  <span m='2158000'>sequencing reaction on the whole PCR product.</span> </p><p><span
  m='2163000'>And random mistakes in one molecule or the other will still be a tiny</span>
  <span m='2167000'>minority of the votes at any given base, right? But suppose I
  were to</span> <span m='2172000'>take my PCR product, all these amplified molecules
  here,</span> <span m='2177000'>and suppose I were to clone them individually and
  I were to sequence</span> <span m='2182000'>each of those individual clones instead
  of sequencing a,</span> <span m='2187000'>a mixture of all the products. I would,
  in fact,</span> <span m='2192000'>see a higher mutation rate. And you're absolutely
  right.</span> </p><p><span m='2196000'>When people clone PCR products they have
  to check them afterwards and</span> <span m='2201000'>throw out the ones that are
  wrong, OK? Absolutely right. Good, good,</span> <span m='2206000'>good. So, you
  guys are, you know, right on top of the important issues</span> <span m='2210000'>about,
  about DNA. So, so I can, I can take a gene and</span> <span m='2215000'>I can re-sequence
  it. I can also do things like take</span> <span m='2220000'>blood and look for the
  presence of a virus.</span> </p><p><span m='2225000'>So, I could re-sequence beta-globin
  and study people and see who's got</span> <span m='2231000'>sickle cell anemia and
  all that. I could take blood and I might want</span> <span m='2238000'>to say do
  I see the HIV virus present in someone's blood?</span> </p><p><span m='2244000'>For
  example, HIV testing can be done by making PCR primers for the</span> <span m='2251000'>sequence
  of the HIV virus. It has a genome.</span> </p><p><span m='2257000'>Taking a human's
  blood sample and PCR-ing it. If you get a positive</span> <span m='2261000'>PCR
  product, a PCR product that is made by these two primers and if,</span> <span m='2265000'>for
  example, you checked that it, that it gives you the HIV sequence</span> <span m='2269000'>then
  you know that that blood sample has, that person has the HIV virus.</span> </p><p><span
  m='2273000'>This is a way to do this. The PCR reaction itself is fast.</span> </p><p><span
  m='2277000'>Typically takes hours. In fact, can be forced to go much</span> <span
  m='2281000'>more quickly by machines that rapidly thermocycle.</span> </p><p><span
  m='2284000'>And you can actually PCR in five minutes, although people don't do it</span>
  <span m='2288000'>very often, but if you put a thin glass capillary and go heat,</span>
  <span m='2291000'>cold, heat, cold very, very quickly, there's a machine from Idaho</span>
  <span m='2294000'>Technologies that can do it in five minutes, but it's usually
  not the</span> <span m='2298000'>trouble. And you just put it in and, you know,
  in a couple hours you'll</span> <span m='2301000'>get an answer there as to whether
  or not somebody has HIV,</span> <span m='2304000'>for example. So, you can do that
  to detect relatively low</span> <span m='2308000'>quantities of virus. How low can
  you go?</span> </p><p><span m='2313000'>Well, it turns out, what's the limit? What's
  the</span> <span m='2319000'>smallest number of molecules you might be able to detect
  in a sample?</span> </p><p><span m='2325000'>Theoretically. One. You can't fewer
  than one molecule,</span> <span m='2331000'>right? So, one might be the limit. So,
  how could I arrange to have a</span> <span m='2337000'>single molecule in a test-tube?
  I would like to have a test-tube</span> <span m='2341000'>that has exactly one copy
  of the beta-globin gene.</span> </p><p><span m='2344000'>What, how's the best, what's
  the best way to get exactly</span> <span m='2348000'>one copy of beta-globin and
  put it in the test-tube?</span> </p><p><span m='2351000'>Sorry? You can't. Why?
  Just one molecule.</span> </p><p><span m='2354000'>I want to get exactly one copy
  of beta-globin. I could,</span> <span m='2358000'>I could just take total DNA and
  dilute it so, on average,</span> <span m='2361000'>there's only one copy. Or, actually,
  is there any way to,</span> <span m='2364000'>I mean can I, I'd just like to buy
  a package that contains exactly</span> <span m='2368000'>one beta-globin. Sorry?
  Bind it to something big.</span> </p><p><span m='2372000'>Let's think biologically.
  Does biology package up a single copy of</span> <span m='2376000'>beta-globin? Sorry?
  Gametes. How about a sperm?</span> </p><p><span m='2381000'>Let's grab a sperm by
  its tail here, put it in the test-tube.</span> </p><p><span m='2386000'>It's one
  copy of beta-globin. So, you can actually take cell</span> <span m='2390000'>sorters
  and have it cell sort sperm into individual test-tubes.</span> </p><p><span m='2395000'>You
  now know there's one copy of beta-globin.</span> </p><p><span m='2400000'>Heat it
  up, it will crack open the sperm, add your primers,</span> <span m='2405000'>you
  can amplify beta-globin, it's a single copy. That proves its</span> <span m='2410000'>extraordinary
  sensitivity. You can do it with a single sperm.</span> </p><p><span m='2416000'>You
  can do it with a single egg also, but harder to come by.</span> </p><p><span m='2421000'>So,
  with that level of sensitivity, you could do the following. So,</span> <span m='2427000'>single
  sperm typing. Now, single sperm typing is cool but</span> <span m='2432000'>sort
  of useless. What are you going to do with it,</span> <span m='2436000'>right? But
  here's another thing you could do. Embryo typing.</span> </p><p><span m='2441000'>Suppose
  someone has a genetic disease in their family,</span> <span m='2445000'>maybe it's
  Huntington's disease. And suppose that the individual with</span> <span m='2450000'>Huntington's
  disease wants to have kids. Or the individual,</span> <span m='2454000'>sorry, the
  individual who is at risk for Huntington's disease or breast</span> <span m='2459000'>cancer
  or whatever wants to have kids.</span> </p><p><span m='2464000'>What you can do
  is with an in vitro fertilization clinic you're able to</span> <span m='2473000'>obtain
  eggs, fertilize eggs in vitro, and grow them up in a Petri plate to</span> <span
  m='2483000'>8 or 16 cell stage before re-implanting embryos</span> <span m='2493000'>back
  in the mother. Wouldn't it be cool if we could</span> <span m='2501000'>choose to
  only re-implant an embryo that did not have the genetic</span> <span m='2508000'>disease?
  How are we going to do that? PCR. How are we,</span> <span m='2516000'>so what do
  we do? We take the embryo.</span> </p><p><span m='2522000'>We make DNA from the
  embryo. We do PCR and we say, ah-ha, this</span> <span m='2526000'>embryo did not
  have the genetic disease. Problem is it has killed</span> <span m='2530000'>the,
  the cells there, right, it killed the embryo.</span> </p><p><span m='2535000'>Any
  ideas? Pull off one cell. Remove a single cell. It turns out</span> <span m='2539000'>that
  at stage the cells are not differentiated.</span> </p><p><span m='2544000'>If I
  remove one cell from an embryo at that very early stage,</span> <span m='2548000'>the
  other cells with make a perfectly happy, healthy baby.</span> </p><p><span m='2553000'>That
  cell is not necessary. This single cell sensitivity is</span> <span m='2557000'>very
  valuable because I can actually do single cell genotyping on in</span> <span m='2561000'>vitro
  fertilized embryos and be able offer parents a chance,</span> <span m='2565000'>the
  opportunity to re-implant only those embryos that do not have the</span> <span m='2569000'>genetic
  defect. That's cool. That's really cool.</span> </p><p><span m='2573000'>There are
  other things you might be able to do. If you're treating a</span> <span m='2577000'>patient
  with cancer, a patient, a cancer patient and</span> <span m='2581000'>you've given
  chemotherapy you want to know have I managed to eradicate</span> <span m='2585000'>the
  cancer cells? And six months later have any of the</span> <span m='2591000'>cancer
  cells come back? I could look for very low</span> <span m='2599000'>quantities of
  cancer cells. I can, I can do surveillance for</span> <span m='2607000'>low quantities
  of cancer cells following chemotherapy.</span> </p><p><span m='2615000'>And, of
  course, I can also do forensics.</span> </p><p><span m='2619000'>I could take a
  small sample of blood from the scene of a crime or</span> <span m='2624000'>saliva
  from the back of an envelope that someone has licked,</span> <span m='2629000'>and
  I could do PCR and look for genetic variations that distinguish</span> <span m='2633000'>people.
  And, presumably, you see all that stuff on television</span> <span m='2638000'>all
  the time. So, that's what PCR is good for.</span> </p><p><span m='2644000'>It's
  good. All right. Last topic, very brief topic, but I do want to</span> <span m='2650000'>mention.
  This was being able to analyze a gene directed mutagenesis.</span> </p><p><span
  m='2657000'>And I won't go through the details of all this, but I just want to at</span>
  <span m='2663000'>least basically describe the concept.</span> </p><p><span m='2670000'>I
  could take any piece of DNA, say from a drosophila, and I can</span> <span m='2676000'>mutate
  the DNA in vitro. I can change this base from a G to</span> <span m='2682000'>a
  C. There's a right, there's a proper protocol and</span> <span m='2688000'>cooking
  trick for doing that. It involves putting a certain oligo</span> <span m='2694000'>over
  it and extending, and it doesn't matter exactly how.</span> </p><p><span m='2700000'>I
  could insert an extra gene into that. I could use a little</span> <span m='2704000'>restriction
  enzyme to open it up and stuff something in.</span> </p><p><span m='2708000'>I could
  delete something from this. Maybe I'll use a restriction enzyme</span> <span m='2713000'>to
  cut it open, et cetera. Basically I could,</span> <span m='2717000'>I can fuse genes
  together. I can do whatever kind of construction of</span> <span m='2722000'>pieces
  of DNA and modifications of pieces of DNA that I would</span> <span m='2726000'>like
  to do in vitro. I can then take that mutated gene,</span> <span m='2731000'>let's
  say the gene is an enzyme, encodes an enzyme,</span> <span m='2736000'>and the enzyme
  has an active site. I could change the code for the</span> <span m='2741000'>amino
  acid right at the active site to see if that amino acid really</span> <span m='2746000'>matters
  or not. I can do any of those things.</span> </p><p><span m='2752000'>And I can
  put this back in an organism. Remember that you,</span> <span m='2757000'>I said
  you could transform DNA back into bacteria?</span> </p><p><span m='2762000'>Well,
  you can also do such things as simply inject DNA into</span> <span m='2767000'>a
  fertilized egg. In fact, at the stage where there's</span> <span m='2771000'>a male
  and a female pronucleus that haven't fused yet right after</span> <span m='2775000'>fertilization.
  You can take your little pipette</span> <span m='2779000'>and a needle and you can
  inject some of the DNA you want into the male</span> <span m='2782000'>pronucleus,
  and then when the male pronucleus and the female pronucleus</span> <span m='2786000'>fuse
  and the embryo grows it will have your DNA.</span> </p><p><span m='2790000'>You
  can make mice that carry whatever gene you've modified like</span> <span m='2796000'>this.
  You can also not, you, you can also not just modify a</span> <span m='2802000'>piece
  of DNA and add, this is gene addition,</span> <span m='2809000'>you can also do
  gene subtraction. You can do gene subtraction and,</span> <span m='2815000'>again,
  I won't worry about the details here, by taking</span> <span m='2821000'>embryonic
  stem cells. Much in the news these days,</span> <span m='2827000'>and we may come
  back to them. And in vitro, working with</span> <span m='2831000'>embryonic stem
  cells, to transform a piece of DNA that has</span> <span m='2835000'>been arranged
  to recombine into the gene of interest and know it out.</span> </p><p><span m='2839000'>So,
  if you build, if you build a piece of DNA in vitro and you put it</span> <span m='2844000'>into
  a whole bunch of embryonic stem cells you can select,</span> <span m='2848000'>by
  various cleaver techniques, for those embryonic stem cells that</span> <span m='2852000'>have
  taken up your gene. And not just taken it up but slammed</span> <span m='2858000'>it
  into the normal locus in place of the normal locus.</span> </p><p><span m='2865000'>And
  that way you can knock out a gene. You can do gene knockout.</span> </p><p><span
  m='2872000'>So, the basic point of this now, to summarize these many lectures is</span>
  <span m='2879000'>we're now at the point where this picture that we saw at the,</span>
  <span m='2886000'>at the beginning, function, gene, protein, that we understood
  now</span> <span m='2893000'>first as a methodology, genetics, biochemistry.</span>
  </p><p><span m='2900000'>And then we understood how genes encode proteins through
  molecular</span> <span m='2906000'>biology. These tools of recombinant DNA allow
  us to move</span> <span m='2912000'>in any direction. You want to find the gene
  underlying</span> <span m='2917000'>a function, find the gene for Huntington's disease?</span>
  </p><p><span m='2921000'>We could do it. Clone it based solely on its linkage.</span>
  </p><p><span m='2925000'>You want to find the gene encoding a protein? If I know
  its amino acid</span> <span m='2929000'>sequence, I can find the DNA sequence that
  corresponds it.</span> </p><p><span m='2933000'>If I want to find what a certain
  protein does, its function,</span> <span m='2937000'>I could get the gene for that
  protein. I could knock out the gene</span> <span m='2941000'>for that protein and
  see what its function is.</span> </p><p><span m='2945000'>Suddenly, for the mathematicians
  amongst the group,</span> <span m='2948000'>this becomes a commutative diagram,
  which you can chase around in any</span> <span m='2952000'>direction. That is, in
  a sense, what the 20th century</span> <span m='2955000'>was about, was intellectually
  these two disciplines merging through</span> <span m='2959000'>molecular biology
  and then recombinant DNA giving you all the</span> <span m='2962000'>tools that
  if you're sitting at any place in this triangle you can move</span> <span m='2966000'>this
  way and that way, from a gene to a protein,</span> <span m='2969000'>from a protein
  to a gene, from a function to a gene, from a</span> <span m='2973000'>function to
  a protein. Much of the rest of the course we'll</span> <span m='2976000'>talk about
  how you use these tools, but this brings to a close this</span> <span m='2980000'>first
  chunk of the course about the concepts and the methodologies of</span> <span m='2984000'>molecular
  biology. Now, if you hang on one more minute,</span> <span m='2988000'>this is my
  last lecture for a while. I won't be, we're having an exam on,</span> <span m='2992000'>we
  have a quiz on Monday, and then Bob's taking over again.</span> </p><p><span m='2996000'>So,
  I won't see you for the next week or so. So, two things.</span> </p><p><span m='3000000'>One,
  I won't see you before the World Series is over so everyone</span> <span m='3004000'>please
  think good thoughts about the Red Sox. Number two,</span> <span m='3009000'>I will
  not see you before the election. Vote.</span> </p><p><span m='3013000'>It's your
  choice who you vote for, but vote. Good-bye.</span> </p>
type: course
uid: eea5db9b7a8ab27b00c9fa42e1e289aa

---
None