---
about_this_resource_text: <p><strong>Topics covered:</strong> Molecular Biology 3</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec12.jpg
  parent_uid: ae2750f05594c53c59122f7f9840031e
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-12-molecular-biology-3/lec12.jpg
  title: lec12.jpg
  type: null
  uid: eb590d51b7e4a852ea8d3878e72bfc52
- id: Video-YouTube-Stream
  media_location: ARjSihLe1K8
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Video-YouTube-Stream
  type: Video
  uid: fc28f4e9b1869d11deb91f4cea712ba4
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/ARjSihLe1K8/default.jpg
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: a628261f8bf827e5545da4eff3dad4d3
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Video-iTunes U-MP4
  type: Video
  uid: c769efb1d5ee5c024e8d6df8a0c3880d
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec12-06oct2004-220k.mp4
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Video-Internet Archive-MP4
  type: Video
  uid: 1ca75aa774e2365fa45a922f41762884
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 685c54f8faaf3541e724cdc413cd24b2
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-06oct2004-1000.mp3
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Video-Internet Archive-MP3
  type: Video
  uid: 886d554d314b399d1bf73241e65a7df2
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 1fba801437d4e293d940ab035d4a669c
- id: Thumbnail-OCW-JPG
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 2f92fc1b68e5d3823bb4cc50272eb2c8
- id: 3Play-3PlayYouTubeid-MP4
  media_location: ARjSihLe1K8
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 33bdff4c08c5157fee0436c97df9dee8
- id: ARjSihLe1K8.srt
  parent_uid: ae2750f05594c53c59122f7f9840031e
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-12-molecular-biology-3/ARjSihLe1K8.srt
  title: 3play caption file
  type: null
  uid: 25dd0e985df867d385ba76391ee08dea
- id: ARjSihLe1K8.pdf
  parent_uid: ae2750f05594c53c59122f7f9840031e
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-12-molecular-biology-3/ARjSihLe1K8.pdf
  title: 3play pdf file
  type: null
  uid: 4c6e249549606c6bae202f7cbc289582
- id: Caption-3Play YouTube id-SRT
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: abd67128e4a6d22414c21b35f44c03a0
- id: Transcript-3Play YouTube id-PDF
  parent_uid: ae2750f05594c53c59122f7f9840031e
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: fe8d9d9c4487d0a796e6b3310d465822
inline_embed_id: 20996404lecture12:molecularbiology363442871
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-12-molecular-biology-3
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-12-molecular-biology-3
template_type: Tabbed
title: 'Lecture 12: Molecular Biology 3'
transcript: <p><span m='1000'>Good morning. Good morning.</span> </p><p><span m='8000'>So,
  I'd like to pick up where we left off last time and just finish</span> <span m='14000'>off
  translation and then step back and look at how this central dogma</span> <span m='20000'>of
  DNA is replicated into DNA, is read into RNA, and is translated</span> <span m='26000'>into
  protein. Or, actually, as Francis Crick</span> <span m='30000'>really put it, all
  information flow from nucleic</span> <span m='33000'>acid to protein. How that varies
  amongst organisms.</span> </p><p><span m='37000'>Because first we're going through
  it and looking at the absolutely</span> <span m='40000'>common features, DNA replication,
  so it's five prime</span> <span m='43000'>to three prime, et cetera, et cetera,</span>
  <span m='46000'>transcription, translation. But in a moment I'd like to turn to</span>
  <span m='50000'>the variations between different kinds of organisms.</span> </p><p><span
  m='53000'>But let me briefly finish up, if I may, the bit about translation</span>
  <span m='56000'>in general so we can look at its variation.</span> </p><p><span
  m='60000'>As we talked about last time, we have a messenger RNA that has</span>
  <span m='66000'>been transcribed from a specific region of the chromosome starting
  at</span> <span m='73000'>a promoter and going to some stop of transcription.</span>
  </p><p><span m='80000'>And that messenger RNA will include some particular sequence,</span>
  <span m='87000'>and I'll copy one here, A-U-A-C-G-A-U-G-A-A-G-A-G-G-C-C-C,</span>
  <span m='94000'>et cetera, et cetera, et cetera, out to a UAG.</span> </p><p><span
  m='101000'>And this is the direction five prime to three prime.</span> </p><p><span
  m='105000'>We'll remember that all nucleic acid polymerization goes five prime</span>
  <span m='109000'>to three prime. So, what happens is the cell begins</span> <span
  m='113000'>scanning this message. And it does that by this message</span> <span
  m='117000'>being exported into the cytoplasm of the cell. The ribosome coming along</span>
  <span m='121000'>and glomming onto this message and scanning on for the</span> <span
  m='125000'>place to start. It looks, it looks,</span> <span m='129000'>it looks,
  it looks, and it finds the first AUG. Footnote,</span> <span m='133000'>this isn't
  100% true. There are occasional messages that start their</span> <span m='138000'>translation
  not at an AUG, and there are even occasional,</span> <span m='142000'>there are
  even more messages that don't quite start at the first AUG</span> <span m='147000'>because
  the ribosome is really is looking for something a little bit</span> <span m='151000'>special,
  but to a first order approximation.</span> </p><p><span m='156000'>Good enough for
  the textbooks. It goes along to the first AUG.</span> </p><p><span m='160000'>In
  reality it's a little more subtle than that.</span> </p><p><span m='164000'>But
  it starts at the first AUG. And what it does is it builds a</span> <span m='168000'>protein
  that corresponds to it according to a three letter genetic</span> <span m='172000'>code.
  And you all know the lookup table. It's in your book.</span> </p><p><span m='176000'>AUG,
  always the first amino acid put in. A methionine.</span> </p><p><span m='180000'>Then
  AAG. Lysine, I think. Then arginine.</span> </p><p><span m='184000'>Then a proline.
  Now, I mean this is this particular sequence.</span> </p><p><span m='188000'>Any
  other sequence would be different. Et cetera.</span> </p><p><span m='192000'>How
  does it accomplish this matching between three letters of</span> <span m='197000'>the
  genetic code? Oh, and when it gets to AUG,</span> <span m='201000'>that is one of
  the three singles for stop, don't put in any more amino</span> <span m='205000'>acids.
  There are three such stop signals.</span> </p><p><span m='210000'>AUG, sorry, UAG,
  UGG and U, oops, what did I just do</span> <span m='217000'>here? Let's get that
  right. UAG, UGG and UGA. Those are the</span> <span m='224000'>three stop codons.
  So, how many total codons are there?</span> </p><p><span m='231000'>64 codons. Three
  of them spell stop. 61 of them spell</span> <span m='239000'>specific amino acids.
  And how many amino acids are there?</span> </p><p><span m='245000'>20. So, the average
  redundancy is three. Some are specified by</span> <span m='251000'>multiple codons.
  The most extreme is some amino</span> <span m='256000'>acids are specified by as
  many as six codons. Did I,</span> <span m='261000'>oh, thank you. Come back down.
  Of course. U-A, so it's UAG, right?</span> </p><p><span m='267000'>Sorry, UAA and
  UGA and UAG.</span> </p><p><span m='272000'>Thank you. Very good. All right. So,
  now, how does it accomplish</span> <span m='277000'>this feat of taking amino acids,
  of taking nucleotide sequence, RNA</span> <span m='282000'>sequence and converting
  it into the sequence of amino acids?</span> </p><p><span m='287000'>As I mentioned
  last time, there was lots of original somewhat</span> <span m='292000'>nutty thinking
  about some looping codes that would make the RNA fold</span> <span m='297000'>up
  in such a way to bind the amino acids and all that.</span> </p><p><span m='303000'>But,
  as Francis Crick thought up, there had to be some kind of an</span> <span m='312000'>adapter
  molecule that would take the RNA sequence and would somehow</span> <span m='321000'>connect
  it up to the correct amino acid, and that was UAC.</span> </p><p><span m='330000'>A
  particular transfer RNA molecule. And the tRNA molecule is an adapter</span> <span
  m='335000'>sequence that has three nucleotides here that match up to the three</span>
  <span m='340000'>nucleotides of the codon that we're trying to translate,</span>
  <span m='345000'>and it has the appropriate amino acid that's been stuck on the
  end of</span> <span m='350000'>it. And how does it get there? How does the right
  tRNA, the tRNA</span> <span m='355000'>to match this codon have the right amino
  acid put on it?</span> </p><p><span m='360000'>There's a dedicated enzyme that recognizes
  that tRNA and puts on</span> <span m='363000'>that amino acid. It's aminoacyl-tRNA
  synthetase.</span> </p><p><span m='367000'>It sticks the right amino on the right
  transfer RNA.</span> </p><p><span m='371000'>So, that's how it accomplishes the
  physical recognition of these three</span> <span m='375000'>bases and has the right
  amino acid attached to it.</span> </p><p><span m='378000'>There's an enzymatic machinery
  that has all of these tRNAs floating</span> <span m='382000'>around in the cell
  which can be used for this translation here.</span> </p><p><span m='386000'>How
  does this actually happen physically?</span> </p><p><span m='390000'>It happens
  in this vast machine called the ribosome.</span> </p><p><span m='395000'>In the
  ribosome, if we have, say, our codon here and we have a</span> <span m='401000'>tRNA
  that, well, we'll put that actually in the</span> <span m='407000'>ribosome that,
  say, has the first amino acid here,</span> <span m='413000'>methionine, there's
  a cavity for this guy and there's a cavity</span> <span m='419000'>for the next
  guy. And other tRNAs come into the cell</span> <span m='425000'>carrying their next
  amino acid. Maybe it will be here a lysine that</span> <span m='431000'>matches
  up with the codon and the anti-codon. And when the right tRNA</span> <span m='437000'>fits
  in the next cavity over, the ribosome itself catalyzes a</span> <span m='443000'>peptide
  bond between these amino acids.</span> </p><p><span m='450000'>Then it chugs over
  by one, it translocates by one moving this</span> <span m='455000'>bit of the complex
  to the left, and the peptide chain continues to</span> <span m='460000'>grow out
  this end as each new codon is moved into position,</span> <span m='465000'>a tRNA
  comes in bringing the right amino acid until finally a stop</span> <span m='470000'>codon
  is hit. And what happens when you hit a stop codon?</span> </p><p><span m='475000'>It
  stops. And is there a tRNA for a stop?</span> </p><p><span m='480000'>It turns out
  there's not. There actually isn't. There's some</span> <span m='482000'>other factor.
  There's a protein factor that helps recognize the</span> <span m='485000'>stops.
  So, that just continues to chug on. Those of you who are</span> <span m='488000'>computer
  scientists or mathematicians will recognize this</span> <span m='491000'>is a two-tape
  Turing machine. It is the small two-tape Turing</span> <span m='494000'>machine
  that I know to exist. If you don't know what that means,</span> <span m='497000'>you
  can forget about that comment. In any case, but some of you know</span> <span m='500000'>what
  that is. So, that's how it proceeds.</span> </p><p><span m='503000'>That is your
  basic protein translation.</span> </p><p><span m='506000'>And, I must say, what
  I really love about this was</span> <span m='508000'>that Francis Crick kind of
  figured out what had to happen just on first</span> <span m='511000'>principles
  and was able to think through it much more clearly and</span> <span m='514000'>direct
  people to know what to look for in the laboratory.</span> </p><p><span m='517000'>And
  if people had not had the clarity of thinking that Crick</span> <span m='520000'>provided
  by saying, look, there's got to be this kind of</span> <span m='523000'>adapter,
  I don't think they would have found it as quickly.</span> </p><p><span m='526000'>But
  once he said this is what you've got to look for,</span> <span m='529000'>golly,
  it was there. You can't do that very often,</span> <span m='532000'>but Francis
  Crick seemed to have a very good track record of doing</span> <span m='535000'>those
  things. OK. So, that was just finishing off</span> <span m='538000'>translation.
  Now what I'd like to do is turn to</span> <span m='542000'>variations on the theme
  as the major issue for today.</span> </p><p><span m='548000'>How does this central
  dogma, DNA replicates, is transcribed into</span> <span m='554000'>RNA and is translated
  into protein, vary amongst the different kinds of</span> <span m='560000'>organisms
  that we might be interested in?</span> </p><p><span m='566000'>The kinds of organisms
  we might be interested in,</span> <span m='572000'>eukaryotes, prokaryotes, viruses.
  Sample eukaryote,</span> <span m='579000'>MIT undergraduate. Prokaryote, E. coli.
  And virus, many possible</span> <span m='586000'>viruses. The eukaryotes' big nucleated
  cells.</span> </p><p><span m='593000'>So, in here we're going to have our nucleated
  cells.</span> </p><p><span m='600000'>DNA living in there. In our prokaryotes we
  have no</span> <span m='604000'>distinct nucleus. The DNA is not in a distinct</span>
  <span m='609000'>nucleus, although it's not entirely freely floating around.</span>
  </p><p><span m='614000'>It tends to be clustered together. In the virus the nucleic
  acid</span> <span m='619000'>resides in some kind of a capsid, some kind of a, it
  could be a</span> <span m='624000'>protein capsid. There are some of them that have</span>
  <span m='629000'>lipid capsids with lipid particles around them, but some kind of
  a coat</span> <span m='634000'>around nucleic acid there. Do they all do exactly
  the same</span> <span m='639000'>things with regard to DNA replication, RNA transcription
  and</span> <span m='644000'>protein translation? Well, not entirely. So,</span>
  <span m='649000'>as a way, in a way to reinforce what we know about these,</span>
  <span m='654000'>let's look at how they differ. DNA replication. Eukaryotes.</span>
  </p><p><span m='660000'>What's the structure of one of your chromosomes? Is it a
  long line,</span> <span m='666000'>a long linear molecule, or is it a circular molecule?</span>
  </p><p><span m='672000'>How many of you have linear chromosomes? How many of you
  have</span> <span m='678000'>circular chromosomes? I heard there were some people
  with</span> <span m='684000'>circular. And how many of you are unsure about your
  chromosomes?</span> </p><p><span m='690000'>OK. That's good. Well, then I'm pleased
  to inform</span> <span m='698000'>you that you have long linear chromosomes. Every
  human chromosome</span> <span m='707000'>is a long double-stranded molecule of DNA.
  Linear double-stranded DNA.</span> </p><p><span m='715000'>They can be extremely
  long. You have 23 chromosomes,</span> <span m='722000'>and together they make up
  three billion nucleotides of DNA.</span> </p><p><span m='728000'>A typical chromosome
  could be 150 million bases long as an average</span> <span m='733000'>size for a
  chromosome. And it's a single connected</span> <span m='739000'>molecule. 150 million
  bases long in the human is a typical chromosome.</span> </p><p><span m='744000'>One
  tricky little bit about replicating DNA.</span> </p><p><span m='750000'>Let's just
  think back to our little model of replicating DNA.</span> </p><p><span m='754000'>Let's
  come to the chromosome end here. It's five prime to three</span> <span m='758000'>prime.
  Five prime to three prime. We're going to start replicating.</span> </p><p><span
  m='762000'>We're getting to the end of chromosome number one.</span> </p><p><span
  m='767000'>We've got a primer here, and the primer is going to be used</span> <span
  m='771000'>to extend, extend, extend. We get right to the end.</span> </p><p><span
  m='775000'>That's good. Tell me how we're going to replicate back.</span> </p><p><span
  m='780000'>We need a little primer to start it, right? And where's that primer</span>
  <span m='784000'>going to land? Maybe over here it will start</span> <span m='789000'>replicating
  back. Oh, boy, we haven't done this</span> <span m='794000'>figure. So, what do
  we have to do there? So, we need to primer a</span> <span m='799000'>little further
  back. OK. But, you know what,</span> <span m='804000'>the chance that we're going
  to get that right at the end,</span> <span m='809000'>that we're going to get a
  primer exactly at the end is pretty low.</span> </p><p><span m='814000'>And if we
  don't have a primer exactly at the end,</span> <span m='817000'>what's going to
  be wrong with that copy of the chromosome?</span> </p><p><span m='821000'>Too short.
  Now, big deal. So, it's short by maybe 20 bases.</span> </p><p><span m='825000'>But
  that's just this cell division. What about next cell division? It</span> <span m='829000'>will
  be short on average by a little bit, and then the next cell division</span> <span
  m='833000'>and the next cell division. It's actually pretty tricky to</span> <span
  m='837000'>replicate a linear chromosome on the lagging strand,</span> <span m='841000'>unless
  you can land the primer in exactly the right place,</span> <span m='845000'>which
  doesn't happen. So, a special little solution is</span> <span m='849000'>used. The
  ends of chromosomes here are called telomeres,</span> <span m='855000'>telo meaning
  end. These telomeres have very specific structures.</span> </p><p><span m='861000'>In
  the human they repeat, T-T-A-G-G-G, again and</span> <span m='867000'>again and
  again. At the end of the chromosome there's</span> <span m='872000'>a special enzyme
  that will come along and add some extra telomere to</span> <span m='876000'>the
  chromosome. That, sorry? Did I say leading</span> <span m='881000'>strand? It's
  the, oh, yeah, sorry. It's the lagging,</span> <span m='886000'>sorry. It's the
  leading strand. No, no, no, this is the lagging strand.</span> </p><p><span m='890000'>This
  is the leading strand because it's running along happily not</span> <span m='895000'>having
  to make a primer. The okazaki fragment should be here.</span> </p><p><span m='900000'>I'll
  stick by that. We'll debate it later.</span> </p><p><span m='904000'>Anyway, they,
  we get the point. But it's lagging because you've got</span> <span m='908000'>the
  ogzocy fragments there. So, anyway, we have a problem of</span> <span m='912000'>replication.
  And the way the cell solves it is the actual replication</span> <span m='916000'>is
  shorter, but since it manages to stick some repeat at the end of the</span> <span
  m='920000'>chromosome it adds back some more T-T-A-G-G-G, T-T-A-G-G-G,</span> <span
  m='924000'>T-T-A-G-G-G, and it keeps dynamically adding more.</span> </p><p><span
  m='929000'>What do you think would happen if you didn't, or what's the enzyme</span>
  <span m='933000'>that adds telomeres? Telomerase. Telomerase adds that.</span> </p><p><span
  m='937000'>What cells do you think need to have active telomerase?</span> </p><p><span
  m='941000'>Rapidly dividing cells would need to have telomerase.</span> </p><p><span
  m='945000'>Cells that are not rapidly dividing, cells that have stopped dividing
  can</span> <span m='949000'>shut off their telomerase. But if a cell is going to
  go</span> <span m='953000'>through lots and lots of cell divisions it's got to,</span>
  <span m='957000'>it's got to tidy up its telomeres each time because they're</span>
  <span m='961000'>getting too short. You've got to have an enzyme that's</span> <span
  m='966000'>adding back ends of chromosomes. What cells do you think</span> <span
  m='970000'>particularly care about having telomerase on them?</span> </p><p><span
  m='974000'>Cancers. It turns out that this is not a trivial point.</span> </p><p><span
  m='979000'>More than 90% of cancers turn on actively the telomerase gene,</span>
  <span m='983000'>which would be a shut off in normal cells because the cell is</span>
  <span m='987000'>not dividing anymore. Part of becoming a cancer is having</span>
  <span m='992000'>to turn on this repair mechanism for the ends, this extension mechanism</span>
  <span m='996000'>for the ends of your chromosomes. And so, various people are trying</span>
  <span m='1000000'>to make drugs to inhibit cancers by inhibiting this telomerase
  enzyme.</span> </p><p><span m='1004000'>So, understanding just your linear replication
  of chromosomes is a kind</span> <span m='1009000'>of useful thing even in dealing
  with things like cancer.</span> </p><p><span m='1013000'>Genome sizes. I mentioned,
  how big was the human genome?</span> </p><p><span m='1017000'>Three times ten to
  the ninth bases. The mouse genome?</span> </p><p><span m='1022000'>It's almost as
  big, about 2.7 times ten to the ninth</span> <span m='1026000'>bases, 2.7 million
  bases. The elephant genome? I actually</span> <span m='1031000'>just found this
  out last week because we just finished sequencing</span> <span m='1035000'>elephant
  DNA, and I can now tell you I think it's 3.</span> </p><p><span m='1040000'>. The
  dog is 2. times ten to the ninth.</span> </p><p><span m='1045000'>Anyway, it's about,
  for most mammals it's pretty close</span> <span m='1049000'>to three billion bases.
  And there is some fluctuation.</span> </p><p><span m='1053000'>Some are a little
  bigger. Some are a little smaller.</span> </p><p><span m='1057000'>It doesn't scale
  with sizing the animal, though,</span> <span m='1061000'>because the dog has a smaller
  genome, for example, than the mouse does,</span> <span m='1065000'>but the elephant
  is a big bigger than us. And check in later in the</span> <span m='1068000'>term,
  I'll tell you about the aardvark. We should know in a</span> <span m='1072000'>little
  while. But here are, for example, fruit flies. The fruit</span> <span m='1076000'>fly,
  it has a genome of two times ten to the eighth.</span> </p><p><span m='1080000'>I'm
  giving, I'm being quite approximate. In fact,</span> <span m='1084000'>I'll make
  it, I'll give you 1. times ten to the eighth. 150</span> <span m='1088000'>million
  bases. Yeast, by contrast,</span> <span m='1092000'>has a genome of 1.2 times ten
  to the seventh. So, that's 12 million,</span> <span m='1097000'>150 million give
  or take, and about three billion,</span> <span m='1101000'>so 3,000 million. So,
  genome sizes can vary quite</span> <span m='1105000'>dramatically amongst different
  eukaryotes.</span> </p><p><span m='1110000'>Now, what about prokaryotes? How do
  the prokaryotes differ?</span> </p><p><span m='1116000'>Prokaryotes differ because
  their genomes are typically not linear</span> <span m='1123000'>chromosomes. The
  typical prokaryotic chromosome is a</span> <span m='1129000'>double-stranded circle.
  It's a double-stranded circular DNA.</span> </p><p><span m='1136000'>Now, the double-stranded
  circular</span> <span m='1142000'>DNA doesn't have this problem of telomeres. You
  just keep</span> <span m='1147000'>replicating around and you get to the end. So,
  there you have a much</span> <span m='1152000'>simpler replication system than having
  to worry about your ends of</span> <span m='1157000'>chromosomes. You also have
  much smaller genomes.</span> </p><p><span m='1162000'>The typical prokaryotic genome
  size, it's on the order of a few million</span> <span m='1167000'>bases. E. coli,
  4.6 million bases. There are, for example,</span> <span m='1171000'>mycobacteria,
  such as the mycobacteria that caused</span> <span m='1175000'>tuberculosis or leprosy,
  have on the order of, well,</span> <span m='1179000'>actually, not quite them, but
  other mycobacteria have on the order of</span> <span m='1183000'>about a million
  bases or so. Mycobacteria, M. genitalia has</span> <span m='1186000'>actually slightly
  less than a million basis.</span> </p><p><span m='1190000'>So, these are basically
  several million bases.</span> </p><p><span m='1194000'>So, there's a huge variation
  in genome size.</span> </p><p><span m='1198000'>Your genome is about a thousand
  times bigger than E.</span> </p><p><span m='1202000'>coli's genome. Now, you do
  actually have one circular chromosome.</span> </p><p><span m='1207000'>Do you know
  what it is? I speak about the 23 pairs of human</span> <span m='1211000'>chromosomes.
  There's actually one more human chromosome.</span> </p><p><span m='1216000'>The
  mitochondria have their own chromosome. It's a circle.</span> </p><p><span m='1220000'>That's
  very odd that you would have one chromosome that's a circle that</span> <span m='1225000'>looks
  like a bacterial chromosome. Do you know why that is?</span> </p><p><span m='1230000'>The
  mitochondria arose as a symbiotic bacterium that became a</span> <span m='1234000'>symbiont
  of eukaryotic cells about 1. billion years ago.</span> </p><p><span m='1238000'>It
  was a bacterium taken up into another cell, and that's how</span> <span m='1242000'>eukaryotes
  evolved. And we can even see that little</span> <span m='1246000'>signature of it
  having been a prokaryote from the fact that it's</span> <span m='1250000'>got one
  of these circular prokaryotic looking chromosomes.</span> </p><p><span m='1254000'>Now,
  it, because it's living in your cells, has thrown out all sorts</span> <span m='1258000'>of
  genes that it doesn't need anymore because the main,</span> <span m='1262000'>the
  nucleus supplies most of the proteins.</span> </p><p><span m='1266000'>So, your
  mitochondrial genome is a circle that's a mere 16,</span> <span m='1270000'>00 bases
  long. It's a very small circle encoding a very limited</span> <span m='1274000'>number
  of genes, but it's, in fact,</span> <span m='1278000'>the residue of the bacterial
  symbiont that lead to the formation</span> <span m='1282000'>of euks. Now, viruses,
  what do viruses have?</span> </p><p><span m='1286000'>Do they have double-strained
  linear chromosomes? Which is it?</span> </p><p><span m='1290000'>Is it double-stranded
  linear DNA or is it double-stranded circular DNA?</span> </p><p><span m='1298000'>Circular
  DNA. So, who votes for linear? Who votes for circular?</span> </p><p><span m='1306000'>Who's
  undecided? Ah, the undecided are very larger here. So,</span> <span m='1314000'>the
  answer is both. Some viruses have double-stranded</span> <span m='1321000'>linear
  DNA. Some viruses have double-stranded circular DNA.</span> </p><p><span m='1327000'>It's
  worse than that, though. Some viruses have single-stranded</span> <span m='1334000'>linear,
  circular DNA. Ha? How does that work?</span> </p><p><span m='1340000'>Some viruses
  actually infect the cell injecting DNA,</span> <span m='1346000'>and it's just single-stranded.
  As soon as it gets into the cell,</span> <span m='1352000'>however, it's replicated
  to make a double-stranded DNA which can then</span> <span m='1356000'>be transcribed,
  et cetera, et cetera,</span> <span m='1361000'>et cetera. But it travels around
  as a single-stranded piece of DNA.</span> </p><p><span m='1366000'>And it's actually
  weirder than that. Some viruses,</span> <span m='1370000'>viruses being very small
  can experiment with all</span> <span m='1375000'>sorts of things. Some viruses actually
  consist not of</span> <span m='1382000'>DNA at all but of RNA, single-stranded RNA.
  How does it do</span> <span m='1390000'>that? So, in other words, in the capsid
  there's a single</span> <span m='1398000'>strand of RNA. When it gets into the cell,</span>
  <span m='1406000'>what does it do? Sorry? It creates DNA.</span> </p><p><span m='1412000'>How
  does it create DNA? From the RNA. How's it going to do</span> <span m='1416000'>that?
  Well, how is it going to turn itself into DNA?</span> </p><p><span m='1421000'>It
  needs an enzyme to do that? Reverse transcriptase. You would</span> <span m='1426000'>like
  to reverse the transcription process, and you would like to name</span> <span m='1430000'>that
  reverse transcriptase. And where are you going to get this</span> <span m='1435000'>reverse
  transcriptase from? Laying around. Laying around where?</span> </p><p><span m='1440000'>I
  mean the cell is just sitting there with reverse transcriptase</span> <span m='1444000'>waiting
  to obligingly reverse transcribe this virus?</span> </p><p><span m='1448000'>Your
  RNA. So make it how? With ribosomes. So, in other words,</span> <span m='1452000'>if
  I'm an RNA, why don't I encode the sequence for</span> <span m='1457000'>reverse
  transcriptase and actually translate myself.</span> </p><p><span m='1461000'>So,
  if you were really cleaver, you might decide to put in the</span> <span m='1465000'>genetic
  code for reverse transcriptase.</span> </p><p><span m='1470000'>And when that message
  gets into the cell, it will first act as an mRNA,</span> <span m='1476000'>a messenger
  RNA, translate, make, here's the reverse transcriptase</span> <span m='1482000'>enzyme,
  which is then going to go, and it's going to reverse transcribe</span> <span m='1488000'>this
  thing into, say, DNA. So, wow.</span> </p><p><span m='1494000'>Now, that's a good
  one. This is a plus strand virus.</span> </p><p><span m='1500000'>It encodes its
  own reverse transcriptase in its instructions.</span> </p><p><span m='1505000'>There
  actually are minus-strand viruses that don't,</span> <span m='1510000'>but what
  they do is instead in their own code, in their own package bring</span> <span m='1515000'>a
  longer reverse transcriptase. So, either you can encode your own</span> <span m='1520000'>reverse
  transcriptase or in the package you can include your own</span> <span m='1525000'>reverse
  transcriptase. Do you know any viruses?</span> </p><p><span m='1530000'>And then
  the reverse transcriptase is then used to transcribe the DNA,</span> <span m='1536000'>the
  RNA into DNA, and eventually into a double-stranded DNA which,</span> <span m='1543000'>in
  some of the viruses, can then be slammed into and inserted into your</span> <span
  m='1549000'>own chromosomes. So, a DNA copy of the virus can be</span> <span m='1556000'>installed
  into your own chromosomes, which is somewhat insidious.</span> </p><p><span m='1563000'>What
  viruses do you know that do this? HIV. More generally</span> <span m='1568000'>retroviruses
  are the class of these viruses that can,</span> <span m='1573000'>in fact, run this
  replication process from RNA to DNA and install</span> <span m='1579000'>DNA copies
  of them in your genome. And how do you then get the DNA</span> <span m='1584000'>copy
  out of your genome? You don't.</span> </p><p><span m='1590000'>It doesn't come out.
  Retroviral insertions don't come</span> <span m='1593000'>out. That's one of the
  issues in dealing with AIDS is once this DNA</span> <span m='1597000'>copy is in
  a cell it's not coming out. We have no way to remove it.</span> </p><p><span m='1600000'>We
  have to make sure that the virus is shut down by other mechanisms</span> <span m='1604000'>that
  might inhibit its products, et cetera, but once its stuck a DNA</span> <span m='1607000'>copy
  into your chromosomes, you know, there's no way of getting</span> <span m='1611000'>it
  out. So, if we had to try to inhibit the</span> <span m='1615000'>action of the
  AIDS virus, we might wish to make inhibitors of</span> <span m='1620000'>this aspect
  of replication, inhibitors or reverse transcription.</span> </p><p><span m='1625000'>And,
  of course, as probably many of you know, some of the important AIDS</span> <span
  m='1630000'>drugs are reverse transcriptase inhibitors, very important to</span>
  <span m='1635000'>limiting the replication of the AIDS virus. And there are many
  other</span> <span m='1640000'>kinds of weirdnesses. Viruses pretty much explore,</span>
  <span m='1645000'>everything you possibly can do, viruses come up with ways to do.</span>
  </p><p><span m='1650000'>Let's take now the process of transcription.</span> </p><p><span
  m='1655000'>We have replication up there. Let's look at transcription. And</span>
  <span m='1660000'>this time let's start with prokaryotes. For the simple aspect</span>
  <span m='1665000'>of transcribing genes, the prokaryotic genome looks just</span>
  <span m='1670000'>like the simple model I gave you. There is some kind of a promoter</span>
  <span m='1675000'>that tells RNA polymerase to come sit down here.</span> </p><p><span
  m='1680000'>RNA polymerase hops on, RNA polymerase begins to copy in RNA,</span>
  <span m='1687000'>and eventually it hits the signal that says to terminate transcription.</span>
  </p><p><span m='1695000'>OK. This is not a stop codon which is about translation.</span>
  </p><p><span m='1702000'>This is a termination of transcription.</span> </p><p><span
  m='1710000'>And this RNA then goes off. A perfectly happy thing, a</span> <span
  m='1715000'>messenger RNA, mRNA. So, there's nothing weird</span> <span m='1720000'>about
  proks compared to the simple description that we gave before.</span> </p><p><span
  m='1726000'>But eukaryotes are different. There are some funny things that</span>
  <span m='1731000'>happen in the eukaryote. Well, first off it starts the same.</span>
  </p><p><span m='1737000'>There's a promoter. RNA polymerase sits down there,</span>
  <span m='1743000'>it starts transcribing, it makes an mRNA, it hits the transcriptional</span>
  <span m='1749000'>termination signal, it stops, and then this RNA gets</span> <span
  m='1756000'>processed in interesting ways. The first thing that happens is</span>
  <span m='1762000'>three modifications happen. The first one is at the five prime</span>
  <span m='1769000'>end, remember five prime to three prime, a funny modification
  is put</span> <span m='1775000'>on. It's a, if the message, say, were A-U-C-U-G-G-C
  et cetera,</span> <span m='1781000'>a G triphosphate is put on backwards. It's actually
  a methyl G</span> <span m='1787000'>triphosphate is put on backwards, so going in
  the other direction.</span> </p><p><span m='1793000'>You have the triphosphate bond
  there, a methyl G.</span> </p><p><span m='1800000'>And the only thing that you share
  care about that,</span> <span m='1804000'>I don't care if you know the structure,
  is that there's a funny</span> <span m='1809000'>cap. This thing is called a cap
  that is put on this message.</span> </p><p><span m='1813000'>And that cap is very
  important to signaling to the cell this is a</span> <span m='1818000'>messenger
  RNA to be dealt with in a certain way, to get the ribosome to</span> <span m='1823000'>hop
  on, to get this thing processed properly, et cetera.</span> </p><p><span m='1827000'>At
  the other end of the message a long string of As is added</span> <span m='1832000'>to
  messenger RNAs. This long string of As is called,</span> <span m='1837000'>very
  sensibly, a poly A tail. The poly A tail is added to the</span> <span m='1841000'>messenger
  RNA, and very often,</span> <span m='1846000'>I mean it's, if you wanted to purify
  messenger RNAs from your own human</span> <span m='1851000'>cells, you can actually
  use poly T as a reagent because it turns out,</span> <span m='1855000'>because messenger
  RNAs have a poly A tail, they'll bind to</span> <span m='1860000'>and stick to poly
  T. So, people actually purify messenger</span> <span m='1864000'>RNAs by binding
  them to poly T and they get the poly A tail.</span> </p><p><span m='1868000'>But
  it is broadly believed that the reason for this poly A tail is not</span> <span
  m='1871000'>to make things convenient for molecular biologists to purify</span>
  <span m='1875000'>messages. To the contrary, it is an important function for the</span>
  <span m='1878000'>cell. And it turns out that this is important in regulating the</span>
  <span m='1882000'>stability of messages. If, in fact, you don't have a poly</span>
  <span m='1885000'>A tail, if you contrive to make the same message without the poly
  A tail,</span> <span m='1889000'>the message will be degraded rather rapidly.</span>
  </p><p><span m='1893000'>And the lengths of the poly A tails control aspects of
  the degradation,</span> <span m='1896000'>et cetera. So, in a complex eukaryotic
  cell,</span> <span m='1899000'>already it's how to attach a little signal at the
  front,</span> <span m='1903000'>some signals at the back that says process me in
  a certain way,</span> <span m='1906000'>et cetera, don't degrade me yet. You could
  even imagine that this</span> <span m='1909000'>poly A tail could serve as a little
  bit of a clock for how long that</span> <span m='1913000'>message sticks around.
  It's not quite that simple but</span> <span m='1916000'>there are ways to do it.
  But all of these pale in comparison</span> <span m='1919000'>to the third way in
  which eukaryotic messages differ from prokaryotic</span> <span m='1923000'>messages.
  These small modifications are,</span> <span m='1930000'>as I say, small. The most
  striking way in which they differ is that</span> <span m='1941000'>only a small
  portion often of the gene, here's my gene,</span> <span m='1953000'>matters for
  the protein that is made. So, my mRNA gets made.</span> </p><p><span m='1964000'>It
  includes the whole long sequence. And then the cell comes along and</span> <span
  m='1974000'>splices this message together. So, this is the immature RNA.</span>
  </p><p><span m='1985000'>It is processed by clipping out this, clipping out this,</span>
  <span m='1993000'>clipping out this. And what you get is a splice where the mature
  message</span> <span m='2002000'>throws this stuff out, splices between here and
  here,</span> <span m='2010000'>splices here, splices here, splices here, and you
  get a</span> <span m='2019000'>much shorter mRNA. And this is a mature mRNA.</span>
  </p><p><span m='2027000'>This splicing is a remarkable phenomenon. In fact,</span>
  <span m='2033000'>it was discovered by Phil Sharp here, for which he won a Nobel
  prize.</span> </p><p><span m='2040000'>This splicing is a very complex operation.
  First off,</span> <span m='2044000'>how does, well, actually, what accomplishes
  splicing?</span> </p><p><span m='2048000'>It should be splicase, right? But it turns
  out it's not a single</span> <span m='2052000'>enzyme. It's a big body of stuff.
  So, instead it's the splicosome, OK.</span> </p><p><span m='2057000'>Everything
  is either ase or some or something like that.</span> </p><p><span m='2061000'>So,
  it turns out it's the splicosome that does that.</span> </p><p><span m='2065000'>It's
  just wonderful how all those names work out. The splicosome.</span> </p><p><span
  m='2070000'>The splicosome comes along and splices it. How does the splicosome</span>
  <span m='2076000'>know how to do this? Well, there are kind of codes.</span> </p><p><span
  m='2082000'>It turns out that there are some information encoded along in these</span>
  <span m='2088000'>messages. It turns out that there is, you know, slight biases.</span>
  </p><p><span m='2094000'>Typically the sequence just after where the slice starts
  here is a GU</span> <span m='2100000'>and the sequence here is an AG, but that's
  obviously not enough</span> <span m='2106000'>information, right? It's not enough
  bases of information</span> <span m='2110000'>to get this right. And so there's
  a little more</span> <span m='2114000'>preferences for what bases use, but the truth
  is we don't fully know.</span> </p><p><span m='2118000'>Our best picture right now
  involves some cellular factors help</span> <span m='2121000'>recognizing the parts
  that are supposed to stay in some sequences</span> <span m='2125000'>here. But the
  truth is we don't have the simple codes.</span> </p><p><span m='2129000'>Because
  if we had the simple codes, we'd be able to take a long stretch</span> <span m='2133000'>of
  DNA and figure out exactly where the splices go based on just</span> <span m='2137000'>computer
  analysis. And we can't do that so well.</span> </p><p><span m='2142000'>These bits
  that stay in are called exons. The bits that go out are</span> <span m='2146000'>called
  introns. This is the source of extraordinary</span> <span m='2151000'>confusion
  for students because you might think that the bits that are</span> <span m='2155000'>excised
  are the exons, but they're not.</span> </p><p><span m='2160000'>The bits that stay
  in are the exons. Why are they called exons if they</span> <span m='2164000'>stay
  in and ex is a prefix meaning out? Well, because the introns are</span> <span m='2168000'>named
  because they're intervening sequences. Once the introns,</span> <span m='2172000'>the
  intervening sequences were named as intervening sequences or introns,</span> <span
  m='2177000'>you were stuck then having to name the things that stay in as exons.</span>
  </p><p><span m='2181000'>This was all done by a Harvard professor, don't blame me.</span>
  </p><p><span m='2185000'>In any case, a good friend Harvard professor.</span> </p><p><span
  m='2190000'>But, nonetheless, I'm not sure that this was the best</span> <span m='2197000'>way
  to name them. But you're stuck with it.</span> </p><p><span m='2204000'>So, for
  a typical human gene, typical human gene, the length of</span> <span m='2212000'>the
  gene itself might be 30, 00 bases. But the mature RNA,</span> <span m='2219000'>the
  mature mRNA might be one and a half, 1,500 bases.</span> </p><p><span m='2227000'>That's
  remarkable. Out of 30,000 letters in the</span> <span m='2231000'>initial transcript
  that is made, the genes start, the promoter, and</span> <span m='2235000'>the transcription
  will stop 30, 00 bases away. The cell goes</span> <span m='2239000'>through the
  trouble of making an RNA of 30,000 bases long,</span> <span m='2244000'>and then
  it trims it down by throwing out 28,</span> <span m='2248000'>00 of the bases, keeping
  only 1, 00 bases at the end.</span> </p><p><span m='2253000'>Now, this may seem
  profligate but it ain't nothing compared to some</span> <span m='2257000'>extreme
  cases. The clotting factor gene,</span> <span m='2262000'>the factor 8 gene, the
  gene that has mutated in individuals with</span> <span m='2267000'>hemophilia, that
  gene is 200, 00 bases long, and it gets spliced</span> <span m='2272000'>down to
  a mere 10, 00 bases. 190,000 bases are thrown</span> <span m='2277000'>away. But
  that's nothing compared to</span> <span m='2282000'>Duchene muscular dystrophy.
  The Duchene muscular dystrophy is</span> <span m='2288000'>the all time winner.
  That gene makes an immature initial</span> <span m='2293000'>RNA of 2 million bases.
  RNA polymerase hops on at the</span> <span m='2299000'>promoter and it gets off
  at the end of the Boston Marathon on here 2</span> <span m='2304000'>million bases
  later having made an RNA of 2 million bases long.</span> </p><p><span m='2310000'>Calculate
  the speed of RNA polymerase and you'll find out that</span> <span m='2316000'>it's
  at it for hours. It hops on and it stays on for</span> <span m='2322000'>hours until
  it gets to the other end. And then for all its troubles this</span> <span m='2328000'>gene
  is spliced down to 16, 00 bases in the mature message.</span> </p><p><span m='2334000'>Yup?
  How would it increase the chance of mutations? Yup.</span> </p><p><span m='2340000'>So,
  splicing mutations could be a problem. Some diseases could arise</span> <span m='2345000'>from
  errors in splicing. Do you think that happens?</span> </p><p><span m='2350000'>Sure
  does. There could be mutations that create,</span> <span m='2355000'>that change
  a splicing, or mutations that create a new</span> <span m='2360000'>splicing, and
  all of that could screw up the gene.</span> </p><p><span m='2365000'>Why do this?
  What in the world is going on?</span> </p><p><span m='2370000'>Just think about
  the energetic cost. I mean count up the ATPs involved</span> <span m='2375000'>in
  synthesizing a nucleotide, and then the ATPs involved in adding</span> <span m='2380000'>nucleotides
  up. You know, think about this totally</span> <span m='2385000'>wasted energy. What
  is the point?</span> </p><p><span m='2390000'>I might be able to encode multiple
  proteins with the same gene.</span> </p><p><span m='2395000'>How would I do that?
  Ooh, wouldn't that be cleaver?</span> </p><p><span m='2400000'>I might be able to
  take a single gene and make a mix and match</span> <span m='2405000'>product. It
  might be, do you mean like one type of cell</span> <span m='2410000'>might splice
  up that message one way to produce a certain protein,</span> <span m='2415000'>but
  a different cell type might splice the same gene another way to</span> <span m='2420000'>produce
  a different protein? Ooh. So, you're proposing, if I</span> <span m='2425000'>understand
  you correctly, alternative splicing.</span> </p><p><span m='2430000'>Alternative
  splicing could create multiple proteins,</span> <span m='2434000'>multiple distinct
  proteins. It might be, for example, that you</span> <span m='2438000'>might make
  one protein that has a cytoplasmic tail and another protein</span> <span m='2442000'>that
  doesn't have cytosplasmic tail or a different tail or,</span> <span m='2446000'>or,
  this is true. This actually happens. It's very cleaver.</span> </p><p><span m='2450000'>Anything
  that can happen does happen somewhere,</span> <span m='2454000'>and it's fairly
  regularly used. A typical gene in the human being</span> <span m='2458000'>has at
  least two alternative splice forms, on average.</span> </p><p><span m='2462000'>Most,
  many don't, but there are some that have large</span> <span m='2465000'>numbers.
  The most extreme is there's a gene known,</span> <span m='2468000'>drosophila, that
  has more than a thousand alternative splice forms.</span> </p><p><span m='2471000'>How
  does it know, how does the cell know whether to splice it one way in</span> <span
  m='2475000'>the liver and one way in a heart or something? We don't fully know but</span>
  <span m='2478000'>there's machinery and signals people are trying to work out for
  that.</span> </p><p><span m='2481000'>Now, I don't want to confuse you too much
  about it.</span> </p><p><span m='2485000'>You know, mostly, when we give you a gene,
  you should think about it</span> <span m='2488000'>spliced out introns, exons. But
  the truth is it is more</span> <span m='2491000'>complicated than that. There can
  be alternative splicing</span> <span m='2495000'>that allows genes to be used in
  multiple ways.</span> </p><p><span m='2499000'>Sometimes they don't make multiple
  proteins. They may splice into</span> <span m='2503000'>portions of the mRNA that
  are not translated, but,</span> <span m='2506000'>there is that, but, boy, it's
  a huge amount of overhead</span> <span m='2510000'>here just to do that. Is it justified?
  Yes?</span> </p><p><span m='2514000'>That is by computer if I just gave you the
  sequence? Not quite.</span> </p><p><span m='2518000'>Almost. Maybe. Sort of. It
  turns out that the</span> <span m='2521000'>computer programs for automatically
  recognizing the matter of the human</span> <span m='2524000'>genome are sort of,
  they're mediocre, not very good.</span> </p><p><span m='2527000'>We have some idea
  of the signals, and various people have trying to</span> <span m='2529000'>write
  better and better algorithms for doing that,</span> <span m='2532000'>but the cell
  knows what it's doing and we don't fully know,</span> <span m='2535000'>as evidenced
  by the fact that we can't write a clean computer program</span> <span m='2538000'>to
  do it yet. We need to get information from the cell or from</span> <span m='2541000'>evolution
  or various other things like that, and that's the ultimate</span> <span m='2544000'>test.
  If we knew what we were talking about we'd just be able to</span> <span m='2547000'>write
  a computer program and splice it out.</span> </p><p><span m='2550000'>And we don't.
  There's another reason why people think these big</span> <span m='2554000'>introns
  and exons, these big introns are helpful,</span> <span m='2558000'>and that is an
  evolutionary reason. The evolutionary reason is a little</span> <span m='2562000'>bit
  harder to follow, but let me try it on you.</span> </p><p><span m='2567000'>Suppose
  a random event happens and a chromosome breaks,</span> <span m='2571000'>that happens,
  and suppose a random breakage sticks one part of a</span> <span m='2575000'>chromosome
  to some other part of the chromosome.</span> </p><p><span m='2580000'>If it lands
  smack dab in the middle of the coding sequence of a gene</span> <span m='2584000'>that's
  bad new. But it turns out that if it lands</span> <span m='2589000'>in the introns
  of two different genes and sticks them together it</span> <span m='2593000'>could
  make a new gene that would still work. By having a random</span> <span m='2598000'>break
  between two genes in their introns and slamming them together,</span> <span m='2603000'>you
  could make a gene that had a bunch of exons from one gene and a</span> <span m='2607000'>bunch
  of exons from another gene. And this intervening sequence in the</span> <span m='2612000'>middle
  and it would get spliced up. Evolution might like that because</span> <span m='2618000'>it
  would be a very easy way for evolution to build new genes that</span> <span m='2624000'>had
  a portion of one protein and a portion of another protein.</span> </p><p><span m='2629000'>This
  kind of mix and match domain swapping could be very useful.</span> </p><p><span
  m='2635000'>And when we look across genomes, we see lots and lots of examples of</span>
  <span m='2641000'>genes that have a similar first half but different second halves.</span>
  </p><p><span m='2647000'>Or have some portion in the middle, a domain that we recognize,
  that we</span> <span m='2650000'>see in multiple proteins. And so, in fact, an argument
  for</span> <span m='2653000'>why we have all of this intronic DNA, one that's impossible
  to prove but</span> <span m='2657000'>is an argument is that from an evolution point
  of view,</span> <span m='2660000'>this allows a great deal of evolutionary innovation.</span>
  </p><p><span m='2664000'>You have to be careful that you say those organisms that
  have this extra</span> <span m='2667000'>space are able to mix and match and create
  more new kinds of combination</span> <span m='2671000'>proteins, et cetera, et cetera,
  and therefore survived</span> <span m='2674000'>better, et cetera, et cetera, et
  cetera.</span> </p><p><span m='2678000'>Why don't bacteria have this? Sorry? They're
  not as complicated.</span> </p><p><span m='2681000'>That's one though is we can
  take a sort of condescending attitude to</span> <span m='2685000'>these bacteria.
  They're not very,</span> <span m='2688000'>they're just not so complicated. There's
  another point of view which</span> <span m='2692000'>is bacteria are far more sophisticated
  than we are because</span> <span m='2696000'>they're under incredibly rigorous evolutionary
  selection.</span> </p><p><span m='2700000'>You might argue that if I'm a bacteria,
  can I really afford all</span> <span m='2704000'>this extra DNA? Now, the metabolic
  cost of all that</span> <span m='2708000'>extra DNA is huge to a bacteria which
  competes on replication.</span> </p><p><span m='2712000'>It's got to divide every
  20 minutes, and trying to put in all these extra</span> <span m='2716000'>bases
  would be very news. So, you might imagine, just to be,</span> <span m='2720000'>you
  know, stand things on its head, that early life all had introns and</span> <span
  m='2724000'>bacteria, in the process of competing to be more and more</span> <span
  m='2728000'>efficient go rid of their introns. There's actually a large camp of</span>
  <span m='2731000'>people who think it went that way, that early life evolved with
  introns,</span> <span m='2734000'>and then bacteria, in the pressure to compete,</span>
  <span m='2737000'>got rid of them. And there's some evidence to support that.</span>
  </p><p><span m='2741000'>Bacteria don't have introns. Small eukaryotes like yeast
  that</span> <span m='2744000'>sort of do compete on replication have some introns,</span>
  <span m='2747000'>but a small number. There are only about 250 introns in</span>
  <span m='2750000'>yeast. Only about 5% of the genes have an intron and they're small.</span>
  </p><p><span m='2753000'>Bigger eukaryotes have bigger introns. And the bigger you
  get,</span> <span m='2757000'>often on average the bigger the genome sizes are the
  more</span> <span m='2760000'>you can tolerate it. And so I actually think,</span>
  <span m='2763000'>I actually probably favor this notion that introns were the</span>
  <span m='2767000'>original state and they've been gotten rid of.</span> </p><p><span
  m='2770000'>And the more pressure you're under to replicate rapidly the less you</span>
  <span m='2774000'>can tolerate this interesting and complicated innovation.</span>
  </p><p><span m='2777000'>Anyway, that's another way that things differ.</span> </p><p><span
  m='2781000'>And then, finally, viruses can do it either way.</span> </p><p><span
  m='2784000'>Viruses, depending on whether they are prokaryotic viruses or</span>
  <span m='2788000'>eukaryotic viruses, are able to replicate,</span> <span m='2791000'>are
  able to either do or don't have splicing.</span> </p><p><span m='2795000'>Last topic.
  Translation. Here eukaryotes are relatively</span> <span m='2801000'>simple. You
  get a message, you get a gene, you get an mRNA.</span> </p><p><span m='2808000'>The
  mRNA goes to the ribosome. Here's a ribosome.</span> </p><p><span m='2814000'>The
  ribosome goes to the mRNA, actually, and it starts turning out</span> <span m='2821000'>one
  protein as it chugs along. Prokaryotes differ in an interesting</span> <span m='2829000'>way.
  I get a promoter here that is transcribed into my mRNA,</span> <span m='2838000'>but
  it turns out that the mRNA can encode multiple independent proteins,</span> <span
  m='2847000'>protein one, protein two, protein three on the same mRNA.</span> </p><p><span
  m='2856000'>And a ribosome will hop on here and synthesize this one.</span> </p><p><span
  m='2860000'>A ribosome will hop on here and synthesize this one,</span> <span m='2864000'>and
  a ribosome will hop on here and synthesize that one.</span> </p><p><span m='2868000'>And
  you have what is called a polycistronic message.</span> </p><p><span m='2872000'>Poly,
  many. Cystronic, cystrons were an old name for coding</span> <span m='2876000'>regions
  of genes here. Polycystronic messages.</span> </p><p><span m='2880000'>Why would
  you want to do that, have a single mRNA that encodes</span> <span m='2883000'>multiple
  distinct proteins, each starting with its own ribosome</span> <span m='2886000'>start
  site there? Efficiency. Maybe,</span> <span m='2889000'>in fact, these would be,
  how about, oh, this would be cleaver,</span> <span m='2892000'>make them multiple
  steps in a biochemical pathway?</span> </p><p><span m='2895000'>Have them coded
  on a single messenger so then you'd only have to</span> <span m='2898000'>worry
  about regulating that once. If you have the regulatory</span> <span m='2901000'>machinery
  to turn on, you'll make all the enzymes for the</span> <span m='2904000'>pathway.
  And that's exactly what bacteria do. They tend to put all</span> <span m='2907000'>the
  enzymes for a pathway on a single message so when they want to</span> <span m='2910000'>call
  up, let's digest hexose this morning, they have a whole thing</span> <span m='2913000'>that
  will let them be able to do that, poly-cystronic.</span> </p><p><span m='2917000'>That's
  because they're small genomes. They're pressed for space.</span> </p><p><span m='2921000'>And,
  because of that, they have to slam a lot into a single unit.</span> </p><p><span
  m='2926000'>And this single unit that has multiple genes encoded in a single</span>
  <span m='2930000'>message is called an operon, and we'll talk more about that.</span>
  </p><p><span m='2935000'>Last of all viruses. Viruses. Viruses have very little
  room.</span> </p><p><span m='2940000'>Their genomes can be tiny. A typical virus
  might have a genome</span> <span m='2945000'>of 5,000 bases to 10, 00 bases to,
  in some cases,</span> <span m='2950000'>200,000 bases, but it hasn't got a lot of
  room. It wants to pack a lot</span> <span m='2955000'>of protein coating information
  in. And some viruses have come up with</span> <span m='2961000'>the most extraordinary
  way of doing that. Some viruses have gone to the</span> <span m='2966000'>extreme
  of having RNAs that get made from them that have a sequence --</span> <span m='2972000'>I'm
  just going to pick up in the middle of the sequence here.</span> </p><p><span m='2979000'>A-C-U-A-C-U-A-C-U-A-C-U.
  You might decide to read the sequence like</span> <span m='2986000'>this, that those
  are the codons, and you'd get a certain protein.</span> </p><p><span m='2993000'>But
  I might also decide to read that sequence C-U-A-C-U-A-C-U-A.</span> </p><p><span
  m='3000000'>And, of course, I'm giving this in a repeating form</span> <span m='3004000'>because
  it's easy to note. I could give you any sequence and I</span> <span m='3008000'>could
  read it in this reading frame, I could read it in this reading</span> <span m='3012000'>frame,
  or I could read it as U-A-C-U-A-C-U-A-C.</span> </p><p><span m='3017000'>In other
  words, there are three reading frames that,</span> <span m='3021000'>in principle,
  you could translate a protein from. In a typical</span> <span m='3025000'>prokaryotic
  gene or eukaryotic gene only one of those is used.</span> </p><p><span m='3030000'>You
  start at the first AUG and that sets the reading frame.</span> </p><p><span m='3034000'>But
  some viruses are so pressed for space and are so cleaver and are so</span> <span
  m='3039000'>efficient that they make messages that have tricks that they actually</span>
  <span m='3043000'>use two or, in some cases, all three reading frames, which is</span>
  <span m='3048000'>an extraordinary packing of information density into a simple</span>
  <span m='3053000'>message. So, the basic point. We have a simple model. DNA is</span>
  <span m='3057000'>replicated. Transcribed into RNA.</span> </p><p><span m='3062000'>Translated
  into protein. But there are a lot of important</span> <span m='3066000'>variations
  between eukaryotes, prokaryotes and viruses. And</span> <span m='3071000'>understanding
  them can be useful for treating cancer,</span> <span m='3075000'>for treating AIDS,
  and for treating viral and bacterial</span> <span m='3080000'>infections. Next time.</span>
  </p>
type: course
uid: ae2750f05594c53c59122f7f9840031e

---
None