---
about_this_resource_text: <p><strong>Topics covered:</strong> Human Genetics</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec09.jpg
  parent_uid: e1ac03022862e5895b1eb80208acd552
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-9-human-genetics/lec09.jpg
  title: lec09.jpg
  type: null
  uid: bd1a5780edfa9caacff684e0a2ae6489
- id: Video-YouTube-Stream
  media_location: PVv4ST8NZaA
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Video-YouTube-Stream
  type: Video
  uid: e51de0fa07ac4b5a0e6b2e2b979a2df3
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/PVv4ST8NZaA/default.jpg
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 5ecc79e64b6188f19433abf33ccf634b
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Video-iTunes U-MP4
  type: Video
  uid: 46869ecfea1673b1b8980c0f360ced5a
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec9-27sep2004-220k.mp4
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Video-Internet Archive-MP4
  type: Video
  uid: b8711b0000d7ab9662b194145029cf28
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 73e5c8e0b0608e9cd852e9a2f9993875
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-27sep2004-1000.mp3
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Video-Internet Archive-MP3
  type: Video
  uid: 6435ab794b4f0e5c58e788de578ae26a
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 9256878da1e0b7138058616a48979a04
- id: Thumbnail-OCW-JPG
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 646ecef601eb505727dee7a99a0a37d2
- id: 3Play-3PlayYouTubeid-MP4
  media_location: PVv4ST8NZaA
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 7094d7e9951921c3e04af45d249e23ff
- id: PVv4ST8NZaA.srt
  parent_uid: e1ac03022862e5895b1eb80208acd552
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-9-human-genetics/PVv4ST8NZaA.srt
  title: 3play caption file
  type: null
  uid: 8b8887fce1893aabda0eb7d8c45f61c3
- id: PVv4ST8NZaA.pdf
  parent_uid: e1ac03022862e5895b1eb80208acd552
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-9-human-genetics/PVv4ST8NZaA.pdf
  title: 3play pdf file
  type: null
  uid: 4a58e575fb577298400528ce7b0a4037
- id: Caption-3Play YouTube id-SRT
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: 68d75914b2ed9710b95730d480e8f3aa
- id: Transcript-3Play YouTube id-PDF
  parent_uid: e1ac03022862e5895b1eb80208acd552
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: aae2b4d31ad23e7c9d1f48065b083d98
inline_embed_id: 39658354lecture9:humangenetics523055
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-9-human-genetics
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-9-human-genetics
template_type: Tabbed
title: 'Lecture 9: Human Genetics'
transcript: '<p><span m=''1000''>I want to go back a second to the end of last time
  because in the</span> <span m=''3000''>closing moments there, we, or at least I,
  got a little bit</span> <span m=''6000''>lost, and where the plusses and</span>
  <span m=''9000''>minuses were at a certain table.</span> </p><p><span m=''12000''>And,
  I want to go back and make sure we''ve got that straight.</span> </p><p><span m=''18000''>We
  were talking about a situation where we were trying to use genetics,</span> <span
  m=''22000''>and the phenotypes that might be observed in mutants to try to</span>
  <span m=''27000''>understand the biochemical pathway because we''re beginning to
  try to</span> <span m=''32000''>unite the geneticist''s point of view who looks
  only at mutants,</span> <span m=''36000''>and the biochemist''s point of view who
  looks at pathways and proteins.</span> </p><p><span m=''40000''>And, I had hypothesized
  that there was some biochemists who had thought</span> <span m=''44000''>up a possible
  pathway for the synthesis of arginine that involved</span> <span m=''49000''>some
  precursor, alpha, beta, gamma,</span> <span m=''53000''>where alpha is turned into
  beta; beta is turned into gamma; and gamma</span> <span m=''57000''>is used to turn
  into arginine. And, hypothetically,</span> <span m=''62000''>there would be some
  enzymes: enzyme A that converts alpha,</span> <span m=''66000''>enzyme B that converts
  beta, and enzyme C that converts gamma.</span> </p><p><span m=''70000''>And, we
  were just thinking about, what would the phenotypes look like</span> <span m=''74000''>of
  different arginine auxotrophs that had blocks at different stages</span> <span m=''78000''>in
  the pathway. If I had an arginine auxotroph that</span> <span m=''82000''>had a
  block here because let''s say a mutation in a gene affecting this</span> <span m=''86000''>enzyme,
  or at a block here at a mutation affecting, say,</span> <span m=''90000''>the gene
  that encodes enzyme C, how would I be able to tell very</span> <span m=''94000''>simply
  that they were in different genes? Last time,</span> <span m=''98000''>we found
  that we could tell they were in different genes by doing a</span> <span m=''102000''>cross
  between a mutant that had the first mutation,</span> <span m=''106000''>and a mutant
  that had the second mutation, and looking at the double</span> <span m=''110000''>heterozygote,
  right? And, if in the double heterozygote</span> <span m=''114000''>you had a wild
  type or a normal phenotype, then they had to be in</span> <span m=''118000''>different
  genes, OK? Remember that?</span> </p><p><span m=''123000''>That was called a test
  of complementation.</span> </p><p><span m=''126000''>That was how we were able to
  sort out which mutations were in the same</span> <span m=''129000''>gene, and which
  mutations were in different genes.</span> </p><p><span m=''132000''>Now we can go
  a step further. When we''ve established that they''re</span> <span m=''135000''>in
  different genes, we can try to begin to think,</span> <span m=''138000''>how do
  these genes relate to a biochemical pathway?</span> </p><p><span m=''141000''>I
  wanted to begin to introduce, because it''ll be relevant for today,</span> <span
  m=''144000''>this notion: so, suppose I had a mutation that affected enzyme A so</span>
  <span m=''147000''>that this enzymatic step couldn''t be carried out.</span> </p><p><span
  m=''151000''>Such a mutant, when I just try to grow it on</span> <span m=''155000''>minimal
  medium won''t be able to grow. If I give it the substrate alpha,</span> <span m=''160000''>it
  doesn''t do it any good because it hasn''t got the enzyme to convert</span> <span
  m=''165000''>alpha. So, given alpha, it won''t grow. But if I give it</span> <span
  m=''170000''>beta, what will happen? It can grow because I''ve bypassed</span> <span
  m=''175000''>the defect. What about if I give it gamma? Arginine?</span> </p><p><span
  m=''180000''>Now, if instead the mutation were affecting enzymatic step here,</span>
  <span m=''198000''>then if I give it on minimal or medium</span> <span m=''216000''>but
  it can grow on gamma. What about this last line?</span> </p><p><span m=''220000''>If
  I have a mutation and the last enzymatic step,</span> <span m=''224000''>minimal
  medium can''t grow with alpha, can''t grow with beta,</span> <span m=''229000''>can''t
  even grow with gamma. But, it can grow with arginine</span> <span m=''233000''>because
  I''ve bypassed that step. So, I get a different phenotype,</span> <span m=''237000''>the
  inability to grow even on gamma,</span> <span m=''242000''>but I can grow on arginine.
  Now, here, if I put together those</span> <span m=''248000''>mutants and make a
  double mutant, a double homozygote, let''s say,</span> <span m=''253000''>that''s
  defective in both A and B, which will it look like? Will it be</span> <span m=''259000''>able
  to grow on minimal medium? Will it be able to grow on alpha?</span> </p><p><span
  m=''264000''>Will it be able to grow on beta?</span> </p><p><span m=''270000''>Will
  it be able to grow on gamma and arginine? What about if I have a</span> <span m=''276000''>double
  mutant in B and C, minus, minus, minus, minus,</span> <span m=''283000''>plus? So
  this looks the same as that. This looks the same as that.</span> </p><p><span m=''289000''>And
  so, by looking at different mutant combinations,</span> <span m=''296000''>I can
  see that the phenotype of B here is what occurs in the double</span> <span m=''302000''>mutant.
  So, this phenotype is epistatic to this phenotype.</span> </p><p><span m=''308000''>Epistatic
  means stands upon, OK? So, phenotypes, just like</span> <span m=''313000''>phenotypes
  can be recessive or dominant, you can also speak about</span> <span m=''319000''>them
  being epistatic. And epistatic means when you have</span> <span m=''324000''>both
  of two mutations together at the epistatic</span> <span m=''330000''>then one of
  them is epistatic to the other, perhaps.</span> </p><p><span m=''333000''>It will,
  in fact, be the one that is present.</span> </p><p><span m=''336000''>So, this is
  not so easy to do in many cases because if I take</span> <span m=''339000''>different
  kinds of mutation affecting wing development,</span> <span m=''343000''>and I put
  them together in the same fly, I may just get a very messed up</span> <span m=''346000''>wing,
  and it''s very hard to tell that the double mutant has a</span> <span m=''349000''>phenotype
  that looks like either of the two single mutants.</span> </p><p><span m=''353000''>But
  sometimes, if they fall very nicely in a pathway where this</span> <span m=''356000''>affects
  the first step, this affects the second step</span> <span m=''360000''>this affects
  the third step, this affects the fourth step,</span> <span m=''364000''>then the
  double mutant will look like one of those,</span> <span m=''368000''>OK? And, that
  way you can somehow order things in a biochemical</span> <span m=''372000''>pathway.
  Now, notice, this is all indirect,</span> <span m=''376000''>right? This is what
  geneticists did in the middle of the 20th century to</span> <span m=''380000''>try
  to figure out how to connect up mutants to biochemistry.</span> </p><p><span m=''384000''>Actually,
  that''s not true. It''s what geneticists still do</span> <span m=''388000''>today
  because you might think that Well, we don''t need to do this</span> <span m=''391000''>anymore,
  but in fact geneticists constantly are looking at mutants</span> <span m=''394000''>and
  making connections trying to say, what does this double combination</span> <span
  m=''398000''>look like? What does that double combination look like,</span> <span
  m=''401000''>and how does that tell us about the developmental pathway,</span> <span
  m=''404000''>which cell signals which cell? This turns out to be one of the</span>
  <span m=''408000''>most powerful ways to figure out what mutations do by saying
  the</span> <span m=''411000''>combination of two mutations looks like the same as
  one of them,</span> <span m=''414000''>allowing you to order the mutations in a
  pathway.</span> </p><p><span m=''418000''>And, there''s no general way to grind
  up a cell and order things in a</span> <span m=''422000''>pathway. Genetics is a
  very powerful tool for doing that.</span> </p><p><span m=''426000''>Now, there are
  some ways to grind up cells and order things,</span> <span m=''430000''>but you
  need both of these techniques to believe stuff.</span> </p><p><span m=''435000''>Anyway,
  I wanted to go over that, because it is an important concept,</span> <span m=''439000''>the
  concept of epistasis, the concept of relating mutations to</span> <span m=''443000''>steps
  and pathways, but what I mostly want to do today</span> <span m=''447000''>is go
  on now to talk about genetics not in organisms like yeast or fruit</span> <span
  m=''453000''>flies or even peas, but genetics in humans.</span> </p><p><span m=''459000''>So,
  what''s different about genetics in humans than genetics in yeast?</span> </p><p><span
  m=''466000''>You can''t choose who mates with whom. Well, you can.</span> </p><p><span
  m=''473000''>I mean, in the days of arranged marriages maybe you couldn''t,</span>
  <span m=''480000''>but you can choose who mates with whom, but only for yourself,</span>
  <span m=''484000''>right? What you can''t do is arrange other crosses in the human</span>
  <span m=''488000''>population as an experimentalist. Now, your own choice of mating,</span>
  <span m=''492000''>unfortunately or fortunately perhaps produces too few progeny
  to be</span> <span m=''496000''>statistically significant. As a parent of three,
  I think about</span> <span m=''500000''>what it would take to raise a statistically
  significant number of</span> <span m=''504000''>offspring to draw any conclusions,
  and I don''t think I could do that.</span> </p><p><span m=''508000''>So, you''re
  absolutely right. We can''t arrange the matings that</span> <span m=''512000''>we
  want in the human population. So, that''s the big difference.</span> </p><p><span
  m=''516000''>So, can we do genetics anyway? How do we do genetics even though</span>
  <span m=''520000''>we can''t arrange the matings the way we''d like to? Sorry?</span>
  </p><p><span m=''525000''>Well, family trees. We have to take the matings as we
  find them in</span> <span m=''529000''>the human population. You can talk to somebody
  who might</span> <span m=''534000''>have an interesting phenotype, I don''t know,
  attached earlobes,</span> <span m=''539000''>or very early heart disease, or some
  unusual color of eyes,</span> <span m=''542000''>and begin to collect a family history
  on that person.</span> </p><p><span m=''545000''>It''s a little bit of a dodgy thing
  because you might just be relying on</span> <span m=''548000''>that person''s recollection.
  So, if you were really industrious</span> <span m=''552000''>about this, you''d
  go check out each of their family members and test for</span> <span m=''555000''>yourself
  whether they have the phenotype. People who do serious</span> <span m=''558000''>human
  genetic studies often go and do that. They have to go confirm,</span> <span m=''561000''>either
  by getting hospital records or interviewing the other members of</span> <span m=''565000''>the
  family, etc. So, this is not as easy as plating</span> <span m=''568000''>out lots
  of yeasts on a Petri plate.</span> </p><p><span m=''579000''>And then you get pedigrees.
  And the pedigrees look like this.</span> </p><p><span m=''598000''>Here''s a pedigree.
  Tell me what you make of it.</span> </p><p><span m=''618000''>Now, symbols: squares
  are males, circles are females by convention,</span> <span m=''626000''>a colored
  in symbol means the phenotype that we''re</span> <span m=''632000''>interested in
  studying at the moment. So, in any given problem,</span> <span m=''636000''>somebody
  will tell you, well, we''re studying some interesting</span> <span m=''641000''>phenotype.
  You often have an index case or a proband,</span> <span m=''646000''>meaning the
  person who comes to clinical attention,</span> <span m=''650000''>and then you chase
  back in the pedigree and try to reconstruct.</span> </p><p><span m=''655000''>So,
  suppose I saw a pedigree like this.</span> </p><p><span m=''660000''>What conclusions
  could I draw? Sorry? Recessive, sex link trait;</span> <span m=''682000''>why sex
  link trait? So, let''s see if we can get your</span> <span m=''695000''>model up
  here. You think that this represents</span> <span m=''700000''>sex-linked inheritance.
  So, what would the genotype be of</span> <span m=''706000''>this male here? Mutant:
  I''ll use M to denote a</span> <span m=''711000''>mutant carried on the X chromosome,
  and a Y on the opposite chromosome.</span> </p><p><span m=''716000''>What''s the
  genotype of the female here?</span> </p><p><span m=''722000''>So, it''s plus over
  plus where I''ll use plus to denote the gene carried</span> <span m=''726000''>on
  the normal X chromosome. OK, and then what do you think</span> <span m=''730000''>happened
  over here? So, mutant over plus,</span> <span m=''734000''>you mate to this male
  who is plus over plus. Why is that male plus</span> <span m=''738000''>over plus?
  Oh, right, good point.</span> </p><p><span m=''742000''>It''s not plus over plus.
  It''s plus over Y. Why is that male</span> <span m=''746000''>plus over Y as opposed
  to mutant over Y?</span> </p><p><span m=''750000''>He''d have the mutant phenotype.
  So, he doesn''t have the mutant</span> <span m=''755000''>phenotype so he can infer
  he''s plus over Y. OK, and then what happens</span> <span m=''760000''>here? Mutant
  over Y; this is plus over Y. How did this person get</span> <span m=''766000''>plus
  over Y? They just the plus for mom, and the daughters,</span> <span m=''771000''>Y
  from dad, and a plus from mom. That''s cool. Now, what about the</span> <span m=''777000''>daughters
  there? They''re plus over plus,</span> <span m=''782000''>or M over plus? Is one,
  one, and one the other? Well,</span> <span m=''786000''>in textbooks it''s always
  plus over plus and M over plus,</span> <span m=''791000''>but in real life? We don''t
  know, right? So, this could be plus over</span> <span m=''796000''>plus, or M over
  plus, we don''t know, OK? Now,</span> <span m=''800000''>what about on this side
  of the pedigree here?</span> </p><p><span m=''805000''>What''s the genotype here?
  Plus over Y, OK.</span> </p><p><span m=''810000''>Why not mutant over Y? Because
  if they got the mutant,</span> <span m=''816000''>it would have to come from the,
  OK, so here, plus over plus, and then</span> <span m=''822000''>here, everybody
  is normal because there''s no mutant allele segregated.</span> </p><p><span m=''828000''>Yes?
  Yeah, couldn''t there just be recessive? I mean,</span> <span m=''834000''>it''s
  a nice story about the sex link</span> <span m=''840000''>but couldn''t it be recessive?
  So, walk me through it being</span> <span m=''847000''>recessive. M over plus, plus
  over plus. Wait, wait,</span> <span m=''855000''>wait, hang on. Could this be M
  over plus, and that person be affected?</span> </p><p><span m=''862000''>It''s got
  to be M over M, right so mutants over mutants</span> <span m=''870000''>but that''s
  possible. Yeah, OK. So, what would this</span> <span m=''877000''>person be? Plus
  over plus, let''s say, come over here. Now,</span> <span m=''884000''>what would
  this person be? M plus. It has to be M plus because, OK,</span> <span m=''891000''>and
  what about this person here? M plus, now what about the</span> <span m=''898000''>offspring?
  So, one of them is M over M,</span> <span m=''905000''>plus over plus, and two M
  pluses. Does it always work out like that?</span> </p><p><span m=''911000''>[LAUGHTER]
  No, it doesn''t always work out like that at all.</span> </p><p><span m=''917000''>So,
  I''m just going to write plus over plus here just to say,</span> <span m=''923000''>tough,
  right? In real life, it doesn''t always come out like that.</span> </p><p><span
  m=''929000''>What about over here? It would have to be plus over plus.</span> </p><p><span
  m=''935000''>Why not? It doesn''t because it could be M over plus and have no</span>
  <span m=''939000''>effect at offspring by chance, right? But, you were going to
  say</span> <span m=''944000''>it''s plus over plus because in the textbooks it''s
  always plus over plus</span> <span m=''948000''>in pictures like this, right? And
  then, it all turns out</span> <span m=''953000''>to be pluses and mutants, and pluses
  and mutants, and all that,</span> <span m=''957000''>right? Well, which picture''s
  right?</span> </p><p><span m=''962000''>Sorry? You don''t know. So, that''s not
  good. There''s supposed</span> <span m=''968000''>to be answers to these things.
  Could either be true? Which is</span> <span m=''974000''>more likely? The one on
  the left? Why? More statistically probable,</span> <span m=''979000''>how come?
  Because it is. It may not quite suffice as a fully</span> <span m=''985000''>complete
  scientific answer though.</span> </p><p><span m=''991000''>Yes? Yep. Well, but I
  have somebody who is affected</span> <span m=''1008000''>here. So, given that I''ve
  gotten affected person in the family --</span> <span m=''1025000''>yeah, so it is
  actually, you''re right, statistically somewhat</span> <span m=''1028000''>less
  likely that you would have two independent M''s entering the same</span> <span m=''1032000''>pedigree
  particularly if M is relatively rare.</span> </p><p><span m=''1036000''>If M is
  quite common, however, suppose M were something</span> <span m=''1040000''>was a
  20% frequency in the population, then it actually might</span> <span m=''1044000''>be
  quite reasonable that this could happen. So, what would you really</span> <span
  m=''1048000''>want to do to test this? Sorry? Well, if you found any females here</span>
  <span m=''1053000''>maybe you''d be able to conclude that it was autosomal recessive
  because</span> <span m=''1059000''>females never show a sex-linked trait. Is that
  true?</span> </p><p><span m=''1066000''>No, that''s not true. Why not? You''re right.
  So, you just have to</span> <span m=''1073000''>be homozygous for it on the X. So,
  having a single</span> <span m=''1080000''>female won''t, I mean, she''s not going
  to take that as</span> <span m=''1089000''>evidence. Get an affected female and
  demonstrate that all of her male</span> <span m=''1098000''>offspring show the trait.
  Cross her with, wait, wait.</span> </p><p><span m=''1108000''>This is a human pedigree
  guys [LAUGHTER]. Whew! There are issues</span> <span m=''1111000''>involved here,
  right? You could introduce her to a</span> <span m=''1115000''>normal guy, [LAUGHTER]
  but whether you can cross her to a normal guy is</span> <span m=''1119000''>not
  actually allowed. So, you see, these are exactly the</span> <span m=''1123000''>issues
  in making sense out of pedigrees like this.</span> </p><p><span m=''1126000''>So,
  what you have to do is you have to collect a lot of data,</span> <span m=''1130000''>and
  the kinds of characteristics that you look for in a pedigree,</span> <span m=''1134000''>but
  they are statistical characteristics, and</span> <span m=''1138000''>notwithstanding
  -- So, this could be colorblindness or</span> <span m=''1142000''>something, but
  notwithstanding the pictures in the textbook of</span> <span m=''1146000''>colorblindness
  and all that, you really do have to take a look at</span> <span m=''1150000''>a
  number of properties. What are some properties?</span> </p><p><span m=''1154000''>One
  you''ve already referred to which is there''s a predominance in</span> <span m=''1159000''>males
  if it''s X-linked. Why is there a predominance in</span> <span m=''1163000''>males?
  Well, there''s a predominance in males because if I</span> <span m=''1167000''>have
  an X over Y and I''ve got a mutation paired on</span> <span m=''1172000''>this X
  chromosome, males only have to get it on one.</span> </p><p><span m=''1176000''>Females
  have to get it on both, and therefore it''s statistically more</span> <span m=''1180000''>likely
  that males will get it. So, for example, the frequency of</span> <span m=''1184000''>colorblindness
  amongst males is what? Yeah, it''s 8-10%,</span> <span m=''1188000''>something like
  that. I think it''s about 8% or so.</span> </p><p><span m=''1192000''>And, amongst
  females, well, if it''s 8% to get one,</span> <span m=''1196000''>what''s the chance
  you''re going to get two?</span> </p><p><span m=''1200000''>It''s 8% times 8% is
  a little less than 1% right?</span> </p><p><span m=''1208000''>It''s 0.64%, OK,
  in females. So, we''ll just go 8%</span> <span m=''1217000''>squared. So in males,
  8% in females, less than one percent.</span> </p><p><span m=''1225000''>So, there
  is a predominance in males</span> <span m=''1233000''>of these sex-linked traits.
  Other things: affected males do not</span> <span m=''1239000''>transmit the trait
  to the kids, in particular do not transmit it to</span> <span m=''1246000''>their
  sons, right, because they are always sending the</span> <span m=''1253000''>Y chromosomes
  to their songs. Carrier females</span> <span m=''1260000''>transmit to half of their
  sons, and affected females transmit to all</span> <span m=''1270000''>of their sons.
  And, the trait appears to skip</span> <span m=''1280000''>generations, although
  I don''t like this terminology.</span> </p><p><span m=''1290000''>It skips generations.
  These are the kinds of properties</span> <span m=''1295000''>that you have. So,
  hemophilia,</span> <span m=''1300000''>a good example of this, if I have a child
  with hemophilia,</span> <span m=''1305000''>male with hemophilia, would you be surprised
  if his uncle had</span> <span m=''1310000''>hemophilia? Which uncle would it be,
  maternal or paternal?</span> </p><p><span m=''1315000''>The maternal uncle would
  have hemophilia most likely.</span> </p><p><span m=''1320000''>It''s always possible
  it could be paternal. This is the problem with</span> <span m=''1324000''>human
  genetics is you''ve got to get enough families so the pattern</span> <span m=''1328000''>becomes
  overwhelmingly clear, OK, because otherwise, as you can</span> <span m=''1332000''>see
  with small numbers, it''s tough to be absolutely certain.</span> </p><p><span m=''1336000''>So,
  these are properties of X linked traits.</span> </p><p><span m=''1340000''>How about
  baldness? Is baldness, that''s a sex-linked</span> <span m=''1344000''>trait? How
  come? You don''t see a lot of bald females.</span> </p><p><span m=''1348000''>Does
  that prove it''s sex linked? Sorry? Guys are stressed more.</span> </p><p><span
  m=''1352000''>[LAUGHTER] Is there evidence that it has anything to do with stress?</span>
  </p><p><span m=''1357000''>Actually, it has to do with excess testosterone it turns
  out,</span> <span m=''1361000''>that high levels of testosterone are correlated
  with male pattern</span> <span m=''1366000''>baldness, but does the fact that males
  become bald indicate that this</span> <span m=''1371000''>is a sex linked trait?
  No. Just because it''s predominant</span> <span m=''1376000''>in male, we have to
  check these other properties.</span> </p><p><span m=''1381000''>Is it the case that
  bald fathers tend to have bald sons?</span> </p><p><span m=''1385000''>Any evidence
  on this point? Common-sensical evidence from</span> <span m=''1389000''>observation?
  It''s pretty clear. It''s very clearly not a sex-linked</span> <span m=''1394000''>trait.
  It''s a sex-limited trait, because in order to show this you</span> <span m=''1398000''>need
  to be male because the high levels of testosterone are not found</span> <span m=''1403000''>in
  females even if they have the genotype that might predispose them</span> <span m=''1407000''>to
  become bald if they were male. So, it actually is not a sex-linked</span> <span
  m=''1413000''>trait at all, and it''s very clear that male pattern baldness does
  run</span> <span m=''1420000''>in families more vertically. So, you''ve got to be
  careful about</span> <span m=''1428000''>the difference between sex linked and sex
  limited,</span> <span m=''1435000''>and sex linked you can really pick out from
  transmission and families.</span> </p><p><span m=''1442000''>OK, here''s another
  one. New pedigree.</span> </p><p><span m=''1450000''>She married twice here. OK,
  what do we got?</span> </p><p><span m=''1483000''>Yep? She married again. She married
  twice. She didn''t have</span> <span m=''1493000''>any offspring the second time.
  But that happens,</span> <span m=''1501000''>and you have to be able to draw it
  in the pedigree.</span> </p><p><span m=''1506000''>She''s entitled, all right. OK,
  so she got married again,</span> <span m=''1512000''>no offspring from this marriage.
  That''s her legal symbol. You guys</span> <span m=''1517000''>think that''s funny.
  It''s real, you know?</span> </p><p><span m=''1522000''>OK, that doesn''t mean she''s
  married to two people at the same time.</span> </p><p><span m=''1528000''>This is
  not a temporal picture. So, what do we got here? Yep?</span> </p><p><span m=''1533000''>Sorry,
  of this person? Well, I''m drawing them as an empty</span> <span m=''1538000''>symbol
  here, indicating that we do not think they have the trait.</span> </p><p><span m=''1544000''>They''re
  not carriers. How do you propose to find that out?</span> </p><p><span m=''1550000''>Look
  at the children. Well, the children are affected. They</span> <span m=''1556000''>could
  be carriers. The data are what they are.</span> </p><p><span m=''1562000''>You''ve
  got to interpret it. Does this person have to be a</span> <span m=''1569000''>carrier?
  What kind of trait do you think this is?</span> </p><p><span m=''1576000''>Dominant?
  Does this look like autosomal dominant to you?</span> </p><p><span m=''1583000''>Yep?
  Oh, not all the kids have the trait</span> <span m=''1590000''>in the first generation,
  and if this was dominant,</span> <span m=''1594000''>they''d all have it? What''s
  a possible genotype for this person?</span> </p><p><span m=''1598000''>Mutant over
  plus. And, these kids could be mutant over plus.</span> </p><p><span m=''1602000''>This
  could be plus over plus, and this could be plus over plus,</span> <span m=''1606000''>mutant
  over plus, plus over plus, mutant over plus,</span> <span m=''1610000''>and plus
  over plus would be one possibility. On average,</span> <span m=''1614000''>what
  fraction of the kids should get the trait? About half</span> <span m=''1618000''>the
  kids, right? So, let''s see what characteristics</span> <span m=''1626000''>we have
  here. We see the trait in every generation.</span> </p><p><span m=''1638000''>On
  average, half the kids get the trait.</span> </p><p><span m=''1650000''>Half of
  the offspring of an affected individual are affected.</span> </p><p><span m=''1662000''>What
  else? Males and females? Roughly equal in males and females?</span> </p><p><span
  m=''1674000''>Sorry? One, two, three,</span> <span m=''1682000''>four, five to two.
  So, it''s a 5:2 ratio?</span> </p><p><span m=''1688000''>Oh, in the offspring it''s
  a 2:1 ratio. So, this is like Mendel.</span> </p><p><span m=''1693000''>You see
  this number and you say, OK, 2:1. Isn''t that trying to tell</span> <span m=''1699000''>me
  something? Not with six offspring. That''s the problem is</span> <span m=''1704000''>with
  six offspring, 2:1 might be trying to tell you 1:1.</span> </p><p><span m=''1710000''>And
  it is. If I had a dominantly inherited trait where there''s a</span> <span m=''1714000''>50/50
  chance of each offspring getting the disease and it was</span> <span m=''1719000''>autosomal,
  not sex linked, there would be very good odds of</span> <span m=''1724000''>getting
  two males and one female because it happens: flip coins and</span> <span m=''1728000''>it
  happens. So, you have to take that into account,</span> <span m=''1733000''>and
  here you see what else we have. Roughly equal numbers of males and</span> <span
  m=''1738000''>females, they transmit equally, and unaffecteds never transmit.</span>
  </p><p><span m=''1743000''>This would be the classic autosomal dominant trait.</span>
  </p><p><span m=''1747000''>Right, here this mutant would go mutant over plus,</span>
  <span m=''1751000''>mutant over plus, plus over plus, mutant over plus, plus over
  plus,</span> <span m=''1755000''>plus over plus, and you''d see here that three
  out of</span> <span m=''1759000''>the five here, and one, two, three out of the
  six</span> <span m=''1763000''>there: that''s a little more than half but it''s
  small numbers</span> <span m=''1767000''>here, right? This is a classic autosomal
  dominant</span> <span m=''1773000''>as in the textbooks. Yes? Turns out not to make
  too</span> <span m=''1779000''>much of a difference. It turns out that there''s
  lots of</span> <span m=''1786000''>genome that''s on either. And so, it is true
  that males are</span> <span m=''1793000''>more susceptible to certain genetic diseases.</span>
  </p><p><span m=''1800000''>So, it''ll be some excess, but it won''t matter for this.</span>
  </p><p><span m=''1804000''>Now, in real life it doesn''t always work so beautifully.</span>
  </p><p><span m=''1809000''>We''ll take an example: colon cancer. There are particular
  autosomal</span> <span m=''1813000''>dominant mutations here that cause a high risk
  of colon cancer.</span> </p><p><span m=''1818000''>People who have mutations in
  a certain gene, MLH-1,</span> <span m=''1823000''>have about a 70% risk of getting
  colon cancer in their life.</span> </p><p><span m=''1827000''>But notice, it''s
  not 100%. You might have incomplete penetrance.</span> </p><p><span m=''1833000''>Incompletely
  penetrance means not everybody who gets the genotype gets</span> <span m=''1841000''>the
  phenotype. Not all people with the M over plus</span> <span m=''1848000''>genotype
  show the phenotype. Once you do that, it messes up our</span> <span m=''1856000''>picture
  colossally, because, tell me,</span> <span m=''1863000''>how do we know that this
  person over here is not actually M over plus.</span> </p><p><span m=''1869000''>Maybe
  they''re cryptic. They haven''t shown the phenotype.</span> </p><p><span m=''1875000''>And
  maybe, it''ll appear in the next generation. That''ll screw up</span> <span m=''1881000''>everything.
  It screws up our rule about not transmitting through</span> <span m=''1887000''>unaffected,
  it screws up the rule about not being shown in</span> <span m=''1892000''>every
  generation, and it will even screw up our 50/50</span> <span m=''1896000''>ratio
  because if half the offspring get M over plus,</span> <span m=''1901000''>but only
  70% of that half show the phenotype, then only 35% of the</span> <span m=''1906000''>offspring
  will show the phenotype. Unfortunately, this is real life.</span> </p><p><span m=''1910000''>When
  human geneticists really look at traits, many mutations,</span> <span m=''1915000''>most
  except the most severe are incompletely penetrant.</span> </p><p><span m=''1920000''>And
  so you have to really begin to gather a lot of data to demonstrate</span> <span
  m=''1924000''>that you''re dealing with an autosomal dominant trait that''s</span>
  <span m=''1928000''>incompletely penetrant. And then there are other issues.</span>
  </p><p><span m=''1932000''>There''s a gene on chromosome number 17 called BRCA-1,</span>
  <span m=''1936000''>mutations in which predisposed to a very high risk of breast
  cancer but</span> <span m=''1940000''>only in women. Males carry the mutation and
  do not</span> <span m=''1944000''>have breast cancer. There are other mutations
  that do</span> <span m=''1948000''>cause breast cancer in males. Males have breast
  tissue,</span> <span m=''1952000''>and can have breast cancer, but the one on chromosome
  17 does</span> <span m=''1956000''>not. And so, there you would only see this transmitted
  through females.</span> </p><p><span m=''1960000''>It would skip into males without
  showing a phenotype,</span> <span m=''1965000''>etc. So, in real life, life''s a
  bit more complicated.</span> </p><p><span m=''1969000''>All right, so autosomal
  dominance. Now, let''s take one more pedigree.</span> </p><p><span m=''1973000''>Sorry?
  Sex limited, but not sex linked.</span> </p><p><span m=''1978000''>So, on chromosome
  17, which is a bona fide autosome,</span> <span m=''1983000''>but it''s sex limited
  in that phenotype can only show itself in an</span> <span m=''1988000''>individual
  who happens to be female. Yes? Sorry? How come autosomal</span> <span m=''1993000''>recessive?
  So, if that left guy up there is</span> <span m=''1999000''>actually a heterozygote,
  and up there that individual,</span> <span m=''2004000''>so if we had a homozygote,
  homozygote, heterozygote,</span> <span m=''2010000''>homozygote, ooh, you can interpret
  that pedigree if</span> <span m=''2013000''>you want to as an autosomal recessive,
  provided that M is pretty</span> <span m=''2017000''>frequent in the population.
  That''s right. Human geneticists,</span> <span m=''2020000''>in fact, to really
  prove that they''ve got the right model,</span> <span m=''2024000''>collect a lot
  of pedigrees and run a computer model.</span> </p><p><span m=''2028000''>The computer
  model first tries out autosomal recessive,</span> <span m=''2031000''>tries out
  autosomal dominant, tries out dominant with incomplete</span> <span m=''2035000''>penetrance,
  and for every possible model figures out the statistical</span> <span m=''2039000''>probability
  that you would see such data under that model.</span> </p><p><span m=''2043000''>And
  when the data become overwhelming and you say,</span> <span m=''2045000''>yeah,
  with one pedigree, any pedigree I draw on the board,</span> <span m=''2048000''>it
  could actually fit almost any for the models. It doesn''t say this in</span> <span
  m=''2051000''>the textbooks, but it''s true. I get enough</span> <span m=''2054000''>pedigrees,
  and eventually I say the odds are 105 times more likely that</span> <span m=''2057000''>this
  collection of pedigrees would arise from autosomal dominance,</span> <span m=''2060000''>inheritance
  with incomplete penetrance of about 80%.</span> </p><p><span m=''2062000''>Then,
  from autosomal recessive inheritance, then I get to write a</span> <span m=''2065000''>paper
  about it. That''s really what human</span> <span m=''2068000''>geneticists do is
  they have to collect enough,</span> <span m=''2072000''>now, any other organism,
  you''d just set up a cross,</span> <span m=''2076000''>but you can''t. And, as long
  as we have nontrivial models,</span> <span m=''2081000''>we really have to collect
  a lot of data. Let''s take the next pedigree,</span> <span m=''2086000''>great,
  that you''re thinking like a human geneticist.</span> </p><p><span m=''2090000''>It''s
  very good. Here''s the next pedigree. Actually,</span> <span m=''2095000''>I''m
  going to reverse it. There we go.</span> </p><p><span m=''2100000''>What''s that?
  Who knows? You can''t tell. Good, I''ve got you</span> <span m=''2103000''>up to
  training to the point where, but in textbooks, this would be</span> <span m=''2107000''>autosomal
  recessive. Or it could be anything.</span> </p><p><span m=''2111000''>You know that,
  right? But the textbooks would show you this</span> <span m=''2115000''>picture
  as an autosomal recessive. But of course, what else could it</span> <span m=''2118000''>be?
  It could be an autosomal dominant with incomplete penetrance.</span> </p><p><span
  m=''2122000''>It could be sex linked. It could be a lot of things.</span> </p><p><span
  m=''2126000''>It could also be, I haven''t told you the phenotype.</span> </p><p><span
  m=''2130000''>What if the phenotype here was getting hit by a truck?</span> </p><p><span
  m=''2134000''>[LAUGHTER] Would you tend to observe this? Yep,</span> <span m=''2138000''>so
  getting hit by a truck, for example, if someone gets hit by</span> <span m=''2142000''>a
  truck, it''s unlikely either their parents were hit by a truck,</span> <span m=''2146000''>or
  going back several generations that their grandparents were hit by</span> <span
  m=''2150000''>a truck. So, how do you tell being hit by a truck from,</span> <span
  m=''2154000''>I mean, that is to say, how do you know that something''s</span> <span
  m=''2158000''>genetic at all? When it''s relatively rare and it</span> <span m=''2161000''>pops
  up in a pedigree, how do you know it''s genetic?</span> </p><p><span m=''2164000''>Because
  of the DNA. But, I mean, it takes a lot of work to find the</span> <span m=''2167000''>gene
  and all that as we''ll come to the course. You might want a little</span> <span
  m=''2170000''>bit of assurance before you go write the grant to the NIH and say
  I''m</span> <span m=''2173000''>going to find the gene for this because you write
  it and say I''m</span> <span m=''2176000''>going to find the gene for getting hit
  by a truck,</span> <span m=''2179000''>and they''re going to write back and say
  show me that it''s worth spending</span> <span m=''2182000''>money to find that
  gene. Show me that it''s true. So,</span> <span m=''2185000''>what kind of things
  would we look for? If we wanted to show something</span> <span m=''2188000''>was
  autosomal recessive in a population, what would we do?</span> </p><p><span m=''2192000''>More
  data. So, we collect a lot of families,</span> <span m=''2200000''>and what would
  we see? As we collected more and more families,</span> <span m=''2208000''>we begin
  to see what things? Sometimes we might see families like</span> <span m=''2216000''>this,
  or we might see families like this. [LAUGHTER]</span> <span m=''2224000''>If both
  parents were mutants, all the children would be mutant,</span> <span m=''2228000''>right?
  We''d color them in mutant. Is that true? Well, first off, it</span> <span m=''2233000''>depends.
  Some of the things we want to study are extremely severe</span> <span m=''2238000''>medical
  genetical phenotypes, and they''re not going to live to</span> <span m=''2243000''>have
  children. So, that''s an issue that you have</span> <span m=''2248000''>to deal
  with. But, it is true that if it was</span> <span m=''2253000''>autosomal recessive,
  a mating between two homozygotes for</span> <span m=''2257000''>that gene would
  transmit. [LAUGHTER] What if they were all in</span> <span m=''2262000''>the same
  car? Which is a very important part,</span> <span m=''2266000''>because we joke
  about the car, but diet, things like that, are</span> <span m=''2271000''>familial
  correlated environmental factors. There are environmental</span> <span m=''2275000''>factors
  that correlate within a family.</span> </p><p><span m=''2280000''>And so, it''s
  not trivial to make this point. So,</span> <span m=''2284000''>all right, we''ll
  be able to demonstrate what''s the real proof of</span> <span m=''2289000''>Mendelian
  inheritance here? Because they could all be in the</span> <span m=''2294000''>same
  car, or they all eat the same kind of food or something like that,</span> <span
  m=''2299000''>which predisposes them a certain way. So, we''re going to want some
  better</span> <span m=''2304000''>proofs of these things. How about Mendelian ratios?</span>
  </p><p><span m=''2309000''>Mendelian ratios anyone? No, because it could be incomplete</span>
  <span m=''2314000''>autosomal dominance. I don''t want to mess you up.</span> </p><p><span
  m=''2318000''>On the exams, you guys can think cleanly about simple things.</span>
  </p><p><span m=''2322000''>But, this could be dominant with incomplete penetrance,</span>
  <span m=''2326000''>though the TA''s are going to hate me because I''m telling you
  that,</span> <span m=''2330000''>anyway, what about Mendelian ratios? How about
  something that''s a pretty</span> <span m=''2334000''>good prediction? What fraction
  of the offspring will</span> <span m=''2338000''>be affected? We get a lot of families,</span>
  <span m=''2342000''>line them all up. What fraction of the offspring?</span> </p><p><span
  m=''2347000''>A quarter. Now, that''s a hard and fast prediction.</span> </p><p><span
  m=''2352000''>One quarter of the offspring are effective. When I have a mating</span>
  <span m=''2357000''>between two homozygotes, so what am I going to do?</span> </p><p><span
  m=''2362000''>I''m going to go out. I''m going to collect a lot of families.</span>
  </p><p><span m=''2368000''>Maybe I''ll collect 100 families because it''ll be a
  particular</span> <span m=''2371000''>disease, diastrophic dysplasia or something
  like that,</span> <span m=''2375000''>xeroderma pygmentosa, ataxia teleangiectasia,</span>
  <span m=''2379000''>and I will go to the disease foundation, and I will get all
  the</span> <span m=''2383000''>pedigrees for all the families, and I''ll see how
  many times it was</span> <span m=''2386000''>one affected, two affected, three affected,
  etc. And on average,</span> <span m=''2390000''>the proportion affecteds will be
  a quarter, except it''s not true.</span> </p><p><span m=''2394000''>If I actually
  do that, I find that the ratio of affecteds is typically</span> <span m=''2398000''>more
  like a third. It isn''t a quarter.</span> </p><p><span m=''2403000''>Now, this should
  disturb you greatly because you know full well</span> <span m=''2409000''>that M
  over plus by M over plus should give you a quarter affecteds.</span> </p><p><span
  m=''2416000''>But when you actually look at human families, it''s not.</span> </p><p><span
  m=''2423000''>Why? In other words, when we count up all the matings</span> <span
  m=''2430000''>between heterozygotes, we''ll collect all the matings that</span>
  <span m=''2433000''>produce one affected child. We''ll collect all the matings that</span>
  <span m=''2437000''>produce two affected children. We''ll collect all the matings
  that</span> <span m=''2441000''>produce three affected children. But, we will fail
  to collect those</span> <span m=''2445000''>matings between homozygotes that produce
  zero affected children.</span> </p><p><span m=''2448000''>And so, we will systematically
  overestimate the proportion.</span> </p><p><span m=''2452000''>Of course, what we
  really have to do is go out and get all of those</span> <span m=''2456000''>couples
  who were both carriers, but because they had a small number</span> <span m=''2460000''>of
  children didn''t happen to have an affected child.</span> </p><p><span m=''2464000''>That''s
  not very easy to do especially when you don''t know the</span> <span m=''2468000''>gene
  in advance. So, when human geneticists try to</span> <span m=''2472000''>go out
  and measure the one-quarter Mendelian ratio,</span> <span m=''2476000''>you can''t.
  But what you can do is the following,</span> <span m=''2481000''>conditional on
  the first trial being affected, now what will be the</span> <span m=''2485000''>proportion
  of subsequent children who are affected?</span> </p><p><span m=''2490000''>A quarter.
  If I make it conditional, conditioning on having a first child</span> <span m=''2493000''>who''s
  affected, number one child who''s affected,</span> <span m=''2497000''>then I know
  I''ve got a mating between heterozygotes.</span> </p><p><span m=''2500000''>Subsequent
  offspring now do not have that bias.</span> </p><p><span m=''2504000''>And so, as
  a matter of fact, you think this pretty cool thought,</span> <span m=''2507000''>right?
  You''ve got a condition on one. It turns out there''s a very</span> <span m=''2511000''>famous
  paper about cystic fibrosis where somebody forgot this point and</span> <span m=''2514000''>made
  a huge big deal in the literature about the fact that a</span> <span m=''2518000''>third
  of the kids on average had cystic fibrosis in these</span> <span m=''2522000''>families,
  and proposed all sorts of models about how cystic fibrosis</span> <span m=''2526000''>might
  be advantageous and would lead to fertility increases and all that.</span> </p><p><span
  m=''2531000''>In fact, it was just a failure to correct for this little statistical</span>
  <span m=''2536000''>bias. OK, this is what human geneticists do is they''ve got
  to</span> <span m=''2540000''>deal with the popular, now, there''s one other trick
  that</span> <span m=''2545000''>you can use to know that something is autosomal
  recessive.</span> </p><p><span m=''2550000''>That trick is this. To site this trick,</span>
  <span m=''2557000''>I have to go back to a person called Archibald Garrett.</span>
  </p><p><span m=''2565000''>Archibald Garrett was a physician in London around 1900.</span>
  </p><p><span m=''2573000''>Garrett studied children with the trait alkoptonuria.</span>
  </p><p><span m=''2581000''>Alkoptonuria was what alkopton means black. Uria means
  urine.</span> </p><p><span m=''2586000''>They had black urine. This was evident
  because their urine turned</span> <span m=''2591000''>black on treatment with alkaline.
  How would you treat urine with</span> <span m=''2596000''>alkaline. How would people
  know this? Sorry? Outhouses with lime,</span> <span m=''2602000''>yeah, and who''s
  going to look at the children''s urine, or something</span> <span m=''2607000''>like
  that? But you''re on the right track.</span> </p><p><span m=''2612000''>How about
  diapers? You wash diapers, cloth diapers,</span> <span m=''2618000''>in alkaline.
  They turn black. This was evident from black diapers.</span> </p><p><span m=''2623000''>The
  kids'' urine would turn black. So, he observed this, and you know</span> <span m=''2629000''>what
  Garrett noticed is when he studied, children alkoptonuria,</span> <span m=''2634000''>he
  found that a very large fraction of affected</span> <span m=''2640000''>offspring
  were in fact produced from matings of first cousins.</span> </p><p><span m=''2650000''>Consanguineous
  matings: now you laugh, but in fact consanguinity has</span> <span m=''2660000''>been
  something that has been favored in many societies,</span> <span m=''2670000''>and
  in Britain, particularly amongst the upper class</span> <span m=''2675000''>in Britain
  in 1900, marriage or first cousins was quite</span> <span m=''2680000''>common,
  but not as common as he observed. He found that eight out</span> <span m=''2685000''>of
  17 alkoptonuria patients were the products of first cousin marriages.</span> </p><p><span
  m=''2690000''>That''s way off the charts because it''s nearly a half,</span> <span
  m=''2695000''>when in fact the typical rate in Britain might have been about 5%.</span>
  </p><p><span m=''2700000''>So, on the basis of that in the early 1900''s, Garrett
  was able to</span> <span m=''2703000''>show only a few years after the rediscovery
  of Mendel''s work that</span> <span m=''2707000''>this property of recessive traits,
  enrichment in the offspring of</span> <span m=''2711000''>consanguineous marriages,
  was a clear demonstration of</span> <span m=''2715000''>Mendelian inheritance. Not
  only did he do that,</span> <span m=''2718000''>but Garrett knew because of the
  work of some biochemists,</span> <span m=''2722000''>and this is way cool, that
  the problem with the urine was</span> <span m=''2726000''>that these patients put
  out in their urine a lot of what''s called</span> <span m=''2730000''>homogentisic
  acid, HGA, which basically is a phenolic</span> <span m=''2742000''>ring. What Garrett
  did was he, and that stuff turns black on</span> <span m=''2754000''>exposure to
  air. What might produce from the things</span> <span m=''2762000''>you''ve learned
  already some kind of ring like that?</span> </p><p><span m=''2767000''>What building
  blocks do you know have rings like that of things</span> <span m=''2772000''>you''ve
  studied already? Phenylalanine, tyrosine both have</span> <span m=''2777000''>rings.
  Suppose somebody had problems breaking down homogentisic</span> <span m=''2782000''>acid.
  Suppose there was some pathway where proteins were</span> <span m=''2787000''>broken
  down into amino acids including phenylalanine</span> <span m=''2792000''>and tyrosine.
  And, they were broken down into</span> <span m=''2797000''>homogentisic acid. And
  they were broken down into I</span> <span m=''2802000''>don''t know what. And, suppose
  like we had up there,</span> <span m=''2807000''>patients had a mutation in that
  enzyme. What would happen if I fed</span> <span m=''2812000''>patients a lot of
  protein? In their urine, you would recover</span> <span m=''2817000''>lots of homogentisic
  acid. Suppose I fed them a lot of tyrosine.</span> </p><p><span m=''2821000''>I''d
  get a lot of homogentisic acid because the body couldn''t break it</span> <span
  m=''2825000''>down. Suppose I fed them a lot of phenylalanine.</span> </p><p><span
  m=''2829000''>They would excrete a lot of homogentisic acid.</span> </p><p><span
  m=''2833000''>Suppose I fed them homogentisic acid. I would get quantitative</span>
  <span m=''2836000''>amounts of homogentisic acid. Garrett did this. These are the</span>
  <span m=''2840000''>days before institutional review boards, you know,</span> <span
  m=''2844000''>informed consent. It turns out it''s harmless feeding them proteins
  and</span> <span m=''2848000''>things like that. But in fact, Garrett,</span> <span
  m=''2853000''>in 1911, worked out that this trait had to be recessive because of
  its</span> <span m=''2859000''>population genetics, and inferred a biochemical pathway</span>
  <span m=''2866000''>by feeding different things along the way and was able to connect
  a</span> <span m=''2873000''>mutation in a gene to a problem with a specific biochemical
  pathway.</span> </p><p><span m=''2880000''>Sorry, 1908: this was his Croonian Lecture
  in 1908.</span> </p><p><span m=''2885000''>Eight years after the rediscovery of
  Mendel, he''s able to connect</span> <span m=''2890000''>genetic defect, showing
  it''s genetic by transmission,</span> <span m=''2895000''>to biochemical defect
  showing that he has a pathway that he can feed</span> <span m=''2900000''>things
  into. And, it all blocks up at the inability to</span> <span m=''2905000''>metabolize
  homogentisic acid. He has connected gene to enzyme by</span> <span m=''2910000''>1908.
  What do you think the reaction to</span> <span m=''2915000''>this was? Polite bewilderment,
  and it sunk like a stone. Nobody</span> <span m=''2919000''>was prepared to hear
  this. This is very much like Mendel in my</span> <span m=''2923000''>opinion. Now,
  he was a distinguished professor.</span> </p><p><span m=''2927000''>It was the Croonian
  Lecture. He got lots of accolades and all that,</span> <span m=''2932000''>and people
  said, what a lovely lecture that was,</span> <span m=''2936000''>and proceeded to
  completely forget this connection between genes and</span> <span m=''2940000''>enzymes,
  genes and proteins. It was not until 40 years later or</span> <span m=''2945000''>so
  that Beadle and Tatum, working with a fungus, actually</span> <span m=''2949000''>rosper
  not yeast, demonstrated that all these mutants</span> <span m=''2953000''>interfered
  with the ability to digest or to make particular amino</span> <span m=''2957000''>acids,
  and wrote this up as the one gene, one enzyme hypothesis of how</span> <span m=''2961000''>genes
  encode enzymes, and won the Nobel Prize for this</span> <span m=''2965000''>work,
  but in fact in their Nobel address,</span> <span m=''2970000''>Beetle and Tatum
  noted, actually, you know, Garrett kind of</span> <span m=''2974000''>knew all this.
  But, people weren''t ready,</span> <span m=''2979000''>yet, to digest it. Genetics
  had just come along,</span> <span m=''2983000''>Biochemistry had just really been
  invented in the last ten years,</span> <span m=''2988000''>and the idea of uniting
  genetics and biochemistry was just something</span> <span m=''2992000''>people weren''t
  prepared for yet. More next time.</span> </p>'
type: course
uid: e1ac03022862e5895b1eb80208acd552

---
None