---
about_this_resource_text: <p><strong>Topics covered:</strong> Genomics</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec25.jpg
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-25-genomics/lec25.jpg
  title: lec25.jpg
  type: null
  uid: d7221a864dabe4d6fc19ef7c714ee8e8
- id: Video-YouTube-Stream
  media_location: 5WhcMXP5yEU
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Video-YouTube-Stream
  type: Video
  uid: 92940d19ba0f7c1928f4b0e7bf8db687
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/5WhcMXP5yEU/default.jpg
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 4f65fcb3c886ef28dc3b8ab78cfef5e6
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Video-iTunes U-MP4
  type: Video
  uid: 3106025d5806e23a650c37a0ab69e8d8
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec25-10nov2004-220k.mp4
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Video-Internet Archive-MP4
  type: Video
  uid: 1609838320255c2da506f732feba7b04
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 124288bb07b23b7774e0978241c7464e
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-10nov2004-1000.mp3
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Video-Internet Archive-MP3
  type: Video
  uid: c185fc6b8daf28a8d850c24ed02c5d87
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: e9631023cf3bef81669978957d18997b
- id: Thumbnail-OCW-JPG
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 9faa2375eea71f7cdf2f2dc03f2fc1ae
- id: 3Play-3PlayYouTubeid-MP4
  media_location: 5WhcMXP5yEU
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: aea9bb6ddfbff5bb4ce1b09eda553ee1
- id: 5WhcMXP5yEU.srt
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-25-genomics/5WhcMXP5yEU.srt
  title: 3play caption file
  type: null
  uid: 7fe3a4c52bac6c6a077a6166146f77b1
- id: 5WhcMXP5yEU.pdf
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-25-genomics/5WhcMXP5yEU.pdf
  title: 3play pdf file
  type: null
  uid: f8cd7c04c980a141e44525db3eea3b70
- id: Caption-3Play YouTube id-SRT
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: b6989582ad3f9ac8d3af03fbe8bb1315
- id: Transcript-3Play YouTube id-PDF
  parent_uid: 73f7ec3d0965a2f067029ba24b282860
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: 24403a563c59e3eb5a30d78e1cb29ec4
inline_embed_id: 9578889lecture25:genomics82352798
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-25-genomics
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-25-genomics
template_type: Tabbed
title: 'Lecture 25: Genomics'
transcript: "<p><span m='1000'>Good morning. Welcome back. So, the Red Sox won, it's\
  \ pretty</span> <span m='6000'>convincing, yeah, very good. Yay Red Sox.</span>\
  \ </p><p><span m='13000'>So, as you can also tell, I have something of a cold,</span>\
  \ <span m='20000'>so I'll see if I, if my voice makes it through, but what I wanted\
  \ to do</span> <span m='27000'>today, if the voice allows, was to talk about genomics.</span>\
  \ </p><p><span m='34000'>Now, this is a little bit different than what we normally\
  \ do in the</span> <span m='38000'>class because, I work on genomics,</span> <span\
  \ m='42000'>it's something I'm extremely interested in.</span> </p><p><span m='46000'>And\
  \ so, what I wanted to do today, and I'll do it one more time before</span> <span\
  \ m='50000'>the end of the term, is to talk about research that's</span> <span m='54000'>going\
  \ on in genomics, give you a sense of what's really</span> <span m='58000'>going\
  \ on. I can assure you that what I say is not going to be in the</span> <span m='62000'>text\
  \ book, or any other text book. And, I'm not entirely sure how this</span> <span\
  \ m='65000'>might appear on an exam, so don't ask, because I'm really</span> <span\
  \ m='68000'>just going to talk about research that's going on today.</span> </p><p><span\
  \ m='72000'>And part of the purpose in doing that is to a, show you that it's</span>\
  \ <span m='75000'>possible for you to understand the kind of research that's going\
  \ on in</span> <span m='78000'>this field, and b, to excite you about what's going\
  \ on</span> <span m='81000'>in this field. So each year I pick different</span>\
  \ <span m='85000'>things to talk about, and I've picked a few things,</span> <span\
  \ m='88000'>and we'll see. So feel free to interrupt and to ask</span> <span m='92000'>questions,\
  \ and all of that, but this is very much more, sort of</span> <span m='96000'>the\
  \ edge of genomics, including stuff that's going on,</span> <span m='100000'>you\
  \ know, right now as we speak. So, we'll fire away.</span> </p><p><span m='104000'>So\
  \ a little introductory stuff. I call this, we can actually keep</span> <span m='108000'>the\
  \ lights up, I think people,</span> <span m='112000'>can people read that? Yeah,\
  \ it's fine, good,</span> <span m='116000'>so we'll leave the lights up and I can\
  \ see people.</span> </p><p><span m='120000'>So, I think the thing that sets apart\
  \ this revolution of biology</span> <span m='124000'>that we're looking through\
  \ right now, is the transformation of biology,</span> <span m='128000'>not just\
  \ from being the study of living organisms,</span> <span m='132000'>to the study\
  \ of chemicals and enzymes, to the study of molecules,</span> <span m='136000'>but\
  \ to the study of biology as information. That is what's</span> <span m='140000'>distinctive\
  \ about this decade, is the idea that the information</span> <span m='144000'>sciences\
  \ have begun to merge with biology, or biology merged with</span> <span m='148000'>information\
  \ sciences, and that it's having a profound</span> <span m='152000'>effect on driving\
  \ biomedicine. In both of the two talks I'll give,</span> <span m='156000'>this\
  \ one and near the end of the term, that will be the common theme,</span> <span\
  \ m='160000'>because I think that's the most important thing that's going on</span>\
  \ <span m='164000'>right now. Now, just to remind you,</span> <span m='168000'>of\
  \ course, the idea that biology is about information is an old one,</span> <span\
  \ m='172000'>it goes back to my hero, Gregor Mendel, with the recognition that</span>\
  \ <span m='176000'>information was passed from parent to offspring, according to\
  \ rules.</span> </p><p><span m='180000'>And, as you know, the history of biology\
  \ in the 20th</span> <span m='184000'>century can be read as the development of\
  \ biology's information.</span> </p><p><span m='188000'>The first quarter of the\
  \ 20th century was the development of the</span> <span m='192000'>idea that the\
  \ information lives in chromosomes. The next quarter of</span> <span m='196000'>the\
  \ 20th century, the idea that the information of the</span> <span m='200000'>chromosomes\
  \ resides in the DNA double-helix, and that information</span> <span m='204000'>was\
  \ contained in this molecule, and somehow in it's sequence, and</span> <span m='208000'>you\
  \ know all of this. And the next quarter of the 20th</span> <span m='211000'>century,\
  \ basically from 1950 to 1975, understanding how it is that the</span> <span m='215000'>cell\
  \ reads out that information, from DNA to RNA to protein, how it</span> <span m='219000'>uses\
  \ a genetic code to translate RNA's into proteins,</span> <span m='223000'>and the\
  \ development of the tools of recombinant DNA that made it</span> <span m='226000'>possible\
  \ for us to read out the information that the cell reads out.</span> </p><p><span\
  \ m='230000'>So that brought us \xBE of the way through the 20th century,</span>\
  \ <span m='234000'>with the ability to read out genetic information, at least in\
  \ little ways,</span> <span m='238000'>but they were little ways. You could write\
  \ a PhD thesis,</span> <span m='242000'>around that time, for sequencing 200 letters\
  \ of DNA.</span> </p><p><span m='247000'>That would be, you know, considered amazingly\
  \ exciting PhD</span> <span m='252000'>thesis. The next quarter of the 20th century,\
  \ the last quarter of</span> <span m='257000'>the 20th century, was characterized\
  \ by a veracious</span> <span m='262000'>appetite to read as much of this information\
  \ as possible.</span> </p><p><span m='267000'>It started, first, with trying to\
  \ read out the sequence of individual</span> <span m='272000'>genes, then sets of\
  \ genes, then genomes of small organisms'</span> <span m='277000'>bacteria, medium-sized\
  \ organisms. And then, you know,</span> <span m='281000'>in a wonderful closure\
  \ to the 20th century, the reading out of the</span> <span m='285000'>nearly complete\
  \ genetic information of the human being in the closing</span> <span m='288000'>weeks\
  \ of the 20th century. When you remember that, that Mendel</span> <span m='292000'>was\
  \ rediscovered in January of 1900, that's when the papers rediscovering</span> <span\
  \ m='295000'>Mendel came out, and you figure you've got perfect</span> <span m='299000'>bookends\
  \ from the rediscovery of Mendel in January of 1900,</span> <span m='302000'>to\
  \ the sequencing of the human genome in around 2000.</span> </p><p><span m='306000'>You\
  \ realize what a century can do. It's not bad, as centuries go, you</span> <span\
  \ m='309000'>know, to accomplish all that, and it gives you know, as students,</span>\
  \ <span m='312000'>you get a point estimate in time of what science knows,</span>\
  \ <span m='315000'>but you guys aren't old enough yet and haven't lived long enough\
  \ yet,</span> <span m='318000'>to measure the derivative, and see how rapidly it's\
  \ changing.</span> </p><p><span m='322000'>But just look at what happened over the\
  \ course of that century,</span> <span m='325000'>and then just project forward\
  \ to what that can mean for</span> <span m='328000'>the next century. So what that's\
  \ done is it's brought</span> <span m='332000'>us to the next picture. I have a\
  \ picture in my head,</span> <span m='336000'>of biology as a vast library of information,\
  \ a library of</span> <span m='340000'>information in which evolution has been taking\
  \ patient notes.</span> </p><p><span m='344000'>Evolution is a very good experimentalist,</span>\
  \ <span m='348000'>and it's a very patient note taker. It's notes, of course, are\
  \ written</span> <span m='352000'>in the genomes, and everyday evolution wakes up,</span>\
  \ <span m='356000'>changes a few nucleotides, sees how the organism works,</span>\
  \ <span m='360000'>if it was an improvement, evolution keeps the notes,</span> <span\
  \ m='364000'>if it was disadvantageous, evolution discards the notes.</span> </p><p><span\
  \ m='368000'>That, by the way, for those of you working in labs,</span> <span m='371000'>is\
  \ no longer considered appropriate laboratory practice.</span> </p><p><span m='374000'>You're\
  \ obliged to keep your laboratory notes from failed</span> <span m='377000'>experiments,\
  \ as well, but evolution got into this before</span> <span m='380000'>those rules\
  \ were codified, and so it discards the notes from</span> <span m='383000'>unsuccessful\
  \ experiments, and keeps the notes from the</span> <span m='386000'>successful experiments.\
  \ But nonetheless, we have all the</span> <span m='389000'>notes from the successful\
  \ experiments, and we can learn a</span> <span m='392000'>tremendous amount from\
  \ it. There's a volume on the shelf</span> <span m='395000'>corresponding to each\
  \ species on the planet. There's a volume on the</span> <span m='398000'>shelf corresponding\
  \ to each individual within each species,</span> <span m='401000'>to each tissue\
  \ within each individual within each species,</span> <span m='404000'>and there's\
  \ information there about the DNA sequence,</span> <span m='407000'>about the RNA\
  \ readouts, about the protein expression levels,</span> <span m='410000'>and in\
  \ principle, even if not yet in practice, we can pull down any</span> <span m='413000'>volume\
  \ we want, and interrogate it,</span> <span m='416000'>and compare it for related\
  \ species, for individuals within a species,</span> <span m='419000'>some of whom\
  \ might have a disease, some of whom might not, for</span> <span m='422000'>different\
  \ kinds of tissues treated in different ways.</span> </p><p><span m='426000'>That\
  \ is, I think, going to be a tremendous theme of</span> <span m='429000'>biology\
  \ going forward, and that's why it's a particular</span> <span m='432000'>pleasure\
  \ to teach biology at MIT, where you guys understand what that</span> <span m='436000'>could\
  \ mean, that fusion could mean. Now, this idea of extracting</span> <span m='439000'>genomic\
  \ information in large-scale, is a relatively new one. In the</span> <span m='443000'>mid-1980's,\
  \ the scientific community began debating what was a pretty</span> <span m='446000'>radical\
  \ idea, sequencing the human genome.</span> </p><p><span m='450000'>This was floated\
  \ in a couple of places, in 1984 at one meeting,</span> <span m='453000'>somebody\
  \ raised the idea, you've got to realize that sequencing itself,</span> <span m='457000'>that\
  \ sequencing DNA, only came from the late 70's,</span> <span m='461000'>so within\
  \ six, seven years of being able to sequence anything,</span> <span m='465000'>people\
  \ were now saying, let's sequence everything.</span> </p><p><span m='469000'>That\
  \ was a reasonably audacious thing to do, and it was</span> <span m='472000'>controversial.\
  \ There were many people who felt</span> <span m='476000'>that the human genome\
  \ project was a terrible idea,</span> <span m='480000'>and with good reason, because\
  \ the initial version of the</span> <span m='484000'>human genome project was, kind\
  \ of, a blunderbuss approach.</span> </p><p><span m='488000'>It was, let's immediately\
  \ mount a massive factory and start sequencing</span> <span m='491000'>the human\
  \ genome with the just horrible technologies of the</span> <span m='495000'>mid-80's,\
  \ with radioactive sequencing gels,</span> <span m='499000'>and you know, lots and\
  \ lots of people doing stuff.</span> </p><p><span m='502000'>And so, you know, many\
  \ people in science were, were concerned that an</span> <span m='506000'>entire\
  \ generation of students would need to be chained to the</span> <span m='510000'>bench,\
  \ sequencing DNA. Sydney Brenner,</span> <span m='513000'>a great molecular biologist,\
  \ proposed the whole thing be done at</span> <span m='517000'>institutions [LAUGHTER],\
  \ because you know, people could be</span> <span m='521000'>sentenced to, 20 million\
  \ bases, with time off for accuracy, or</span> <span m='525000'>things like that\
  \ [LAUGHTER]. And so what happened was, the</span> <span m='528000'>scientific community\
  \ came together well, in it's best form.</span> </p><p><span m='532000'>Group, a\
  \ group was put together by the National Academy of Sciences,</span> <span m='536000'>who\
  \ said, well look, this is a really good idea,</span> <span m='540000'>but we also\
  \ need a carefully thought-through program to do it.</span> </p><p><span m='544000'>We\
  \ need intermediate goals that will get us things that will advance the</span> <span\
  \ m='547000'>science along the way, we need to improve the technologies,</span>\
  \ <span m='550000'>and laid out a plan. The goals of that plan, to develop a genetic\
  \ map,</span> <span m='553000'>a map showing the locations of DNA polymorphisms,</span>\
  \ <span m='556000'>sites of variation, genetic markers, just like Sturdiman</span>\
  \ <span m='559000'>did with fruit flies, but to do it with humans,</span> <span\
  \ m='562000'>and with DNA sequence differences, to be used to trace inheritance.</span>\
  \ </p><p><span m='565000'>That, that genetic map could be used to map human diseases,</span>\
  \ <span m='568000'>and if all you accomplish was, got a human map of the human being,</span>\
  \ <span m='571000'>that would be a good thing. Then you could get a physical map\
  \ of</span> <span m='574000'>the human being, all the pieces of DNA overlapping</span>\
  \ <span m='578000'>each other, so that you would know if you had a genetic marker\
  \ linked</span> <span m='581000'>to cystic fibrosis, you would be able to get the\
  \ piece</span> <span m='584000'>of DNA that contains the gene. Then, if we managed\
  \ to pull that</span> <span m='588000'>off, we could get a sequence of the human\
  \ genome, all three billion</span> <span m='591000'>nucleotides, on the web, so\
  \ that you could go to just any</span> <span m='594000'>place on the genome, double-click,\
  \ and up would pop the</span> <span m='598000'>sequence. Now, you guys of course,</span>\
  \ <span m='601000'>don't laugh at that, but about eight years ago, when I would\
  \ give talks</span> <span m='604000'>about this, I would speak about, oh you'll\
  \ be able to go double-click</span> <span m='607000'>and up will pop the sequence,\
  \ and of course, everybody thought</span> <span m='610000'>that was really funny,\
  \ and that, that was something people</span> <span m='613000'>laughed at. But of\
  \ course, you can just do that today, if</span> <span m='616000'>anybody has a wireless\
  \ you can just double-click, and up will pop the</span> <span m='619000'>sequence.\
  \ And then, of course, a complete inventory of</span> <span m='622000'>all the genes\
  \ within that sequence. And a very importantly, and from</span> <span m='625000'>the\
  \ very beginning, the notion that all this information</span> <span m='628000'>should\
  \ be completely, freely available to anybody,</span> <span m='631000'>regardless\
  \ of where they were, whether in academia, or industry,</span> <span m='634000'>in\
  \ first world, third world countries, that everybody should</span> <span m='637000'>have\
  \ free and unrestricted access to that information.</span> </p><p><span m='640000'>So\
  \ a plan was laid out, I won't go into the details here,</span> <span m='643000'>but\
  \ the plan was laid out that involved work constructing genetic</span> <span m='646000'>maps,\
  \ physical maps, sequence maps, in the human,</span> <span m='649000'>the mouse,\
  \ and some model organisms, including the bacteria yeast, fruit</span> <span m='653000'>flies,\
  \ worms. And, quite remarkably, it largely went</span> <span m='656000'>according\
  \ to plan, over the course of about 15 years.</span> </p><p><span m='660000'>A lot\
  \ of people in the scientific community came together and took up</span> <span m='663000'>different\
  \ tasks. I should say, with some pride,</span> <span m='666000'>that MIT was by\
  \ far, one of the leading contributors to this effort,</span> <span m='669000'>having\
  \ been involved in essentially every stage of this,</span> <span m='673000'>the\
  \ genetic mapping of human and mouse, the physical mapping of human</span> <span\
  \ m='676000'>and mouse, and the sequencing of human and mouse,</span> <span m='679000'>and\
  \ having been the leading contributor to the latter,</span> <span m='683000'>and\
  \ it's not an accident because MIT's a marvelous environment in</span> <span m='686000'>which\
  \ to undertake this kind of research.</span> </p><p><span m='690000'>It involved\
  \ changing the way we do biology. Back in the mid-80's,</span> <span m='693000'>when\
  \ we sequenced DNA, we did it with radioactivity,</span> <span m='697000'>remember\
  \ I taught you how to sequence using radioactive label of</span> <span m='700000'>a\
  \ gel, and all that. That's how we did it,</span> <span m='704000'>stood behind\
  \ this plastic shield, and you loaded the gels. Of course,</span> <span m='708000'>now\
  \ it's done in a highly automated fashion. This is the production</span> <span m='711000'>floor\
  \ at the Broad Institute, which is here at MIT, where robots</span> <span m='715000'>prepare\
  \ all the DNA samples, so E. coli's grown up, and then you</span> <span m='719000'>have\
  \ to crack open the cells, purify the DNA, purify the plasmid,</span> <span m='722000'>do\
  \ a sequencing reaction, etc., etc. it's all done robotically there,</span> <span\
  \ m='726000'>and this is capable of processing, and does process, in a given day,</span>\
  \ <span m='730000'>about 200,000 samples per day. They then go, and this is all</span>\
  \ <span m='733000'>equipment designed by people here at MIT, and then commercially\
  \ built for</span> <span m='737000'>us. They then go to the back room where, actually,</span>\
  \ <span m='741000'>these are the previous generation of DNA sequencers,</span> <span\
  \ m='744000'>commercial detectors, those capillary detectors that have</span> <span\
  \ m='748000'>little lasers on them, there's a whole farm of them that</span> <span\
  \ m='752000'>sit there, and are able to get data out.</span> </p><p><span m='756000'>In\
  \ the course of a single day, we can now generate about 40 billion</span> <span\
  \ m='759000'>bases, I'm sorry, in the course of a single year we</span> <span m='763000'>can\
  \ generate about 40 billion bases of DNA sequence.</span> </p><p><span m='766000'>The\
  \ genome project itself, was a collaboration involving 20</span> <span m='770000'>different\
  \ groups around the world, groups in the United States, United</span> <span m='773000'>Kingdom,\
  \ France, Germany, and Japan,</span> <span m='777000'>and China. They were of different\
  \ sizes, they used different</span> <span m='781000'>approaches, but everybody was\
  \ committed to one common cause of</span> <span m='784000'>producing this information,\
  \ and making it freely available,</span> <span m='788000'>and everybody worked together.\
  \ And for the rest of my life,</span> <span m='791000'>when it comes to Friday,\
  \ at 11 o'clock, I will always think genome</span> <span m='795000'>project, because\
  \ we had a weekly conference call of all the groups in</span> <span m='799000'>the\
  \ world working on this Fridays, at eleven, and it was a fascinating</span> <span\
  \ m='803000'>experience, there were many, many years of that. So a draft</span>\
  \ <span m='806000'>sequence, a rough draft sequence of the human genome,</span>\
  \ <span m='810000'>was published in the year, in February of 2001, it was</span>\
  \ <span m='814000'>announced with some fanfare in June of 2000, but the real scientific</span>\
  \ <span m='818000'>paper came out in February of 2001.</span> </p><p><span m='822000'>This\
  \ was not a perfect sequence of the human genome,</span> <span m='825000'>by any\
  \ means. We discovered about 90% of the sequence of the human</span> <span m='828000'>genome.\
  \ It still had about 150, 00 gaps in it, it had errors. But,</span> <span m='831000'>it\
  \ still did have 90% of the sequence of the human genome.</span> </p><p><span m='834000'>For\
  \ the next three years, people worked very hard,</span> <span m='837000'>and, as\
  \ of last April, a finished sequence of the human</span> <span m='840000'>genome\
  \ was produced, and was published a couple weeks ago,</span> <span m='843000'>and\
  \ it contains, our best guess, about 99.</span> </p><p><span m='846000'>% of the\
  \ human genome, and it still has about 343 gaps,</span> <span m='849000'>they're,\
  \ we know what they are, we know where they are, but they're</span> <span m='852000'>not\
  \ sequence able with current technology.</span> </p><p><span m='856000'>That's the\
  \ \x93finished human genome\x94. What is it like? Well, this is a</span> <span m='859000'>picture\
  \ of the genome, do we have a pointer, yes,</span> <span m='863000'>I see here we\
  \ do have a pointer. This is your genome here, this is</span> <span m='867000'>chromosome\
  \ number 11, and I'll call attention to some</span> <span m='871000'>interesting\
  \ bits. So these colored lines here,</span> <span m='874000'>represent genes, or\
  \ gene-predictions, based on both,</span> <span m='878000'>sequencing of the DNA,\
  \ and mapping them back to the genome,</span> <span m='882000'>as well as computer\
  \ programs that analyze the genome.</span> </p><p><span m='886000'>And, right here,\
  \ you have a big pileup of lots of</span> <span m='889000'>genes, very few genes\
  \ of here. Lots of genes, few genes. Notice</span> <span m='892000'>the places where\
  \ there are lots of genes, match up with these</span> <span m='895000'>light-grey\
  \ bands, which are the light-grey bands of</span> <span m='898000'>the microscope,\
  \ on chromosomes. The places with</span> <span m='901000'>very few genes match up\
  \ with the dark bands in the chromosome.</span> </p><p><span m='904000'>Do you know\
  \ why that is, that the gene-rich regions are these</span> <span m='908000'>light\
  \ bands, and the gene-poor regions are the chromosome dark</span> <span m='912000'>bands?\
  \ Me neither. Nobody has a clue. It's really,</span> <span m='916000'>it's really\
  \ just one of these things. We had no reason to expect that</span> <span m='920000'>we'd\
  \ see these striking patterns, and other genomes, e-coli, doesn't</span> <span m='924000'>have\
  \ this dense, urban cluster, and these big,</span> <span m='928000'>rural plains\
  \ that are gene-poor. This is very weird, and it's</span> <span m='932000'>distinctive\
  \ to mammals. You'll also notice that the</span> <span m='935000'>gene-rich regions,\
  \ here, are rich in G's and C's,</span> <span m='938000'>they have different distributions\
  \ of some repeat elements,</span> <span m='941000'>it's all sorts of weirdness that\
  \ comes from just looking at the</span> <span m='943000'>genome. The biggest weirdness\
  \ was the number of genes,</span> <span m='946000'>the count of genes is, our best\
  \ guess, about 22,</span> <span m='949000'>00 genes, if I had to pick a number today,\
  \ it would be our count of</span> <span m='952000'>genes, and of course, that's\
  \ down from the 100,</span> <span m='955000'>00 that was in some textbooks, and\
  \ it's down from even 30 to 40,</span> <span m='958000'>00 that was in the genome\
  \ paper of February, 2001.</span> </p><p><span m='961000'>Our best guess is that\
  \ it's really just about that range.</span> </p><p><span m='964000'>Genes, themselves,\
  \ are very interesting.</span> </p><p><span m='967000'>When you look at, you know,\
  \ if we only have 22,000 genes we know</span> <span m='971000'>of, how do we manage\
  \ to run a human being with so few genes?</span> </p><p><span m='975000'>It is,\
  \ by the way, probably fewer genes than the mustard weed,</span> <span m='979000'>or\
  \ Arabidopsis thaliana. So, what do we do? Well, humans, one</span> <span m='982000'>thing\
  \ we may take comfort in, is that we, although we only have</span> <span m='986000'>about\
  \ 22,000 genes, there's a lot of alternative</span> <span m='990000'>splicing, on\
  \ average the typical gene, on average,</span> <span m='994000'>has about two alternative\
  \ splice products.</span> </p><p><span m='998000'>Some have many, some have few,\
  \ but probably,</span> <span m='1001000'>when you're all done, those 22, 00 genes\
  \ may encode 70-80,000</span> <span m='1005000'>different proteins, and it could\
  \ be more than that</span> <span m='1008000'>because we don't know all the alternative\
  \ splice products,</span> <span m='1012000'>and what they do. But, if you ask, humans\
  \ get credit for being really</span> <span m='1015000'>inventive or creative, for\
  \ having lots of new genes that</span> <span m='1019000'>make us human, the answer\
  \ is, no.</span> </p><p><span m='1023000'>Not only are humans not different in their\
  \ gene complement from other</span> <span m='1026000'>mammals, mammals, as a group,\
  \ really haven't invented</span> <span m='1030000'>that much, when you get down\
  \ to it. Most of the recognizable sub-domains</span> <span m='1034000'>of proteins,\
  \ proteins are built up of sub-domains,</span> <span m='1038000'>recognizable sequences\
  \ that have certain motifs that fold up in</span> <span m='1042000'>certain ways,\
  \ or carry out certain enzymatic functions.</span> </p><p><span m='1046000'>And\
  \ it looks like our genomes, our genes, are mixed-and-matched</span> <span m='1050000'>combinations\
  \ of many domains that were invented a long time ago,</span> <span m='1054000'>in\
  \ invertebrates and before, and that most of evolutionary innovation</span> <span\
  \ m='1058000'>in the more complex, multi-cellular animals,</span> <span m='1062000'>has\
  \ simply been mixing-and-matching these domains in new ways,</span> <span m='1066000'>to\
  \ get slightly different functions.</span> </p><p><span m='1070000'>You don't get\
  \ a lot of points for creativity, but it does seem to work.</span> </p><p><span\
  \ m='1074000'>By far, the most derivative of all, and what characterizes our genome</span>\
  \ <span m='1078000'>tremendously is, when a gene works,</span> <span m='1082000'>make\
  \ extra copies of it, and let it diverge slightly,</span> <span m='1087000'>and\
  \ take up new functions. Really, your genome is just characterized by</span> <span\
  \ m='1091000'>large expansions of families, immunoglobulin-like genes,</span> <span\
  \ m='1095000'>intermediate filament proteins holding together the cytoskeleton.</span>\
  \ </p><p><span m='1100000'>There are 111 different keratin-like genes in your genome.</span>\
  \ </p><p><span m='1103000'>They're all different, they do different things,</span>\
  \ <span m='1106000'>but they all came from one gene that was copied, copied,</span>\
  \ <span m='1109000'>copied, at random, randomly duplicated, and then diverged to</span>\
  \ <span m='1113000'>take up new functions. Growth factors, flies and worms</span>\
  \ <span m='1116000'>managed to get by just fine, thank you, with two growth factors</span>\
  \ <span m='1119000'>of the TGF beta-class, whatever that is. You have 42</span>\
  \ <span m='1123000'>growth factors of this TGF beta-class, all of which help</span>\
  \ <span m='1126000'>communicate, cells communicate, in different ways.</span> </p><p><span\
  \ m='1130000'>And then, of course, all the olfactory receptors.</span> </p><p><span\
  \ m='1133000'>In your genome, you have about 1, 00 genes for olfactory, for smell</span>\
  \ <span m='1137000'>receptors. This is what Richard Axel and Linda Buck won a Nobel</span>\
  \ <span m='1140000'>Prize for this year, was their work on the olfactory</span>\
  \ <span m='1144000'>receptors. Sad to say though, out of all your olfactory receptors,</span>\
  \ <span m='1147000'>genes, most of them are broken. They're most pseudo-genes.</span>\
  \ </p><p><span m='1151000'>It's not true in dogs and mice, who keep their olfactory\
  \ receptor</span> <span m='1155000'>genes in pretty fine-working order, but it's\
  \ very clear that in primates</span> <span m='1158000'>with color vision, our olfactory\
  \ receptor genes have</span> <span m='1162000'>been going to seed. They've been\
  \ piling up mutations,</span> <span m='1165000'>and there's no selective pressure\
  \ to keep many of them.</span> </p><p><span m='1168000'>And, in fact, we've now\
  \ shown, in a paper that will come out soon,</span> <span m='1172000'>that this\
  \ process is accelerating dramatically in the last 7 million</span> <span m='1175000'>years\
  \ since we diverged from chimps. And so, humans have almost</span> <span m='1178000'>completely\
  \ lost interest in smell, that's not totally true, some of</span> <span m='1182000'>these\
  \ olfactory receptors surely matter for various processes,</span> <span m='1185000'>but\
  \ most of them are probably irrelevant right now.</span> </p><p><span m='1188000'>And\
  \ so, anyway, that's the nature of the genes there.</span> </p><p><span m='1192000'>Anyway,\
  \ another interesting fact that's worth mentioning about your</span> <span m='1196000'>genome\
  \ is half of your genome consists of transposable elements,</span> <span m='1201000'>elements\
  \ that simply duplicate themselves, and hop around the</span> <span m='1205000'>genome.\
  \ Elements that are like viruses, they make a copy,</span> <span m='1210000'>sometimes\
  \ in RNA, the RNA is copied back into DNA and slammed elsewhere</span> <span m='1214000'>in\
  \ your genome. These elements,</span> <span m='1219000'>well the, there are four\
  \ classes.</span> </p><p><span m='1224000'>Alo elements, Line elements, Retro-Virus\
  \ like elements, all these</span> <span m='1227000'>go through RNA intermediates,\
  \ and use reverse transcription.</span> </p><p><span m='1230000'>And then there's\
  \ certain DNA transposons, that go through DNA</span> <span m='1234000'>intermediate.\
  \ The number of copies of the aloe element,</span> <span m='1237000'>the aloe element\
  \ that's hopped around your genome,</span> <span m='1240000'>you have about a million,\
  \ you have a million fossils of this</span> <span m='1244000'>element. You say,\
  \ why is it there, and the answer is,</span> <span m='1247000'>because it's there.\
  \ Because anything that knows how to make a</span> <span m='1250000'>copy of itself,\
  \ and insert it itself in it's genome,</span> <span m='1254000'>you can't get rid\
  \ of. You can consider it,</span> <span m='1257000'>if you wish, an infection, but\
  \ half of your genome consists of</span> <span m='1260000'>an infection, with these\
  \ kinds of transposable elements.</span> </p><p><span m='1263000'>Now that's it,\
  \ yes?</span> </p><p><span m='1272000'>Well, it's very interesting, what's the effect?\
  \ Well, they do,</span> <span m='1276000'>some of them are transcribed and, it's\
  \ very interesting.</span> </p><p><span m='1280000'>Sometimes it's bad, one of them\
  \ will hop into a gene and mutate it,</span> <span m='1284000'>and that's bad, that\
  \ person will have a lethal mutation,</span> <span m='1288000'>but the genome has\
  \ probably begun to use them, and count on their being</span> <span m='1292000'>there.\
  \ So, when a bunch, when a transposable goes in,</span> <span m='1296000'>and creates\
  \ a spacing, if you, for example, if an engineering</span> <span m='1300000'>committee\
  \ came in and cleaned up the genome by getting rid of all the</span> <span m='1304000'>transposable\
  \ elements, it would surely not work.</span> </p><p><span m='1308000'>Because we\
  \ have evolutionarily come to count on the spacing there.</span> </p><p><span m='1311000'>It's\
  \ sort of like, if in some very, some very messy attic, you put a cup</span> <span\
  \ m='1315000'>of coffee down on top of a stack of papers, those papers may be utterly</span>\
  \ <span m='1318000'>irrelevant, but now they're holding up that cup of coffee that\
  \ you put</span> <span m='1322000'>down on it. And if you were to just, poof, magically\
  \ get rid of them,</span> <span m='1326000'>the cup of coffee would come crashing\
  \ to the ground.</span> </p><p><span m='1330000'>So, you know it, they're just there,</span>\
  \ <span m='1333000'>taking up space. Now sometimes, even more than that, a few of\
  \ them</span> <span m='1337000'>have actually been co-opted into being human genes.</span>\
  \ </p><p><span m='1340000'>We know that a few of these transposable elements have\
  \ mutated</span> <span m='1344000'>into being our genes that do something for us.</span>\
  \ </p><p><span m='1347000'>And others of them may do things in affecting the general\
  \ neighborhood</span> <span m='1351000'>with regard to transcription, and so, instead\
  \ of it being a</span> <span m='1355000'>parasite, think of them as a symbiont,\
  \ that's a genomic symbiont,</span> <span m='1358000'>which takes some advantage\
  \ of us, and we may, you know, have worked</span> <span m='1362000'>out a compromise\
  \ to take some advantage of it.</span> </p><p><span m='1366000'>Every time a copy\
  \ is made of these, and it hops in the genome, some</span> <span m='1369000'>mutations\
  \ may happen in the master element, but when it lands in the</span> <span m='1373000'>new\
  \ place, we have a record of that hop. And if you reconstruct the</span> <span m='1376000'>sequence\
  \ of the million AluI elements, you can see which ones are</span> <span m='1380000'>very\
  \ close relatives of each other, and had to have hopped recently, and</span> <span\
  \ m='1384000'>which ones are somewhat more distant relatives.</span> </p><p><span\
  \ m='1388000'>And you can build an evolutionary tree connecting all of the repeat</span>\
  \ <span m='1391000'>elements that have hopped around your genome, and thereby attaching\
  \ a</span> <span m='1394000'>date to each of them, as to when they hopped.</span>\
  \ </p><p><span m='1397000'>So it really is a fossil record, and you can figure out\
  \ how many of</span> <span m='1400000'>them have been hopping at different times\
  \ over history.</span> </p><p><span m='1403000'>And we can even make a plot of that,\
  \ this is long ago,</span> <span m='1407000'>sometime here, some 30 million years\
  \ ago, there was a huge explosion and</span> <span m='1410000'>in transposion, transposons,\
  \ in our genome.</span> </p><p><span m='1413000'>We don't know why that happened,\
  \ but it's very interesting, it does</span> <span m='1416000'>correspond to very\
  \ interesting periods of primate evolution.</span> </p><p><span m='1420000'>And\
  \ then, interestingly, there's been a huge crash,</span> <span m='1423000'>and transposition\
  \ has dropped dramatically. We have no clue why</span> <span m='1426000'>this is,\
  \ but we have a whole fossil record here of the rate of</span> <span m='1429000'>transposition\
  \ of different kinds of repeat elements around our genome,</span> <span m='1432000'>and\
  \ people are now starting to try to figure out what in the world this</span> <span\
  \ m='1435000'>means. So all this is sort of there, inherent in the sequence,</span>\
  \ <span m='1438000'>and if you want the sequence, as I say, you can go to the web\
  \ and</span> <span m='1441000'>pull all this stuff now. So how do we understand\
  \ the</span> <span m='1444000'>sequence? Well, I've told you a little bit about\
  \ it,</span> <span m='1447000'>from the simple things that we've done, but there's\
  \ a lot more that</span> <span m='1450000'>needs to be learned about the sequence,\
  \ so what I really want to</span> <span m='1453000'>turn to, is how we're extracting\
  \ information out of this sequence.</span> </p><p><span m='1456000'>So, DNA sequence\
  \ is long and boring, it's only marginally more</span> <span m='1459000'>interesting\
  \ than reading your hard disk, because it has four letters,</span> <span m='1463000'>instead\
  \ of ones and zeros, but it's, you know, well, it's pretty really</span> <span m='1467000'>boring\
  \ if you take a look at it. How do you attach meaning to all</span> <span m='1470000'>this\
  \ stuff? One of the most powerful ways is by comparison with</span> <span m='1474000'>other\
  \ genomes. And so, comparing the human genome</span> <span m='1478000'>to the mouse\
  \ genome is very informative in many ways.</span> </p><p><span m='1482000'>So, as\
  \ soon as the human genome was far along, a portion of the</span> <span m='1485000'>international\
  \ consortium, set to work getting a sequence of</span> <span m='1489000'>the mouse\
  \ genome. And that was published in December</span> <span m='1492000'>of 2002. We\
  \ have a nice map of the mouse genome, with all these things,</span> <span m='1496000'>it,\
  \ too, shows these gene-rich regions, gene-poor regions,</span> <span m='1499000'>all\
  \ sorts of funny things. And if we look closely at a portion of the</span> <span\
  \ m='1503000'>human genome over here, I've picked about a million bases of</span>\
  \ <span m='1506000'>the human genome, and we take any little spot in that</span>\
  \ <span m='1510000'>million bases of the human genome, let's say over here.</span>\
  \ </p><p><span m='1514000'>And we take half the DNA sequence corresponding to this\
  \ spot,</span> <span m='1517000'>and we run it in the computer against the mouse\
  \ genome,</span> <span m='1520000'>and ask where in the mouse genome do we get the\
  \ best match for this,</span> <span m='1523000'>the best match to this is here.\
  \ Now let's do it for this piece,</span> <span m='1526000'>here. The best match\
  \ anywhere in the mouse genome lands in the same</span> <span m='1529000'>million\
  \ bases here as the mouse genome. In fact,</span> <span m='1532000'>for every single\
  \ sequence that we pull out from this million bases in</span> <span m='1536000'>the\
  \ human genome, the best match is in this million</span> <span m='1539000'>bases\
  \ of the mouse genome. That's very interesting. Why is</span> <span m='1542000'>that?\
  \ Sorry? No, people do know.</span> </p><p><span m='1545000'>It, it was a good try,\
  \ though. [LAUGHTER]. This million bases in</span> <span m='1549000'>the mouse genome,\
  \ and this million bases in the human</span> <span m='1552000'>genome, represent\
  \ the evolutionary descendents of a common million</span> <span m='1556000'>bases\
  \ that occurred in our common ancestor 75-million years ago.</span> </p><p><span\
  \ m='1559000'>This is a clear evidence of the evolution here,</span> <span m='1563000'>because\
  \ we can see that this is a segment of DNA from our common</span> <span m='1566000'>ancestor\
  \ that really hasn't undergone much rearrangement,</span> <span m='1570000'>and\
  \ we can just line up the sequences and see.</span> </p><p><span m='1574000'>In\
  \ fact, we can build a whole map across the mouse genome like this.</span> </p><p><span\
  \ m='1577000'>For any bit of the mouse genome, I don't know, here's a bit on mouse</span>\
  \ <span m='1580000'>chromosome 17, this whole stretch corresponds to a</span> <span\
  \ m='1584000'>portion of human chromosome number eight. This stretch here,</span>\
  \ <span m='1587000'>I don't know, this green color here on chromosome number six,</span>\
  \ <span m='1590000'>corresponds to chromosome four in the human. And so,</span>\
  \ <span m='1594000'>we can build a look-up table that says, for any portion of the\
  \ human</span> <span m='1597000'>genome, what's the corresponding portion of the\
  \ mouse genome that</span> <span m='1600000'>came from the same ancestor, has basically\
  \ the same complement of</span> <span m='1604000'>genes in it. And there's only\
  \ about 330 such</span> <span m='1607000'>regions that we need to cut-and-paste\
  \ the human genome order</span> <span m='1610000'>to the mouse genome order, roughly\
  \ speaking. There's a lot of</span> <span m='1613000'>little local rearrangements,\
  \ but at this gross level. So now,</span> <span m='1616000'>if we go back more closely\
  \ and we look at this, and we say,</span> <span m='1619000'>OK, so now we look at\
  \ this region, we now know these two regions</span> <span m='1623000'>descend from\
  \ a common ancestor, if we do a careful evolutionary</span> <span m='1626000'>analysis,\
  \ lining up all the sequences, and see how</span> <span m='1629000'>well-preserved\
  \ the sequences are, some are much better preserved than</span> <span m='1632000'>others.\
  \ Evolution has been much more</span> <span m='1636000'>lovingly conserving other\
  \ sequences than others, and so,</span> <span m='1640000'>so let's now zoom-in on\
  \ a gene, this is a gene that goes by the name,</span> <span m='1644000'>PP-Gama,\
  \ I'm fond of this gene but, it doesn't matter. If we look, I've</span> <span m='1648000'>indicated\
  \ all the regions here, in which there's a heightened degree</span> <span m='1652000'>of\
  \ conservation. The sequence is well-conserved here,</span> <span m='1656000'>here,\
  \ here, here, here, here, here, and here,</span> <span m='1660000'>here, here, here,\
  \ here, here. These correspond to the exons of</span> <span m='1664000'>the PPR-Gama\
  \ gene, they encode the protein of the gene,</span> <span m='1668000'>then the splicing\
  \ goes like this, OK? These things here do not correspond</span> <span m='1672000'>to\
  \ the exons. People have no idea what they are,</span> <span m='1676000'>in fact,\
  \ this is not supposed to be here. The official textbook picture</span> <span m='1680000'>says,\
  \ the vast majority of what matters for a gene,</span> <span m='1684000'>what evolution\
  \ should preserve, is the exons plus the promoter.</span> </p><p><span m='1689000'>Here's\
  \ the promoter. But in fact, what we found is that</span> <span m='1693000'>an awful\
  \ lot more is being preserved. In fact, across the genome,</span> <span m='1697000'>our\
  \ best estimate is there are about 500,000 conserved elements across</span> <span\
  \ m='1701000'>the genome, and only 1/3 of them are protein-coding exons.</span>\
  \ </p><p><span m='1706000'>That means 2/3 of the stuff evolution has been interested\
  \ in,</span> <span m='1710000'>is not protein-coding exons, and the truth is, we\
  \ do not know what it is,</span> <span m='1714000'>this was a very radical finding,\
  \ when this mouse paper came out,</span> <span m='1718000'>about a year and a half,\
  \ about two years ago now.</span> </p><p><span m='1723000'>What it must be, I think,\
  \ but we're guessing,</span> <span m='1727000'>are regulatory signals, the structural\
  \ elements in chromosomes,</span> <span m='1731000'>RNA genes, but there's an awful\
  \ lot more of it than we had imagined.</span> </p><p><span m='1736000'>And we've,\
  \ now we're in this fascinating situation,</span> <span m='1739000'>where computational\
  \ analysis has told us what's on evolution's mind,</span> <span m='1743000'>and\
  \ now we have to go to the lab and figure out what in the world it does.</span>\
  \ </p><p><span m='1747000'>But there's no doubt that it must do something, because\
  \ evolution has</span> <span m='1750000'>preserved it quite well. Now, I oversimplified\
  \ greatly in</span> <span m='1754000'>this discussion, let me first say,</span>\
  \ <span m='1758000'>and I'll come back to that. We do know, if we take some of those</span>\
  \ <span m='1761000'>elements, here's one, there's a 481 base-pair elements</span>\
  \ <span m='1764000'>that's 84% identical between human and mouse. You could write\
  \ yourself</span> <span m='1767000'>a little statistical model to say that's way\
  \ unusual to have something</span> <span m='1771000'>that's so well preserved. When\
  \ Eddie Ruben and his colleagues</span> <span m='1774000'>from Berkley made a knockout\
  \ mouse that deleted that segment,</span> <span m='1777000'>this knockout mouse\
  \ loses regulation of three different genes in the</span> <span m='1780000'>neighborhood,\
  \ saying that this must be a regulatory sequence that</span> <span m='1783000'>affects\
  \ multiple genes in the neighborhood. That,</span> <span m='1787000'>that's one,\
  \ with about 300, 00 such elements to go, in order to</span> <span m='1790000'>attach\
  \ meaning to them. So doing this entirely by knocking</span> <span m='1794000'>out\
  \ mice will be a slow process, one's going to need other ways to be</span> <span\
  \ m='1798000'>able to attach meaning, but there's no doubt. Now,</span> <span m='1802000'>there's\
  \ some other interesting papers where people have knocked</span> <span m='1806000'>some\
  \ of these things out, and they've seen no effect on the</span> <span m='1810000'>mouse.\
  \ They get a totally viable mouse. Can you conclude from that,</span> <span m='1814000'>that\
  \ they have no function? Why not? The knockout mouse is viable.</span> </p><p><span\
  \ m='1818000'>Could be redundant, it could even not be redundant,</span> <span m='1822000'>but\
  \ yes, it could be redundant, but you couldn't knock out both of</span> <span m='1826000'>two\
  \ things. It turns out, suppose knocking it</span> <span m='1829000'>out affected\
  \ the mouse's viability by part, ten to the third,</span> <span m='1833000'>it was\
  \ only 99.9% as fertile, would you be able to see that in the</span> <span m='1837000'>laboratory?\
  \ No. Would that matter to evolution?</span> </p><p><span m='1841000'>It would be\
  \ lethal, in an evolutionary sense.</span> </p><p><span m='1844000'>Such mutation\
  \ could never propagate through a population.</span> </p><p><span m='1848000'>One\
  \ part, and ten to the third, is massive selection against, from</span> <span m='1852000'>an\
  \ evolutionary point of view, but almost undetectable in a</span> <span m='1856000'>laboratory\
  \ batch. Evolution has a far more sensitive</span> <span m='1860000'>assay than\
  \ we do. Now, I won't go into detail,</span> <span m='1864000'>but for the mathematically\
  \ inclined here, showing that there really were</span> <span m='1869000'>about 5%\
  \ of the human genome under, under evolutionary selection, it was</span> <span m='1873000'>a\
  \ complicated affair, because with only two genomes,</span> <span m='1878000'>what\
  \ we really had to do, and if this doesn't make sense, ignore it.</span> </p><p><span\
  \ m='1883000'>We looked at the background distribution of conservation of the</span>\
  \ <span m='1886000'>genome in unimportant elements, in those repeat elements that\
  \ we</span> <span m='1889000'>knew to be functionally broken. We looked at the overall</span>\
  \ <span m='1892000'>conservation of the genome, and found that the overall genome</span>\
  \ <span m='1895000'>has this rightward tail, by subtracting the distributions we</span>\
  \ <span m='1898000'>were able to see how much excess conservation there was.</span>\
  \ </p><p><span m='1901000'>That's because we only had two genomes, we had to draw\
  \ inferences.</span> </p><p><span m='1904000'>If we had more genomes, like the mouse\
  \ and the rat,</span> <span m='1907000'>and the dog and the-this-and-the-that,</span>\
  \ <span m='1910000'>we would be able to extract signal from noise.</span> </p><p><span\
  \ m='1914000'>We would be able to see right away, which bits were well-conserved,\
  \ and</span> <span m='1917000'>we wouldn't have to do this as a sensitive statistical\
  \ analysis.</span> </p><p><span m='1921000'>So, in fact, we need more mammalian\
  \ genomes, so, so right now there's</span> <span m='1925000'>been a sequence of\
  \ the rat genome in the past year or so,</span> <span m='1929000'>there's a sequence\
  \ of the dog genome, we're writing up that paper now,</span> <span m='1932000'>but\
  \ it's on the web already. There's a sequence of the chimpanzee</span> <span m='1936000'>genome\
  \ we're writing up a paper on that, in collaboration with our</span> <span m='1940000'>friends\
  \ in the genome-sequencing community.</span> </p><p><span m='1944000'>We're currently\
  \ sequencing a variety of other organisms,</span> <span m='1947000'>as well. And\
  \ if you had enough organisms, you ought to be able to</span> <span m='1950000'>just\
  \ line it up and say, what has evolution preserved,</span> <span m='1954000'>and\
  \ figure out exactly which nucleotides matter,</span> <span m='1957000'>and which\
  \ nucleotides don't, are allowed to drift freely, at the</span> <span m='1960000'>background\
  \ rate. How far could you go with this?</span> </p><p><span m='1964000'>Well, we\
  \ decided to try an interesting experiment.</span> </p><p><span m='1967000'>We said,\
  \ since mammals are very big, then we're going to need a lot of</span> <span m='1970000'>genome\
  \ sequences, how about we try a small organism,</span> <span m='1974000'>like yeast?\
  \ What if we were to try to do this,</span> <span m='1978000'>this kind of evolutionary,\
  \ genomic analysis on something like the yeast</span> <span m='1982000'>genome?\
  \ And so, this is work that I'll describe,</span> <span m='1986000'>that was between\
  \ a bunch of people here at MIT who do genome-sequencing,</span> <span m='1990000'>and\
  \ a student in computer science, Manolis Kellis, was PhD student in</span> <span\
  \ m='1994000'>computer science, he now just joined the faculty here</span> <span\
  \ m='1998000'>at MIT in computer science. But it was a really great example of</span>\
  \ <span m='2001000'>how biology and computer science could come together.</span>\
  \ </p><p><span m='2005000'>So, the genome-sequencing folks sequenced three related\
  \ species,</span> <span m='2008000'>through our friend, the baker's yeast, Saccharomyces\
  \ cerevisiae,</span> <span m='2012000'>workhorse of geneticist. These three different\
  \ species are</span> <span m='2015000'>separated by different evolutionary distances,\
  \ from Saccharomyces</span> <span m='2019000'>cerevisiae. When you line up their\
  \ genomes, just like with human and</span> <span m='2022000'>mouse, you find the\
  \ genes occur largely in the same order,</span> <span m='2026000'>and it's not hard\
  \ to pick out, oh there's this gene there, there,</span> <span m='2029000'>it's\
  \ all lined up, you've got these evolutionary segments,</span> <span m='2033000'>and\
  \ very few rearrangements have occurred across these species,</span> <span m='2036000'>despite\
  \ the fact that they're about 20 million years apart in history.</span> </p><p><span\
  \ m='2040000'>But here's an interesting thing. When the yeast genome,</span> <span\
  \ m='2045000'>Saccharomyces cerevisiae, was first published in 1995,</span> <span\
  \ m='2051000'>the paper describing it reported 6, 00 genes. Now, how did they know</span>\
  \ <span m='2056000'>there were 6,200 genes? They ran a computer program looking</span>\
  \ <span m='2062000'>for open reading frames. Any open reading frame, consecutive</span>\
  \ <span m='2068000'>codons without a stop sufficiently long, was called a gene.</span>\
  \ </p><p><span m='2074000'>But statistically, you could, by chance,</span> <span\
  \ m='2077000'>just have a long stretch of codons without a stop codon.</span> </p><p><span\
  \ m='2081000'>And so, if I saw 100 codons in a row, without a stop,</span> <span\
  \ m='2084000'>they called it a gene, but it might just be chance.</span> </p><p><span\
  \ m='2088000'>And they knew that, of course, they wrote that in the paper, but</span>\
  \ <span m='2092000'>for many years, people then had 6,</span> <span m='2095000'>00\
  \ open reading frames, which were the yeast's genes.</span> </p><p><span m='2099000'>Could\
  \ evolution now tell us which one of them were real and which</span> <span m='2102000'>weren't?\
  \ Well, it turns out that evolution was</span> <span m='2106000'>tremendously powerful\
  \ in doing that.</span> </p><p><span m='2110000'>If you take something that's a\
  \ well-known gene that has been</span> <span m='2114000'>extensively studied by\
  \ yeast geneticists, you line it up across</span> <span m='2119000'>all four species,\
  \ you almost never see deletions.</span> </p><p><span m='2123000'>And when you do\
  \ see the lesions, here in grey, they're always a</span> <span m='2128000'>multiple\
  \ of three. Why are they a multiple of three?</span> </p><p><span m='2133000'>They\
  \ preserve the reading frame. By contrast, if I take some clear,</span> <span m='2137000'>intergenetic\
  \ DNA, that's not protein-coding,</span> <span m='2142000'>and I compare it across\
  \ these four species, I see lots and lots of</span> <span m='2147000'>frame shifting\
  \ deletions that occur,</span> <span m='2152000'>Evolution tolerates frame shifting\
  \ deletions, and if I juts write down</span> <span m='2154000'>the rates, frame\
  \ shifting deletions are 75x more common in intergenic</span> <span m='2157000'>DNA,\
  \ than genic DNA. This provides a very powerful test.</span> </p><p><span m='2160000'>Run\
  \ this test across the genome, looking for the density of frame</span> <span m='2163000'>shifting\
  \ deletions, any place that doesn't tolerate</span> <span m='2166000'>frame shifting\
  \ deletions is probably a real gene, anything that does</span> <span m='2169000'>tolerate\
  \ it is probably not. When you sorted through all this,</span> <span m='2172000'>it\
  \ turned out that 528 of the official yeast genes were clearly</span> <span m='2175000'>not\
  \ real, not real genes. They were just chock-a-block full</span> <span m='2178000'>of\
  \ these frame shifting deletions. And, and a bunch of others could be</span> <span\
  \ m='2182000'>confirmed. So the yeast gene count, and I won't tell you all the</span>\
  \ <span m='2186000'>experimental and other that shows this is right,</span> <span\
  \ m='2190000'>but the yeast genome has now been revised downward to 5,</span> <span\
  \ m='2194000'>00 genes, and we have great confidence that almost all of those</span>\
  \ <span m='2198000'>are real genes, there are 20 whose origins that</span> <span\
  \ m='2202000'>we're not sure of, and new genes could be found in this</span> <span\
  \ m='2206000'>way. Here's a really audacious thing.</span> </p><p><span m='2210000'>This\
  \ graduate student in computer science said, I think,</span> <span m='2211000'>based\
  \ on these other species, there was a mistake made in the</span> <span m='2213000'>sequencing\
  \ of the first yeast, and that the reason these things are</span> <span m='2215000'>called\
  \ two separate genes, is that somebody made a sequencing</span> <span m='2217000'>error\
  \ that got a stop codon here, but I think these are really part of</span> <span\
  \ m='2218000'>one gene. And so, somebody went back and re-sequenced</span> <span\
  \ m='2220000'>some of these, and sure enough,</span> <span m='2222000'>he had correctly\
  \ predicted that there had been a mistake made at</span> <span m='2224000'>that\
  \ letter, and that these were in fact, a single gene.</span> </p><p><span m='2226000'>The\
  \ computational analysis was incredibly powerful in this regard,</span> <span m='2231000'>it\
  \ could go further than this, you could ask, could I also figure out</span> <span\
  \ m='2237000'>the way genes are regulated in this fashion, could I work out the</span>\
  \ <span m='2243000'>intergenic signals in the promoter regions? Remember that lac</span>\
  \ <span m='2249000'>repressor to a certain operator site, well, all of these regulatory</span>\
  \ <span m='2255000'>proteins bind to different sequences, could we figure out what\
  \ the</span> <span m='2261000'>sequences were, computational? Well, if we look closely\
  \ at a genic,</span> <span m='2266000'>intergenic region, here's one where there's\
  \ two genes being transcribed</span> <span m='2270000'>in opposite directions, gal-1\
  \ and gal-10, both involved in</span> <span m='2274000'>galactose metabolism, and\
  \ there's a particular protein,</span> <span m='2278000'>a transcription factor\
  \ here, called Gal-4, in this region,</span> <span m='2283000'>and it has a particular\
  \ sequence that it likes,</span> <span m='2287000'>CCG, 11 bases, GGC. So, that\
  \ Gal-4 we see,</span> <span m='2291000'>is very well preserved across all of the\
  \ species.</span> </p><p><span m='2296000'>So, in no regulatory sequence is well-preserved,</span>\
  \ <span m='2300000'>now let's look at that closely. This Gal-4 binding site is a\
  \ measly,</span> <span m='2304000'>crummy, six nucleotides of information. At random,</span>\
  \ <span m='2309000'>it's going to occur in many places in the yeast genome,</span>\
  \ <span m='2313000'>but not be a real, important Gal-4, right? Some of them matter,\
  \ some of</span> <span m='2318000'>them don't. How do we figure out which of these\
  \ occurrences are real</span> <span m='2322000'>Gal-4, well, if we look across all\
  \ four species, what we find is that</span> <span m='2326000'>those occurrences\
  \ that occur in promoter regions,</span> <span m='2331000'>are much more likely\
  \ to be conserved by evolution than those</span> <span m='2335000'>that don't. So\
  \ there's a special property here,</span> <span m='2340000'>conservation of the\
  \ motif and the motor regions.</span> </p><p><span m='2344000'>In fact, this particular\
  \ sequence is four times more likely to be</span> <span m='2348000'>preserved when\
  \ it occurs in a promoter region,</span> <span m='2352000'>than when it occurs in\
  \ a coded region. And for a typical control</span> <span m='2356000'>region, the\
  \ opposite is true. Since genes, since coding sequences</span> <span m='2360000'>are\
  \ better preserved in general, for a randomly chosen sequence, I</span> <span m='2364000'>don't\
  \ know, ATGGCAT, it's more likely to be preserved in</span> <span m='2368000'>coding\
  \ regions than non-coding regions.</span> </p><p><span m='2372000'>So this Gal-4\
  \ motif has a very funky property that,</span> <span m='2375000'>on average, it's\
  \ 12x more likely than background,</span> <span m='2378000'>to be preserved when\
  \ it occurs in a promoter. Now,</span> <span m='2381000'>that's a test you apply\
  \ to another motif, and another motif.</span> </p><p><span m='2384000'>In fact,\
  \ you could, by computer, test all possible motifs, and ask</span> <span m='2387000'>which\
  \ ones have that property? Make a scatter plot, most motifs</span> <span m='2390000'>are\
  \ better conserved when they occur in promoter regions,</span> <span m='2393000'>than\
  \ when they occur in coding regions, some however,</span> <span m='2396000'>are\
  \ better preserved in promoter regions than in coding regions.</span> </p><p><span\
  \ m='2400000'>Our friend, Gal-4, is up there, but there are a lot</span> <span m='2404000'>more\
  \ things like it, that are better preserved by</span> <span m='2409000'>evolution\
  \ than promoters are. You can make a list of them. You</span> <span m='2414000'>can\
  \ get about 72 well-conserved, regulatory motifs and it turns out</span> <span m='2419000'>that\
  \ 20 years of yeast work produced knowledge about things like the</span> <span m='2424000'>Gal-4\
  \ site, and other sites. Almost all the known regulatory</span> <span m='2429000'>sites\
  \ that had been discovered over the course of 20 years of</span> <span m='2434000'>experimental\
  \ work appear on this list that falls out of the computer</span> <span m='2439000'>analysis\
  \ of evolutionary comparison of genomes.</span> </p><p><span m='2444000'>You can\
  \ actually go a step further, I'll hesitate to tell you, but I'll</span> <span m='2448000'>try\
  \ anyway. If you wanted to find out, without knowing in advance,</span> <span m='2453000'>what\
  \ these motifs were doing, what their biological function was,</span> <span m='2457000'>you\
  \ can do that informationally, too. It turns out that if I take my</span> <span\
  \ m='2462000'>motif, Gal-4, and I ask, which chains does it occur in front</span>\
  \ <span m='2466000'>of? Well, across Saccharomyces cerevisiae, you find this crummy</span>\
  \ <span m='2471000'>little motif in many, many places because, as I said,</span>\
  \ <span m='2475000'>most of it's just noise. But if I ask, which genes have this</span>\
  \ <span m='2480000'>motif in all four species, these genes, there's a huge overlap</span>\
  \ <span m='2484000'>with a class of genes involved in carbohydrate metabolism.</span>\
  \ </p><p><span m='2488000'>So, if I didn't know in advance that the Gal-4 motif\
  \ was involved in</span> <span m='2493000'>regulating genes in carbohydrate metabolism,\
  \ I could tell,</span> <span m='2497000'>just from the fact that the genes that'd\
  \ conserved it,</span> <span m='2501000'>are genes involved in carbohydrate metabolism.</span>\
  \ </p><p><span m='2506000'>You can do that using all sorts of tricks, expression\
  \ of genes,</span> <span m='2510000'>protein mass spec, blah, blah, blah, and the\
  \ short answer is, for</span> <span m='2514000'>almost all of those motifs that\
  \ you can find in the computer,</span> <span m='2518000'>by consulting public data\
  \ bases of sets of genes that are co-expressed,</span> <span m='2522000'>or have\
  \ similar properties and all that, the computer can also offer</span> <span m='2526000'>you\
  \ a pretty good hypothesis about what that motif is associated with.</span> </p><p><span\
  \ m='2530000'>You can even go a step further than that. You can begin to look at</span>\
  \ <span m='2534000'>pairs of motifs, you can say, if I have a certain</span> <span\
  \ m='2538000'>regulatory sequence, number one, and a second regulatory</span> <span\
  \ m='2543000'>sequence, number two, do they tend to be preserved in</span> <span\
  \ m='2547000'>front of the same genes as each other? Is their conservation</span>\
  \ <span m='2551000'>correlated? And you can build a map of these two</span> <span\
  \ m='2556000'>guys tend, when this guy's correlated, this guy tends to be</span>\
  \ <span m='2560000'>correlated. And you can say, oh those proteins must be talking\
  \ to</span> <span m='2564000'>each other, and you can read that off from the patterns\
  \ of evolution,</span> <span m='2568000'>as well. There are two regulators, one\
  \ called Sterile 12,</span> <span m='2572000'>one called Tec1. This computational\
  \ analysis shows that they tend to</span> <span m='2577000'>co-occur in a conserved\
  \ fashion, far more often then you'd expect by</span> <span m='2581000'>chance.\
  \ And when you do the analysis, you find that those genes</span> <span m='2585000'>that\
  \ just have a conserved Sterile 12, those genes tend to</span> <span m='2589000'>be\
  \ involved in mating. Genes that just have a conserved</span> <span m='2593000'>instance\
  \ of Tec1 tend to be involved in the budding of the yeast,</span> <span m='2596000'>and\
  \ those genes that have conserved the occurrences of both tend to be</span> <span\
  \ m='2600000'>involved in fillamentation. Now all that can be read out,</span> <span\
  \ m='2603000'>which is way cool, this is not the way we used to do biology.</span>\
  \ </p><p><span m='2606000'>Now don't get me wrong, there's a ton of experiments\
  \ that underlay</span> <span m='2610000'>creating these databases, and there's a\
  \ ton of experiments</span> <span m='2613000'>that have to be done to check any\
  \ of these things. But what we have is</span> <span m='2616000'>one of the most\
  \ powerful hypothesis generators that's ever</span> <span m='2620000'>been seen\
  \ here. Evolution, by telling us what to</span> <span m='2624000'>focus on, is giving\
  \ us, on a silver platter, hundreds of</span> <span m='2628000'>hypothesis about\
  \ who's interacting with whom, and sending us back to</span> <span m='2632000'>the\
  \ lab then, to test these hypotheses. Now,</span> <span m='2636000'>what are the\
  \ implications of all of this for the human genome?</span> </p><p><span m='2640000'>Could\
  \ we do this for the human genome? Well,</span> <span m='2644000'>these species,\
  \ Saccharomyces cerevisiase, S.</span> </p><p><span m='2648000'>paradoxus, S. mikatae\
  \ and S. bayanus, are they a good model for</span> <span m='2652000'>mammals? Well\
  \ it turns out that their</span> <span m='2655000'>evolutionary distance from each\
  \ other is the same as the distance of</span> <span m='2659000'>human to lemur,\
  \ to dog, to mouse.</span> </p><p><span m='2663000'>So they were chosen with a purpose.\
  \ Those are actually fairly good</span> <span m='2667000'>models for the human.\
  \ So could we do exactly the same</span> <span m='2670000'>analysis for the human,\
  \ for the entire human genome?</span> </p><p><span m='2674000'>If we had, human,\
  \ lemur, dog, and mouse, are basically four</span> <span m='2678000'>species, human,\
  \ mouse, rat, and dog.</span> </p><p><span m='2682000'>Well, there's one little\
  \ fly in the ointment. The human genome is 20x</span> <span m='2686000'>bigger than\
  \ the yeast genome. If I want to analyze the whole</span> <span m='2690000'>human\
  \ genome, I have a problem of signal-to-noise.</span> </p><p><span m='2694000'>The\
  \ genome is 20x bigger, I've got 20x as much noise to get</span> <span m='2698000'>rid\
  \ of. I won't walk you through it, but I need more evolutionary</span> <span m='2703000'>information\
  \ to get rid of all that noise. And, you can do a simple</span> <span m='2707000'>calculation\
  \ that says, my evolutionary tree needs to be</span> <span m='2711000'>bigger, it's\
  \ branch length needs to be bigger by about the natural log</span> <span m='2715000'>of\
  \ 20, to get rid of 20 fold more noise.</span> </p><p><span m='2720000'>And that\
  \ would mean I'd need more species, I'd need about 16 species,</span> <span m='2724000'>or\
  \ something like that to be able to do that. But if I built an</span> <span m='2728000'>evolutionary\
  \ tree that had a branch length of four,</span> <span m='2732000'>that is, four\
  \ substitutions per base across this evolutionary tree,</span> <span m='2736000'>as\
  \ indicated by these colored lines here, I should have enough power to</span> <span\
  \ m='2740000'>analyze the entire human genome, the way we just did the yeast genome.</span>\
  \ </p><p><span m='2744000'>So we currently have human, chimp, mouse, rat, dog. As\
  \ of this</span> <span m='2748000'>fall, during in fact, right at the beginning\
  \ of this term,</span> <span m='2752000'>the National Institute of Health signed\
  \ off on the sequencing of</span> <span m='2756000'>these additional eight mammals.\
  \ These mammals are now in process,</span> <span m='2760000'>and in fact, the elephant\
  \ is done, and the armadillo is in process,</span> <span m='2764000'>and the tree\
  \ shrew, I think, is being caught at the moment.</span> </p><p><span m='2768000'>[LAUGHTER].\
  \ The ten-, don't talk about the tree</span> <span m='2772000'>shrews. The tenrec\
  \ is actually being tested right now,</span> <span m='2778000'>etc, and all this\
  \ is going on right now, as we speak,</span> <span m='2784000'>and I think that\
  \ by next summer, we should have much of, and by</span> <span m='2789000'>certainly,\
  \ by a year from now, we should have all this information</span> <span m='2795000'>to\
  \ do such an analysis. That said, we're of course,</span> <span m='2801000'>very\
  \ impatient people, you could just take the human,</span> <span m='2807000'>the\
  \ mouse, the rat, and the dog. And I said that's not enough if you</span> <span\
  \ m='2811000'>wanted to analyze the whole genome, but suppose you just wanted to</span>\
  \ <span m='2815000'>analyze a portion of the genome, maybe about a yeast-size piece\
  \ of</span> <span m='2819000'>the genome, well let's see, at 20,000 genes, I don't\
  \ know,</span> <span m='2823000'>suppose I take, I don't know, two kilo bases around\
  \ each 20,</span> <span m='2826000'>00 genes, well that's you know, 40 mega bases\
  \ of DNA, it's only a</span> <span m='2830000'>couple-fold more than yeast. Maybe,\
  \ if I just focus on a limited</span> <span m='2834000'>region around each promoter,\
  \ I could start reading out these</span> <span m='2838000'>regulatory signals, with\
  \ just four species.</span> </p><p><span m='2842000'>So in fact, the post-doctorate\
  \ fellow is, has been working on this</span> <span m='2846000'>problem over the\
  \ summer, and a little bit, too, through the</span> <span m='2850000'>spring and\
  \ summer, together with Manolis Kellis,</span> <span m='2854000'>who's now in the\
  \ computer science department. And I think we have a</span> <span m='2858000'>preliminary\
  \ list for the human genome that's fallen out over the</span> <span m='2862000'>course\
  \ of the past couple of months, and we're in the process, right now,</span> <span\
  \ m='2866000'>of finishing up a paper that we're hoping to get submitted by Friday,</span>\
  \ <span m='2870000'>with a preliminary list of regulatory signals in the human</span>\
  \ <span m='2874000'>genome, read out from evolution of human, mouse, rat, and dog.</span>\
  \ </p><p><span m='2878000'>It won't be everything, we don't have full power to pick\
  \ up</span> <span m='2881000'>all possible signals, but we're picking up a lot of\
  \ the</span> <span m='2884000'>signals, we're picking up a very large fraction of\
  \ previously</span> <span m='2888000'>discovered signals, and lots more new signals,</span>\
  \ <span m='2891000'>as well, are falling out of that analysis. So anyway,</span>\
  \ <span m='2894000'>I can assure you that that's not in the textbooks because,</span>\
  \ <span m='2898000'>actually, it hasn't been submitted yet. This other stuff I've</span>\
  \ <span m='2901000'>described about the yeast analysis, this, you do want to look\
  \ it up,</span> <span m='2905000'>there's a paper in nature about a year and change\
  \ ago,</span> <span m='2908000'>Kellis et. al. describes this yeast work. This is\
  \ what's going on.</span> </p><p><span m='2912000'>This is what's fun about teaching\
  \ at MIT, as I can tell you this stuff,</span> <span m='2916000'>and you guys have\
  \ a sense for the convergence that's going on in our</span> <span m='2921000'>field.\
  \ Much of what I've tried to make the biology,</span> <span m='2925000'>you know,\
  \ in making the biology clear, I've talked about how the</span> <span m='2930000'>different\
  \ directions, genetics, biochemistry,</span> <span m='2934000'>have converged together.\
  \ What we're really seeing now is</span> <span m='2939000'>information sciences\
  \ converging with that as well, and I've got to say,</span> <span m='2943000'>it's\
  \ a tremendous amount of fun. See you on Monday, good</span> <span m='2948000'>luck\
  \ on the quiz.</span> </p>"
type: course
uid: 73f7ec3d0965a2f067029ba24b282860

---
None