---
about_this_resource_text: <p><strong>Topics covered:</strong> Recombinant DNA 3</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec17.jpg
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-17-recombinant-dna-3/lec17.jpg
  title: lec17.jpg
  type: null
  uid: fb1adcaa777f35257408a2387e4d6982
- id: Video-YouTube-Stream
  media_location: Rqs_zVh5sr8
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Video-YouTube-Stream
  type: Video
  uid: 3f22090d47262f0d24b449a8ed5d0c95
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/Rqs_zVh5sr8/default.jpg
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 8ad77f7ab2d95f23458a10b93877b6ac
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Video-iTunes U-MP4
  type: Video
  uid: fddefeac6f470c5e37dd4be4be4c7115
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec17-20oct2004-220k.mp4
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Video-Internet Archive-MP4
  type: Video
  uid: a76aea815348ac1dc882403c4a2e2090
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Audio-iTunes U-MP3
  type: Audio
  uid: e4e9cde2f490c2d167f58eb991f9d1a0
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-20oct2004-1000.mp3
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Video-Internet Archive-MP3
  type: Video
  uid: b50616e58fb986678940bbe8ff797490
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 822c90da7e5a4b0ffd429293f9a31250
- id: Thumbnail-OCW-JPG
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: d1cab4f9604e99b6758f239102dddf7e
- id: 3Play-3PlayYouTubeid-MP4
  media_location: Rqs_zVh5sr8
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: 84c052e81f407d9d9b33fc61fe2a77cb
- id: Rqs_zVh5sr8.srt
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-17-recombinant-dna-3/Rqs_zVh5sr8.srt
  title: 3play caption file
  type: null
  uid: 619e55a3b8f4e729213f0cc341cfc3b0
- id: Rqs_zVh5sr8.pdf
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-17-recombinant-dna-3/Rqs_zVh5sr8.pdf
  title: 3play pdf file
  type: null
  uid: e3a035c56e739e03fdf3bf72109e0792
- id: Caption-3Play YouTube id-SRT
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: bbade54f0b583b485432b54e76cc6e36
- id: Transcript-3Play YouTube id-PDF
  parent_uid: 15a752d2d7ccfb4cea41ceb95b805b34
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: 6575cddc65227ba96334893fada1abab
inline_embed_id: 85646088lecture17:recombinantdna312505920
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-17-recombinant-dna-3
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-17-recombinant-dna-3
template_type: Tabbed
title: 'Lecture 17: Recombinant DNA 3'
transcript: '<p><span m=''1000''>Good morning. Good morning.</span> </p><p><span m=''5000''>I
  don''t know about you, but I can''t take too many more nights like this.</span>
  </p><p><span m=''10000''>I confess, I haven''t gotten a thing done for so many nights
  in a row now,</span> <span m=''15000''>but what a game! How many of you saw the
  game? Excellent.</span> </p><p><span m=''20000''>Very good, very good. You have
  your priorities straight in</span> <span m=''25000''>the world. Very good. Well,
  if it''s possible to get your</span> <span m=''30000''>minds off Curt Schilling
  last night, and off more importantly tonight.</span> </p><p><span m=''35000''>Perhaps
  we can spend a bit of time this morning in the meanwhile with</span> <span m=''40000''>whatever
  spare neurons you have talking about recombinant DNA for a</span> <span m=''45000''>bit,
  OK? What we talked about last time was different ways to clone</span> <span m=''50000''>your
  gene based on its properties. We started off with cloning by</span> <span m=''55000''>complementation,
  right, the idea that if you took a</span> <span m=''60000''>library of clones, you
  would be able to put it into</span> <span m=''65000''>bacteria and select a bacterium
  whose phenotype had been restored by</span> <span m=''70000''>virtue of having the
  plasmid. You would complement the defect.</span> </p><p><span m=''74000''>You''d
  find the clone you wanted because it complemented the defect.</span> </p><p><span
  m=''78000''>That''s great if you can put it into an organism that has a defect.</span>
  </p><p><span m=''82000''>You can do it with bacteria. You can do that with yeast.</span>
  </p><p><span m=''85000''>It''s harder to do with large organisms because you can''t
  inject</span> <span m=''89000''>enough of them with different clones to be able
  to make that practical</span> <span m=''93000''>unless you''re working in cell culture
  or some very small,</span> <span m=''97000''>fast growing organism. We talked about
  being able to use a</span> <span m=''101000''>protein sequence, reverse translating
  that protein</span> <span m=''105000''>sequence in the computer from amino acid
  sequence to nucleotide sequence,</span> <span m=''110000''>and using the nucleotide
  sequence to design a probe to hybridize back to</span> <span m=''114000''>the genome.
  That works fine if you have a protein sequence.</span> </p><p><span m=''119000''>But
  the last topic we talked about that I wanted to just touch on again</span> <span
  m=''122000''>this morning was suppose you were trying to clone the gene that causes</span>
  <span m=''125000''>a certain human disease, and you have no idea what the</span>
  <span m=''128000''>protein was. Then, you can''t use its amino acid</span> <span
  m=''131000''>sequence because you don''t have the protein. What can you possibly
  do</span> <span m=''135000''>when all you know is that you have a gene which causes
  a genetic defect</span> <span m=''138000''>that causes a disease? And I said you
  could clone it using</span> <span m=''141000''>the ideas of genetic mapping, position,
  the things that Sturtevant</span> <span m=''144000''>developed. And, I touched on
  it briefly,</span> <span m=''148000''>and I want to just touch on it a bit more
  because some people had some</span> <span m=''151000''>questions about it. And I''ve
  set up a very simple</span> <span m=''154000''>example to show you. Suppose that,
  to make it easy,</span> <span m=''158000''>we''re working in a fruit fly first.
  We''re working drosophila, and</span> <span m=''161000''>suppose that the true picture
  of the underlying chromosome is like this.</span> </p><p><span m=''164000''>There''s
  a locus that could either have a mutant allele M or the wild</span> <span m=''168000''>type
  allele plus. There''s a bunch of other loci along</span> <span m=''171000''>the
  chromosome. And, let''s suppose we know all of</span> <span m=''174000''>where they
  are and all that. And, they have two alternative</span> <span m=''178000''>alleles.
  At this locus the alleles are orange</span> <span m=''181000''>or pink. At this
  locus I''ll call the alleles orange or pink.</span> </p><p><span m=''185000''>Now,
  these are different loci. These are different alleles. I''ve</span> <span m=''188000''>just
  called them orange and pink in both cases so I don''t have a rainbow</span> <span
  m=''191000''>of colors up here to confuse us. But all I mean is there''s two</span>
  <span m=''195000''>possible alleles here, two alleles here, two alleles here.</span>
  </p><p><span m=''198000''>This is the diseased gene we''re interested in,</span>
  <span m=''201000''>and these are passive markers. These are other markers along
  the</span> <span m=''205000''>chromosome. If we were to set up a cross between</span>
  <span m=''208000''>heterozygotes, a heterozygote here,</span> <span m=''212000''>and
  a heterozygote here, and it were the case that on the</span> <span m=''216000''>chromosome
  bearing the mutant allele, it happened that at these three</span> <span m=''220000''>markers
  we had orange alleles. I don''t know what they are, but</span> <span m=''224000''>whatever
  these orange alleles are, they might be a visible phenotype,</span> <span m=''228000''>forked
  or yellow or bristled. They could be a DNA sequence</span> <span m=''232000''>difference.
  They could be whatever you want, but let''s suppose the M</span> <span m=''236000''>chromosome
  has a set of alleles that are different in each location than</span> <span m=''240000''>the
  plus chromosome. Then, when we look at the offspring</span> <span m=''243000''>that
  come out of this cross, let''s only, for the sake of</span> <span m=''247000''>simplicity,
  look at those offspring who are homozygous mutants.</span> </p><p><span m=''251000''>Well,
  in general, if there''s been no crossover here,</span> <span m=''255000''>then the
  M chromosome will have orange, orange,</span> <span m=''258000''>orange, orange,
  orange, orange. If there''s been a crossover,</span> <span m=''262000''>however,
  it could go orange, orange, pink on one of those</span> <span m=''266000''>chromosomes.
  Or if there''s been crossovers like this,</span> <span m=''270000''>it could go
  orange, orange, pink on one chromosome, and orange,</span> <span m=''274000''>pink,
  pink on the other chromosome. It could even,</span> <span m=''277000''>in the extreme,
  have had crossovers very close to</span> <span m=''281000''>the gene maybe here,
  and even maybe here. And you''ve got</span> <span m=''284000''>orange, pink, pink,
  and pink, pink, pink.</span> </p><p><span m=''287000''>But if we look at the many
  segregates, you know from genetic</span> <span m=''291000''>mapping that the closer
  the locus is to the disease gene,</span> <span m=''294000''>the more strongly correlated
  the inheritance will be,</span> <span m=''297000''>the tighter the linkage will
  be. This is nothing more than linkage</span> <span m=''301000''>mapping. But now,
  suppose we were doing</span> <span m=''304000''>linkage mapping, but for the sake
  of argument the</span> <span m=''308000''>whole genome had already been sequenced.
  Suppose the genome had</span> <span m=''312000''>been sequenced in a cross, and
  the whole genome of the fruit</span> <span m=''315000''>fly had been sequenced which
  it has been sequenced.</span> </p><p><span m=''319000''>And, we looked at a cross
  and we looked at the mutants.</span> </p><p><span m=''322000''>And what we did was
  we tried different positions along the genome.</span> </p><p><span m=''326000''>And
  at each position, we had some genetic marker.</span> </p><p><span m=''330000''>And
  that genetic marker might be as simple as the fact that at that</span> <span m=''334000''>position,
  maybe there is an A in the DNA sequence on one of the</span> <span m=''338000''>chromosomes,
  and maybe I don''t know a G in the other sequence.</span> </p><p><span m=''342000''>And
  over here, this marker might be, there''s a T in some particular</span> <span m=''347000''>position,
  and there''s a C in some particular position.</span> </p><p><span m=''351000''>If
  we could assay that, if we could tell, we could look</span> <span m=''355000''>whether
  this spelling variation is closely correlated with the mutant.</span> </p><p><span
  m=''360000''>And this spelling variation is closely correlated with the</span> <span
  m=''363000''>inheritance of the mutant allele. And we could just try up and down</span>
  <span m=''367000''>the genome, different sites of spelling difference as if they
  were</span> <span m=''371000''>genetic markers in our cross because they are genetic
  markers in our</span> <span m=''375000''>cross, and see which one is most tightly
  correlated.</span> </p><p><span m=''378000''>The minute we get any genetic sequence
  difference,</span> <span m=''382000''>that shows co-inheritance linkage in this
  cross, we know that this spot</span> <span m=''386000''>in the genome must be nearby
  our mutation.</span> </p><p><span m=''390000''>So, we''ll try one closer, and we''ll
  try one on the other side.</span> </p><p><span m=''393000''>And, what you do is
  you test sites of genetic variation,</span> <span m=''397000''>first to find one
  that shows any co-inheritance.</span> </p><p><span m=''400000''>And once you''ve
  got that, you try ones closer, and closer,</span> <span m=''404000''>and closer.
  Last time I talked about the process of,</span> <span m=''408000''>if you had one
  of those markers you could use it to isolate the next</span> <span m=''411000''>clone
  and the next clone and the next clone. But you know what I</span> <span m=''415000''>realized?
  That''s so old fashioned. We might as well deal with the fact</span> <span m=''419000''>we
  have a sequence of the genome. No more would you ever isolate the</span> <span m=''422000''>next
  clone and the next clone and the next clone.</span> </p><p><span m=''425000''>You
  just look it up in the computer. So, even if you have the whole</span> <span m=''428000''>sequence
  of the genome, we have to figure out what part of</span> <span m=''431000''>it was
  co-inherited along with this disease, and that''s the way you do</span> <span m=''434000''>it,
  OK? Genetic mapping, ust as Sturtevant invented it,</span> <span m=''437000''>can
  be applied if you have a whole sequence of the genome,</span> <span m=''440000''>and
  enough sites of variation. And, I''ve drawn it for a fruit fly</span> <span m=''443000''>cross,
  but this could equally well be cystic fibrosis.</span> </p><p><span m=''446000''>The
  only difference is if we''re doing this in human families and</span> <span m=''449000''>it''s
  cystic fibrosis we don''t have as many offspring.</span> </p><p><span m=''453000''>So,
  we have to pool data from many families. And,</span> <span m=''456000''>we can''t
  arrange it so that every family has exactly the same orange</span> <span m=''460000''>alleles
  up here and pink alleles down there, but computers can deal</span> <span m=''463000''>with
  that. They can still figure out the correlation across many</span> <span m=''467000''>families,
  and you find the spot in the genome where for many,</span> <span m=''470000''>many,
  many families the kids who all got the disease show correlated</span> <span m=''474000''>inheritance
  with this marker. And that eventually pins you down</span> <span m=''477000''>to
  a region of the genome. It pins you down to those genetic</span> <span m=''481000''>markers
  that show the absolute tightest correlation,</span> <span m=''484000''>tight correlation,
  and that''s where you look.</span> </p><p><span m=''488000''>And in that fashion,
  people went being able to map the</span> <span m=''492000''>location of Huntington''s
  Disease in 1984 to, by now,</span> <span m=''496000''>mapping the locations of more
  than 1, 00 different human genetic diseases</span> <span m=''500000''>where people
  didn''t know the protein in advance. They did it entirely</span> <span m=''505000''>based
  on this positional mapping. So, Sturtevant''s idea, which I like</span> <span m=''509000''>so
  much, has played itself out so beautifully now in the area of</span> <span m=''513000''>modern
  molecular medicine. OK. So, onward.</span> </p><p><span m=''518000''>I want to talk
  about a few other variations on the theme rather</span> <span m=''523000''>quickly,
  and then I think I want to talk about how you analyze your</span> <span m=''528000''>clones.
  First, variations on cloning,</span> <span m=''533000''>I should just at least mention
  it. We talked about cloning in an</span> <span m=''538000''>autonomously replicating
  plasmid in a bacteria.</span> </p><p><span m=''544000''>So, you go to a bacteria.
  They have some autonomously</span> <span m=''547000''>replicated pieces of DNA.
  There are circles. You can clone</span> <span m=''550000''>in them, and you can
  typically, these things are on the order of,</span> <span m=''553000''>I don''t
  know, 1,000 to 2,000 to 5, 00 bases can be readily cloned in</span> <span m=''557000''>these
  plasmids. You can do more,</span> <span m=''560000''>but that''s a typical kind
  of number is the insert size,</span> <span m=''563000''>typically. But we in the
  lab go up to much higher numbers</span> <span m=''567000''>like 10,000 sometimes.
  You can also, if you wanted to study</span> <span m=''571000''>yeast, it turns out
  yeast happily have plasmids as well,</span> <span m=''576000''>and you can do a
  similar sort of thing for yeast.</span> </p><p><span m=''581000''>It turns out that
  instead of using plasmids, you can use bacterial</span> <span m=''587000''>viruses.
  These bacterial viruses have all different shapes as we''ve</span> <span m=''592000''>talked
  about, circular or linear, and they can typically hold, oh, 15,</span> <span m=''597000''>00-40,000.
  Some of these viruses are quite big.</span> </p><p><span m=''602000''>The bacteriophage
  lambda tends to carry a lot of stuff.</span> </p><p><span m=''606000''>And, it can
  replicate. So, you could do the same thing to</span> <span m=''611000''>that. You
  can even use viruses that infect mammalian cells and there are</span> <span m=''615000''>all
  sorts of viruses now that people clone in again,</span> <span m=''619000''>linear
  or circular. I don''t know, for mammalian cells,</span> <span m=''624000''>you often,
  the viruses like 1,000-5, 00. You can even make artificial</span> <span m=''628000''>whole
  chromosomes now. You can do this in yeast.</span> </p><p><span m=''633000''>Artificial
  chromosomes are called YACs. They have all the little</span> <span m=''639000''>machinery,
  little telomeres on them, little centromeres. They have a</span> <span m=''644000''>selectable
  marker, and then you can clone into it your</span> <span m=''650000''>piece of DNA.
  And these can take up to a million bases of DNA.</span> </p><p><span m=''655000''>So,
  if you wanted, there are bacterial artificial chromosomes.</span> </p><p><span m=''661000''>They''re
  called BACs if they''re in bacteria. And recently,</span> <span m=''665000''>people
  have developed artificial chromosome systems for mammalian</span> <span m=''669000''>cells,
  and specifically human cells. And they''re called unfortunately</span> <span m=''673000''>MACs
  and HACs and things like that. Basically, any molecule that can</span> <span m=''677000''>replicate
  in any system, some smart molecular biologist will</span> <span m=''681000''>come
  along and say, how do I use that for my purpose,</span> <span m=''685000''>to stick
  my DNA in it, and get it to replicate in this organism?</span> </p><p><span m=''690000''>And
  so, if something''s not on this list, it will be soon,</span> <span m=''696000''>OK?
  Now, here''s another thing. This is cloning chunks of DNA.</span> </p><p><span m=''703000''>Just
  to have the piece of DNA in a library, but suppose we want to do</span> <span m=''710000''>more
  than just have the DNA sitting there in the bacterium,</span> <span m=''717000''>suppose
  what I''d really like to do is take a bacterium,</span> <span m=''724000''>E coli,
  and put it to work for us. Maybe what I''d like to do is take a</span> <span m=''730000''>plasmid
  and insert in that plasmid the gene for human insulin.</span> </p><p><span m=''734000''>So,
  I''m going to take the DNA locus corresponding to human insulin,</span> <span m=''739000''>clone
  it into my plasmid. Maybe I''ll have isolated it from my</span> <span m=''743000''>library
  because, let''s see, insulin''s protein</span> <span m=''748000''>sequence is known
  so I could reverse translate it to a nucleotide</span> <span m=''752000''>sequence.
  So, I could probe a library.</span> </p><p><span m=''756000''>So, I could find the
  clone that has insulin. Now what I''d like to do is</span> <span m=''760000''>persuade
  this bacteria not just to carry the DNA but to make insulin</span> <span m=''763000''>for
  me. Would that be useful? Yeah, how did people used to get</span> <span m=''767000''>insulin?
  Cadavers, dead bodies; it would be much easier</span> <span m=''771000''>to get
  them from a fermenter, right, to get insulin from a</span> <span m=''774000''>fermenter,
  if you could just ask E coli to make it.</span> </p><p><span m=''778000''>So, if
  we put it into E coli, will it make insulin for us?</span> </p><p><span m=''782000''>Here''s
  the human locus, DNA for insulin. Will it make</span> <span m=''789000''>insulin?
  Let''s see, how do you make a protein?</span> </p><p><span m=''797000''>You''ve
  got to start by making RNA, right? You''ve got to transcribe the</span> <span m=''804000''>gene.
  Will E coli transcribe this gene?</span> </p><p><span m=''812000''>Well, why? It''s
  got a promoter, right? It''s got the insulin</span> <span m=''816000''>promoter.
  There we go. The insulin promoter is here.</span> </p><p><span m=''821000''>So,
  E coli will come along to the insulin promoter and start making</span> <span m=''825000''>RNA?
  No, it turns out that promoters in humans and promoters in</span> <span m=''830000''>bacteria
  are sufficiently different. They don''t work across species.</span> </p><p><span
  m=''835000''>They won''t recognize the human promoter. Too bad. Any ideas?</span>
  </p><p><span m=''840000''>Yep? Stick a bacterial promoter there. Good, you''re acting
  like a</span> <span m=''845000''>good molecular biology designer here. Let''s put
  a bacterial promoter here.</span> </p><p><span m=''850000''>It will recognize its
  own promoter. That''s great. Then, let''s put the</span> <span m=''855000''>DNA
  for the human insulin gene here. And now, maybe we''ll put the Lac</span> <span
  m=''861000''>operon, and when it has lactose it''ll start making RNA from the</span>
  <span m=''866000''>human insulin gene. And it''ll start translating it.</span> </p><p><span
  m=''872000''>And, we get insulin. Any problems? Well,</span> <span m=''878000''>will
  it make any, for starters? What''s another aspect of mammalian</span> <span m=''884000''>genes
  that''s different from bacterial genes?</span> </p><p><span m=''890000''>Processing,
  what kind of processing with the RNA? And the splicing,</span> <span m=''896000''>ooh,
  the insulin gene has introns that have to be spliced out.</span> </p><p><span m=''902000''>So,
  this is going to make some RNA, insulin RNA, and it needs to be</span> <span m=''906000''>processed
  like this. Will bacteria carry on our splicing</span> <span m=''910000''>for us?
  They don''t do splicing. Yep? Well, that''s a very</span> <span m=''914000''>interesting
  question because we haven''t. But,</span> <span m=''918000''>what do you propose?
  You see, I''ve just taken a piece of</span> <span m=''922000''>human DNA from the
  human genome, which encodes the introns and the</span> <span m=''926000''>exons.
  But, you seem to have a solution to our problem,</span> <span m=''930000''>and what
  would that be? So, instead of making a library of</span> <span m=''935000''>genomic
  DNA, what you''re suggesting is a radical idea.</span> </p><p><span m=''942000''>Let''s
  instead take human RNA. Here''s some human RNA, lots of</span> <span m=''950000''>human
  RNA, a big collection of human RNA. What was at the end of the</span> <span m=''957000''>human
  RNA: a poly(A) tail. And what I understand you to be</span> <span m=''962000''>suggesting
  is if we take human mRNAs, a whole collection of them,</span> <span m=''966000''>you
  want me to turn these mRNAs back into DNA and clone them instead of</span> <span
  m=''970000''>using the chromosomal DNA. How do I turn an RNA back to DNA?</span>
  </p><p><span m=''974000''>Is that possible? What do you use: reverse transcriptase.</span>
  </p><p><span m=''978000''>We have to give it a primer. So remember, five prime to
  three</span> <span m=''982000''>prime, we''d like to put a primer going over here.</span>
  </p><p><span m=''986000''>Any ideas for a good primer? Poly(T), isn''t that convenient?</span>
  </p><p><span m=''990000''>One of the reasons that mammalian messages have poly(A)
  tails is so</span> <span m=''995000''>that we are able to reverse transcribe them
  using poly(T)</span> <span m=''1001000''>primers. No, that''s actually not true.
  So, we use reverse</span> <span m=''1006000''>transcriptase. And what we can do
  is we''ll copy</span> <span m=''1012000''>this RNA into a strand of DNA. There we
  go.</span> </p><p><span m=''1018000''>Then what we''ll do, next step, is we''ll
  take the DNA,</span> <span m=''1023000''>and we''ll copy back into a second strand
  of DNA.</span> </p><p><span m=''1029000''>And now, we have double-stranded DNA whose
  sequence matches the</span> <span m=''1035000''>already-processed mRNAs. Sorry?
  So, the sequences would</span> <span m=''1041000''>match the mRNAs. So what you
  could do is instead of</span> <span m=''1047000''>taking human DNA from the nucleus,
  you could take RNAs,</span> <span m=''1052000''>turn them back into DNA by reverse
  transcriptase,</span> <span m=''1058000''>and make a library now that consists of
  zillions of inserts,</span> <span m=''1063000''>each of which has what''s called
  a cDNA, a copied DNA,</span> <span m=''1069000''>copied back from the RNA. The great
  advantage of this is that</span> <span m=''1074000''>the human cell has already
  done the splicing, and so there</span> <span m=''1080000''>are no introns left.
  Now, when you stick it in a</span> <span m=''1085000''>bacterium, the bacterium
  is able to express this. It''s able,</span> <span m=''1089000''>if you give it its
  own bacterial promoter, to make an RNA.</span> </p><p><span m=''1093000''>And if
  you don''t ask the bacteria to have to splice,</span> <span m=''1097000''>if you
  just give it a pre-spliced piece of DNA that doesn''t need</span> <span m=''1101000''>splicing,
  it can translate that DNA. Now, notice we used all of our</span> <span m=''1105000''>tricks.
  You had to know about reverse transcriptase,</span> <span m=''1109000''>poly(A)
  tails, structures of genes, introns, exons, yes, question?</span> </p><p><span m=''1114000''>It
  doesn''t. You do this in the test tube. You purify human mRNA in the</span> <span
  m=''1118000''>test tube. You take that mRNA in a test tube, add reverse transcriptase,</span>
  <span m=''1122000''>add poly(T), make this reaction of RNA to DNA in the test tube
  go back.</span> </p><p><span m=''1127000''>Where does it come from? Viruses that
  copy themselves back for a</span> <span m=''1131000''>living, right? So, again,
  every single thing we''re</span> <span m=''1136000''>using comes from some living
  organism that does this</span> <span m=''1140000''>kind of stuff. And, when I teach
  you about the</span> <span m=''1144000''>facts of how viruses replicate or what
  the structure of mRNAs look</span> <span m=''1148000''>like or whatever, it''s because
  every bit of knowledge</span> <span m=''1151000''>we get about the way biology works
  turns into an incredibly powerful</span> <span m=''1154000''>tool as it''s turning
  out for us to actually be able to further study</span> <span m=''1158000''>biology.
  So, great. So, where does reverse</span> <span m=''1161000''>transcriptase come
  from now? Originally they come from viruses</span> <span m=''1164000''>that turn
  themselves back from RNA to DNA. Now, how do you get reverse</span> <span m=''1168000''>transcriptase?
  Catalog, right,</span> <span m=''1172000''>very good. All right, so this is called,
  finally,</span> <span m=''1178000''>a cDNA library. And, if you had made a cDNA
  library,</span> <span m=''1184000''>you would be able to screen the cDNA library
  to find the gene for insulin.</span> </p><p><span m=''1189000''>Is this useful?
  This happens to be,</span> <span m=''1195000''>for example, one of the consequences
  of this was the biotechnology</span> <span m=''1201000''>industry. OK, so if you
  have any doubts about</span> <span m=''1206000''>the usefulness of understanding
  these abstract things about E coli</span> <span m=''1210000''>and bacteria and stuff
  like that, one of the consequences was</span> <span m=''1214000''>Genentech, Biogen,
  and Amgen, and if you just simply</span> <span m=''1218000''>walk around Kendall
  Square, within a mile of this place you will</span> <span m=''1222000''>see laid
  out before you the consequences of this ability,</span> <span m=''1226000''>OK?
  It''s transforming Cambridge. Yes?</span> </p><p><span m=''1230000''>And the world.
  Yeah. Indeed.</span> </p><p><span m=''1236000''>It might be that producing large
  amounts of insulin was bad for the</span> <span m=''1243000''>bacteria because there
  would be so much protein it would clump and kill</span> <span m=''1250000''>the
  bacteria. It might be that insulin, for various reasons,</span> <span m=''1257000''>might
  not fold appropriately in the bacterial environment.</span> </p><p><span m=''1264000''>And,
  this is why the biotechnology industry has lots of smart people</span> <span m=''1267000''>working
  in it because you''re totally, 100% right. You might decide that</span> <span m=''1270000''>instead
  of cloning it in bacteria it''s better to clone it in some</span> <span m=''1273000''>insect
  cell in culture which, in fact, people like to work with,</span> <span m=''1276000''>or
  some other cell, or a mammalian cell. And so,</span> <span m=''1279000''>I simplify
  by saying put it in coli, but in fact that might test six</span> <span m=''1283000''>different
  cell lines, six different host possibilities.</span> </p><p><span m=''1286000''>They
  might have to take the insulin out and refold it in vitro</span> <span m=''1289000''>and
  things like that. You''re totally right.</span> </p><p><span m=''1293000''>This
  is actually something that requires work to do it right,</span> <span m=''1297000''>just
  like building an airplane requires work.</span> </p><p><span m=''1302000''>I could
  tell you Bernoulli''s principles, but then Boeing does</span> <span m=''1307000''>more
  than just writes down Bernoulli''s principles.</span> </p><p><span m=''1311000''>OK,
  so onward. Now, I''d like to turn next to analyzing your clone.</span> </p><p><span
  m=''1316000''>Analyzing the clone, so suppose we have, maybe it''s by positional</span>
  <span m=''1320000''>cloning, maybe it''s by cDNA cloning, but one way or the other
  we''ve got</span> <span m=''1325000''>us a clone that we''re very interested in.</span>
  </p><p><span m=''1330000''>Maybe it has the insulin gene. Maybe it has the Huntington''s</span>
  <span m=''1334000''>disease gene. Whatever it is, we''re going to want to study
  it.</span> </p><p><span m=''1338000''>And at the moment, I haven''t told you how
  I would even read its DNA</span> <span m=''1342000''>sequence or analyze its DNA.
  So, the first step is, of course,</span> <span m=''1346000''>I have to purify the
  plasmid. And, it turns out that that can be done.</span> </p><p><span m=''1351000''>There
  are simple biochemical techniques, as I mentioned in a</span> <span m=''1354000''>previous
  lecture, that allow you to grow up a lot of</span> <span m=''1357000''>the bacteria,
  crack them open, and the plasmid being a little</span> <span m=''1360000''>circle,
  and being a little more tightly super-coiled and wound up</span> <span m=''1363000''>has
  somewhat different physical properties. And you can use those</span> <span m=''1366000''>to
  purify the plasmid. So, plasmid preps are not hard to</span> <span m=''1370000''>do.
  You can get a fairly pure collection of the plasmid.</span> </p><p><span m=''1373000''>Now,
  suppose I''ve done this for, oh, I don''t know, let''s take my</span> <span m=''1376000''>first
  example, orange mutants. Suppose I tried to rescue bacteria</span> <span m=''1380000''>that
  were orange minus, and suppose I found that 50</span> <span m=''1384000''>different
  plasmids rescued my orange mutant because I transformed a lot</span> <span m=''1388000''>of
  plasmids in, I plated it, and 50 colonies grew up.</span> </p><p><span m=''1392000''>Are
  they all the same thing or are they different?</span> </p><p><span m=''1396000''>Is
  there any quickie way to take a look at these 50 plasmids and see if</span> <span
  m=''1400000''>they''re identical or fairly close, or obviously different? Well,
  I''d</span> <span m=''1404000''>like to take some way to take the DNA from the plasmid
  and analyze it</span> <span m=''1408000''>kind of easily. I might want to see,</span>
  <span m=''1412000''>like, how big is the insert? Right, that''d be one way,</span>
  <span m=''1417000''>if they had different sized inserts so they couldn''t be the
  same thing.</span> </p><p><span m=''1423000''>So, maybe what I could do is how do
  I clone this? I used EcoRI sites I</span> <span m=''1429000''>recall. So, I have
  EcoRI sites here. Suppose I were to take this DNA,</span> <span m=''1435000''>and
  I were to now cut the DNA from the plasmid with EcoRI.</span> </p><p><span m=''1442000''>Then,
  what I would get is two separate molecules.</span> </p><p><span m=''1449000''>I
  would get the vector and the insert. How could I see how big</span> <span m=''1456000''>they
  were? Gels, gel electrophoresis is the way to do</span> <span m=''1464000''>that.
  So, I take a gel. A gel is a slab of gelatin,</span> <span m=''1469000''>Jell-O,
  OK, and normally it''s laid flat, but I''m going to do it</span> <span m=''1473000''>vertically
  here. I load into the top of it here a</span> <span m=''1477000''>little bit of
  my DNA, this whole mixture. I take the</span> <span m=''1481000''>plasmid. I cut
  it. I put it in here. DNA''s positive</span> <span m=''1485000''>charge or negative
  charge? Negative. So, where should I put</span> <span m=''1489000''>the positive
  pull? On the bottom, well done.</span> </p><p><span m=''1493000''>That''s often
  not done, and to the detriment of the experiment.</span> </p><p><span m=''1497000''>If
  you put the positive pull here, it goes the wrong way, and everybody</span> <span
  m=''1501000''>has to do that at least once. So, what''ll happen is the DNA</span>
  <span m=''1505000''>fragments move through, and the smaller fragments move</span>
  <span m=''1511000''>faster than the big fragments, right? If something''s little,
  it''ll</span> <span m=''1516000''>move fast. If something''s big, it moves slowly:
  little, big.</span> </p><p><span m=''1522000''>Smaller moves faster because it wiggles
  through the little pores in</span> <span m=''1527000''>the gel better. So, suppose
  I were to do this for a</span> <span m=''1533000''>bunch of plasmids, and what I
  saw was this.</span> </p><p><span m=''1539000''>First order, what do you guess?
  Sorry? Top road''s probably the</span> <span m=''1547000''>plasmid vector. This
  is probably the vector,</span> <span m=''1555000''>and what do I know about the
  inserts? At least two inserts,</span> <span m=''1563000''>at least two distinct
  inserts. Now, if I wanted to be sure that was</span> <span m=''1569000''>the vector,
  maybe what I could do is take another row,</span> <span m=''1573000''>and run a
  known amount of the vector, take the vector alone and I could</span> <span m=''1577000''>check
  that the vector alone runs over here. And maybe I might take</span> <span m=''1581000''>some
  other known molecules. These would be called molecular</span> <span m=''1585000''>weight
  standards. So, if I run some knowns in one of</span> <span m=''1589000''>the lanes
  of the gel, I can even measure and say,</span> <span m=''1593000''>ah-ha, the insert
  is somewhere between the size of this one and the</span> <span m=''1597000''>size
  of that one. And so, I get a little ruler that I</span> <span m=''1600000''>can
  put on the gel. So, in fact, that''s the first thing</span> <span m=''1603000''>you
  would do is you digest your clone that way.</span> </p><p><span m=''1606000''>Now,
  does the fact that these guys have exactly the same,</span> <span m=''1609000''>apparently,
  size on the gel mean that they''re the exact same piece of</span> <span m=''1612000''>DNA?
  No, because you can''t even actually tell it''s exactly the same.</span> </p><p><span
  m=''1615000''>There''s a limit to how precisely you can measure it.</span> </p><p><span
  m=''1619000''>So, what else could you do? You could try another restriction</span>
  <span m=''1624000''>enzyme. It turns out that since there are so many restriction</span>
  <span m=''1630000''>enzymes in the catalog, if I take a piece of DNA,</span> <span
  m=''1635000''>maybe that Eco fragment, I could try cutting it with HinDIII.</span>
  </p><p><span m=''1641000''>And when I cut it with HinDIII, I''m going to get three
  distinct</span> <span m=''1646000''>lengths. I could try cutting it with, oh, I
  don''t know,</span> <span m=''1652000''>pick another enzyme, BamHI. When I cut it
  with BamHI,</span> <span m=''1657000''>I''ll get some other lengths. And, how to
  get these lengths by</span> <span m=''1663000''>adding these, by running them out
  on a gel and looking at their sizes.</span> </p><p><span m=''1668000''>What if I
  added both HinDIII and BamHI to my test tube?</span> </p><p><span m=''1674000''>I''d
  cut at both sites. So, I''d cut here, here,</span> <span m=''1680000''>here, here,
  here. So, this is cut with HinDIII,</span> <span m=''1686000''>here cut with BamHI,
  here cut with both and I could measure these</span> <span m=''1692000''>lengths.
  So, suppose I gave you this as a computer problem,</span> <span m=''1699000''>I
  have a string and it''s an unknown string, and I cut it at two places</span> <span
  m=''1705000''>and I get these lengths, X1, X2, X3. And then I take that same string
  and</span> <span m=''1711000''>I cut it at other positions, Y1, Y2, and Y3 are the
  lengths that</span> <span m=''1715000''>result. And then suppose I now cut it at
  both of the sites,</span> <span m=''1719000''>and I measure it, and I get Z1, Z2,
  Z3, Z4, Z5. If I gave you all</span> <span m=''1723000''>those numbers, could you
  figure out where the sites</span> <span m=''1728000''>must be? Probably. It turns
  out to be a reasonably</span> <span m=''1732000''>doable computer problem, although
  it can get a little hard in</span> <span m=''1736000''>places. And you could try
  a third enzyme and</span> <span m=''1740000''>a fourth enzyme, and it''s a cute
  exercise to write</span> <span m=''1743000''>yourself a little piece of code that
  will figure out where the sites are</span> <span m=''1747000''>based on the lengths.
  The reason it occasionally gets</span> <span m=''1750000''>funny what if Z3 and
  Z4 are exactly the same length and they run on top</span> <span m=''1753000''>of
  each other in the gel, and there are special cases.</span> </p><p><span m=''1756000''>But
  you can kind of reconstruct where those restriction sites must</span> <span m=''1760000''>be
  just by writing a good piece of code that''ll put these pieces</span> <span m=''1763000''>together.
  This is called restriction mapping,</span> <span m=''1766000''>and it''s great fun.
  Everybody likes to do this once.</span> </p><p><span m=''1770000''>But, it''s only
  a limited amount of information, right,</span> <span m=''1773000''>because you get
  where the sites are, and I guess if I gave you ten clones</span> <span m=''1776000''>and
  they all had exactly the same restriction maps,</span> <span m=''1780000''>the exact
  same positions of these restriction sites,</span> <span m=''1783000''>you''d feel
  pretty confident they were the same clone.</span> </p><p><span m=''1787000''>But
  you still wouldn''t really know much about the clone other than it</span> <span
  m=''1790000''>had two HinDIII sites and two BamHI sites, and here''s where they
  were.</span> </p><p><span m=''1793000''>What do you really want to know about this
  clone? It''s</span> <span m=''1797000''>DNA sequence, right? Let''s not settle for
  anything less</span> <span m=''1802000''>than the exact nucleotide sequence of the
  clone. So,</span> <span m=''1810000''>that''s really the last key topic is sequencing
  DNA.</span> </p><p><span m=''1818000''>How are you going to sequence DNA? Well,
  suppose I give you some</span> <span m=''1826000''>double strand of DNA, five prime
  to three prime,</span> <span m=''1834000''>five prime, three prime, double stranded
  DNA.</span> </p><p><span m=''1842000''>Let me heat it up. What happens when I heat
  up DNA?</span> </p><p><span m=''1847000''>It melts the hydrogen bonds, the non-covalent
  hydrogen bonds here</span> <span m=''1852000''>break, and I got my two strands separated.
  Now,</span> <span m=''1857000''>what I''d like to do is I want to start reading
  out this DNA sequence.</span> </p><p><span m=''1862000''>So, I''m going to make
  me a primer. Now, golly, here''s a primer.</span> </p><p><span m=''1868000''>You''re
  going to ask me, how did I even know what primer to</span> <span m=''1874000''>use
  if I don''t know the DNA sequence? How can I make a primer?</span> </p><p><span
  m=''1881000''>Hold that question. Make sure I remember to come back and answer</span>
  <span m=''1887000''>that, OK? But for the moment, grant me that I have a primer
  here.</span> </p><p><span m=''1894000''>What I''d like to do is add DNA polymerase.
  So,</span> <span m=''1899000''>let''s add some DNA polymerase. And, I''d like to
  add nucleotide</span> <span m=''1905000''>triphosphates, dNTPs, the dATP,</span>
  <span m=''1910000''>dCTP, the dGTP, dTTP, and if I add DNA polymerase and I</span>
  <span m=''1916000''>add my nucleotides, what does Arthur Kornberg tell us</span>
  <span m=''1922000''>will happen? It''ll start polymerizing,</span> <span m=''1927000''>right?
  And, it''ll stop there. So, the polymerase knows the bases,</span> <span m=''1933000''>right?
  It knows what base to put in because polymerase is very smart.</span> </p><p><span
  m=''1938000''>So, the bases get put in correctly. The only problem is, how do we
  get</span> <span m=''1944000''>polymerase to tell us what it just did?</span> </p><p><span
  m=''1950000''>Here''s a cute trick. This is, by the way,</span> <span m=''1957000''>a
  cute trick that won the Nobel Prize. So, suppose my primer is</span> <span m=''1964000''>like
  this: five prime, T, A, A, T, T, C, T, and the</span> <span m=''1971000''>template
  strand here, A, T, T, A, A, G, A, now let''s keep</span> <span m=''1978000''>going,
  A, T, G, C, C, A, A, T, G,</span> <span m=''1985000''>G, A, T, T, A, five prime.
  So, there''s my primer.</span> </p><p><span m=''1994000''>There''s my template.
  I''m going to start adding. Well, let''s</span> <span m=''2006000''>add our polymerase.
  Let''s add our dNTPs,</span> <span m=''2017000''>polymerase, dATP, dCTP, dTTP, dGTP,
  and then I want to add a</span> <span m=''2027000''>special extra good old ingredient
  into this.</span> </p><p><span m=''2038000''>The special extra ingredient I want
  to add is a defective T,</span> <span m=''2047000''>a defective dTTP. What do I
  mean by defective? I mean chemically</span> <span m=''2057000''>modified in such
  a way that it can''t be extended, that you</span> <span m=''2067000''>can''t extend
  past it. So now, let''s follow my reaction.</span> </p><p><span m=''2076000''>I''m
  going to start with, I''m just going to write them down here,</span> <span m=''2084000''>T,
  A, A, T, T, C, T. What''s the next base I''m going to put in?</span> </p><p><span
  m=''2092000''>T, OK? Is that a defective T or a good T? I don''t know.</span> </p><p><span
  m=''2100000''>It could be. Maybe it''s a defective T, which I''ll put a little star</span>
  <span m=''2105000''>there, OK? If so, what happens to my polymerase?</span> </p><p><span
  m=''2111000''>It stops. It can''t go any further. It can''t go any further because
  the</span> <span m=''2117000''>T''s defective. But what if it wasn''t a defective
  T?</span> </p><p><span m=''2122000''>What if it was a good T? Then what goes on?
  The polymerase</span> <span m=''2128000''>will put in, keep going guys. A, C, G,
  G, and what does it put in</span> <span m=''2134000''>now? T, right? Now,</span>
  <span m=''2139000''>is that a defective T? Maybe. We don''t know.</span> </p><p><span
  m=''2145000''>If it is a defective T, it stops there. Otherwise,</span> <span m=''2150000''>polymerase
  goes here, and the next space is what?</span> </p><p><span m=''2156000''>T, and
  is that a defective T? Maybe.</span> </p><p><span m=''2162000''>And, if it''s not
  a defective T, then polymerase goes on, puts in an</span> <span m=''2169000''>A,
  puts in a G, a C, C, and then a T.</span> </p><p><span m=''2176000''>And maybe that''s
  defective. All right, which of these</span> <span m=''2183000''>possibilities is
  what polymerase does when I throw it in?</span> </p><p><span m=''2191000''>Well,
  all of them. There''s a lot of molecules there.</span> </p><p><span m=''2195000''>Some
  of the molecules, by chance, happen to install a defective T,</span> <span m=''2199000''>and
  they grind to a halt here. Sometimes, a good T''s put in and the</span> <span m=''2203000''>molecules
  stop here. Sometimes they stop here,</span> <span m=''2207000''>and if I start with
  a big collection of primers in a lot of my template</span> <span m=''2211000''>DNA,
  I''m going to get this whole collection of different molecules of</span> <span m=''2215000''>different
  lengths. What lengths do I get?</span> </p><p><span m=''2219000''>The lengths correspond
  precisely to the positions of the Ts.</span> </p><p><span m=''2224000''>I get a
  series of molecules whose lengths perfectly match the</span> <span m=''2230000''>positions
  of Ts. Well, first off,</span> <span m=''2236000''>how do I measure their lengths?
  Run a gel, bingo, run a gel.</span> </p><p><span m=''2243000''>So, if I could run
  a gel that could separate nucleotides based on length</span> <span m=''2250000''>that
  two next to each other, another one up there, I''d see a</span> <span m=''2257000''>small
  molecule, length one, two,</span> <span m=''2264000''>three, four, five, three,
  six, eight, I''d see one of</span> <span m=''2271000''>length eight. I''d see one
  of length, what''s the next one,</span> <span m=''2278000''>13, eight, nine, ten,
  13, 14, so eight, nine,</span> <span m=''2285000''>ten, 11, 12, 13, 14, 15, what''s
  that, 13, 14, 15, 16,</span> <span m=''2292000''>17, 18. OK, those would be the
  positions at</span> <span m=''2298000''>which I would see this T. So, I''d need
  to have a special kind</span> <span m=''2302000''>of gel that''s so accurate that
  it can separate single nucleotides,</span> <span m=''2306000''>right, that the lengths,
  but that can be done.</span> </p><p><span m=''2311000''>There''s acrylamide, the
  polymer that will do that.</span> </p><p><span m=''2316000''>That''ll tell me the
  exact lengths of the T''s. What else do I do?</span> </p><p><span m=''2321000''>Well,
  let''s obviously do it from the other bases.</span> </p><p><span m=''2326000''>Let''s
  try defective A, defective C, defective G.</span> </p><p><span m=''2332000''>Let''s
  see, if I got it right, which we''ll try, it ought to end up</span> <span m=''2337000''>looking
  something like that. And if not, you get the picture,</span> <span m=''2342000''>that
  this ought to match up as to which columns have which lengths.</span> </p><p><span
  m=''2347000''>OK, I think I got it right. That tells me the lengths of the</span>
  <span m=''2353000''>molecules. So, I could read off at sequence.</span> </p><p><span
  m=''2358000''>The sequence of that molecule ought to be, starting over there,</span>
  <span m=''2363000''>the sequence of what I''ve added in, ought to be something like
  T, A, C,</span> <span m=''2369000''>G, G, T, T, A, C, C, T, yep, it worked.</span>
  </p><p><span m=''2374000''>It''s exactly right. Bingo. I can now read the sequence.</span>
  </p><p><span m=''2379000''>Fred Sanger, a brilliant scientist, thought up this method
  of just</span> <span m=''2384000''>exploiting E coli''s own polymerase or other
  organism''s own polymerases.</span> </p><p><span m=''2389000''>So, copying and all
  the chemistry that had to be done was thinking up</span> <span m=''2394000''>a defective
  nucleotide that could not be extended.</span> </p><p><span m=''2400000''>It could
  obviously be inserted. It can''t be extended. So, one</span> <span m=''2410000''>question
  is, what''s a defective nucleotide? Well,</span> <span m=''2420000''>you will recall
  that our nucleotide in the sugar phosphate chain</span> <span m=''2430000''>is sitting
  like this. Let''s see, hanging off the one prime</span> <span m=''2437000''>carbon
  is the base. This is the one prime carbon,</span> <span m=''2442000''>the two prime
  carbon, the three prime carbon, the four prime carbon,</span> <span m=''2447000''>the
  five prime carbon. What do we know in DNA at the two prime carbon?</span> </p><p><span
  m=''2452000''>Normally in ribose there would be a hydroxyl here,</span> <span m=''2457000''>right?
  But in deoxyribose, there''s just a hydrogen.</span> </p><p><span m=''2462000''>So,
  if this is deoxyribose, so a dNTP really means a two prime</span> <span m=''2471000''>deoxyribose,
  where do I now attach my next base in the sugar phosphate</span> <span m=''2480000''>train?
  Three prime ends, and what do I attach it to: the OH.</span> </p><p><span m=''2490000''>What
  do you think would happen if there''s no OH there?</span> </p><p><span m=''2495000''>You''re
  stuck. All you''ve got to do is take off that hydroxyl.</span> </p><p><span m=''2500000''>No
  hydroxyl group. If you made nucleotides that don''t have that</span> <span m=''2505000''>hydroxyl
  group, they can''t be extended.</span> </p><p><span m=''2510000''>So, instead of
  these being just deoxy at the two prime position,</span> <span m=''2515000''>they
  are dideoxy, deoxy at two positions.</span> </p><p><span m=''2520000''>They are
  two prime, three prime, dideoxynucleotides.</span> </p><p><span m=''2524000''>That''s
  it. Now, if you needed to get two prime three prime</span> <span m=''2529000''>dideoxynucleotides,
  they''re in the catalogue of course,</span> <span m=''2533000''>right, because Fred
  Sanger had to make them himself and all that,</span> <span m=''2538000''>but you
  can just buy them now. And so, you can do the sequence.</span> </p><p><span m=''2543000''>A
  few other little details here, though, guys.</span> </p><p><span m=''2548000''>How
  do we see the DNA and the gel? One possibility would be staining</span> <span m=''2552000''>it.
  There are some dies like ethidium bromide,</span> <span m=''2557000''>and for doing
  your restriction mapping, using a dye that sticks to</span> <span m=''2562000''>DNA
  like ethidium bromide does is pretty good. And then you put it</span> <span m=''2567000''>under
  fluorescent light and you look. For sequencing,</span> <span m=''2572000''>the amount
  of DNA is so little that it''s hard to see with a dye by the</span> <span m=''2577000''>naked
  eye, which is what you do with restriction map. So, sorry?</span> </p><p><span m=''2582000''>So,
  the first thing people did was radioactive. What they did was they</span> <span
  m=''2586000''>took a primer, made it radioactive,</span> <span m=''2590000''>and
  you did this whole sequencing reaction with radioactive primer.</span> </p><p><span
  m=''2594000''>Then, when you run the gel, you take your gel and you expose it for</span>
  <span m=''2598000''>some number of hours, eight hours maybe, a piece of x-ray</span>
  <span m=''2602000''>film, develop the x-ray film, and you''ll see that picture.
  So,</span> <span m=''2606000''>one solution that you could do to visualize is using
  radioactive</span> <span m=''2610000''>nucleotides. So, we got the defective nucleotide.</span>
  </p><p><span m=''2617000''>We now need to visualize our DNA. Let''s visualize the
  sequence.</span> </p><p><span m=''2625000''>One possibility: radioactive. The second
  possibility, someone</span> <span m=''2634000''>already mentioned it, a fluorescent
  dye.</span> </p><p><span m=''2643000''>Now, here, a fluorescent dye could be put
  on, and you can''t read it</span> <span m=''2650000''>with your eye, but lasers
  are very good at reading.</span> </p><p><span m=''2657000''>So, you might run a
  whole gel here and have lasers scan it.</span> </p><p><span m=''2664000''>But, you
  can actually do better than that. Suppose I put my</span> <span m=''2671000''>fluorescent
  dye on my dideoxynucleotides.</span> </p><p><span m=''2679000''>Suppose I put it
  on my dideoxynucleotides,</span> <span m=''2685000''>and suppose I even had enough
  chemistry at my disposal that I</span> <span m=''2692000''>could put a different
  color of fluorescent dye on each</span> <span m=''2699000''>of my nucleotides. Then,
  whenever the dideoxy is put in</span> <span m=''2705000''>to terminate the chain,
  it carries with it its own color.</span> </p><p><span m=''2709000''>Wouldn''t that
  be cool? And, that''s what''s done. Not just can you buy</span> <span m=''2714000''>dideoxynucleotides
  now, but you can buy the four different</span> <span m=''2718000''>dideoxynucleotides
  each with its own dye attached to it.</span> </p><p><span m=''2723000''>So, there
  are di-dideoxies I guess, sorry, but it''s different di''s,</span> <span m=''2727000''>right?
  They''re dye-dideoxies. So, you could do that.</span> </p><p><span m=''2732000''>And
  then what you get would be that in this column you get this color.</span> </p><p><span
  m=''2736000''>And in this column, you''d get this color. And in this column you''d
  get</span> <span m=''2740000''>this color, etc. I''m not worrying about where they</span>
  <span m=''2744000''>are here. And they''d all be different colors and it would be</span>
  <span m=''2748000''>very pretty. You know what? Why do we need to run separate</span>
  <span m=''2752000''>lanes anymore? If we got a laser,</span> <span m=''2756000''>we
  can tell the laser scan it to tell it different. Stick</span> <span m=''2760000''>it
  in one way. In fact, what''s done is stick it in</span> <span m=''2765000''>a capillary
  tube, throw in all four at the same time</span> <span m=''2773000''>now, and as
  these fragments come by, each has its own color. And all we</span> <span m=''2780000''>need
  is a laser scanner capable of sitting right here. Here''s</span> <span m=''2788000''>my
  laser scanner. And the laser scanner,</span> <span m=''2794000''>positive here,
  negative here, as the DNA flows by through this</span> <span m=''2800000''>polymer,
  the laser scanner reads off which colors just went by.</span> </p><p><span m=''2806000''>And
  it goes A color, C color, T color, G color. That''s it. So,</span> <span m=''2811000''>there
  are actually machines now that have 96 different capillaries.</span> </p><p><span
  m=''2817000''>These are called capillary tubes. And you can have 96 of them with</span>
  <span m=''2824000''>laser scanning across, and in each column now,</span> <span
  m=''2832000''>it turns out that you can read almost 1,000 letters,</span> <span
  m=''2840000''>1,000 bases per column per capillary times about 100 capillaries.</span>
  </p><p><span m=''2848000''>Or in other words, you can read out about 10^5 bases
  of information.</span> </p><p><span m=''2856000''>You can read out 10^5 bases of
  information in about two hours.</span> </p><p><span m=''2860000''>Of course, you
  can do that ten times a day. So,</span> <span m=''2865000''>you can actually read
  out 10^6 or about a million bases of information</span> <span m=''2870000''>per
  machine. And here at MIT, we have 100 of these machines. So,</span> <span m=''2875000''>we
  actually can read out a little shy of 100 million letters of DNA</span> <span m=''2880000''>sequence
  per day, which I mean is a lot.</span> </p><p><span m=''2885000''>We read about
  40 billion letters per year here at MIT,</span> <span m=''2891000''>and this is
  how we do it. How much does a machine cost?</span> </p><p><span m=''2897000''>List,
  or do you want a deal? They list for $300,</span> <span m=''2903000''>00, but if
  you buy in bulk, I can do better. [LAUGHTER] We buy</span> <span m=''2909000''>it
  in bulk, by the way. So now, how are we going to get our</span> <span m=''2914000''>primer
  there? That was the only little bit we were missing is where</span> <span m=''2918000''>did
  our primer come from? The last little detail: here''s my</span> <span m=''2922000''>vector,
  remember, and I want to sequence this insert.</span> </p><p><span m=''2927000''>How
  am I going to get a primer in the insert? I don''t know what its</span> <span m=''2931000''>sequence
  is. How do I even start this? Sorry? Well,</span> <span m=''2936000''>but that won''t
  tell me what the sequence is that I have to,</span> <span m=''2940000''>I mean,
  I was looking to try to get a primer that matches the insert.</span> </p><p><span
  m=''2945000''>And I don''t know what the insert is. So, how am I going to get a
  primer?</span> </p><p><span m=''2949000''>Oh, I know the vector. The vector is well
  known.</span> </p><p><span m=''2953000''>It sequence is in the catalog. Let me instead
  just use a primer</span> <span m=''2957000''>that happens to sit in the vector,
  and I''ll match to a known sequence</span> <span m=''2961000''>to start with, and
  then I''ll sequence into my</span> <span m=''2965000''>unknown territory. So, this
  is how you get the initial</span> <span m=''2969000''>primer was you arrange that
  your initial primer is sitting in known</span> <span m=''2973000''>vector sequence.
  All right, so you can now sequence</span> <span m=''2977000''>DNA. I''ve got to
  say, I''ve taught this course for a little</span> <span m=''2980000''>more than
  a decade, and being able to say,</span> <span m=''2984000''>now we can routinely
  sequence about a million letters per machine,</span> <span m=''2987000''>and 100
  million letters per day, and things like this was not</span> <span m=''2990000''>routinely
  the case. When we started teaching this</span> <span m=''2993000''>course, I was
  describing what we di</span> </p>'
type: course
uid: 15a752d2d7ccfb4cea41ceb95b805b34

---
None