---
about_this_resource_text: <p><strong>Topics covered:</strong> Gene Regulation</p>  <p><strong>Instructors:</strong>
  Prof. Eric Lander</p>
course_id: 7-012-introduction-to-biology-fall-2004
embedded_media:
- id: lec13.jpg
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-13-gene-regulation/lec13.jpg
  title: lec13.jpg
  type: null
  uid: 9808c02e47ce3bfcaaae17c66a5e1183
- id: Video-YouTube-Stream
  media_location: 00LNy0Q_i6c
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Video-YouTube-Stream
  type: Video
  uid: 9c6653afc48d64b6a691a57060ac6ece
- id: Thumbnail-YouTube-JPG
  media_location: https://img.youtube.com/vi/00LNy0Q_i6c/default.jpg
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Thumbnail-YouTube-JPG
  type: Thumbnail
  uid: 22b60ef38a17b5bc9ea21f300f1e11a1
- id: Video-iTunesU-MP4
  media_location: https://itunes.apple.com/us/itunes-u/id341596582
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Video-iTunes U-MP4
  type: Video
  uid: fe598d6e61aba78c3203a0d8ce835acd
- id: Video-InternetArchive-MP4
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec13-08oct2004-220k.mp4
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Video-Internet Archive-MP4
  type: Video
  uid: d2569f93dcceef095db642764e15247d
- id: Audio-iTunesU-MP3
  media_location: https://itunes.apple.com/us/itunes-u/id341596147
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Audio-iTunes U-MP3
  type: Audio
  uid: 176c4f34ea31f81237fb872ef0f7287f
- id: Video-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT7.012F04/ocw-7.012-lec-mit-10250-08oct2004-1000.mp3
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Video-Internet Archive-MP3
  type: Video
  uid: 888f68d1f86aa7a73d69a2931412c2e7
- id: Video-VideoLecturesnet-Stream
  media_location: http://videolectures.net/mit7012f04_introduction_biology/
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Video-VideoLectures.net-Stream
  type: Video
  uid: 7fac26c84c8d7a2cb9576884a0f4afc2
- id: Thumbnail-OCW-JPG
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Thumbnail-OCW-JPG
  type: Thumbnail
  uid: 1e4e305b7f499386f3b102f0b5cef46c
- id: 3Play-3PlayYouTubeid-MP4
  media_location: 00LNy0Q_i6c
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: 3Play-3Play YouTube id
  type: 3Play
  uid: a3746e8b122fa319e2d5d46c165b4a7c
- id: 00LNy0Q_i6c.srt
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-13-gene-regulation/00LNy0Q_i6c.srt
  title: 3play caption file
  type: null
  uid: 9cdf6218f76ae115c45184fca2387e06
- id: 00LNy0Q_i6c.pdf
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-13-gene-regulation/00LNy0Q_i6c.pdf
  title: 3play pdf file
  type: null
  uid: ebda1131d71422d2b6a60f17ad89409b
- id: Caption-3Play YouTube id-SRT
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Caption-3Play YouTube id-SRT-English - US
  type: Caption
  uid: 5238e76aca820e971fb9ae6730c46475
- id: Transcript-3Play YouTube id-PDF
  parent_uid: b3f4c1cb67c92f2728cc8f833d4d0bd2
  title: Transcript-3Play YouTube id-PDF-English - US
  type: Transcript
  uid: f591ff98a791ce9b0c202d884307bbe1
inline_embed_id: 35610206lecture13:generegulation39229658
layout: video
order_index: null
parent_uid: 53e57148b5fdab332e2a4c75bf512f31
related_resources_text: ''
short_url: lecture-13-gene-regulation
technical_location: https://ocw.mit.edu/courses/biology/7-012-introduction-to-biology-fall-2004/video-lectures/lecture-13-gene-regulation
template_type: Tabbed
title: 'Lecture 13: Gene Regulation'
transcript: '<p><span m=''33000''>Good morning. Good morning.</span> </p><p><span
  m=''42000''>So, what I would like to do today is pick up on our basic theme of</span>
  <span m=''49000''>molecular biology. We''ve talked about DNA replication.</span>
  </p><p><span m=''56000''>The transcription of DNA into RNA, and the translation
  of RNA</span> <span m=''63000''>into protein. We discussed last time some of the
  variations between</span> <span m=''69000''>different types of organisms: viruses,
  prokaryotes,</span> <span m=''75000''>eukaryotes, with respect to the details of
  how they do that in</span> <span m=''81000''>general that bacteria have circular
  DNA chromosomes typically</span> <span m=''87000''>that eukaryotes have linear chromosomes,</span>
  <span m=''92000''>etc. What I''d like to talk about today is variation,</span> <span
  m=''96000''>but variation not between organisms but within an organism from time
  to</span> <span m=''101000''>time and place to place, namely, how it is that some
  genes or</span> <span m=''106000''>gene activities are turned on, on some occasions,
  and turned off on</span> <span m=''110000''>other occasions. This is, obviously,</span>
  <span m=''115000''>a very important problem to an organism, particularly</span>
  <span m=''120000''>to somebody like you who''s a multi-cellular organism,</span>
  <span m=''123000''>and has the same DNA instruction set in all of your cells.</span>
  </p><p><span m=''126000''>It''s obviously quite important to make sure that the
  same basic code</span> <span m=''129000''>is doing different things in different
  cells.</span> </p><p><span m=''133000''>It''s important, also, to a bacterium to
  make sure that</span> <span m=''136000''>it''s doing different things at different
  times,</span> <span m=''139000''>depending on its environment. So, I''m going to
  talk about a very</span> <span m=''143000''>particular system today as an illustration
  of how genes are</span> <span m=''146000''>regulated, but before we do that, let''s</span>
  <span m=''150000''>Ask, where are the different places in this picture?</span> </p><p><span
  m=''155000''>DNA goes to DNA goes to RNA goes to protein, in which you might,</span>
  <span m=''160000''>in principle, regulate the activity of a gene. Could you regulate
  the</span> <span m=''165000''>activity of a gene by actually changing the DNA encoded
  in the</span> <span m=''170000''>genome? So, why not? Because what? It becomes a</span>
  <span m=''175000''>different gene. Yeah, that''s just a definition.</span> </p><p><span
  m=''180000''>Why couldn''t the cell just decide that I want this gene now to change</span>
  <span m=''185000''>in some way? Oh, I don''t know, I''ll alter the DNA</span> <span
  m=''190000''>sequence in some way. And, that''ll make the gene work.</span> </p><p><span
  m=''196000''>Could that happen? Is that allowed? Yeah, it turns out to happen.</span>
  </p><p><span m=''201000''>It''s not the most common thing, and it''s not the thing
  they''ll talk</span> <span m=''207000''>about in the textbooks a lot but you can
  actually do regulation.</span> </p><p><span m=''212000''>So, the levels of regulation
  are many, and one is actually at the</span> <span m=''217000''>level of DNA rearrangement.
  As we''ll come to later in the</span> <span m=''222000''>course, for example, your
  immune system creates new,</span> <span m=''227000''>functional genes by rearranging
  locally some pieces of DNA,</span> <span m=''232000''>some bacteria, particularly
  infectious organisms control whether</span> <span m=''237000''>genes are turned
  on or off by actually going in there,</span> <span m=''241000''>and flipping around
  a piece of DNA in their chromosome.</span> </p><p><span m=''244000''>And, that''s
  how they turn the gene on or off is they actually go in and</span> <span m=''247000''>change
  the genome. There''s some protein that actually</span> <span m=''250000''>flips
  the orientation of a segment of DNA. Now, these are a little</span> <span m=''253000''>funky,
  and we''re not going to talk a lot about them,</span> <span m=''256000''>but you
  should know, almost anything that can happen does</span> <span m=''259000''>happen
  and gets exploited in different ways by organisms.</span> </p><p><span m=''262000''>So,
  DNA rearrangement certainly happens. It''s rare,</span> <span m=''265000''>but it''s
  always cool when it happens.</span> </p><p><span m=''269000''>So, it''s fun to look
  at. And, something like the immune</span> <span m=''274000''>system can''t be dismissed
  as simply an oddity. That''s an incredibly</span> <span m=''280000''>important thing.
  The most common form is at the</span> <span m=''285000''>level of transcriptional
  regulation, where whether or not a transcript</span> <span m=''291000''>gets made
  is how it''s processed can be different. First off,</span> <span m=''297000''>the
  initiation of transcription that RNA polymerase should happen to</span> <span m=''302000''>sit
  down at this gene on this occasion and start transcribing it</span> <span m=''306000''>is
  a potentially regulatable (sic) step that maybe you''re only going to</span> <span
  m=''310000''>turn on the gene for beta-globin and alpha-globin that together make
  the</span> <span m=''314000''>two components of hemoglobin, and you''re only going
  to turn them</span> <span m=''318000''>on in red blood cells, or red blood cell
  precursors,</span> <span m=''322000''>and that could be done at the level of whether
  or not you make the</span> <span m=''326000''>message in the first place. That''s
  one place it can be done.</span> </p><p><span m=''330000''>Another place is the
  splicing choices that you make.</span> </p><p><span m=''335000''>With respect to
  your message, you get this thing with a number of</span> <span m=''340000''>different
  potential exons, and you can regulate how this gene</span> <span m=''345000''>is
  used by deciding to splice it this way, and skip over that exon</span> <span m=''350000''>perhaps,
  or not skip over that exon. That alternative spicing is a</span> <span m=''355000''>powerful
  way to regulate. And then finally, you can also</span> <span m=''360000''>regulate
  at the level of mRNA stability.</span> </p><p><span m=''366000''>Stability means
  the persistence of the message, the degradation of the</span> <span m=''369000''>message.
  It could be that in certain cells,</span> <span m=''372000''>the message is protected
  so that it hangs around longer.</span> </p><p><span m=''375000''>And, in other cells,
  perhaps, it''s unprotected and it''s</span> <span m=''378000''>degraded very rapidly.
  If it''s degraded very rapidly,</span> <span m=''381000''>it doesn''t get a chance
  to make a protein or maybe it doesn''t get to</span> <span m=''384000''>make too
  many copies of the protein. If it''s persistent for a long time,</span> <span m=''387000''>it
  can make a lot of copies of protein.</span> </p><p><span m=''390000''>All of those
  things can and do occur. Then, of course,</span> <span m=''395000''>there is the
  regulation at the level of translation.</span> </p><p><span m=''400000''>Translation,
  if I give you an mRNA, is it automatically going to be</span> <span m=''406000''>translated?
  Maybe the cell has a way to sequester the RNA to ramp it</span> <span m=''411000''>up
  in some way so that it doesn''t get to the ribosome under</span> <span m=''417000''>some
  conditions, and under other conditions it does</span> <span m=''421000''>get to
  the ribosome, or some ways to block in other</span> <span m=''425000''>manners than
  just sequestering it, but to physically block whether or</span> <span m=''428000''>not
  this message gets translated, what turns out that there''s a</span> <span m=''432000''>tremendous
  amount of that. It''s, again, not the most common,</span> <span m=''435000''>but
  we''re learning, particularly over the last couple of years,</span> <span m=''439000''>that
  regulation of the translation of an mRNA is important.</span> </p><p><span m=''442000''>There
  are, although I won''t talk about them at length,</span> <span m=''446000''>an exciting
  new set of genes called micro RNA''s,</span> <span m=''450000''>teeny little RNAs
  that encode 21-22 base pair segments that are able to</span> <span m=''454000''>pair
  with a messenger RNA and interfere in some ways partially</span> <span m=''459000''>with
  its translatability. And so, by the number and the kinds</span> <span m=''463000''>of
  little micro RNAs that are there, organisms can tweak up or down how</span> <span
  m=''468000''>actively a particular message is being translated.</span> </p><p><span
  m=''472000''>So, the ability to regulate translation in a number of different</span>
  <span m=''477000''>ways is important. And then, of course,</span> <span m=''481000''>there''s
  post-translational control. Once a protein is made,</span> <span m=''486000''>there''s
  post-translational regulation that could happen.</span> </p><p><span m=''490000''>It
  could be that the protein is modified in some way.</span> </p><p><span m=''494000''>The
  proteins say completely inactive unless you put a phosphate</span> <span m=''498000''>group
  on it, and some enzyme comes along and puts a phosphate group on</span> <span m=''502000''>it.
  Or, it''s inactive until you take off the phosphate group.</span> </p><p><span m=''506000''>All
  sorts of post-translational modifications can occur to proteins</span> <span m=''510000''>after
  the amino acid chain is made that can affect whether or not the</span> <span m=''513000''>protein
  is active. Every one of these is potentially a</span> <span m=''517000''>step by
  which an organism can regulate whether or not you have a</span> <span m=''521000''>certain
  biochemical activity present in a certain amount at a certain</span> <span m=''525000''>time.
  And, every one of these gets used. This is the thing about</span> <span m=''529000''>coming
  to a system that has been in the process of evolution for three</span> <span m=''533000''>and
  a half billion years is that even little differences can be</span> <span m=''537000''>fought
  over as competitive advantages, and can be fixed</span> <span m=''541000''>by an
  organism. So, if a tiny little thing began to</span> <span m=''545000''>help the
  organism slightly, it could reach fixation. And,</span> <span m=''550000''>you''re
  coming along to this system, which has had about three and a half</span> <span m=''555000''>billion
  years of patches to the software code,</span> <span m=''560000''>and it''s just
  got all sorts of layers and regulation piled on top</span> <span m=''565000''>of
  it. All of these things happen. But, what we think is the most</span> <span m=''570000''>important
  out of this whole collection is this guy.</span> </p><p><span m=''575000''>The fundamental
  place at which you''re going to regulate whether or</span> <span m=''580000''>not
  you have the product of a gene is whether you bother to transcribe</span> <span
  m=''585000''>its RNA. But I do want to say because, yes? And,</span> <span m=''590000''>which
  exons you used and which aren''t? Yeah,</span> <span m=''595000''>well, there are
  tissue-specific factors that are gene-specific</span> <span m=''600000''>that can
  influence that. And, surprisingly little is known</span> <span m=''603000''>about
  the details. There are a couple of cases where</span> <span m=''606000''>people
  know, but as you''d imagine, you actually need a regulatory</span> <span m=''610000''>system
  in that tissue to be able to decide to skip over that exon.</span> </p><p><span
  m=''613000''>And, the mechanics of that surprisingly are understood in very</span>
  <span m=''617000''>few cases. And, you might think that evolution</span> <span m=''620000''>wouldn''t
  like to use that as the most common thing because you really</span> <span m=''623000''>do
  have to make a specialized thing to do that. So, that''s what</span> <span m=''627000''>happens
  on these. That''s one in particular where I</span> <span m=''630000''>think a tremendous
  amount of more work has to happen.</span> </p><p><span m=''633000''>mRNA stability,
  we understand some of it but not all the factors in</span> <span m=''636000''>this
  business. I was telling you about translation</span> <span m=''639000''>with these
  little micro-RNAs is stuff that''s really only a few years</span> <span m=''642000''>old
  that people have come to understand. So,</span> <span m=''645000''>there''s a lot
  to be understood about these things. I''m going to tell you</span> <span m=''648000''>about
  initiation of mRNAs, because it''s the area where we know</span> <span m=''651000''>the
  most, and I think it''ll give you a good idea of the general paradigm.</span> </p><p><span
  m=''654000''>But, any of you who want to go into this will find that there''s a</span>
  <span m=''657000''>tremendous amount more to still be discovered about these things.</span>
  </p><p><span m=''660000''>So, the amount of protein that a cell might make varies
  wildly.</span> </p><p><span m=''665000''>Your red blood cells, 80% of your red blood
  cells,</span> <span m=''670000''>protein, is alpha or beta-globin. It''s a huge
  amount. That''s not</span> <span m=''675000''>true in any other cell in your body.
  So, we were talking about pretty</span> <span m=''681000''>significant ranges of
  difference as to how much protein is made.</span> </p><p><span m=''686000''>How
  do things like that happen? Well, I''m going to describe the</span> <span m=''691000''>simplest
  and classic case of gene regulation and bacteria,</span> <span m=''697000''>and
  in particular, the famous lack operon of E coli.</span> </p><p><span m=''703000''>So,
  this was the first case in which regulation was ever really</span> <span m=''709000''>worked
  out, and it stands today as a very good paradigm of how</span> <span m=''715000''>regulation
  works. E coli, in order to grow,</span> <span m=''720000''>needs a carbon source.
  In particular, E coli is fond of sugar.</span> </p><p><span m=''725000''>It would
  like to have a sugar to grow on. Given a choice,</span> <span m=''730000''>what''s
  E coli''s favorite sugar? It''s glucose, right, because we</span> <span m=''735000''>have
  the whole cycle of glucose. The whole pathway of glucose goes</span> <span m=''740000''>to
  pyruvate, which we''ve talked about, and glucose is the preferred</span> <span m=''745000''>sugar
  to go into that pathway, OK, of glycolysis.</span> </p><p><span m=''750000''>Glycolysis:
  the breakdown of glucose. But, suppose there''s no glucose</span> <span m=''756000''>available.
  Is E coli willing to have a different sugar?</span> </p><p><span m=''762000''>Sure,
  because E coli''s not stupid. If it were to refuse another sugar,</span> <span m=''768000''>it
  wouldn''t be able to grow. So, it has a variety of pathways that will</span> <span
  m=''774000''>shunt other sugars to glucose, which will then allow you to go</span>
  <span m=''780000''>through glycolysis, etc. Now, given a choice,</span> <span m=''788000''>it
  would prefer to use the glucose. But if not, suppose you gave it</span> <span m=''796000''>lactose.
  Lactose is a disaccharide. It''s milk sugar, and I''ll just</span> <span m=''804000''>briefly
  sketch, so lactose is a disaccharide where</span> <span m=''812000''>you''ve got
  a glucose and a galactose.</span> </p><p><span m=''820000''>Glucose plus galactose
  equals lactose. So, if E coli is given</span> <span m=''830000''>galactose, it is
  able to break it down into glucose plus galactose.</span> </p><p><span m=''840000''>And
  it does that by a particular enzyme called beta galactosidase,</span> <span m=''848000''>which
  breaks down glactosides. And, it''ll give you galactose plus</span> <span m=''857000''>glucose.
  How much beta-galactosidase does an E coli</span> <span m=''865000''>cell have around?
  Sorry? None? But how does it do this?</span> </p><p><span m=''872000''>When it needs
  it, it''ll synthesize it. When it needs it,</span> <span m=''878000''>like, there''s
  no glucose and there''s a lot of galactose around,</span> <span m=''883000''>how
  much of it will there be? A lot. It turns out that in</span> <span m=''889000''>circumstances
  where E coli is dependent on galactose as its fuel,</span> <span m=''894000''>something
  like 10% of total protein</span> <span m=''900000''>can be beta-gal under the circumstances
  when you have</span> <span m=''906000''>galactose but no glucose. Sorry? Sorry,
  when you have</span> <span m=''913000''>lactose but no glucose. Thank you. So, when
  you have</span> <span m=''919000''>lactose but no glucose, E coli has 10% of its
  protein weight</span> <span m=''926000''>as beta-galactosidase. Wow. But when you
  have glucose</span> <span m=''933000''>around or you don''t have lactose around,
  you have very little.</span> </p><p><span m=''939000''>It could be almost none,
  trace amounts. So, why do this?</span> </p><p><span m=''945000''>Why not, for example,
  just have a far more reasonable some compromise?</span> </p><p><span m=''951000''>Like,
  let''s always just have 1% of beta-galactosidase.</span> </p><p><span m=''957000''>Why
  do we need the 0-10%? 10%''s actually extremely high.</span> </p><p><span m=''964000''>So
  what. It''s a good insurance policy. So, if I only have</span> <span m=''969000''>galactose,
  I need more. Well, I mean, 1% will still digest</span> <span m=''975000''>it. I''ll
  still do it. What''s the problem? Sorry?</span> </p><p><span m=''981000''>So what,
  I do it at a slower rate. Life''s long. Why not? Ah, it has</span> <span m=''987000''>to
  compete. So, if the cell to the left had a</span> <span m=''991000''>mutation that
  got it to produce four times as much,</span> <span m=''995000''>then it would soak
  up the lactose in the environment,</span> <span m=''998000''>grow faster, etc. etc.,
  and we could have competed.</span> </p><p><span m=''1002000''>So, these little tuning
  mutations have a huge effect amongst this</span> <span m=''1005000''>competing population
  of bacteria. And so, if E coli currently thinks</span> <span m=''1009000''>that
  it''s really good to have almost non at sometimes and 10% at other</span> <span
  m=''1013000''>times, you can bet that it''s worked that out through the product
  of</span> <span m=''1016000''>pretty rigorous competition, that it doesn''t want
  to waste the</span> <span m=''1020000''>energy making this when you don''t need
  it, and that when you do need</span> <span m=''1023000''>it, you really have to
  compete hard by growing as fast as you can when</span> <span m=''1027000''>you have
  that lactose around. OK. So, how does it actually get</span> <span m=''1031000''>the
  lactose, sorry, keep me honest on lactose versus</span> <span m=''1036000''>galactose,
  into the cell? It turns out that it also has</span> <span m=''1042000''>another
  gene product, another protein, which is a lactose</span> <span m=''1047000''>permease.
  And, any guesses as to what a</span> <span m=''1052000''>lactose permease does?
  It makes the cell permeable to</span> <span m=''1058000''>lactose, right, good.
  So, the lactose can get into</span> <span m=''1063000''>the cell, and then beta-gal
  can break it down into galactose plus</span> <span m=''1069000''>glucose. These
  two things, in fact, both get regulated,</span> <span m=''1074000''>beta-gal and
  this lactose permease. So, how does it work?</span> </p><p><span m=''1080000''>Let''s
  take a look now at the structure of the lack operon.</span> </p><p><span m=''1087000''>So,
  I mentioned briefly last time, what''s an operon? Remember we said</span> <span
  m=''1094000''>that in bacteria, you often made a transcript that had</span> <span
  m=''1101000''>multiple proteins that were encoded on it.</span> </p><p><span m=''1109000''>A
  single mRNA could get made, and multiple starts for translation</span> <span m=''1113000''>could
  occur, and you could make multiple proteins.</span> </p><p><span m=''1117000''>And,
  this would be a good thing if you wanted to make a bunch of</span> <span m=''1121000''>proteins
  that were a part of the same biochemical pathway.</span> </p><p><span m=''1125000''>Such
  an object, a regulated piece of DNA that makes a transcript</span> <span m=''1129000''>encoding
  multiple polypeptides is called an operon because they''re</span> <span m=''1133000''>operated
  together. So, let''s take a look here at the</span> <span m=''1137000''>lack operon.
  I said there was a promoter.</span> </p><p><span m=''1143000''>Here is a promoter
  for the operon, and we''ll call it P lack, promoter</span> <span m=''1149000''>for
  the lack operon. Here is the first gene that is</span> <span m=''1156000''>encoded.
  So, the message will start here, actually about here,</span> <span m=''1163000''>and
  start going off. And, the first gene is given the name lack Z.</span> </p><p><span
  m=''1170000''>It happens to encode beta-galactosidase enzyme.</span> </p><p><span
  m=''1174000''>Remember, they did a mutant hunt, and when they did the mutant hunt,</span>
  <span m=''1178000''>they didn''t know what each gene was as they isolated mutants.</span>
  </p><p><span m=''1182000''>So, they just gave them names of letters. And so,</span>
  <span m=''1186000''>it''s called lack Z. And, everybody in molecular biology</span>
  <span m=''1190000''>knows this is the lack Z gene, although Z has nothing to do
  with</span> <span m=''1194000''>beta-galactosidase. It was just the letter given
  to it.</span> </p><p><span m=''1198000''>But, it''s stuck. Next is lack Y.</span>
  </p><p><span m=''1202000''>And, that encodes the permease. And, there is also lack
  A, which</span> <span m=''1207000''>encodes a transacetylase, and as far as I''m
  concerned you can</span> <span m=''1212000''>forget about it. OK, but I just mentioned
  that it is</span> <span m=''1217000''>there, and it actually does make three polypeptides.</span>
  </p><p><span m=''1222000''>We won''t worry about it, OK, but it does make a</span>
  <span m=''1227000''>transacetylase, OK? But it won''t figure in what we''re</span>
  <span m=''1232000''>going to talk about, and actually remarkably little is</span>
  <span m=''1236000''>known about the transacetylase. There''s also one other gene
  I need</span> <span m=''1241000''>to talk about, and that''s over here,</span> <span
  m=''1246000''>and that''s called lack I. And, it too has a promoter,</span> <span
  m=''1250000''>which we can call PI, for the promoter for lack I.</span> </p><p><span
  m=''1255000''>And, this encodes a very interesting protein.</span> </p><p><span
  m=''1260000''>So, we get here one message encoding one polypeptide here.</span>
  </p><p><span m=''1265000''>This mRNA encodes one polypeptide. It is monocystronic.
  This guy here</span> <span m=''1271000''>is a polycystronic message. It has multiple
  cystrons, which is</span> <span m=''1277000''>the dusty old name for these regions
  that were translated into distinct</span> <span m=''1283000''>proteins. And so,
  that''s that mRNA.</span> </p><p><span m=''1289000''>So, lack I, this encodes a
  very interesting protein,</span> <span m=''1300000''>which is called the lack repressor.
  The lack repressor, actually I''ll</span> <span m=''1312000''>bring this down a
  moment, is not an enzyme.</span> </p><p><span m=''1324000''>It''s not a self-surface
  channel for putting in galactose.</span> </p><p><span m=''1330000''>It is a DNA
  binding protein. It binds to DNA. But, it''s not a</span> <span m=''1337000''>nonspecific
  DNA binding protein that binds to any old DNA.</span> </p><p><span m=''1343000''>It
  has a sequence-specific preference.</span> </p><p><span m=''1350000''>It''s a protein
  that has a particular confirmation, a particular shape,</span> <span m=''1355000''>a
  particular set of amino acids sticking out, that it combined into</span> <span m=''1360000''>the
  major groove of DNA in a sequence-specific fashion such that</span> <span m=''1366000''>it
  particularly likes to recognize a certain sequence of nucleotides and</span> <span
  m=''1371000''>binds there. Where is the specific sequence of nucleotides where this</span>
  <span m=''1377000''>guy likes to bind? It so happens that it''s there.</span> </p><p><span
  m=''1383000''>And this is called the operator sequence or the operator site.</span>
  </p><p><span m=''1391000''>So, this protein likes to go and bind there. Now,</span>
  <span m=''1398000''>I''ve drawn this, by the way, so that this operator site is</span>
  <span m=''1406000''>actually right overlapping the promoter site.</span> </p><p><span
  m=''1414000''>Who likes to bind at the promoter site? RNA polymerase.</span> </p><p><span
  m=''1420000''>What''s going to happen if the lack repressor protein is sitting there?</span>
  </p><p><span m=''1426000''>RNA polymerase can''t bind. It''s just physically,</span>
  <span m=''1432000''>blocked from binding. So, let''s examine some cases here.</span>
  </p><p><span m=''1438000''>Let''s suppose that we look at here</span> <span m=''1446000''>at
  our gene. We''ve got our promoter, P lack. We''ve got the operator site</span> <span
  m=''1454000''>here. We''ve got the lack Z gene here, and we''ve got the lack</span>
  <span m=''1462000''>repressor, lack I, the repressor sitting there.</span> </p><p><span
  m=''1471000''>Polymerase tries to come along to this, and it''s blocked.</span>
  </p><p><span m=''1478000''>So, what will happen in terms of the transcription of
  the lack</span> <span m=''1485000''>operon: no mRNA. So, that''s great.</span> </p><p><span
  m=''1492000''>So, we''ve solved one problem right off the bat.</span> </p><p><span
  m=''1499000''>We want to be sure that sometimes there''s going to be no mRNA made.</span>
  </p><p><span m=''1503000''>This way, we''re not going to waste any metabolic energy,</span>
  <span m=''1507000''>making beta-galactosidase. Are we done? No? Why not.</span>
  </p><p><span m=''1511000''>We''ve got to sometimes make beta-galactosidase.</span>
  </p><p><span m=''1516000''>So, we''ve got to get that repressor off there. Well,</span>
  <span m=''1520000''>how is the repressor going to come off there? When do we want
  the</span> <span m=''1524000''>repressor off there: when there''s lactose present.</span>
  </p><p><span m=''1529000''>So, somehow we need to build some kind of an elaborate
  sensory</span> <span m=''1534000''>mechanism that is able to tell when lactose is
  present,</span> <span m=''1539000''>and send a signal to the repressor protein saying,</span>
  <span m=''1544000''>hey, lactose is around. The signal gets transmitted all the</span>
  <span m=''1549000''>way to the repressor protein, and the repressor protein comes
  off.</span> </p><p><span m=''1555000''>What kind of an elaborate sensory mechanism
  might be built?</span> </p><p><span m=''1560000''>Use lactose as what? So, this
  is actually pretty simple.</span> </p><p><span m=''1565000''>You''re saying just
  take lactose, and you want lactose to be its own</span> <span m=''1571000''>signal?
  So, if lactose were to just bind to the repressor,</span> <span m=''1576000''>the
  repressor might then know that there was lactose around.</span> </p><p><span m=''1581000''>Well,
  what would it do if lactose bound to it? Sorry? Why</span> <span m=''1587000''>would
  it fall off? Yep. More interested in the lactose.</span> </p><p><span m=''1593000''>So,
  if you''re suggestion, this is good. I like the design</span> <span m=''1599000''>work
  going on here. The suggestion is that if lactose</span> <span m=''1605000''>binds
  to this here, binds to our repressor,</span> <span m=''1611000''>it''s going to
  fall off because it''s more interested in lactose</span> <span m=''1617000''>than
  in the DNA. Now, how is the interest actually</span> <span m=''1623000''>conveyed
  into something material? Because the actual level of</span> <span m=''1627000''>cognitive
  like or dislike for DNA on the part of this polypeptide is</span> <span m=''1631000''>unclear,
  you may be anthropomorphizing slightly with</span> <span m=''1635000''>regard to
  this polypeptide chain. So, mechanistically, what''s going</span> <span m=''1639000''>to
  happen? Shape. Yes, shape? Change confirmation,</span> <span m=''1643000''>the binding
  act, the act of binding lactose creates some energy,</span> <span m=''1647000''>may
  change the shape of the protein,</span> <span m=''1651000''>and that shape of the
  protein may, in the process of wiggling around to</span> <span m=''1655000''>bind
  lactose may de-wiggle some other part of it that now no longer</span> <span m=''1660000''>binds
  so well to DNA. That is exactly what happens.</span> </p><p><span m=''1664000''>Good
  job. So, you guys have designed, in fact,</span> <span m=''1669000''>what really
  happens. What happens is what''s called an</span> <span m=''1674000''>allosteric
  change. It just means other shape.</span> </p><p><span m=''1678000''>So, it just
  changes its shape, that it changes shape on binding of</span> <span m=''1683000''>lactose.
  And it falls off because it''s less</span> <span m=''1690000''>suitable for binding
  this particular DNA sequence when it''s bound to</span> <span m=''1698000''>lactose
  there. So, in this case,</span> <span m=''1706000''>in the presence of lactose,
  lack I does not bind.</span> </p><p><span m=''1714000''>And, the lack operon is
  transcribed. Yes? Uh-oh. OK, all right</span> <span m=''1724000''>designers, here
  we''ve got a problem. You have such a cool system,</span> <span m=''1734000''>right?
  You were going to sense lactose.</span> </p><p><span m=''1743000''>Lactose was going
  to bind to the lack repressor,</span> <span m=''1750000''>change its confirmation
  falloff: uh-oh. But, as you point out,</span> <span m=''1757000''>how''s it going
  to get any lactose, because there''s not a lactose</span> <span m=''1765000''>permease
  because the lactose permease is made by</span> <span m=''1772000''>the same operon.
  So, what if, in fact,</span> <span m=''1777000''>instead of getting one of these
  DOD mill speck kind of things of some</span> <span m=''1780000''>repressor that
  is absolutely so tight that it never falls off under</span> <span m=''1783000''>any
  circumstances, what if we build a slightly sloppy</span> <span m=''1787000''>repressor
  that occasionally falls off, and occasionally allows</span> <span m=''1790000''>transcription
  of the lack operon? Then, we''ll have some trace</span> <span m=''1793000''>quantities
  of permease around. With a little bit of permease</span> <span m=''1796000''>around,
  a little lactose will get in.</span> </p><p><span m=''1800000''>And, as long as
  even a little lactose gets in,</span> <span m=''1804000''>it''ll now shift the equilibrium
  so that the repressor is off more,</span> <span m=''1808000''>and of course that
  will make more permease, and shift,</span> <span m=''1812000''>and shift, and shift,
  and shift. So, as long as it''s not</span> <span m=''1816000''>so perfectly engineered
  as to have nothing being transcribed,</span> <span m=''1821000''>so no mRNA is really
  very little mRNA. See, this is what''s so good,</span> <span m=''1825000''>I think,
  about having MIT students learn this stuff because there are</span> <span m=''1829000''>all
  sorts of wonderful design principles here about how</span> <span m=''1833000''>you
  build systems. And, I think this is just a very</span> <span m=''1838000''>good
  example of how you build a system like this.</span> </p><p><span m=''1843000''>Now,
  all right, so we now have the ability to have lack on and lack off,</span> <span
  m=''1848000''>and that is lack off, mostly off because of your permease</span> <span
  m=''1852000''>problem: very good. Now, let''s take a little digression</span> <span
  m=''1857000''>about, how do we know this? This kind of reasoning,</span> <span m=''1862000''>I''ve
  now told you the answer. But let''s actually take a look at</span> <span m=''1866000''>understanding
  the evidence that lets you conclude this.</span> </p><p><span m=''1870000''>So,
  in order to do this, and this is the famous work in</span> <span m=''1874000''>molecular
  biology of Jacobin Manoux in the late ''50s for which they won</span> <span m=''1878000''>a
  Nobel Prize, they wanted to collect some mutants.</span> </p><p><span m=''1882000''>Remember,
  this is before the time of DNA sequence or anything like</span> <span m=''1886000''>that,
  and wanted to collect mutants that affected this process.</span> </p><p><span m=''1890000''>So,
  in order to collect mutants that screwed up the regulation,</span> <span m=''1897000''>they
  knew that beta-galactosidase was produced in much higher quantity</span> <span m=''1905000''>if
  lactose was around. The difficulty with that was that</span> <span m=''1913000''>wild
  type E coli, when you had no lactose would</span> <span m=''1921000''>produce very
  little beta-gal, one unit of beta-gal,</span> <span m=''1927000''>and in the presence
  of lactose, would produce a lot, let''s call it 1,</span> <span m=''1931000''>00
  units of beta-gal. But, the problem with playing</span> <span m=''1936000''>around
  with this is lactose is serving two different roles.</span> </p><p><span m=''1940000''>Lactose
  is both the inducer of the expression of the gene by virtue of</span> <span m=''1945000''>binding
  to the repressor, etc., etc.</span> </p><p><span m=''1950000''>But, it''s also the
  substrate for the enzyme because as beta-galactosidase</span> <span m=''1954000''>gets
  made, it breaks down the lactose. So, there''s less lactose</span> <span m=''1958000''>in
  binding, and if you wanted to really study the regulatory controls,</span> <span
  m=''1963000''>you have the problem that the thing that''s inducing the gene by binding</span>
  <span m=''1967000''>to the repressor is the thing that''s getting destroyed by the
  product of</span> <span m=''1972000''>the gene. So, it''s going to make the kinetics
  of</span> <span m=''1976000''>studying such a process really messy. It would be
  very nice if you could</span> <span m=''1981000''>make a form of lactose that could
  induce beta-galactosidase by binding</span> <span m=''1985000''>to the repressor,
  but wasn''t itself digested.</span> </p><p><span m=''1990000''>Chemically, in fact,
  you can do that. Chemically,</span> <span m=''1997000''>it''s possible to make a
  molecule called IPTG, which is a</span> <span m=''2004000''>galactoside analog.
  And, what it does is this molecule</span> <span m=''2012000''>here which I''ll just
  sketch very quickly here, it''s a sulfur there,</span> <span m=''2021000''>and you
  can see vaguely similar, this is able to be an inducer.</span> </p><p><span m=''2030000''>It''ll
  induce beta-gal, but not a substrate. It won''t get digested.</span> </p><p><span
  m=''2039000''>So, it''ll stick around as long as you want. It''s also very convenient</span>
  <span m=''2044000''>to use a molecule that was developed called ex-gal.</span> </p><p><span
  m=''2049000''>Ex-gal again has a sugar moiety, and then it also has this kind of
  a</span> <span m=''2055000''>funny double ring here, which is a chlorine, and a
  bromine,</span> <span m=''2060000''>and etc. And, this guy here is not an inducer.
  It''s not capable of</span> <span m=''2065000''>being induced, of inducing beta-galactosidase</span>
  <span m=''2071000''>expression. But, it is a substrate.</span> </p><p><span m=''2077000''>It
  will be broken down by the enzyme, and rather neatly when it''s</span> <span m=''2083000''>broken
  down it turns blue. These two chemicals turned out to</span> <span m=''2089000''>be
  very handy in trying to work out the regulation of the</span> <span m=''2095000''>lack
  operon. So, if I, instead of adding lactose,</span> <span m=''2100000''>if I think
  about adding IPTG, my inducer, when I add IPTG I''m going</span> <span m=''2106000''>to
  get beta-gal produced. When I don''t have IPTG, I won''t</span> <span m=''2112000''>produce
  beta-gal. But then I don''t have a problem of</span> <span m=''2118000''>this getting
  used up. So now, what kind of a mutant might</span> <span m=''2124000''>I look for?
  I might look for a mutant that even</span> <span m=''2129000''>in the absence of
  the inducer, IPTG, still produces a lot of</span> <span m=''2134000''>beta-gal.
  Now, I can also look for mutants that no</span> <span m=''2139000''>matter what
  never produce beta-gal, right? But, what would they likely</span> <span m=''2144000''>be?
  They''d likely be structural mutations affecting the coding</span> <span m=''2150000''>sequence
  of beta-gal, right? Those will happen.</span> </p><p><span m=''2155000''>I can collect
  mutations that cause the E coli never to</span> <span m=''2160000''>produce beta-gal.
  But that''s not as interesting as</span> <span m=''2165000''>collecting mutations
  that block the repression that cause beta-gal to be</span> <span m=''2171000''>produced
  all of the time. So, how would I find such a mutant?</span> </p><p><span m=''2176000''>I
  want to find a mutant that''s producing a lot of beta-gal even</span> <span m=''2182000''>when
  there''s no IPTG. So, let''s place some E coli on a</span> <span m=''2187000''>plate.
  Should we put IPTG on a plate? No, so no IPTG.</span> </p><p><span m=''2193000''>What
  do I look for? How do I tell whether or not any of</span> <span m=''2197000''>these
  guys here is producing a lot of beta-gal? Yep?</span> </p><p><span m=''2202000''>So,
  no IPTG, but put on ex-gal, and if anybody''s producing a lot of</span> <span m=''2206000''>beta-gal,
  what happens? They turn blue: very easy to go</span> <span m=''2211000''>through
  lots of E coli like that looking for something blue.</span> </p><p><span m=''2215000''>And
  so, lots of mutants were collected that were blue.</span> </p><p><span m=''2220000''>And,
  these chemicals are still used today. They''re routinely used in</span> <span m=''2225000''>labs,
  ex-gal and stuff like that, making bugs turn blue because this</span> <span m=''2230000''>has
  turned out to be such a well-studied system that we use it</span> <span m=''2235000''>for
  a lot of things. So, mutants were found that were</span> <span m=''2240000''>constituative.
  So, mutants were found that were</span> <span m=''2245000''>constituative mutants.
  Constituative mutants: meaning</span> <span m=''2250000''>expressing all the time,
  no longer regulated, so,</span> <span m=''2255000''>characterizing these constituative
  mutants.</span> </p><p><span m=''2260000''>It turns out that they fell into two
  different classes of constituative</span> <span m=''2264000''>mutants. If we had
  enough time, and you could read the papers and</span> <span m=''2268000''>all, what
  I would do is give you the descriptions that Jacobin Maneaux</span> <span m=''2272000''>had
  of these funny mutants which they''d isolated and were trying to</span> <span m=''2276000''>characterize,
  and how to puzzle out what was going on.</span> </p><p><span m=''2280000''>But,
  it''s complicated and hard, and makes your head hurt if you</span> <span m=''2284000''>don''t
  know what the answer is. So, I''m going to first tell you the</span> <span m=''2288000''>answer
  of what''s going on, and then sort of see how you would</span> <span m=''2292000''>know
  that this was the case. But, imagine that you didn''t know</span> <span m=''2297000''>this
  answer, and had to puzzle this out from the data.</span> </p><p><span m=''2301000''>So,
  suppose we had, so if there were going to be two</span> <span m=''2305000''>kinds
  of mutants: mutant number one are operator constituents.</span> </p><p><span m=''2310000''>They
  have a defective operator sequence. Mutations have occurred</span> <span m=''2318000''>at
  the operator site. Mutant number two have a defective</span> <span m=''2326000''>repressor
  protein, the gene for the repressor protein.</span> </p><p><span m=''2334000''>How
  can I tell the difference?</span> </p><p><span m=''2340000''>So, I could have a
  problem in my operator site.</span> </p><p><span m=''2344000''>What would be the
  problem with the operator site?</span> </p><p><span m=''2348000''>Some mutation
  to the sequence causes the repressor not to bind</span> <span m=''2352000''>there
  anymore, OK? So, a defective operator site</span> <span m=''2356000''>doesn''t bind
  repressors. Defective repressor, the operator</span> <span m=''2360000''>site is
  just fine, but I don''t have a repressor to bind</span> <span m=''2364000''>at it.
  So how do I tell the difference? One way to tell the</span> <span m=''2368000''>difference
  is to begin crossing the mutants together to wild type,</span> <span m=''2372000''>and
  asking, are they dominant or recessive, or things like that?</span> </p><p><span
  m=''2376000''>Now, here''s a little problem. E Coli is not a diploid, so you</span>
  <span m=''2379000''>can''t cross together two E colis and make a diploid E coli,</span>
  <span m=''2383000''>right? It''s a prokaryote. It only has one genome. But,</span>
  <span m=''2386000''>it turns out that you can make temporary diploids,</span> <span
  m=''2390000''>partial diploids out of E coli because it turns out you can mate</span>
  <span m=''2393000''>bacteria. Bacteria, which have a bacterial chromosome</span>
  <span m=''2397000''>here also engage in sex and in the course of bacterial sex,</span>
  <span m=''2401000''>plasmids can be transferred called, for example, an F factor,
  is able to</span> <span m=''2405000''>be transferred from another bacteria. And,
  through the wonders of partial</span> <span m=''2410000''>merodiploid, you can temporarily
  get E colis, or you can permanently get</span> <span m=''2415000''>E colis, that
  are partially diploid. So, you can do what I''m about to</span> <span m=''2420000''>say.
  But, in case you were worried about my writing diploid genotypes</span> <span m=''2425000''>for
  E coli, you can actually do this.</span> </p><p><span m=''2430000''>You can make
  partial diploids. So, let''s try out a genotype here.</span> </p><p><span m=''2436000''>Suppose
  the repressor is a wild type, the operator is wild type,</span> <span m=''2443000''>and
  the lack Z gene is wild type. And, suppose I have no IPTG, I''m</span> <span m=''2449000''>un-induced.
  I have one unit of beta-gal. When I add my inducer,</span> <span m=''2456000''>what
  happens? I get 1,000 units of beta-gal.</span> </p><p><span m=''2463000''>Now, suppose
  I would have an operator constituative mutation.</span> </p><p><span m=''2467000''>Then,
  the operator site is defective. It doesn''t bind the</span> <span m=''2472000''>repressor.
  Beta-gal is going to be expressed all the time,</span> <span m=''2477000''>even
  in the absence. All right, well that was, of course, what we</span> <span m=''2482000''>selected
  for. Now, suppose I made the following diploid.</span> </p><p><span m=''2487000''>I
  plus, O plus, Z plus, over I plus,</span> <span m=''2492000''>O constituative, Z
  plus. So, here''s my diploid. What would</span> <span m=''2498000''>be the phenotype?
  So, in other words,</span> <span m=''2504000''>one of the chromosomes has an operator
  problem.</span> </p><p><span m=''2510000''>Well, that means that this chromosome
  here is always going to</span> <span m=''2516000''>be constituatively expressing
  beta-gal.</span> </p><p><span m=''2522000''>But, what about this chromosome here?
  It won''t. So,</span> <span m=''2526000''>this would be about 1, 01, give or take,
  because it''s got</span> <span m=''2530000''>one chromosome doing that and one chromosome
  doing this,</span> <span m=''2534000''>and this one would be about 2, 00. Now, that
  quantitative</span> <span m=''2538000''>difference doesn''t matter a lot. What you
  really saw when you did</span> <span m=''2542000''>the molecular biology was that
  when you had one copy of the operator</span> <span m=''2546000''>constituative mutation,
  you still got a lot of beta-gal here</span> <span m=''2550000''>even in the absence
  of IPTG. So, that operator constituative site</span> <span m=''2556000''>looked
  like it was dominant to this plus site here.</span> </p><p><span m=''2564000''>But
  now, let''s try this one here. I plus, O plus, Z plus, over I plus,</span> <span
  m=''2572000''>operator constituative, Z minus. What happens then?</span> </p><p><span
  m=''2580000''>This operator constituative site allows constant transcription of</span>
  <span m=''2585000''>this particular copy. But, can this particular copy make</span>
  <span m=''2590000''>a working, functional beta-gal? No. So, this looks, when you
  do</span> <span m=''2595000''>your genetic crosses, you find that the operator</span>
  <span m=''2600000''>constituative, now, if I reverse these here,</span> <span m=''2605000''>suppose
  I reverse these, I plus, O plus, Z minus, I plus, O</span> <span m=''2610000''>constituative,
  Z plus, same genotypes,</span> <span m=''2615000''>right, except that I flipped
  which chromosome these are on.</span> </p><p><span m=''2620000''>Now, what happens?
  This chromosome here: always making beta-gal and it</span> <span m=''2626000''>works.
  This chromosome here: not making beta-gal.</span> </p><p><span m=''2631000''>Even
  though it''s regulated, it''s a mutant. So, in other words,</span> <span m=''2636000''>from
  this very experiment, you can tell that the operator site is only</span> <span m=''2641000''>affecting
  the chromosome that it''s physically on,</span> <span m=''2647000''>that it doesn''t
  make a protein that floats around.</span> </p><p><span m=''2653000''>What it does
  is it''s said to work in cys. In cys means on the same</span> <span m=''2659000''>chromosome.
  It physically works on the same chromosome.</span> </p><p><span m=''2666000''>Now,
  let''s take a look, by contrast, of the properties of</span> <span m=''2672000''>the
  lack repressor mutants. If I give you a lack repressor</span> <span m=''2678000''>mutant,
  I plus, O plus, Z plus is the wild type.</span> </p><p><span m=''2683000''>I constituative,
  O plus, Z plus: what happens here?</span> </p><p><span m=''2689000''>This wild type
  is one in 1, 00. This guy here: 1,000 and 1,</span> <span m=''2694000''>00, and
  then here let''s look at a diploid:</span> <span m=''2700000''>I plus, O plus, Z
  plus, I constituative,</span> <span m=''2705000''>O plus, Z plus. What''s the effect?
  The I constituative doesn''t make a</span> <span m=''2710000''>functioning repressor.
  But, I plus makes a functioning</span> <span m=''2715000''>repressor. So, will this
  show regulation?</span> </p><p><span m=''2721000''>Yeah, this will be regulated
  just fine. This works out just fine,</span> <span m=''2726000''>and in fact it''ll
  make 2,000, and it''ll make two copies there.</span> </p><p><span m=''2732000''>But
  again, the units don''t matter too much. And,</span> <span m=''2737000''>by contrast,
  if I give you I plus, O plus, Z minus, and I constituative,</span> <span m=''2742000''>O
  plus, Z plus, what will happen?</span> </p><p><span m=''2747000''>Here, I have my
  mutation on this chromosome. But,</span> <span m=''2753000''>it doesn''t matter
  because I''ve got my mutation on this chromosome in</span> <span m=''2758000''>the
  repressor. I''ve got a mutation on lack Z here,</span> <span m=''2763000''>but as
  long as I have a functional copy, one functional copy of the</span> <span m=''2769000''>lack
  repressor, it works on both chromosomes.</span> </p><p><span m=''2775000''>It will
  work on both chromosomes, and so in other words this lack</span> <span m=''2780000''>repressor,
  one copy works on both chromosomes. In other words,</span> <span m=''2786000''>it
  makes a product that diffuses around, and can work on either</span> <span m=''2792000''>chromosome,
  and it''s said to work in trans, that is, across.</span> </p><p><span m=''2798000''>So,
  the operator is working in cys. It''s operating on its own</span> <span m=''2801000''>chromosome
  only. A mutation in the operator only</span> <span m=''2805000''>affects the chromosome
  it lives on, whereas a functional copy of the</span> <span m=''2808000''>lack repressor
  will float around because it''s a protein,</span> <span m=''2812000''>and that''s
  how Jacobin Maneaux knew the difference.</span> </p><p><span m=''2815000''>They
  proved their model by showing that these two kinds of mutations</span> <span m=''2819000''>had
  very different properties. Operator mutations affected only the</span> <span m=''2823000''>physical
  chromosome on which they occurred, which of course they had</span> <span m=''2827000''>to
  infer from the genetics they did, whereas repressor, a functional copy</span> <span
  m=''2831000''>repressor, could act on any chromosome in the cell.</span> </p><p><span
  m=''2835000''>So, OK, we''ve got that. Now, last point, what about glucose?</span>
  </p><p><span m=''2840000''>I haven''t said a word about glucose. See, this was a
  big deal to people.</span> </p><p><span m=''2844000''>This model, the repressor
  model, we have this repressor. What</span> <span m=''2848000''>about glucose? What''s
  glucose doing in this picture?</span> </p><p><span m=''2856000''>So, glucose control:
  so here''s my gene. Here''s my promoter,</span> <span m=''2867000''>P lack. Here''s
  my operator, beta-gal.</span> </p><p><span m=''2878000''>It''s encoded by lack Z.
  You''ve got all that. When this guy</span> <span m=''2883000''>is present, sorry,
  when lactose is present,</span> <span m=''2888000''>the repressor comes off. Polymerase
  sits down. Wait a second,</span> <span m=''2893000''>polymerase isn''t supposed
  to sit down unless there''s no glucose.</span> </p><p><span m=''2898000''>We need
  another sensor to tell if there''s glucose,</span> <span m=''2903000''>or if there''s
  low glucose. So, we''re going to need us a sensor</span> <span m=''2908000''>that
  tells that. Any ideas? Yep?</span> </p><p><span m=''2914000''>Yeah, if you work
  that one through, I don''t think it quite works. But,</span> <span m=''2920000''>you''ve
  got the basic idea. You''re going to want another</span> <span m=''2926000''>something,
  and it turns out there''s another site over here,</span> <span m=''2932000''>OK?
  There''s a second site on which a completely different</span> <span m=''2938000''>protein
  binds. And, this protein is the cyclic AMP</span> <span m=''2945000''>regulatory
  protein, and it so happens that in the cell,</span> <span m=''2953000''>when there''s
  low amounts of glucose, let me make sure I''ve got this right,</span> <span m=''2960000''>when
  there''s low amounts of glucose, what we have is high amounts of</span> <span m=''2968000''>cyclic
  AMP. Cyclic AMP turns out,</span> <span m=''2974000''>whereas lactose is used directly
  as the signal, cyclic AMP is used as</span> <span m=''2978000''>the signal here.
  When the cell has low amounts of</span> <span m=''2982000''>glucose, it has high
  amounts of cyclic AMP. Now,</span> <span m=''2986000''>what do you want your cyclic
  AMP to do? How are we going to design this?</span> </p><p><span m=''2990000''>It''s
  going to bind to a protein, cyclic AMP regulatory protein, it''s</span> <span m=''2994000''>going
  to sit down, and now what''s it going to do?</span> </p><p><span m=''2998000''>Is
  it going to block RNA polymerase?</span> </p><p><span m=''3002000''>What do we want
  to do? If there''s low glucose,</span> <span m=''3006000''>high cyclic AMP, we sit
  down at the site, we want to turn on</span> <span m=''3010000''>transcription now,
  right? So, what it''s got to do is</span> <span m=''3015000''>not block RNA polymerase,
  but help RNA polymerase. So,</span> <span m=''3019000''>what it actually does is
  instead of being a repressor,</span> <span m=''3024000''>it''s an activator. And
  what it does is it makes it more attractive for</span> <span m=''3028000''>RNA polymerase
  to bind, and it actually does that by,</span> <span m=''3032000''>actually it does
  it slightly by bending the DNA.</span> </p><p><span m=''3036000''>But, what it does
  is it makes it easier for RNA polymerase to bind.</span> </p><p><span m=''3040000''>It
  turns out that the promoter is kind of a crummy promoter.</span> </p><p><span m=''3044000''>It''s
  actually just like, remember the repressor wasn''t perfect; the</span> <span m=''3048000''>promoter''s
  not perfect either. The promoter''s kind of crummy.</span> </p><p><span m=''3052000''>And,
  unless RNA polymerase gets a little help from this other</span> <span m=''3056000''>regulatory
  protein, it doesn''t work.</span> </p><p><span m=''3060000''>We have two controls:
  a negative regulator responding to an</span> <span m=''3064000''>environmental cue,
  a positive activator responding to</span> <span m=''3069000''>an environmental cue,
  helping polymerase decide whether to</span> <span m=''3073000''>transcribe or not,
  and basically that''s how a human egg</span> <span m=''3078000''>goes to a complete
  adult and lives its entire life,</span> <span m=''3082000''>minus a few other details.
  There are some details left out,</span> <span m=''3087000''>but that''s a sketch
  of how you turn genes on and off.</span> </p>'
type: course
uid: b3f4c1cb67c92f2728cc8f833d4d0bd2

---
None